<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 537 from Anon (session_user_id: a232eeae6033133cc7308dcd12faa6d2c79bcbd0)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 537 from Anon (session_user_id: a232eeae6033133cc7308dcd12faa6d2c79bcbd0)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>The epigenetic machinery of a cell organizes its chromatin in order to establish and maintain cell type specific gene expression. DNA methylation is one component of this machinery. Usually 70 - 80% of the CpGs found in somatic cells of a mammalian genome are methylated. In <strong>CpG-islands</strong>, the CpG-content is increased and they are usually <em>not</em> <em>methylated</em>, due to active repression of methylation. CpG-island <em>methylation</em> usually -not always- correlates with a <em>silent</em> state of the corresponding gene.</p>
<p>Cancer cells are found to be hypermethylated at CpG-islands. As the malignant character of a cancer cell progresses, tumor suppressor genes become less active and oncogenes are activated, either by genetic or epigenetic alterations. Hypermethylation of CpG-islands is a possible way of silencing tumor-suppressors (or other otherwise active genes), being one step of the multiple hits needed for the malignant transformation of cells. Silencing of tumor-suppressors lead to impaired cell cycle control, loss of apoptosis inducing signals or altered DNA repair.</p>
<p>The genome-wide methylation of DNA in cancer cells progressively decreases correlating with the malignancy of the cell. CpG-islands behave inversely - they tend to become more methylated as the cancer progresses.</p>
<p>In normal cells <strong>repetitive elements</strong> are highly methylated in order to be kept silent. <strong>Intergenic regions</strong> are methylated to silence transcriptional noise from additional promoters, which could lead to disrupted transcription by interference of RNA-Polymerases. Mehtylation at intergenic regions also influence splicing sites and replication timing. Both elements are <em>hypomethylated in cancer</em>.</p>
<p>Repetitive elements can be highly mutagenic if activated. They cause illegitimate recombinations or transpose into other regions of the genome where they potentially activate oncogenes or interrupt tumor-suppressor genes. Upon activation by loss of methylation repetitive elements lead to genomic instability. Hypomethylation of intergenic regions lead to impaired transcription/splicing.  </p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>Loss-of-imprinting due to hyper- or hypomethylation at ICRs is found very commonly in cancer cells. Imprinted genes are often involved in growth control, which explains why they are frequently disturbed in neoplastic cells. In normal cells, the ICR of the paternal allele at the Igf2 locus is methylated, thereby preventing CTCF-binding. CTCF acts as an insulator protein, preventing downstream enhancers to interact with and activate the upstream Igf2 gene. Upon paternal imprinted methylation of the ICR and thereby inhibition of CTCF binding, the enhancers are able to allow Igf2 expression. On the maternal allele, the ICR is not methylated and CTCF binds. Here Igf2 expression is prevented by enhancer-blocking. H19 is reciprocally activated. On the paternal allele, H19 is silenced by spreading of heterochromatin from the ICR into the gene. On the maternal allele, H19 interacts with the downstream enhancers and remains active.</p>
<p>In Wilm's tumor, both parental alleles are methylated and display paternal expression pattern. This can be caused by mutations that lead to loss-of-imprinting, by uniparental disomy or rarely by epigenetic disruption. The result is always a double dose of Igf2, which is growth promoting. It's overexpression therefore enhances malignancies.  </p>
<table style="text-align:left;" width="19" cellspacing="0" cellpadding="0"><tbody><tr><td style="width:200px;height:114px;"> </td>
<td style="width:522px;"> </td>
<td style="width:1px;"> </td>
</tr><tr><td style="height:24px;"> </td>
<td align="left" valign="top"> </td>
</tr><tr><td style="height:6px;"> </td>
</tr><tr><td style="height:25px;"> </td>
<td colspan="2" align="left" valign="top"> </td>
</tr></tbody></table></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine is a DNA-demethylating agent belonging to DNA-Methyltransferase inhibitors. These drugs are nucleotide-analogons, which irreversibly bind to the active site of DNMT1, thereby inhibiting the enzyme. DNMT1 is specific for hemimethylated DNA and only displays it's function in replicating cells. If DNMT1 is irreversible bound to a nucleotid, it no longer methylates DNA. After a few cycles of replication, the DNA in a cell affected becomes progressively less methylated. Because a hallmark of cancer cells is their ability to replicate, they are more likely to be hit by the drug than normal cells.</p>
<p>Improper high methylation at CpG-islands in promotor regions of tumor-suppressor genes can lead to decreased transcription. This results in enhanced malignancy of the cell. By inhibiting DNMT1, hypermethylation of these regions can be prevented. The outcome could be a higher expression level of tumor-suppressor genes. </p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>DNA methylation is a way to mitotically heritable set epigenetic marks. After the <em>de novo</em> methylation by DNMT3a and DNMT3b, which are recruited by transcription factors, DNA is replicated. Hemimethylated DNA results after replication, which is a substrate for DNMT1. DNMT1 lays down methylation on the other strand of DNA. This pathway ensures a stable, mitotically heritable DNA-methylation pattern. If the methylation pattern changes for some reason in a somatic cell, the pattern will be inherited by the same mechanism.</p>
<p>The periods of <strong>epigenetic reprogramming</strong> are called the <strong>sensitive periods</strong>. Alterations in methylation pattern due to intake of Dnmt1-inhibitors could be set and then kept for the rest of the life of the individual or could have effects on the offspring.</p>
<p>The first sensitive period is during early development, when paternal DNA is actively demethylated by enzymes and maternal DNA becomes less methylated by dilution due to Dnmt1 exclusion of the nucleus. Shortly after implantation, methylation is reestablished. The second sensitive phase is during primordial germ cell development, where somatic epigenetic marks are removed and reset. There also are naturally occuring sensitive periods for single tissues and organs, or <em>in vitro</em> after induction of artificial reprogramming.</p></div>
  </body>
</html>