<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1948 from Anon (session_user_id: 851d025e4705472acd8b62f2e25b59231b433dd1)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1948 from Anon (session_user_id: 851d025e4705472acd8b62f2e25b59231b433dd1)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>Normally, on an expressed gene, CpG islands are not highly methylated upstream of the gene. In some cancer, the CpG islands are hypermethylated in the promoter region of a tumor suppressor gene.  This is locus specific and not genome-wide. This can then silence the gene which results in less of the tumor suppressor, and can be one of the hits that will then result in growth, <em>and because it is mitotically heritable</em>, in <em>uncontrolled</em> growth, as it is selected for.  As an aside, taking out specific tumor suppressor genes via epigenetic mechanisms is far more common than via mutation.</p>
<p>Normally, there is DNA methylation in the intergenic regions and especially in the repetitive elements.  This results in these regions also being heterochromatized. In this state, they are not mistakenly taken by the mitotic machinery to be the same locus. </p>
<p>In cancer, the repetitive elements could be hypomethylated, and not heterochromatized, so a recombination mistake could happen due to the fact that they are matched with other such regions which are <em>not </em>at the same locus. Also, when the repetitive elements are hypomethylated, they also are then active, and can transpose elsewhere in the genome causing a number of different processing errors. This is not locus specific, but genome-wide, and is found in some degree in all cancers.</p>
<p>Both these cause genomic instability, a cancer hit.</p>
<p>Both of these DNA methylation aberrations appear to progress with time, and therefore are associated with age.</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>Normally, the paternal allele has DNA methylation at the Imprint Control Region.  This methylation can spread to the H19 promoter and silence it.  Finally, the downstream enhancers then act on the Igf2 and express it rather than H19.  Since it is the paternal allele that has a methylated ICR, it is ‘paternally imprinted’.</p>
<p>In the maternal allele, there is no methylation at the ICR.  A protein CTCF binds there, and insulates the downstream enhancers from accessing the Igf2.  Instead, they enhance the H19 to produce the lncRNA</p>
<p>(It is thought that chromatin looping is the mechanism that determines which locus is operated on by the enhancers.)</p>
<p>In Wilm’s tumor (kidney) the maternal allele is also hypermethylated, which causes too much Igf2, a growth factor, which is then mitotically passed on causing the cancer.</p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine (azacitidine) is a DNA-demethylating agent.  It works by binding to DNA Methyl Transferease (DNMT) and inhibiting its function, which is to lay down Methyl marks.  It therefore works in cancer by reversing hypermethylation.</p>
<p>Since the methylation pattern is mitotically heritable, this has an anti tumor effect.  The uncontrolled growth is stalled.</p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Changing the DNA methylation pattern has enduring effects on the epigenome precisely because the methylation pattern is passed on to daughter cells.  During some developmental periods of the organism, large scale changes in the methylation pattern occur.  These are sort of top-level modifications of the organism to differentiate types of organs and tissues, and to choose whether or not active genes will be inherited from the father or mother. These are termed sensitive periods, because there is a lot of epigenetic processing going on, which is presumably finely tuned by complex genetic machinery.  Blunt external disruptions, like a DNA demethylation drug, are likely to have wider impact and unintended consequences at this stage.  It is in fact more <strong>sensitive</strong> to external modifications, and this is why attempting to modify epigenetic marks during these periods (at our current level of understanding) is unadvisable. The two most sensitive periods are pre-implantation development and primordial germ cell development</p></div>
  </body>
</html>