<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 735 from Anon (session_user_id: da80f116afea3ef3f412755142cafa7f6427455e)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 735 from Anon (session_user_id: da80f116afea3ef3f412755142cafa7f6427455e)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>CpG islands, in normal cells, are mostly devoid of methylation, allowing for gene expression in proper regulatory contexts. In cancer, however, many CpG islands appear methylated, forcing silencing of nearby genes. These silenced genes are often tumour suppressor genes, the silencing of which favour cancer progression by allowing unchecked growth of cancer cells. On the other hand, cancer displays the reverse effect in intergenic regions and repetitive elements. These elements, which are usually methylated in normal cells, appear often with reduced methylation in cancer cells. Repetitive elements are a hotspot for transposition events and subsequent structural genomic errors, and are thus kept silenced via methylation. Removal of this mark brings about overall genomic instability, which can change the expression of some genes by changing their regulatory environment (either by moving reguatory elements around, or by changing the chromatin state). This genomic instability can then be used in a evolutionary process that benefits structural changes that cause silencing of tumour suppressor and promote the activation of oncogenes. These disturbances in methylation are often brought about by the occurence of genetic mutations in elements of the methylation machinery, like DNA methylases. </p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>The H19/Igf2 cluster is an interesting case of tug-of-war between maternal and paternal forces. The paternal allele is imprinted with a methylation mark in an imprinting control region (ICR) where the insulating factor CTCF usually binds. By preventing the binding of CTCF to this ICR, it allows the action of nearby enhancers in activating the expression of the Igf2 growth factor. The maternal allele, on the other  hand, is usually unmethylated, thus allowing the binding of CTCF and preventing the activation of Igf2 by the enhancers, which activate instead the H19 gene. In Wilm's tumour, the maternal alelle behaves like the paternal alelle, e.g. due to a mutation that causes permanent methylation of the ICR. This leads to an excess of Igf2 and consequent excessive growth promotion. Disruption of the imprinting in the ICR of the H19/Igf2 cluster can cause an inbalanced gene expression, either promoting too much growth (like in Wilm's tumour), or completely abrogating expression of Igf2 (in the case where the paternal allele behaves like the maternal alelle). </p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p><span>Decitabine is an inhibitor of DNA methyltransferases. By inhibiting the action of DNA methylases, it leads to hypomethylation, e.g. preventing the hypermethylation of CpG islands that is observed in cancers like acute myeloid leukemia.  By preventing hypermethylation of CpG islands or of certain imprinting control regions, it can prevent the inhibition of tumour suppressors, or imprinting inbalances that can lead to cancer.</span></p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>DNA methylation can have enduring effects on the epigenome because this mark can be copied following cell division. Specific DNA methyltransferases, like DNMT1, can copy the methylation mark from a single strand of DNA to the naked strand being copied from it, this way making methylation mitotically inheritable. Nonetheless, the presence of these marks is undesirable in germ cells or early embryonic cells, since these cells must be able to produce any cell in the organism. Therefore, there are specific phases in the life cycle of the organism where methylation marks (either globally, or locally e.g. from imprinting) are removed. These phases are the so called sensitive periods of development, and consist of the lineage leading to the germ cells (eggs and sperm), and early embryonic development (pre-implantantion). Treating patients during early embryonic development (e.g. a pregnant woman) can lead to the death or long term consequences to the baby. Treating patients while they are fertile can lead to fertility problems and issues with the generated offspring. Ideally, only more elderly patients should have these treatments, since these patients will not have more children, and the benefits of expanded lifetime surmount the possible longer term consequences.</p></div>
  </body>
</html>