<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1038 from Anon (session_user_id: 44109681fef65c634af22708f4b7d4009a8416c3)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1038 from Anon (session_user_id: 44109681fef65c634af22708f4b7d4009a8416c3)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>CpG islands are generally methylated within promoters; CpG-methylated regions attract a protein that then silences transcription of the associated genes. In cancer, while global methylation decreases, CpG island methylation can increase, thereby silencing specific tumor suppressor genes that would stop tumorigenesis. Hypermethylation patterns apparently vary by tumor type, and these epimutations spread more rapidly than mutations. DNA methylation in intergenic regions appears to lead to heterochromatin formation and represses the spread of transposons; in short, methylation increases the stability of these intergenic and repeat sequences. Hypomethylation in the intergenic region will therefore decrease the stability of these regions, leading to more transpositions, deletions, and inappropriate genetic expression. The movement of genetic elements, either by transposition or recombinations, can link certain genes under new control elements, thereby changing expression patterns. CpG-poor promoter regions tend to overexpress when hypomethylated; this overexpression could turn a proto-oncogene into an oncogene, increasing the growth of a tumor (such as with Igf2). Furthermore, cryptic start sites and other elements in intergenic regions that were repressed by methylation could become active.</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>Paternal methylation of the H19 gene causes formation of heterochromatin in that region, blocking the binding of the CTCF protein to a downstream region of DNA. CTCF is an insulator protein that keeps certain transcription factors bound to upstream enhancers from working on downstream genes, specifically the gene for Igf2. Igf2 is a proto-oncogene (growth factor), so its overexpression can lead to tumorigenesis. Paternally, Igf2 is expressed because of H19 methylation. Maternally, the lack of methylation keeps CTCF from binding to the DNA and allows enhancers to act on the H19 gene rather than the Igf2 gene. In this mechanism, methylation indirectly leads to maternal silencing of Igf2. In Wilm's tumor, the maternal methylation of H19 causes expression of Igf2 (just as in paternal imprinting of that locus). Too much Igf2 is linked to excessive cell growth (normal imprinting would lead to a comparative half dosage of Igf2).</p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine is a DNA-demethylating agent, which means that it removes overmethylation in cancer. We have learned that tumorigenic overmethylation can occur on CpG islands in promoters of certain genes and on histones; these instances of overmethylation tend to shut down gene expression by blocking the promoters or by forming more heterochromatin (causing position effect variegation or PEV), respectively. If suppressed genes are themselves tumor suppressors, the excessive methylation can be a step towards carcinogenesis. Since decitabine decreases methylation, it can keep these tumor suppressor genes active and thereby protect the cells from too much growth.</p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Methylation patterns tends to be inherited by mitosis, so methylation patterns set during sensitive periods, gametogenesis or pre-implant embryogenesis, are maintained. Therefore, once a methylation pattern is switched to become a carcinogenic one, it will persist; if drugs can cause a generation of cells to break that methylation pattern, then future cells will also be altered. A sensitive period is a time in an organism's life cycle when methylation patterns are being set, and therefore outside insults (chemicals, radiation, etc.) are more likely to have lasting effects during these periods. Since these anti-cancer drugs are looking to inhibit methylation or block histone-deacteylation, their use during an individual's sensitive periods could block the formation of the proper imprinting, X-inactivation, intergenic methylation patterns or other epigenetic functions that are required for the organism's healthy development and existence.</p></div>
  </body>
</html>