<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 347 from Anon (session_user_id: ef1cd4c49c8f3834b08954a999e88777572b1d8d)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 347 from Anon (session_user_id: ef1cd4c49c8f3834b08954a999e88777572b1d8d)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>During normal function, CpG islands when methylated result in a silencing of the gene downstream i.e in the direction of transcription by tRNA. Unmethylated CpG islands allow the transcription of the downstream gene to be expressed. With cancer, there is a general methylation of CpG islands with a genome wide unmethylation of the chromatin. This results in a general silencing of the downstream affected genes, some of which are tumor supressing genes which would otherwise help protect against developing a cancerous cell.</p>
<p>Intergenic regions and repetitive regions are methylated during normal function but are unmethylated in cancer cells. Repetitive elements by their very nature are more easily transcribed erroneously because it is not easy for the normal cellular enzymes and proteins to correctly identify the sequences. Repetitive elements close to the centromere may result in an incorrect number of chromosomes after mitosis. Another pattern is to have chromatin sections swapping to their trans counterpart or to another area of the chromosome in cis. This leads to unexpected chromatin structure and a general increase in genetic instability in the cell.</p>
<p>A general unmethylation may lead to an over activation in the expression of a protein which affects the activity of a epigenetic enzyme unfavourably.</p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Altering the methlyation of DNA can have an enduring effect as epigenetic marks are inheritable mitotically over a cell lineage. This is the case in normal function whereby the epigenetic markers from a developing foetus are generally preserved throughout the lineage of a particular somatic cell type e.g. blood cells, nerve. If this were not the case, many cells would be unable to perform their cell function.</p>
<p>DNA methlyation is normally preserved by proteins such as DNMT1 so that if a drug alters the methylation of a cell, this alteration can be passed onto daughter cells - especially if this change happens during or shortly before mitosis.</p>
<p>The genomic cells have their epigenetic markers removed during two prinicpal stage. Firstly, during early development shortly after fertilisation and secondly during germ cell development during embryo development.</p>
<p>These two sensitive periods of development have significant epigenetic activity so drugs or exposure to environmental factors such as in-vitro or radiation can result in abnormal epigenetic markers that can affect the survival of the embryo. In germ cell development these alterations can increase susceptibility to disease if the sperm/egg go on to form a baby as imprinted genes are preserved across generations.</p>
<p> </p>
<p> </p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine is a dna demethylating agent and part of a class of epigenetic inhibitors called DNMT inhibitors. Decitabine inhibits the function of DNA methyltransferase (DNMT) which is responsible for methylatiing DNA. As a result, it is indirectly reducing the methlyation of DNA.</p>
<p>DNA methlyation is one of the common mechanisms involved in cancer formation so if this drug can be applied to cancer forming cells, it can inhibit methlyation and result in cell death or a failure for the tumor cell to grow. Each daughter cell from a cancer cell would not preserve the methylation marks as DNMT would not be able to methylate the daughter chromosome strand and lead to passive demythlation and removal of the cancer expression.</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>The paternal allele is methylated and inhibits the binding of a protein called CTCF. The methlyation spreads to the downstream H19 promotor and silences it. Downstream enhancers are uninpaired by H19 activity and are free to access the igf2 gene and active it. By contrast, the maternal allele is not methylated. This allows the insulator protein CTCF to bind to the allele and prevent methlyation of the H19 promotor - which in turn inhibits the enhancers from activating igf2. As a result, igf2 remains inactive.</p>
<p>In Wilm's tumour, the maternal allele reverts to a normal paternal allele epigenotype. There is a methylation of the H19 promotor and subsequent activation of igf2. This results in over expression of growth factors in the cell as there is reduced H19 which inhibits growth and an increase in igf2 which promotes growth. One of the markers of a cancer is that thay have permanent growth, so they can continue to cause damage to their sorrounding tissue and ignore signals that would otherwise result in cell death or apoptosis. Having growth factors like igf2 is oncogenic and allow cells to become cancerous. </p></div>
  </body>
</html>