<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 81 from Anon (session_user_id: 5cbfb4e028fa5561f243f5bfdd98005d8d89a539)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 81 from Anon (session_user_id: 5cbfb4e028fa5561f243f5bfdd98005d8d89a539)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p><a title="DNA methylation" href="http://en.wikipedia.org/wiki/DNA_methylation">DNA methylation</a> is probably the most well researched epigenetic mark that differs between normal cells and tumor cells in humans. The "normal" <a title="CpG site" href="http://en.wikipedia.org/wiki/CpG_site">CpG</a> methylation profile is often inverted in cells that become tumorigenic.<sup class="reference"><a href="http://en.wikipedia.org/wiki/Cancer_epigenetics#cite_note-Esteller2007-2">[2]</a></sup> In normal cells, CpG Islands preceding<a class="mw-redirect" title="Gene promoter" href="http://en.wikipedia.org/wiki/Gene_promoter">gene promoters</a> are generally unmethylated, while other individual CpG <a class="mw-redirect" title="Dinucleotide" href="http://en.wikipedia.org/wiki/Dinucleotide">dinucleotides</a> throughout the genome tend to be methylated. However, in cancer cells, <a title="CpG island" href="http://en.wikipedia.org/wiki/CpG_island">CpG islands</a> preceding <a title="Tumor suppressor gene" href="http://en.wikipedia.org/wiki/Tumor_suppressor_gene">tumor suppressor gene</a> promoters are often hypermethylated, while CpG methylation of oncogene promoter regions and parasitic <a class="mw-redirect" title="Repeat sequences" href="http://en.wikipedia.org/wiki/Repeat_sequences">repeat sequences</a> is often decreased.</p>
<p>Hypermethylation of gene promoters can result in silencing of those genes. This type of epigenetic mutation is dangerous when genes that regulate the cell cycle are silenced, allowing cells to grow and reproduce uncontrollably, leading to tumorigenesis.<sup class="reference"><a href="http://en.wikipedia.org/wiki/Cancer_epigenetics#cite_note-Esteller2007-2">[2]</a></sup>Genes commonly found to be transcriptionally silenced due to promoter hypermethylation include: Cyclin-dependent kinase inhibitor<a title="P16 (gene)" href="http://en.wikipedia.org/wiki/P16_(gene)">p16</a>, a cell-cycle inhibitor; <a title="P53" href="http://en.wikipedia.org/wiki/P53">p53</a>, a tumor suppressor gene; <a title="O-6-methylguanine-DNA methyltransferase" href="http://en.wikipedia.org/wiki/O-6-methylguanine-DNA_methyltransferase">MGMT</a>, a <a title="DNA repair" href="http://en.wikipedia.org/wiki/DNA_repair">DNA repair</a> gene; <a title="Adenomatous polyposis coli" href="http://en.wikipedia.org/wiki/Adenomatous_polyposis_coli">APC</a>, a cell cycle regulator; <a title="MLH1" href="http://en.wikipedia.org/wiki/MLH1">MLH1</a>, a DNA-repair gene; and <a title="BRCA1" href="http://en.wikipedia.org/wiki/BRCA1">BRCA1</a>, another DNA-repair gene.<sup class="reference"><a href="http://en.wikipedia.org/wiki/Cancer_epigenetics#cite_note-Esteller2007-2">[2]</a></sup><sup class="reference"><a href="http://en.wikipedia.org/wiki/Cancer_epigenetics#cite_note-Jones_2002-3">[3]</a></sup> Indeed, cancer cells can become addicted to the transcriptional silencing, due to promoter hypermethylation, of some key tumor suppressor genes, a process known as epigenetic addiction<sup class="reference"><a href="http://en.wikipedia.org/wiki/Cancer_epigenetics#cite_note-De_Carvalho_et_al_2012-4">[4]</a></sup></p>
<p>Hypomethylation of CpG dinucleotides in other parts of the genome leads to <a class="new" title="Chromosome instability (page does not exist)" href="http://en.wikipedia.org/w/index.php?title=Chromosome_instability&amp;action=edit&amp;redlink=1">chromosome instability</a> due to mechanisms such as loss of imprinting and reactivation of <a title="Transposable element" href="http://en.wikipedia.org/wiki/Transposable_element">transposable elements</a>.<sup class="reference"><a href="http://en.wikipedia.org/wiki/Cancer_epigenetics#cite_note-HermanBaylin2003-5">[5]</a></sup><sup class="reference"><a href="http://en.wikipedia.org/wiki/Cancer_epigenetics#cite_note-FeinbergTycko2004-6">[6]</a></sup><sup class="reference"><a href="http://en.wikipedia.org/wiki/Cancer_epigenetics#cite_note-EggerLiangAparicio2004-7">[7]</a></sup><sup class="reference"><a href="http://en.wikipedia.org/wiki/Cancer_epigenetics#cite_note-Esteller2005-8">[8]</a></sup> Loss of imprinting of <a title="Insulin-like growth factor" href="http://en.wikipedia.org/wiki/Insulin-like_growth_factor">insulin-like growth factor</a> gene (IGF2) increases risk of <a title="Colorectal cancer" href="http://en.wikipedia.org/wiki/Colorectal_cancer">colorectal cancer</a> and is associated with <a class="mw-redirect" title="Beckwith-Wiedemann syndrome" href="http://en.wikipedia.org/wiki/Beckwith-Wiedemann_syndrome">Beckwith-Wiedemann syndrome</a> which significantly increases the risk of cancer for newborns.<sup class="reference"><a href="http://en.wikipedia.org/wiki/Cancer_epigenetics#cite_note-BaylinJones_2011-9">[9]</a></sup> In healthy cells, CpG dinucleotides of lower densities are found within coding and <a title="Noncoding DNA" href="http://en.wikipedia.org/wiki/Noncoding_DNA">non-coding</a> intergenic regions. Parasitic repetitive sequences, <a title="Centromere" href="http://en.wikipedia.org/wiki/Centromere">centromeres</a> and <a title="Oncogene" href="http://en.wikipedia.org/wiki/Oncogene">oncogenes</a> are often repressed through methylation.</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p><a class="new" title="Eukaryotic DNA (page does not exist)" href="http://en.wikipedia.org/w/index.php?title=Eukaryotic_DNA&amp;action=edit&amp;redlink=1">Eukaryotic DNA</a> has a complex structure. It is generally wrapped around special proteins called <a title="Histone" href="http://en.wikipedia.org/wiki/Histone">histones</a> to form a structure called a<a title="Nucleosome" href="http://en.wikipedia.org/wiki/Nucleosome">nucleosome</a>. A nucleosome consists of 2 sets of 4 histones: <a title="Histone H2A" href="http://en.wikipedia.org/wiki/Histone_H2A">H2A</a>, <a title="Histone H2B" href="http://en.wikipedia.org/wiki/Histone_H2B">H2B</a>, <a title="Histone H3" href="http://en.wikipedia.org/wiki/Histone_H3">H3</a>, and <a title="Histone H4" href="http://en.wikipedia.org/wiki/Histone_H4">H4</a>. Additionally, <a title="Histone H1" href="http://en.wikipedia.org/wiki/Histone_H1">histone H1</a> contributes to <a class="mw-redirect" title="DNA packaging" href="http://en.wikipedia.org/wiki/DNA_packaging">DNA packaging</a> outside of the nucleosome. Certain histone modifying enzymes can add or remove functional groups to the histones, and these modifications influence the level of transcription of the genes wrapped around those histones and the level of DNA replication. Therefore, one might expect the histone modification profiles of healthy and cancerous cells to differ.</p>
<p>In comparison to healthy cells, cancerous cells have been seen to exhibit decreased monoacetylated and trimethylated forms of <a title="Histone H4" href="http://en.wikipedia.org/wiki/Histone_H4">histone H4</a> (decreased H4ac and H4me3).<sup class="reference"><a href="http://en.wikipedia.org/wiki/Cancer_epigenetics#cite_note-Fraga_et_al_2005-10">[10]</a></sup>In mouse models, scientists have noticed that the loss of histone H4 acetylation and trimethylation actually increases as tumor growth continues.<sup class="reference"><a href="http://en.wikipedia.org/wiki/Cancer_epigenetics#cite_note-Fraga_et_al_2005-10">[10]</a></sup> Interestingly, loss of histone H4 Lysine 16 acetylation (<a class="new" title="H4K16ac (page does not exist)" href="http://en.wikipedia.org/w/index.php?title=H4K16ac&amp;action=edit&amp;redlink=1">H4K16ac</a>), which is a mark of aging at the <a title="Telomere" href="http://en.wikipedia.org/wiki/Telomere">telomeres</a>, specifically loses its acetylation. Some scientists hope this particular loss of histone acetylation might be battled with a <a title="Histone deacetylase" href="http://en.wikipedia.org/wiki/Histone_deacetylase">histone deacetylase</a> (HDAC) inhibitor specific for <a class="mw-redirect" title="SIRT1" href="http://en.wikipedia.org/wiki/SIRT1">SIRT1</a>, an HDAC specific for H4K16.<sup class="reference"><a href="http://en.wikipedia.org/wiki/Cancer_epigenetics#cite_note-Esteller2007-2">[2]</a></sup><sup class="reference"><a href="http://en.wikipedia.org/wiki/Cancer_epigenetics#cite_note-Dang_et_al_2009-11">[11]</a></sup></p>
<p> </p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p><span>patients with breast cancer, hypermethylation of CpG islands located proximal to the transcription start site was observed. Loss of </span><a title="CTCF" href="http://en.wikipedia.org/wiki/CTCF">CTCF</a><span> binding and an increase in repressive histone marks, H3K9me3 and H3K27me3, correlates with </span><a title="DNA methylation" href="http://en.wikipedia.org/wiki/DNA_methylation">DNA methylation</a><span> and miR-125b1 silencing. Mechanistically, CTCF may function as a boundary element to stop the spread of DNA methylation. Results from experiments conducted by Soto-Reyes indicate a negative effect of methylation on the function and expression of miR-125b1, therefore Soto-Reyes and his team were able to conclude that DNA methylation has a part in silencing the gene. Furthermore, results show that some miRNA’s are epigenetically silenced early on in breast cancer, and therefore these miRNA’s could potentially be useful as tumor markers.</span><sup class="reference"><a href="http://en.wikipedia.org/wiki/Cancer_epigenetics#cite_note-Soto-Reyes_et_al_2012-20">[20]</a></sup></p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p><a title="Prostate cancer" href="http://en.wikipedia.org/wiki/Prostate_cancer">Prostate cancer</a><span> kills around 35,000 men yearly, and about 220,000 men are diagnosed with prostate cancer per year, in North America alone.</span><sup class="reference"><a href="http://en.wikipedia.org/wiki/Cancer_epigenetics#cite_note-Collins_et_al_2012-21">[21]</a></sup><span> Prostate cancer is the second leading cause of cancer-caused fatalities in men, and within a man’s lifetime, one in six men will have the disease.</span><sup class="reference"><a href="http://en.wikipedia.org/wiki/Cancer_epigenetics#cite_note-Collins_et_al_2012-21">[21]</a></sup><span> Prostate cancer has been associated with </span><a title="Gene silencing" href="http://en.wikipedia.org/wiki/Gene_silencing">gene silencing</a><span> by </span><a class="new" title="CpG island hypermethylation (page does not exist)" href="http://en.wikipedia.org/w/index.php?title=CpG_island_hypermethylation&amp;action=edit&amp;redlink=1">CpG island hypermethylation</a><span>. The </span><a title="GSTP1" href="http://en.wikipedia.org/wiki/GSTP1">GSTP1</a><span> gene has been found to defend </span><a title="Prostate" href="http://en.wikipedia.org/wiki/Prostate">prostate</a><span> cells against genomic damage that is caused by different oxidants or</span><a title="Carcinogen" href="http://en.wikipedia.org/wiki/Carcinogen">carcinogens</a><span>.</span><sup class="reference"><a href="http://en.wikipedia.org/wiki/Cancer_epigenetics#cite_note-Gurel_et_al_2008-22">[22]</a></sup><span> This suggests that the silencing of this gene will permit genetic damage to the prostate by oxidants and carcinogens. Methylation of </span><a title="Cytosine" href="http://en.wikipedia.org/wiki/Cytosine">cytosines</a><span> within a CpG island is a causative factor in gene silencing. </span></p></div>
  </body>
</html>