<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 502 from Anon (session_user_id: 9ed78e87aafcbe898c06d2ed13dfd36379320de6)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 502 from Anon (session_user_id: 9ed78e87aafcbe898c06d2ed13dfd36379320de6)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p><span> CpG Island (CGI) methylation functions to regulate
the gene expression by blocking the binding of transcription factors and by recruiting
the methyl CpG binding domain proteins. CGI methylation also blocks the
expression of foreign and harmful genes. </span><span>In cancer, disruption of CGI methylation can be explained by – selective
advantage and selective targeting. Selective advantage explains aberrant DNA methylation
to cause random DNA methylation throughout the genome which dysregulates DNA
methylation machinery and the <i>de novo</i>
methylated tumour suppressor genes (TSG) get selected for tumour progression. Selective
targeting explains that aberration in DNA methyltransferases (DNMT) causes
hypermethylation in specific regions which serves as DNMT targets as they
possess cis-acting elements. </span><span>Polycomb group (PcG) proteins
are associated with CGI promoters. EZH2, a PcG subunit, is abundant in tumours and
it recruits DNMTs to hypermethylate the promoters. Sadikovic <i>et.al.,</i> 2006 identified that in </span>breast cancer<span> environmental carcinogen
like-benzopyrene can </span>disrupts DNA methylation patterns. In cancer, hypermethylation usually occurs at CGIs of TSG silencing
which promotes metastasis whereas global hypomethylation of repeat elements
such as satellites and retrotransposons leads to genomic instability and activation
of oncogenes.</p>

<p><span>Methylation of
intergenic region and repetitive elements silences transposable and viral
elements which if expressed could cause genomic instability, insertion mutagenesis,
deletion and gene rearrangements. Disruption of intergenic regions and
repetitive elements (IRRE) methylation in cancer arises by aberrant DNA methylation
due to loss of function of epigenetic enzymes (DNMT) by mutation leading to
global hypomethylation. Alterations in methylation pattern could also result
from aberrant targeting or alterations in epigenetic complexes. Global hypomethylation
in cancer leads to loss of methylation silencing of IRRE, increasing genomic instability
by chromosomal rearrangement, activates retrotransposons further increasing
genomic instability and activates proto-oncogenes. </span></p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><span>In a
normal cell, on maternal allele the H19/Igf2 locus has unmethylated ICR to which
CTCF binds thus enabling downstream enhancers to access and express H19 gene. While
on paternal allele, ICR is methylated as a result CTCF does not binds to ICR thus
expressing Igf2 gene and silencing H19 gene. Thus in a normal cell both H19 on maternal
allele while Igf2 on paternal allele is expressed. Wilm’s tumour is marked by hyperacetylation
of ICR leading to biallelic expression of <i>IGF2</i>
and no H19 expression. Also loss of methylation at <i>IGF2</i> DMR leads to expression of igf2 on maternal allele. IGF2
activates either PI3K-Akt or Ras-MAPK pathway controlling growth, differentiation,
metabolism and apoptosis. Loss of imprinting of IGF2 leads to over-expression
of IGF2 which drives excessive cellular growth leading to Wilm’s tumour.</span></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><span>Decitabine
is DNA methyltransferase inhibitor, it leads to hypomethylation by inhibiting
the methylation of DNA by DNMTs. Decitabine is used to treat myelodysplastic
syndromes but recent articles also indicates it to be effective against
adernocortical carcinoma. Antitumour activity of decitabine is primarily due to
its ability to activate the previously hypermethylation-silenced  tumour suppressor genes by inhibiting DNMTs.</span></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p><span>DNA methylation
manipulation produces long lasting effects as DNA methylation pattern once altered
are diligently preserved by DNA methyltransferases during mitosis. Period
considered epigenetically sensitive  are
time windows in the life of an organism when extensive epigenetic reprogramming
or resetting takes place. The sensitive periods for treatment includes time before
and after conception in pregnant mothers as removal and resetting of epigenetic
marks takes place during early fetus development and during formation of
gametes. Hence this period is inadvisable for treatment.</span></p></div>
  </body>
</html>