<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1661 from Anon (session_user_id: 88b2101033fe335680a345e89ce425c98feaa26d)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1661 from Anon (session_user_id: 88b2101033fe335680a345e89ce425c98feaa26d)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p><span style="line-height:21px;font-size:14px;">In a normal cell, there is a lack of DNA methylation at CpG islands (often within promoter regions). Methylation of this region silences the gene under control by the promoter because the promoter region becomes heterochromatic upon methylation. Methylation may also alter transcription factor (repressors and activator) binding to their target sequences. </span><span style="font-size:14px;line-height:21px;">CpG islands increase in their DNA methylation in cancer and this h</span><span style="line-height:21px;font-size:14px;">ypermethylation is often found in tumour supressor genes in cancer, thus the genes that would stop the progression of cancer at one stage are reduced. </span></p>
<p><br /><span style="line-height:21px;font-size:14px;">Intergenic and repetitive elements are often methylated; this reduces genome instability by preventing repetative element transposition, and erronious recombination. Repeats can also be reduced because methylated C, is often mutated to T, allowing for less transposition. Futhermore, methylation reduces interference from nearby promoters which have a strong activating role.  </span><br /><span style="line-height:21px;font-size:14px;">In cancer, these regions are often hypomethylated. In cancer these regions are hypomethylated and t</span><span style="line-height:21px;font-size:14px;">he protection from methylation in these regions as described above is lost and thus recombination and transposition of these regions increases. This leads to an increase in insertions, and deletions, often inactivating oncogenes. Further, genes favourable for cancers can increase in their number through the same mechanism acting under positive selection. </span><span style="line-height:21px;font-size:14px;">Finally, strong promoters may also activate or indeed repress nearby genes, as well as alternate splice sites, anti-sense transcription or splice variants of genes. </span></p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p><span style="line-height:21px;font-size:14px;">Imprinting is the pre-determined inactivation of one of the alleles from a set of parents. In the H19/Igf2 cluster the paternal allele is methylated, including the imprint control region - CTCF is not binding to the imprint control region, and Igf2 is activated by the enhancers. The H19 gene is also methylated and is not expressed. In contrast, the m</span><span style="line-height:21px;font-size:14px;">aternal allele is unmethylated, CTCF can bind the imprint control region, insulating the enhancers from Igf2 and thus only H19 is expressed from this locus.</span></p>
<p><span style="line-height:21px;font-size:14px;">In Wilm's tumour hypermethylation occurs at the H19/Igf2, the maternal and paternal alleles are both methylated. </span><span style="line-height:21px;font-size:14px;">This hypomethylation increases the expression of Igf2 (as described in the case of the paternal allele) as both alleles are expressing the protein, promoting growth. </span></p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine is a cytidine analogue, acting as a DNMT (DNA methyl-transferase) inhibibitor, that is to say that they irriversibly bind to DNMTs upon when incorporated into the DNA, stopping DNMTs being free to copy more methylation to the new strand of DNA upon replication. This reduced DNA methylation iis therefor replication dependent. The mechanism of action is still unclear, but low doses appear to be effective, success in non-solid tumours may be due to a dependence on tumour suppressor gene hypermethylation in these cancers. Overall a reduction in methylation would be thought to act on promotor regions of tumour supressor genes and thereby activate cellular pathways that would stop the progression of the disease.  </p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Epigenetic drugs can alter tumours, even after treatment. For DNA methylation, if daughter cell methylation is decreased due to DNMTis, then the nascent DNA strands will have less DNA methylation in actively dividing cells. Ongoing treatment may not be necessary as once the methylation marks are lost, they will not be transferred to daughter cells upon replication assuming that methylation is not added de novo. "Sensitive periods" are periods where the environment has the ability to influence the epigenetic landscape of an organism. These would be from primordial germ cell development to the period to mature gametes, as well as the period from fertilised egg to epiblast. These periods are generally points where epigenetic marks are most dyamic. Treating patients and altering their epigentic landscape at this point is likely to alter their development in a manner that is abnormal. It would be expected that epigenetic marks at these points are necessary for a healthy development and thus it would be a risky window of time to treat a patient. </p></div>
  </body>
</html>