<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 906 from Anon (session_user_id: 05eff1afb43a59ca09aee06ad3d1a810289cb4ff)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 906 from Anon (session_user_id: 05eff1afb43a59ca09aee06ad3d1a810289cb4ff)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p><span>The CpG islands, which are found in about 60% of promoters, </span><span>have a high CpG density and are usually kept free of methylation independent of their activity state,</span><span> but in cancer cells promoter CpG islands tend to become hypermethylated, which then causes silencing of the underlying gene. <span>Changes in DNA methylation consist of genome-wide loss and a regional gain of DNA methylation causing, on one hand, genomic instability and deregulation of tissue specific and imprinted genes, and on the other hand silencing of tumor suppressor genes – controlling cell cycle, apoptosis or DNA repair – by hypermethylation of their promoter CpG islands. <span>One class of non-coding RNAs constitute the recently discovered transcribed ultraconserved regions (T-UCRs)</span><span>, which are transcribed from evolutionary ultraconserved regions found in the human, rat and mouse genome:</span><span> they show aberrant expression in several cancers, including adult chronic lymphocytic leukemia, colorectal and hepatocellular carcinomas and neuroblastomas.</span></span></span></p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>Hyper- and hypomethylation at the IGF2-H19 imprinting control region (ICR) result in reciprocal changes in IGF2-H19 expression. DNA methylation of the ICR controls the reciprocal imprinting of IGF2 and H19 by preventing the binding of the insulator protein, CTCF. Local changes in histone modiﬁcations and CTCF–cohesin binding at the ICR in BWS and SRS together with DNA methylation correlate with the higher order chromatin structure at the locus. In patient-derived cell lines, the mat/pat asymmetric distribution of these epigenetic marks was lost with H3K9me3 and H4K20me3 becoming biallelic in the BWS and H3K4me2, H3K27me3 and H3K9ac together with CTCF–cohesin becoming biallelic in the SRS. In normal cells, lack of CTCF–cohesin binding at the paternal ICR is associated with monoallelic interaction between two CTCF sites ﬂanking the locus. CTCF–cohesin binding at the maternal ICR blocks this interaction by associating with the CTCF site downstream of the enhancers. The two alternative chromatin conformations are differently favoured in BWS and SRS likely predisposing the locus to the activation of IGF2 or H19, respectively. <span style="font-size:14px;line-height:21px;">A </span><span style="font-size:14px;line-height:21px;">transcript that is antisense to IGF2, IGF2-antisense (IGF2-AS), is transcribed from within IGF2 in a reverse orientation. This transcript is also </span><span style="font-size:14px;line-height:21px;">maternally imprinted and overexpressed in Wilms’ tumor. I</span></p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p><span>In vivo effects of the DNA hypomethylating agent 5-aza-2′-deoxycytidine (DAC) upon the epigenome of acute myeloid leukemia (AML) patients are scarcely studied in primary blasts. Here, we sequentially assessed DNA methylation and transcriptome changes in myeloblasts of DAC-treated AML patients. DNA methylation changes were detected in all patients already after the first series of infusion (median: 6 days). LINE1 analysis indicated global DNA methylation decrease in 7/8 patients. Gene-specific hypomethylating effects were not directly linked to mRNA expression changes. In conclusion, complex DAC-induced DNA methylation changes occur very early and imply mechanisms distinct from non-hypomethylating cytosine analogs.</span></p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Baylin has shown that a combination of a histone-deacetylase inhibitor and azacitidine slowed tumour growth in some people with advanced lung cancer. This result was important for two reasons: it was the first time epigenetic drugs had been deployed successfully against a solid tumour, rather than a leukaemia or a lymphoma (solid tumours are harder to treat, because the drug has to penetrate them); and, second, some of the participants in Dr Baylin's study who did not show much response to the trial itself then went on to show an unexpectedly good reaction to the routine chemotherapeutic drugs which were employed on them next. Although it is too early to say for sure, Dr Baylin speculates that his epigenetic drugs altered the tumour cells in some lasting way that made them more susceptible to standard chemotherapy.<span style="font-size:14px;line-height:21px;"> Unlike other forms of gene regulation, epigenetic changes are passed on during cell division to daughter and granddaughter cells until they are actively erased. Once erased, though, they do not return. It might therefore be that epigenetic therapies can effect changes which stop a cancer growing without having to kill all its cells.</span><span style="font-size:14px;line-height:21px;"> If it is, it would truly be a conceptual breakthrough, and epigenetics might justly take its place alongside genetics in the analysis and treatment of cancer.</span></p></div>
  </body>
</html>