<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 864 from Anon (session_user_id: 2b09a97eec38446ab026bd02f56ad7e277383d3e)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 864 from Anon (session_user_id: 2b09a97eec38446ab026bd02f56ad7e277383d3e)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>Altered DNA methylation is a hallmark of cancer, including locus-specific hypermethylation at CpG islands &amp; hypomethylation at intergenic regions &amp; repetitive elements throughout the genome. </p>
<p>CpG islands tend to be protected from methylation in normal cells.  There are a few exceptions to this general rule however, as seen with dynamic methylation at CpG islands between different cell types. In such instances, methylation of CpG islands is almost universally synonymous with the silencing of gene expression.</p>
<p>In normal cells, unmethylated CpG islands are found in the promoters of many tumor suppressor genes—allowing these important “housekeeping” genes to be expressed. However, CpG islands become hypermethylated in cancer cells, which promotes the formation of a repressive chromatin structure. This effectively silences the tumor suppressor genes normally involved in controlling cell cycle, apoptosis or DNA repair. Thus, the loss of these vital cellular processes increases genomic instability &amp; contributes to diseases like cancer. </p>
<p>Unlike CpG islands, both intergenic regions &amp; repetitive elements are typically methylated in normal cells. DNA methylation at intergenic regions contributes to the maintenance of genomic integrity by helping to maintain a normal karyotype—ensuring the appropriate number of chromosomes &amp; preventing illegitimate recombinations between chromosomes, including minimizing reciprocal translocations, insertions, &amp; deletions. DNA methylation at intergenic regions is also associated with the silencing of cryptic transcription start sites or cryptic splice sites. </p>
<p>Likewise, DNA methylation helps to maintain genomic integrity &amp; stability by silencing repetitive elements. In this way, DNA methylation prevents mutagenic transposition of repeats &amp; helps to avoid transcriptional interference from strong promoters.  Moreover, DNA methylation helps foster genomic stability through its mutagenic properties. Because methyl-cytosine mutates to thymine, such mutations may, over time, prevent transposition by disabling the promoters of repetitive elements. And finally, methylation of repeats may also prevent illegitimate recombination by allowing for compaction of the chromatin structure.</p>
<p>It's therefore clear that DNA methylation at intergenic intervals and repetitive elements confers genomic stability in a variety of ways—which are lost when these regions become hypomethylated in cancer cells. </p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>In normal cells, the imprint control region (ICR) for the H19/Igf2 cluster is methylated on the paternal allele. This prevents CTCF—an insulator protein—from binding the ICR. Without CTCF, DNA methylation spreads to the H19 promoter &amp; silences it, thereby allowing downstream enhancers to access &amp; express Igf2 from the paternal allele.</p>
<p>Conversely, the ICR on the maternal allele is unmethylated in normal cells, which allows CTCF to bind to the ICR. This insulates Igf2 from downstream enhancers &amp; silences Igf2 expression from the maternal allele. Instead, the enhancers in this case will act on the H19 promoter.  </p>
<p>Wilm’s tumor is marked by a loss of imprinting at the H19/Igf2 cluster, such that the ICR on the maternal allele is now methylated. This causes Igf2 expression from the maternal allele as well as from the paternal allele, leading to a double dose of growth-promoting Igf2.  </p>
<p>Alterations in DNA methylation at ICRs—which can be hypermethylated or hypomethylated in cancer—often result in loss of expression of growth-restricting genes, or over-expression of growth-promoting genes (since many imprinted genes impact growth). Thus, loss of imprinting is common in tumors, as genes that should display monoallelic parent-of-origin-specific expression are either expressed from both parental alleles, or silent from both parental alleles—which contributes to disease.</p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine is a cytidine nucleoside analog used for the treatment of patients with myelodysplastic syndromes and acute myeloid leukemia (AML). It is in a class of medications called DNA-demethylating or DNA-hypomethylating agents. As a cytidine nucleoside analog, decitabine gets incorporated into DNA upon replication. The enzyme DNA methyltransferase 1 (DNMT1)—which recognizes hemi-methylated strands of DNA—then irreversibly binds to the nucleoside analog as it attempts to methylate the daughter strand of DNA. Consequently, DNMT1 is not released and can longer act. In this way, decitabine prevents methylation of daughter cells during replication—making it replication dependent. And because cancer cells divide much more rapidly than most other cells in the body, they are more severely affected by the action of this drug. Accordingly, DNMT1 inhibitors such as decitabine, when used at very low doses at the right time during tumorigenesis, have very good anti-neoplastic effects (i.e. tumor cells are effectively killed). </p>
<p> </p>
<p> </p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Unlike other forms of gene regulation, epigenetic changes such as alterations in DNA methylation are mitotically heritable. This means that they are passed on during cell division to daughter &amp; granddaughter cells; but once erased, they do not return. For this reason, drugs that alter DNA methylation can have an enduring effect on the epigenome beyond the period of drug treatment. For example, DNA-demethylating agents that work by inhibiting DNMT1 can prevent DNA methylation during daughter cell replication. And as replication continues, granddaughter and great-grandaughter cells will be increasingly unmethylated without further drug treatment. Consequently, epigenetic therapies that effect changes through altered DNA methylation may be able to stop the replication of cancer cells, allowing chemotherapeutic agents to then be more effective.</p>
<p>Sensitive periods occur when epigenetic marks are reprogrammed during early development, primordial germ cell development, &amp; later during specific stages of differentiation. Because epigenetic drugs are used throughout the system &amp; every cell can be affected by them, a drug that inhibits epigenetic machinery may present a major insult to a patient's environment if administered during a sensitive period (if given to younger patients with developing germ cells, for example). Accordingly, because the silencing function of DNA methylation is essential for embryogenesis/gametogenesis, it is inadvisable to treat patients with drugs that alter DNA methylation during these times. </p></div>
  </body>
</html>