<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 693 from Anon (session_user_id: 9793db794345f439c468d2fa44ea3e670e32852b)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 693 from Anon (session_user_id: 9793db794345f439c468d2fa44ea3e670e32852b)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>DNA methylation can knock out genes functionality just like DNa itself mutation, and it is actuallly mor common in macro scale then the mutations. In normal CpG islands would be unmethylated for the genes to remain active, since they normally promote genes that have tumor suppressive functionality. In cancer, CpG islands become methylated more frequently than in normal cells. CpG methylation would cause those genes to be inactivated and that could induce the cells divide uncontrllably or not die in programmed way. Different types of tumor may have different patterns of CpG Islands methylation across the genome.</p>
<p>In contrast, the intergenic, repetitive elements and introns are usually methylated in a normal cell to increase the stability of the genome structure. In cancer cells these areas become more frequently unmethylated and contribute to genome instability. That instability is represented in deletions, insertions and reciprocal translocations. These modifications are because the mechanisms of those changes require similarity between DNA chains and that is more likely to happen when the repetitive elements are unmethylated and in euchromatic, active DNA location in the nucleus. Hypomethylation of those intergenic regions may also create strong promoters which may activate the nearby genes, like in the agouti yellow allelle case. Hypomethylation may alos activate the transposons, the jumping genes, which may translocate themselves into other genes area and disrupt their expression.</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>In a normal cell the paternal allelle has methylation downstream from the lgf2 gene which is a growth promoter. that region is a region for CTCF protein binding which is not possible to be bound when the methylation is present. further down the stream from that region there are enhancers for lgf2 gene expression. when the methylated are is not bound by CTCF protein, those enhancers have access to lgf2 and so it is expressed from the paternal chromosome.</p>
<p>in a maternal chromosome the area downstream of lgf2 is not methylated and so that allows the CTCF to bind onto the DNA. the enhancers that are found downstream from that area in that situation have obstructed access to the lgf2 and instead they promote H19 region which is found in between the CTCF binding site and the enhancers. that hypomethylation in that region makes the lgf2 on the maternal chromosome to be silenced.</p>
<p>in cancer cells both the paternal and the maternal site are methylated, which disrupts CTCF binding, and so the enhancers have unobstructed access to lgf2 sites on maternal and paternal chromosomes. that leads to overexpression of lgf2, which in turn leads to tissue overgrowth and tumorgenesis. </p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine belongs to DNA demethylating agents group, acting as DNMT (DNA methylatrasfrase enzyme) inhibitor, together with azactidine is one of two FDA approved DNMTi epigenetic drugs. It is approved for use on myelodysplastic syndrome (precursor of acute leukemia) but it could potentialy be effective on other neoplasia that is driven by CpG islands hypermethylation at the promoter sites of tumor supressing genes.</p>
<p>Decitabine blocks DNMT activity hence the DNA methylation is reduced.</p>
<p>If the tumor is driven by the CpG hypermethylation, then a reduction in methylation at that/those sites would allow the tumor supressing genes expression which in turn would supress the tumorgenesis like it is done in normal cells. However if the tumor is driven by other factors, it may not be as effective, and in case when the tumor is driven by DNA instability, genome-wide demethylation could have potentially adverse impact on clinical outcome. </p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Drugs targeting DNA methylation do not have specificity to tumor cells but act on all the tissues that are exposed to it. Since DNA methylation is mitotically heritable in cells lineage, drug induced alteration of the DNA methylation would be carried forward.</p>
<p>Especially sensitive period to DNA methylation is when the cells are re-setting the epigenetic marks, which happens during embryonic development but also during gametogenesis. Hence the use of drug by a preganant woman or by young people who are still forming their gametes, it could alter the epigenetic reprogramming in their offspring with effects on their furture life that may not be easy to predict. The effects of methylation changes could also have effect on their own health since the action of the drug would have effect in all their tissues. Eg if the drug was a DNMT inhibitor like in a previous case above, it could suppress the targeted tumor that was driven by hypermethylation but it would decrease methylation across the whole genome and so it could increase the risk of other tumors that would be driven by hypomethylation and/or genomic instability. </p></div>
  </body>
</html>