<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 200 from Anon (session_user_id: d7e1980d4d8c6a51fe75ca7dc80803659380c539)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 200 from Anon (session_user_id: d7e1980d4d8c6a51fe75ca7dc80803659380c539)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value">Cancers disrupt the normal functioning of DNA methylation in cells. Two of the ways in which this occurs are hypermethylation of CpG islands and hypomethylation of repetitive elements and intergenic regions.<br /><br />CpG islands are genomic regions of abnormally frequent CpG dinucleotides, typically found at promoters. They are normally unmethylated in noncancerous cells regardless of the activity level of the promoter's gene. However, in various cancers there tends to be overall CpG island hypermethylation relative to normal cells. This is particularly acute at the promoters of tumor supressor genes. The methylation at these regions silences the expression of the tumor supressors. In turn, this dysregulates such functions as apoptosis, cell proliferation, and DNA repair, thereby contributing to the cancer phenotype.<br /><br />The genomes of noncancerous cells are frequently methylated in intergenic regions and repetitive elements. In the case of repetitive elements, this functions to silence repeats to prevent their activation and transposition in the genome. In addition, keeping these repeats silenced guards against illegitimate recombination with repetitive elements elsewhere in the genome. Moreover, silencing repeats and intergenic regions prevents transcription from cryptic promoters that may disrupt normal gene expression. In cancer there is a loss of methylation at these regions. Accordingly, illegitimate recombinations and transpositions become more common and the genome becomes increasingly unstable, which can exacerbate the cancer phenotype.</div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value">Disruption of normal methylation at parental imprint regions can also contribute to cancer. One such example is the H19/Igf2 cluster. Expression of these two genes is controlled by the methylation at a locus in between them. In normally functioning cells, this locus is methylated on the paternal allele and unmethylated on the maternal allele. In the methylated allele, Igf2 and H19 share downstream enhancers, but Igf2 apparently attracts these enhancers with much higher affinity. Thus, Igf2 is expressed from this allele, but H19 is not expressed appreciably. In contrast, the unmethylated allele binds an insulator protein, CTCF. This prevents the downstream enhancers from accessing Igf2, so they enhance H19 instead. Thus, H19 is expressed from this allele but not Igf2.<br /><br />As a growth factor, Igf2 is clearly a proto-oncogene. That is, it has the capacity to dysregulate the cell cycle and contribute to cancer. In Wilm's tumor, this happens through aberrant methylation. The maternal allele becomes methylated, which makes it act the same as the paternal allele. Accordingly, Igf2 is overexpressed, which helps fuel the tumor.</div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value">Decitabine is an example of a DNMT inibitor. It is a nucleoside analogue that becomes incorporated into DNA during replication. After incorporation, it can suicidally inhibit DNMTs by covalently binding to them. As a DMNT inhibitor, it reduces DNA methylation, but in a non-specific manner. Since its mechanism of effect is clearly proportional to the rate of DNA replication, decitabine is somewhat specific for regions of rapid proliferation, namely tumors. Moreover, the fact that it can disrupt methylation in cancerous cells, which have pathological methylation, suggests possible therapeutic benefits. These benefits have been borne out in clinical trials, but the precise mechanism of action is not yet known.</div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value">A key feature of DNA methylation is that it is mitotically heritable. Once a locus is methylated on a one DNA strand, DNMT1 copies the mark at the same locus on the other stand. Thus, daughter cells each receive one strand methylated at that locus, and DMNT1 can copy the mark to the other strand once again. Thus, a modification in methylation (such as the ones presumably made by these drugs) can persist in an organism long after its cause has left.<br /><br />There are, however, sensitive periods in which these sorts of epigenetic treatments would be inadvisable. These are normal periods of epigenetic reprogramming  that occur during the bastocyst stage and in the primordial germ cells. Maturative alterations of methylation during these periods are important for many processes, including parental imprinting and tissue differentiation. Modifying methylation patterns during these periods would result in unpredictable changes that could easily affect core biological functions. Moreover, the mitotic heritability of these changes would ensure that they would be virtually permanent.</div>
  </body>
</html>