<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1488 from Anon (session_user_id: 7eeb18d8b6504007e1b06d1b610a96056a3bf926)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1488 from Anon (session_user_id: 7eeb18d8b6504007e1b06d1b610a96056a3bf926)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>Of all CpG dinucleotides in a mammalian genome, 70-80% are methylated in somatic cells. However, CpG islands (CGI), found in 60% of the promoters, are under-methylated irrespective of their expression state, whereas bodies of highly expressed genes are methylated.</p>
<p>Intragenic methylation is found at repeat elements—satellite DNA and retroviral elements like LINEs and SINEs. Analysis of the methylome shows lineage-specific methylated or demethylated regions. Methylation at non-CGI sites is generally inversely related to expression; it may repress transcription from alternative start sites and/or antisense strand, control alternate RNA splicing, and influence start of DNA replication within S phase.</p>
<p>Aberrant methylation contributes to disease by inappropriate activation of silencing of affected regions.</p>
<p>Cancer epigenomes show global hypomethylation—preferentially at repeat elements and CGI shores—but local hypermethylation--usually at CGI, termed CIMP (for CpG island methylator phenotype). Overexpression of satellite repeats can cause genome instability. CGI hypomethylation is associated with silencing of associated genes; CGI of tumor suppressor genes are commonly hypomethylated in neoplasms. </p>
<p>Cancer specific differentially methylated regions (cDMRs) can include large hypervariable blocks covering over half the genome; such regions show differences (upregulation or downregulation) in gene expression patterns, involving genes that regulate tumor-associated processes such as cell division and matrix remodeling. In some cancers, large hypomethylated regions cover late replication foci and nuclear lamina-associated regions. <span style="font-size:14px;line-height:21px;">Another region of aberrant methylation in cancer cells are T-UCR (transcribed ultraconserved regions) which produce ncRNA, but their function is unknown.</span></p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>H19 and Igf2 genes are located 90 kb apart on human chromosome 11p15.5. H19 produces a 2.5-kb noncoding, spliced, and polyadenylated RNA of yet-unknown function. Igf2 encodes a fetal growth factor: insulin-like growth factor 2.<span style="font-size:14px;line-height:21px;"> </span></p>
<p><span style="font-size:14px;line-height:21px;">H19 and Igf2 are oppositely imprinted by a imprinting control region (ICR) located between the two genes. This ICR is a cis-acting CTCF-dependent insulator. </span></p>
<p><span style="font-size:14px;line-height:21px;">The ICR is hypomethylated on maternal chromosome; thus CTCF can bind to the 4 binding sites in ICR and prevent enhancers located downstream of H19 from activating Igf2. </span><span style="font-size:14px;line-height:21px;">ICR is hypermethylated on paternal chromosome, so it cannot bind CTCF and enhancers can activate expression of IgF2. Thus, H19 is maternally expressed and Igf2 paternally expressed.</span></p>
<p><span style="font-size:14px;line-height:21px;">Imprinting of these genes is lost in many cancers – like Wilm’s nephroblastoma, and some lung, colon, and ovarian tumors – where ICR on maternal chromosome also gets hypermethylated. Then level of H19 RNA is reduced by upto 20-fold and level of Igf2 increases. One hypothesis is that if H19 represses IGF2, then loss of H19 results in overexpression of IGF2, promoting tumorigenesis.</span></p>
<p>Aberrant imprinting at IGF2 and CDKN1C genes causes <span>Beckwith-Wiedemann fetal overgrowth syndrome.</span></p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine (DAC) is a cytidine analog (5-aza-2'-deoxycytidine)<span>, used to</span><span> treat myelodysplastic syndromes and acute myeloid leukemia. </span></p>
<p><span>DAC inhibits DNA methyltransferase, resulting in hypomethylation of DNA. <span>DAC causes an increase of cells in G2/M phases.</span></span></p>
<p><span>Since many tumors have hypermethylation of promoters and CGI of tumor suppressor genes which is associated with reduced expression, hypomethylation may restore their expression to normal level and restore control over cell division.</span></p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>DNA methylation is mitotically heritable in somatic cells through the action of DNA methyltransferases. Hence hypomethylation of tumor cells by demethylating agents can be transmitted to their daughter cells and make them less "tumor-like". </p>
<p>In normal cells, methylation sites are established during two critical periods of life: (a) early embryogenesis (fertilization to blastocyst), and (b) establishment of primordial germ cells and differentiation into gametes. </p>
<p>During embyrogenesis, paternal chromosomes are actively demethylated and maternal chromosomes are passively demethylated (imprinted loci may escape this fate); this corresponds with de-differentiation into totipotent and multipotent stages. Tissue-specific methylation patterns are established as cells become progressively more differentiated. </p>
<p>In primordial germ cells, epigenetic marks specific for differentiating somatic cells are erased and gamete-specific methylation patterns are established. </p>
<p>Exposing patients to methylation inhibitors (drugs or environmental factors like stress) during these sensitive periods can produce aberrant methylation during embryonic development (e.g., maternal grooming affects methylation of steroid receptors in mouse pups; parental smoking or starvation affects metabolic profile in human children and grandchildren), and these patterns are then transmitted to somatic cells and germ cells of the offspring. Hence exposure to demethylating agents during sensitive periods carries risk for future generations.</p>
<p> </p></div>
  </body>
</html>