<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1100 from Anon (session_user_id: a683f77f6e410fe177850762f923704b0b6ddc8b)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1100 from Anon (session_user_id: a683f77f6e410fe177850762f923704b0b6ddc8b)</h1>
    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine is a histone-deacetylase (HDAC) inibitor. HDACs remove acetyl groups from histones enabling them to wrap DNA more tightly. So, decitabine impacts histone acetylation rather than DNA methylation.  </p>
<p>Inhibition of HDAC results in greater histone acetylation, therefore the histones are not able to pack the DNA as tightly and the resulting chromatin structure is more open. </p>
<p>The simplest, or most naive, hypothesis is that with a more open chromatin structure, genes in the open regions are more transcriptionally accessible and thus more highly expressed. If these genes include tumor supressor genes, this could account for the anti-tumor effect. Of course, without more detailed studies, we shouldn't discount more subtle or complex mechanisms that result in the same effect.  </p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Changes to the state of methylation of DNA are mitotically heritable. This means that the initial change in methylation, e.g. to either add or remove methyl groups from DNA, is passed on to the daughter cells and all subsequent cell generations. These durg induced changes can therefore be made permanent.</p>
<p>During a senstive period, a cell undergoes signficant epigenetic reprogramming: existing epigentic marks are cleared followed by setting of new marks.</p>
<p>There are two sensitive periods of development: primoridal germ cell development and early embryonic development. </p>
<p>Treating patients during a sensitive period with drugs that alter the epigenetic state would likely cause adverse impacts. The incomplete clearing of previous epigenetic marks or improper formation of new marks would likley result in a gamete or embryo with significant epigenetic deviations resulting in signficant developmental defects or death. For example, correct genomic imprinting is known to be critical for embryonic viability.  </p>
<p>Note that treating a woman who is pregnant with epigenetic drugs could have an impact on both her embryo (i.e. during early development) but also the embryo's future offspring (i.e. during primoridal germ cell development) depending on the timing and duration of therapy.  </p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>In the paternal allele of a normal cell, the imprint control region (ICR) upstream of H19 is methylated. This prevents the binding of CTCF to the ICR. Without CTCF acting as an insulator, the downstream enhancers are free to act on Igf2 which results in expression of Igf2 from the paternal allele.</p>
<p>In the maternal allele of a normal cell, the ICR of H19 is <em>not</em> methylated. This allows CTCF to bind to the ICR and act as an insulator, that is, the downstream enhancers act on H19 rather than Igf2, and so Igf2 is silenced in the maternal allele. </p>
<p>In Wilm's tumor, the maternal allele is also methylated resulting in the same expression pattern for both the matermal and paternal alleles: H19 is slienced and Igf2 is expressed with a double dose. </p>
<p>With loss of imprinting, the growth promoting gene Igf2 is over expressed. Over expression of a growth promoting gene can contribute, or lead, to tumorgenesis. Note that loss of imprinting is seen in pre-neoplastic tissues and could therefore be an effective, early diagnostic. </p></div>

    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>In normal cells CpG islands, dense CpG regions found in approximately 60% of promotors, are normally kept free of methylation independent of gene acitvity or expression. </p>
<p>In contrast, the CpG islands of cancer cells tend to be hypermethylated. This higher frequency of methylation at CpG islands in cancer cells <span>was termed CIMP (CpG island methylator phenotype). </span></p>
<p>Methylation of CpG islands within gene promoters results in silencing of the underlying gene. Genes that are silenced include <span>tumor suppressor genes controlling cell cycle, apoptosis and DNA repair. Loss of expression of tumor suppressing genes is clearly a contributing factor leading to tumorgenesis or cancer.</span></p>
<p><span>In normal cells, intergenic regions and repeats tend to be methylated. This ensures genomic stability, for example, by: silencing crytpic transcription start or splice sites; silencing of repeats to prevent transposition; preventing or inhibiting illegitimate recombination. </span></p>
<p><span>In cancer cells, general genome wide hypomethylation is observed which includes intergenic regions and repetitive elements. This was the earliest epigenetic anamoly associated with cancer and has been observed in all cancers studied to date. </span></p>
<p>Hypomethylation leads to genomic instalbility characterized by abnormal karytotypes (deletions, insertions, and recirpocal translocations), specifically: illegitimate recombination between repeats, activation of repeats and tranposition, and activation of cryptic promoters and transcription start and splice sites.</p>
<p>Evidence that hypomethylation leads to genomic instability is supported by: mouse models where Dnmt1 has been deleted and leads to genomic instalbility; mutation of DNMT3B in human ICF syndrome which is characteried by genomic instability; and observational evidence from human cancer studies. </p></div>
  </body>
</html>