<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 511 from Anon (session_user_id: a7bce430d52095b4909ab8745d3f22cc1a15aad9)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 511 from Anon (session_user_id: a7bce430d52095b4909ab8745d3f22cc1a15aad9)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value">In normal cells with regard to DNA Methylation you would expect to find hypomethylation of CpG islands and hypermethylation of repetitive and intergenic regions. In cancer cells this is reversed and you get hypermethylation of CpG Islands and hypomethylation of repetitive and intergenic regions. Not all CpG islands will be methylated, but CpG islands found in the promoter regions of tumour suppressor cells are found to be methylated in most tumour cells. DNA methylation at CpG islands is normally associated with silencing of gene expression, this is done by condensation of the chromatin in these areas either by binding with methylated CpG proteins, MeCP1 and MeCP2, or by these MeCP proteins recruiting other factors that condense the chromatin. Methylation of the promoters of tumour suppressor genes acts like a genetic mutation preventing cell death and allowing proliferation of the tumour cell. The normal function of DNA Methylation at intergenic  regions is to maintain genomic stability and at repetitive regions it is also to do this by silencing repeats thus preventing transpositions, illegitimate recombinations and transcriptional interference.. DNA hypomethylation of intergenic and repetitive regions of the genome in cancer cells leads to genomic instability. This then allows illegitimate recombinations between repeats that would not be able to occur under normal methylated conditions and reciprocal translocations between chromosomes. This genomic instability can also lead to insertion and deletion abnormalities in the genome. DNA hypomethylation of these areas and genomic instability is a common feature in many cancers</div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value">Changes in DNA methylation at Imprint Control regions can lead to loss of expression of growth restricting genes and over expression of growth promoting genes. One of the problems with cancer cells is reduced cell death and increased proliferation. For an example the H19/Igf2 cluster;  normally on the maternal allele, binding of CTCF ( an insulator protein) , at the Imprint Control Region (ICR) for Igf2, (which is an oncogene i.e. promotes cell growth), insulates downstream enhancers from the action of Igf2 and allows expression of H19. On the paternal allele DNA Methylation of this ICR prevents the CTCF from binding which then spreads to the H19 promoter causing methylation there and hence silencing and allowing the downstream enhancers to enhance expression of Igf2. In Wilm's tumour there is loss of imprinting in the maternal allele causing methylation of the ICR on the maternal allele (it behaves like the paternal allele) which then means that there is no expression of H19 (which helps restrict growth) and a double dose of Igf2 increasing tumour cell proliferation</div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value">Decitabine is a DNA methyltransferase inhibitor -  it is incorporated into the DNA strands after cell division and once this occurs prevents the DNA methyltransferase from methylating hemi-methylated daughter strands and therefore leads to demethylation in subsequent cell division.  It is used to treat tumours in Myelodysplastic syndrome by reducing the methylation in tumour suppressor cells. This restores their normal function which is to maintain control over cell growth. It also belongs to a group of chemotherapy called  antimetabolites which interact with a number of targets to produce a direct cytotoxic effect on the fast dividing cancer cells causing cell death</div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value">Epigenetic marks are mitotically heritable therefore if we give someone a drug which alters the DNA Methylation in the genome this may then be passed down into the next generation of cells. We do not know what effect removing the DNA Methylation genome wide may have or if  The sensitive periods are the two periods of epigenetic reprogramming.  During primordial germ cell development all epigenetic marks are removed from the somatic cells from which the egg or sperm came and a fresh set of  unique to egg or sperm is laid down to allow the expression of genes needed for these cell. During early development the epigenetic marks from the zygote are removed to allow the development of a todipotent cell that can develop into the different types of cell needed in the adult. To administer treatment that may affect the whole epigenome at such times may interfere with these vital processes, inhibiting normal growth  and may have a deleterious effect. Thus treating pregnant women who are growing a foetus with immature epigenetic marks and young children whose germ cells will not yet be mature would be inadvisable</div>
  </body>
</html>