<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1368 from Anon (session_user_id: 8c7e17ea2318f732cba359e87e80439b41f435c2)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1368 from Anon (session_user_id: 8c7e17ea2318f732cba359e87e80439b41f435c2)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p class="s8"><span><span class="s3">For normal cell functioning, appropriate DNA methylation is critical.   In the developing embryo, DNA methylation patterns are laid out in a tissue-specific manner by targeting </span></span>CpG islands in regulatory/promoter regions of genes and by regulating chromatin remodeling.  In addition, DNA methylation targets repetitive elements throughout the genome, particularly in CpG poor regions, which helps suppress the expression of potentially harmful tranposon sequences.</p>
<p class="s8"><span><span class="s3">In cancer, methylation patterns are abnormal, with DNA exhibiting global hypomethylation accompanied by region-specific hypermethylation.  Global hypomethylation targets the normally inactive (ie methylated) intergenic regions throughout the genome; this induces activation of transposable elements.  The expression of these elements alone may have deleterious effects; however these elements may also insert at random into other parts of the genome, with the potential of creating genomic instability.  Additionally, methylation changes, particularly hypomethylation, have been obserrved at imprint control regions (ICR) with subsequent loss of imprinting of parental gene.</span></span></p>
<p class="s8"><span><span class="s3">Cancer cells also display unusual gains in DNA methlyation in CpG islands at promoter regions of genes, which </span></span><span class="s3">results in dysregulation of gene expression including the silencing of tumor-suppresor genes.  Recent studies suggest that in almost all cancer cells, DNA hypermethylation affects 5-10% of CpG islands in promotor regions that never contained DNA methylation from embryogenesis onwards.   In several cancers, selective hypermethylation at promotor CpG islands causes silencing of genes involved in many critical process including DNA repair (MLH1, BRCA1) and cell-cycle regulation (eg.p16INK4a, RB).  </span> </p>
<div>
<div><span class="s3"><br /></span></div>
</div></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p class="s8"><span><span class="s3">Imprinting at the H19/igf2 locus involves the simultaneous silencing of the maternal allele for Igf2 and paternal allele for H19</span><span class="s3"> and</span><span class="s3"> </span><span class="s3">transcription of maternal H19 and paternal Igf2 genes.  </span><span class="s3">On the maternal chromosome, the</span><span class="s3"> imprinting control region (ICR) upstream from </span><span class="s3">the</span><span class="s3"> H19 gene</span><span class="s3">  is u</span><span class="s3">nmethylated and acts as an insulator, allowing </span><span class="s3">CRC</span><span class="s3">T protein binding; this in turn blocks</span><span class="s3"> activity of the Igf2</span><span class="s3"> in cis</span><span class="s3"> by</span><span class="s3"> </span><span class="s3">in</span><span class="s3">activatin</span><span class="s3">g</span><span class="s3"> distal </span><span class="s3">enhancers</span><span class="s3"> of Igf2</span><span class="s3">.</span><span class="s3">  On the paternal chromosome, the ICR is methylated.  This inhibits CRCF binding; thus the paternal ICR lacks insulator activity, allowing the distal enhancers to promote Igf2 expression.  At the same time, the methylation of the paternal ICR spreads to the H19 promoter, thereby silencing paternal H19 allele.  </span></span></p>
<div>
<p class="s8"><span><span class="s3">In Wilms tumor, which is fetal in origin, alterations in the methylation patterns of Igf2 and H19 genes</span><span class="s3"> </span><span class="s3">have been observed, specifically a gain in methylation at the maternal H19 allele (normally silenced) and a loss of methylation at maternal Igf2 allele (normally silent).  This results in biallelic expression of Igf2 leading to over production of IGF2, a growth factor suspecting of enhancing tumor development.</span></span></p>
<div>
<div><span class="s3"><br /></span></div>
</div>
</div></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p class="s13">The loss of function of a particular gene may be due to genetic mutations and \or epigenetic alterations, but at present, only epigenetic changes are reversible. Decitabine, a hypomethylating agent that acts by inhibiting DNA methyltransferase is used primarily in the treatment of hematopoetic cancers, triggering demethylation, which leads to reactivation of epigenetically silenced genes.  </p>
<p class="s13">Decitabine has demonstrated efficacy in clinical trials, however, the mechanism of its actions are not entirely clear.  It is not known whether the demethylation effects of this drug are gene\region specific, thus it may result in widespread DNA hypomethylation, with unknown consequences.  For example, hypmethylation may activate previously silent tumor promoters (oncogenes), and as discussed in question 1 above, global hypomethylation (leading to genomic imbalance) is a feature of carcinogenesis; thus it may be that seconday cancer may be a long term consequence of this drug (given the long induction period for carcinogenesis).  </p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>The use of epigenetic drugs to restore cell sensitivity to conventional chemotherapeutic drugs is a promising development in cancer treatment, since drug resistance is often a significant obstacle to successful treatment.  Drugs targetting epigenetic marks may silence or reactive abnormally regulated genes and thus induce reactivation of genes involved in a successful drug response. </p>
<p>As an example, drugs that alter DNA methylation reduce DNMT activity, and in subsequent cell division(s) DNA methylation is reduced.  With reduced DNMT activity, daughter cells are increasingly unmethylated, and after repeated cycles of cell division, passive demethylation occurs.  Once this occurs, the original activity of the epigenetic target region is restored.  Notably, these drugs do not affect methylation in resting (non-dividing) cells; as rapid cell replication is a feature of cancer, drugs that target replicating cells would likely spare non-cancerous cells. </p>
<p>Caution should be used in using these drugs during sensitive periods of development where lasting epigenetic markings are laid down.  There are two critical windows in which epigenetic marks are erased - during early embryogenesis (pregnancy) and again in the developing primordial germ cell (ie childhood, early adolescence).  These drugs should not be used during these sensitive windows, since epigenetic marks (such as occurs with DNA demethylating agents) laid down during these times will be permanently carried in all daughter cells.</p>
<p> </p>
<p> </p></div>
  </body>
</html>