<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 463 from Anon (session_user_id: bfec39c9afeec3365bc7a7ca8aff7a3aad31b92c)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 463 from Anon (session_user_id: bfec39c9afeec3365bc7a7ca8aff7a3aad31b92c)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value">2 types of changes in DNA methylation are seen in cancer.<br />1. Locus specific hypermethylation: CpG islands (CGIs) are found in 60% of promoters and are normally unmethylated independent of their activity state; hypermethylation in CGIs in cancer cells results in silencing of tumour suppressor genes as CGIs are normally found in promoters of these genes, which control cell cycle, apoptosis and DNA repair. Conversely. intergenic regions and repetitive elements are methylated in normal tissue; these regions maintain genomic integrity by silencing cryptic  transcription start sites, and preventing transpositions and illegitimate recombinations. All these processes are hallmarks of cancer.  As methylation is mitotically heritable, these epigenetic marks can be transmitted into succeeding generations of cancer cells. Aberrant DNA methylation could arise stochastically due to a loss of fidelity or mutation of epigenetic enzymes such as DNMTs.<br /><br />2. Genome wide DNA hypomethylation: This occurs to some degree in all tumour types tested and the consequence depends on the location affected. Hypomethylation results in genomic instability if present in intergenic regions or repeats, and  activation of oncogenes associated with CpG poor promoters.<br /><br /><br /></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value">Alterations in DNA methylation in imprint control regions, which can either be hyper- or hypomethylation  can result in loss of expression in growth restricting genes or overexpression of growth promoting genes. This has been noted as an early event in many types of neoplasia, and has also been seen in precancerous tissue.<br />In the H19/ IgF2 cluster, the paternal allele is methylated while the maternal allele is unmethylated. In the maternal allele, CTCF is an  insulator element, insulating IGF2 from the downstream enhancer elements which then act on H19 causing its expression. in the paternal allele, methylation of the ICR results in non binding of CTCF, therefore the downstream enhancers cause preferential expression of IgF2. With loss of imprinting, there is expression of IgF2 from the maternal allele as well i.e. there is a double dose of expression of a growth promoting gene. This process has been noted in Wilm's tumour.<br /></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value">Decitabine is a DNA methytransferase (DNMT) inhibitor. which has been approved for the treatment of the haematologic disorder, myelodysplastic syndrome. Unmethylated CpG islands associated with a variety of genes become 
partially or fully methylated in tumors and can be reactivated by decitabine, which is a nucleoside analogue that is incorporated into DNA; it is replication dependent (Christman, Oncogene, 2002) and results in irreversible binding of DNMT. It has been shown to cause depletion of DNA methyltransferases, hypomethylation and direct DNA damage in acute myeloid leukemia cells (Hollenbach, 2010, PLoS ONE 5(2): e9001). This results in inhibition of cell proliferation at high doses and a DNA
 hypomethylation-mediated effect on gene re-expression at low doses, 
affecting processes of cell differentiation, tumor suppression, and 
stimulation of immune mechanisms. All these mechanisms contribute to its antitumour effect.<br /><div>Haematological malignancies are the most sensitive to decitabine, since they are dependent on tumour suppressor gene hypermethylation.</div><br /><br /></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value">Changes in DNA methylation are known to be mitotically heritable,hence they can be passed on to succeeding generations of both normal and cancer cells. Therefore, it is postulated that treatment with epigenetic modifiers could potentially sensitize/ re-sensitize tumours to the effects of standard chemotherapy, used either sequentially or concurrently. This finding was noted in the study by Dr Stephen Baylin. <br /><div>Sensitive periods are those when altered environments have an effect on epigenetic control. </div>The effects on germ cells need to be considered in younger patients as epigenetic reprogramming of DNA methylation occurs during gametogenesis and after fertilization. Clearing of epigenetic marks in germ cell development and early embryonic development means that these are sensitive periods of development, hence treatment during this time i.e in early pregnancy and younger patients (prepubertal) may cause heritable changes, which may manifest in their offspring over many generations.<br /><br /></div>
  </body>
</html>