<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1432 from Anon (session_user_id: e85de0858b4b3752f5235ba00c1daaada84d849b)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1432 from Anon (session_user_id: e85de0858b4b3752f5235ba00c1daaada84d849b)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p style="text-align:justify;">DNA methylation is an epigenetic modification involving the addition of a methyl-group at cytosines within CG dinucleotides found in tandem in CpG islands, or dispersed throughout the genome. It is catalysed by de novo DNA methyltransferases, DNMT3A/3B, and maintained by DNMT1. CGs are clustered into CpG islands (CGI) often at promoters of genes, and their methylation is associated with gene expression silencing and chromatin changes.</p>
<p style="text-align:justify;">In normal cells, CGIs are protected from methylation, which also occurs in intergenic regions and repetitive elements. Methylation in intergenic regions silences cryptic transcription start sites and cryptic splice sites, whereas repetitive element methylation silences them to prevent transposition, transcriptional interference from strong promoters (LTRs) and llegitimate recombination, hence it maintains genome integrity. Methylation at repetitive elements protects the genome from transposable elements (Genome defense model).</p>
<p style="text-align:justify;">Cancer involves activation of oncogenes and inactivation of tumour suppressors. In contrast to normal cells, cancer cells display genome-wide hypomethylation, an early epigenetic aberration, more consistent than genetic mutations (a hallmark of cancer). Global methylation levels decrease progressing from normal through to metastatic tissue; but some CGIs are hypermethylated, particularly in tumour suppressors (locus-specific hypermentylation) leading to their silencing. These epimutations are rapidly selected for, allowing cancer cells to have a competitive proliferative advantage. So DNA methylation can be one of the hits in Knudson hypothesis. In contrast, intergenic intervals/repetitive elements, and CpG-poor promoters, are hypomethylated. So, genomic instability increases in cancer with illegitimate deletions, insertions, reciprocal translocations and chromosome duplications or loss. So hypomethylation at intergenic intervals, and repetitive elements contributes to genomic instability in cancer. Hypomethylation of GpG-poor promoters leads to oncogene activation or tumor suppressor downregulation through activation of certain miRs.</p>
<p style="text-align:justify;">However, the role of DNA methylation is context- and stage-dependent with implications for effectiveness of epigenetic therapy.</p>
<p><br /><a href="https://coursera-uploads.s3.amazonaws.com/user-0a7a2d915d8c170b41487f48/970238/asst-5/970238-5211d54ef36469.26173854.pdf">Q1-A1.pdf</a></p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p> </p>
<p style="text-align:justify;">The imprint control region (ICR) (oval) is unmethylated on the maternal and methylated on the paternal allele (paternally-imprinted).</p>
<p style="text-align:justify;">On the maternal allele, ICR is unmethylated and bound by the CTCF insulator protein which insulates Igf2 from the downstream enhancers (green stars) acting on H19 lncRNA and enhance H19 expression (Igf2 is silent), occurring because of chromatin looping between the enhancers and Igf2.</p>
<p style="text-align:justify;">On the paternal allele ICR is methylated and CTCF doesn't bind to the insulator, so the enhancers act on Igf2 and promote Igf2's expression only from the paternal allele. This is the mechanism of imprinted expression of Igf2. But H19 is no longer active on the paternal allele because of heterochromatin or DNA methylation spreading at the ICR downstream into the H19 promoter, and when it becomes methylated, it is silenced because it is a CGI promoter.</p>
<p style="text-align:justify;">ICR hypermethylation-loss of imprinting is a common, early event in preneoplastic tissue.</p>
<p style="text-align:justify;">With loss of imprinting in cancer, ICR is hypermethylated on the maternal allele as well, so Igf2 is expressed. So Igf2 is overexpressed (double dose) compared to normal cells, and Igf2 is growth promoting; this is associated with Wilm's tumour, a childhood kidney tumour.</p>
<p><br /><a href="https://coursera-uploads.s3.amazonaws.com/user-0a7a2d915d8c170b41487f48/970238/asst-5/970238-5211d63b7ec282.47600417.pdf">Q2-A2.pdf</a></p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p style="text-align:justify;">Decitabine, a DNA methyltransferase inhibitor (DNMTi), acts as a DNA-demethylating agent, and sold as Dacogen by Eisai, a Japanese company. Decitabine is well-tolerated and approved for use in myelodysplastic syndromes (MDS) that progressed to acute myeloid leukemia (AML).</p>
<p style="text-align:justify;">DNMTi's are nucleoside analogs, and get incorporated into the DNA upon replication, and when DNMT1 comes along to bind that nucleotide to copy the methylation to the daughter strand, DNMT1 is then bound irreversibly by decitabine and it can no longer be released. So the action of decitabine would be division-dependent, that is the cells should be replicating. Because cancer cells are dividing much more rapidly than most other cells in the body, they will be more severely affected. MDS is characterised by heavy CGI hypermethylation which is a poor prognostic factor. Decitabine effectively kills MDS cancer cells by acting on hypermethylated CGIs, and perhaps activating tumour suppressors.</p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p style="text-align:justify;">Alterations of DNA methylation in cancer involve genome-wide hypomethylation leading to genomic instability and oncogene activation, and CGI hypermethylation silencing (locus-specific) leading to tumour suppressor (TS) silencing. But the driving role of DNA methylation is context-dependent: Depletion of DNA methylation suppresses tumourigenesis if driven by TS hypermethylation, but enhances tumourigenesis if driven by chromosomal instability. DNA methylation depletion is also stage-specific (early vs. late tumorigenesis), as some tumours change their dependencies throughout their lifespan. These should be addressed in individual tumour types at different stages.</p>
<p style="text-align:justify;">Epigenetic therapy is complicated by mutations in epigenetic modifiers. There are implications for effectiveness of low dose, nontoxic epigenetic therapy in tumours with these dependencies; by minimizing off-target effects will enable cancer cells to undergo reprogramming associated with loss of long-term tumorigenicity.</p>
<p style="text-align:justify;">Periods of epigenetic reprogramming are sensitive periods, predominantly during establishment of epigenetic marks, rather than maintenance. There are two waves of epigenetic reprogramming occurring during primordial and germ cell development, and early embryonic development. Epigenetic therapy of younger patients with developing germ cells in sensitive periods should be considered with caution as it can lead to: loss of imprinting marks, increased genomic instabilty, and altered gene expression.</p>
<p style="text-align:justify;"><br /><a href="https://coursera-uploads.s3.amazonaws.com/user-0a7a2d915d8c170b41487f48/970238/asst-5/970238-5211d8d27b1eb5.45669626.pdf">Q4-A4.pdf</a></p></div>
  </body>
</html>