<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 783 from Anon (session_user_id: 770f1f3c5e0a60bff6bb188c68ad3d3d8b9af2a9)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 783 from Anon (session_user_id: 770f1f3c5e0a60bff6bb188c68ad3d3d8b9af2a9)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><div><span><u>Genetic and cancer</u><br />Cellular transformation manifested in the carcinogenic processes is a broad spectrum disorder triggered through both endogenous and exogenous factors and is sinergistically mediated by genetic and epigenetic mechanisms.</span></div><div><br /></div><div><span><u>DNA methylation</u> </span></div><div>The best-studied epigenetic alterations in cancer are the methylation changes that occur in DNA. These are covalent chemical modifications that in mammals occur mostly at the CpG dinucleotides.</div><div>The modification of the C5 position of the cytosine base is found in approximately 70-80% of CpG dinucleotides in somatic mammalian cells. DNA methylation is the basis for different epigenetic phenomena as the formation of heterochromatin, imprinting or X chromosome inactivation.</div><div>DNA methylation patterns are controlled by a family of DNA methyltransferases and active/passive demethylation . Ussually, methylation is associated with gene expression and unmethylation with gene silencing. Majority sequences susceptible to methylation in DNA are:</div><div>A) CpG islands. Regions with a high frequency of CpG sites, which are present in ~ 70% of human promoters). Usually unmethylated.</div><div>B) Intergenic regions. Regions associated with the maintenance of genomic instability and silencing cryptic transcription start sites or cryptic splice sites. Usually methylated.</div><div>C) Repetitive elements. Also are associated with the maintenance of genomic instability. U<span>sually methylated.</span></div><div><br /></div><div><u>Methylation and cancer</u></div><div>The epigenome of cancer cells display numerous aberrations. Epigenetic changes are involved in every step of the cascade of development and progression of cancer. Best known alterations in the normal methylation patterns are:</div><div>A) Locus specific DNA hypermethylation. Predominantly involves CpG islands and CpG islands shores, resulting in abnormal silencing of tumor suppressor genes, or alterations in DNA methylation at ICRs, resulting in lost of imprinting.</div><div>B) Genome-wide DNA hypomethylation. Predominantly involves repetitive/intergenic regions, causing genomic instability; or CpG poor promoters, activating proto-oncogenes. It can also affect imprinting of growth restricting genes resulting in lost of imprinting or lost of expression.</div></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><div>Genomic imprinting is a monoallelic specific expresion based on parent-of-origin of the allele and associated with DNA methylation established in the primordial germline.<br /></div><div>Expression of imprinted genes is controlled by Imprint Control Regions (ICRs) where the methylation patterns differ between the parental alleles, known as Differentially Methylated Regions (DMRs).</div><div>H19/IGF2 (H19/Igf2 in mouse) cluster of imprinted genes is located on human chromosome 11p15.5 containing maternally expressed H19 and paternally expressed IGF2. In the maternal allele, an insulator protein binds to unmethylated DMR, insulating IGF2 from downstream enhancers. In paternal allele, DNA methylation at ICR blocks binding and spreads to H19 promotor to silence and enhancers can access IGF2 to activate.</div><div>The majority of imprinted genes have roles in growth regulation and are, therefore, potencial oncogenes or tumour suppressors. Lost of imprinting (LOI) results in biallelic expression.</div><div>Lost of IGF2 imprinting due to gain of methylation at the ICR is often associated with Wilms' tumour, the most common paediatric kidney cancer in children. But overexpression of the IGF2 gene is not dependent on LOI. Wilms' tumours with LOI differ in three ways from those with normal imprinting: pathologic subtype, expression of IGF2 and age of the patient.</div></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><div><span><br />Locus especific DNA hypermethylation of tumour suppressor genes is a frequent event in all types of cancer. It is a reversible epigenetic change, </span>thus constituting a good target for therapy. The epigenome is responsive to environmental signals and it is known that epigenetic drugs can cause changes that persist long after exposure has ceased. Genes silenced by aberrant DNA methylation can be reactivated by treatment with new chemical substances. <br />An example is Decitabine (DAC) is a prodrug indicated for the treatment of patients with myelodysplastic syndrome (MDS). Treatment for acute myeloid leukemia and others alterations are under investigation. Chemically as 5-aza-2'-deoxycytidine, DAC is a cytidine analog. The prodrug has to be activated by photophosforylation by deoxycytidine kinasa. Aberrant differentiation in MDS can often be traced to abnormal DNA methylation and alterated cells in the presence of DAC incorporate it into DNA during replication. The incorporation inhibits methyltransferase thereby causing demethylation.</div></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><span>The paradox of the study by Dr. Baylin is controversial. This is an epidemiological review, which mainly are analyzed phenotypic variables. However, involving both the versatility of epigenetic mechanisms, due to the stability and reversibility of its marks, such as different degrees of sensitivity present in the population (sensitive periods vs refractory periods).<br />On the one hand, epigenetic drugs can prolong their effects beyond the exposure. For example,  altering DNA methylation can have enduring effects because DNA methyltransferase enzymes preserve methyl marks during mitosis.<br />Moreover, s</span><span>ensitive period is a phase in the life spam during which an organism  has vulnerability to exogenous stimuli. Particularly, it refers to critical period which epigenome shows increased sensitivity to therapeutic agent. Evidence from animal and human research sugest two sensitive periods surrounding birth (Zhang et al. 2013). Concretely germcell epigenetic reprograming, during gonadal sex determination, and gestation and newborn offspring. In them, drug use would be inadvisable as it could alter the epigenetic reprogramming. S</span>o it is contraindicated to drug administration to the pregnant mother because there would be risks to the fetus.<span><br /></span>In this type of treatment, there is also the possibility <span>of cross-antagonistic effects, for example those produced by genome-wide DNA hypomethylation.<br /></span></div>
  </body>
</html>