<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1410 from Anon (session_user_id: b3bd21f27c5e5b141c22a143931249722c62b9b2)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1410 from Anon (session_user_id: b3bd21f27c5e5b141c22a143931249722c62b9b2)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>The modification of the C5 position of the cytosine base (5 mC) is found in approximately 70–80% of CpG dinucleotides in somatic mammalian cells and to some extend in non-CpG sequences in embryonic stem cells. DNA methylation of promoter regions inversely correlates with gene expression. In cancer cells promoter CpG islands tend to become hypermethylated, which then causes silencing of the underlying gene. Intragenic methylation is found at repetitive sequences such as satellite repeats and remnants of retroviral insertions such as LINEs and SINEs in human DNA. High levels of 5hmC have been detected in brain tissue and ESCs, which might be a prerequisite for active demethylation by the action of DNA glycosylases or passive demethylation during replication. Differentiation specific demethylation was observed in hematopoietic progenitors. The deciphering of the mouse methylome revealed regions of low methylation at distal regulatory elements, which was linked to binding of tissue specific transcription factors. Deposition of histone variants can be replication coupled or independent and an interesting function has been attested to H3.3. H3.3 deposition via the chaperone HIRA was linked locally to RNAPII occupancy at sites of active transcription and a more global gap-filling mechanism to protect genome integrity during transcription or replication.</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>lncRNAs can also contribute to modulation of cell cycle networks in the cell, such as the recently discovered lincRNA-p21, which is induced by p53 and mediates gene repression of p53 target genes by associating with repressor complex heterogeneous nuclear ribonucleoprotein K (hnRNP-K) and targeting the complex to previously active genes. lncRNAs involved in imprinting processes are the paternally expressed lncRNA AIR, which is required for silencing the maternally expressed protein-coding genes Igf2r. H3K27me3 seems to occur mutually exclusive to DNA methylation and promote <em>de novo</em> silencing of genes in different cancers. ncRNA involved in targeting of PCR complexes to tumor suppressor genes is antisense non-coding RNA in the INK4 locus. PTENP1 has been shown to be able to regulate cellular levels of PTEN by detracting miRNAs from PTEN mRNA and is selectively lost in cancer, indicating a tumor suppressor function. Mutations associated with DNA methylation have also been described in hematological disorders. The <em>de novo</em> methyltransferase <em>DNMT3a</em> was found mutated in acute myeloid leukemia (AML) and myelodysplastic syndrome.</p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>The pattern in question is that many of the genes whose breakage leads to cancer are themselves involved in a specific sort of genetic regulation, known as epigenetics. This switches genes on and off by plastering either their DNA or the proteins which support that DNA in chromosomes with clusters of atoms called methyl and acetyl groups. The nature of these reactions means epigenetic processes are susceptible to chemical intervention in a way that genetic mutations are not. A combination of a histone-deacetylase inhibitor and azacitidine slowed tumour growth in some people with advanced lung cancer. This result was notable for two reasons. It was the first time epigenetic drugs had been deployed successfully against a solid tumour, rather than a leukaemia or a lymphoma.</p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>EZH2 enzyme attaches methyl groups to histone proteins, which are part of the chromosomal packaging. A lot of lymphomas cancers of the immune system are caused by mutations that make EZH2 overactive. Such overactivity methylates histones more than they should be and thus silences the genes they surround, including so-called tumour-suppressor genes whose job is to stop the uncontrolled cell growth that causes cancer. Lymphoma cells with a newly developed inhibitor, currently referred to by the unmemorable name GSK2816126, they found that the amount of histone overmethylation declined dramatically. And when they treated both cell cultures and laboratory animals with GSK2816126, they found it also reduces the proliferation of tumour cells while, crucially, having no apparent effect on nearby normal cells. Although neither GSK2816126 nor JQ1 is ready for human trials, two other sorts of epigenetic drugs are already on the market. DNA-demethylating agents, in the form of azacitidine, sold as Vidaza by Celgene, of Summit, New Jersey, and decitabine, sold as Dacogen by Eisai, a Japanese company, are used to treat myelodysplastic syndromes, the precursors of acute myelogenous leukaemia.</p></div>
  </body>
</html>