<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1355 from Anon (session_user_id: 227103a17a95399ce7f5b99d71ea97606d0f0e71)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1355 from Anon (session_user_id: 227103a17a95399ce7f5b99d71ea97606d0f0e71)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>    CpG islands are generally unmethylated in the genome. A small subset of CpG loci are methylated, for these the methylation serves to differentiate between developmental cell fates. Methylation at CpG islands is then one mechanism by which cell differentiation is established via the differential control of gene expression.</p>
<p>    CpG dinucleotides in intergenic regions and in repetitive elements are generally methylated in the genome. Methylation at these sites causes transcriptional repression, and serves to silence inappropriate transcription from transposon genes, cryptic start sites and other deleterious promoter sites. The 'Genome Defense' model put forth by Tim Bestor proposes that the silencing of transposable elements is the primary evolutionary driving force behind DNA methylation.  Methylation of repeats is also important for the maintenance of the integrity of the genome via various mechanisms, such as reducing illegitimate recombination between the repeat regions.</p>
<p>    Most of the genomic DNA of cancer cells is hypomethylated, however CpG islands are hypermethylated. The altered methylation has consequences for the progress of the cancer, as the expression of some key tumor supressors is surpressed by the methylation of CpG islands. The widespread hypomethylation of DNA, which includes intergenic regions and transposons has significant consequences for the oncogenic tranformation in cancer. Inappropriate transcription can occur from formerly supressed cryptic genes present in the intergenic region, and these genes may be regulators of cell growth. Also, and probably more central to the progress of cancer, the loss of methylation can cause instability of the genomic DNA, along with the accumulation of mutational 'hits' per the Knudson hypothesis.</p>
<p> </p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine is a DNA-Methyltransferase inhibitor (DNMTi), which leads to nonspecific reduction in cellular DNA methylation. In most cases this is not a desired effect, as the usual hypomethylation found in cancer cells is generally deleterious as discussed above. However, DNMTi's are effective against some specific malignancies, when hypermethylation at some imprinted loci is a necessary step in the oncogenic progression. In these cases, use of these drugs may cause the loss of inappropriate imprinting at these loci.</p>
<p> </p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Drugs which alter DNA methylation can have lasting effects on a cell and its clonal decendents by altering cellular imprinting, which can otherwise be stably maintained in somatic growth. These drugs can have significant side effects on reproductive cells - either germ cells or embryonic cells. Sensitive periods are those periods in the normal developmental cycle when epigenetic marks are being laid down, either in the germ cells or the (especially early/blastocyst) embryo. The use of epigenetic drugs during these periods can interfere with the normal maintenance of epigenetic imprinting, and result in inviable germ cells or pathologies in offspring.</p>
<p> </p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>In the H19/Igf2 cluster, the imprinting control region (ICR) of the paternal allele is normally methylated, and the maternal ICR is normally unmethylated. In the normal paternal cell, the methylation of the ICR surpresses binding of the CTCF protein and transcription of the H19 locus. Instead, a group of promoters downstream of the locus upregulates expression of the negihboring Igf2 gene. In the maternal cellls, the binding of CTCF causes the association of the promoters with the H19 promoter, and H19 is expressed rather than Igf2.</p>
<p>In Wilm's tumor, both maternal and paternal alleles are methylated, so cells get no H19 gene product, and a double dose of the growth promoting Igf2 growth factor, which appears to be a step in neoplastic transformation and is common in pre-neoplastic tissue.</p></div>
  </body>
</html>