<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1032 from Anon (session_user_id: 1eb6543cb1123781e901c616888e8dfa4f7cb6fc)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1032 from Anon (session_user_id: 1eb6543cb1123781e901c616888e8dfa4f7cb6fc)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><div>CpG islands acts as promoters for gene expression. In normal cells, we see a hypomethylated CpG island, which facilitates gene expression. In cancer, tumor cells are more likely to have hypermethylated CpG islands, which would silence gene expression. Methylated CpG in a CpG island is associated with the formation of a repressive chromatin structure, and with recruitment of other factors that condense the chromatin. This makes the gene almost impossible to transcribe. Hypermethylated CpG islands silencing a tumor suppressor gene is undesirable because it enhances tumorigenesis.<br /></div><div><br /></div><div>Normal cells show methylation of repetitive elements, intergenic regions and introns. In cancer cells, repetitive elements, intergenic regions and introns are more-likely to be hypomethylated or unmethylated. Such a change to intergenic regions causes genomic instability resulting in abnormal karyotypes. Unmethylated repetitive elements can cause transcriptional interference, by virtue of allowing transposition or illegitimate recombination. It also causes activation of cryptic promoters and disruption to neighboring genes.</div></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><div>In maternal allele, ICR and the H19 promoter region is unmethylated. This allows CTCF insulator protein to bind to ICR. CTCF prevents downstream enhancers from acting upon ICR. As a result, they act upon H19, thereby enhancing its expression. Igf2 is maternally imprinted.</div><div><br /></div><div>In paternal allele, ICR is methylated. Therefore, CTCF is unable to bind causing methylation to spread to H19 promoter, thereby silencing it. Enhancers are no longer able to act upon ICR or H19 region. They instead go further upstream enhancing expression of Igf2.    </div><div><br /></div><div>In Wilm's tumor, hypermethylation of ICR in maternal allele prevents CTCF insulator protein from binding to it. Methylation spreads to H19 promoter and silences it. Downstream enhancers therefore activate Igf2 in maternal allele causing overexpression of Igf2 (remember that paternal allele is already expressing Igf2). </div><div><br /></div><div>Disrupting imprinting at H19/Igf2 cluster can cause over or under expression of Igf2 which can cause genomic instability. If there is over expression, we may see excessive uncontrolled growth of cancerous cells.</div></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><div>There are several kinds of epigenetic inhibitors like chromatin remodelling inhibitors, DNMTi (DNA methyl transferase inhibitors), HDACi (histone deacetylase inhibitors), HMTi (histone methyl transferase inhibitors), HATi (histone acetyl transferase inhibitors), HDMTi (histone demthyl transferase inhibitors) etc. </div><div><br />decitabine is a DNMTi. By definition, it reduces DNA methylation. It is to be used when the cancer cells are hypermethylated. When administered in right context and quantity, it can help increase expression of tumor-suppressor genes. Also, once a cancer cell is demethylated, the epigenetic mark remains mitotically heritable, which means the first generation of daughter cells thereafter will not be cancerous. This approach allows us to control cancer without having to kill cancerous cells. It can also help reduce expression of certain imprinted genes like Igf2. All of these benefits collectively give us the anti-tumour effect.</div></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><div>Once a cancer cell is demethylated, the epigenetic mark remains mitotically heritable, which means the first generation of daughter cells thereafter will not be cancerous. This approach allows us to control cancer without having to kill cancerous cells. This is an enduring effect on the epigenome.</div><div><br /></div><div>A sensitive period is a period of active remodelling of the epigenome when food and environment can have a permanent genentic or epigenetic effect on one or more types of developing cells. The genetic or epigenetic change may stay unchanged for the lifetime of the individual.</div><div><br /></div><div>Sensitive periods of development are :</div><div>- period of primordial germ cell development all the way to development of gametes</div><div>- pre-implantation period</div><div>- early post-implanation period</div><div><br /></div><div>Treating patients during sensitive periods is inadvisable because there is no way to precisely target the drug. In other words, we dont have a way to prevent the drug from impacting other epigenetic machinery which is being actively remodelled during the sensitive period. It is also possible that the impact of the drug may be genome wide, which may not be desirable.</div></div>
  </body>
</html>