<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1529 from Anon (session_user_id: 0e332040a2371f6b7fbad0cb1222e5ae6979a78f)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1529 from Anon (session_user_id: 0e332040a2371f6b7fbad0cb1222e5ae6979a78f)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>DNA methylation at CpG islands usually function to control X-inactivation, chromatin configuation and/or to pass heritable genetic information (imprinting) from parents to children. In cancer however, these CpG islands become hypermethylated and silenced, as methylation marks are laid down on these CpG islands which are usually not methylated. Development of the cell then becomes abnormal. since approximately 70% of annotated gene promoters are associated with these CpG islands (<a class="xref-bibr" href="http://genesdev.cshlp.org/content/25/10/1010.full#ref-97">Saxonov et al. 2006</a>), many important genes such as the tumour suppresor genes may be silenced and cell/tumour growth becomes uncontrolled. Other diseases apart from cancer, such as mental retardation may occur as well.</p>
<p>DNA methylation at repititive elements and intergenic regions functions to silence reptitive elements (which can cause overexpression of a certain gene if active) and control transciption start sites at intergnic regions. This ensure genomic stability, where transcriptional interference and transpositions are prevented. In cancer, DNA methylation in these two regions are disrupted and methylation marks are lossed. Hypomethylation is said to have occured. Hence repetitive elements become active and causes overexpression of the particular gene. As for the loss of methylation at intergenic regions, promoters are then active and genes which have a two-way transcriptional direction will experience interference when the RNAs clash with each other during transcription. </p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>On the <strong>paternal</strong> allele the imprint control region (ICR) is methylated, and this epigenetic mark blocks the binding of the CTCF protein.The methylation then spreads to the nearby H19 cluster and silences it. Now the enhancers can acctivate Igf2 and it is expressed. On the other hand, the ICR on the<strong> maternal</strong> allele is NOT methylated, hence allowing the insulator protein CTCF to bind and block the enhancers from activating Igf2. A a secondary preference of the enhancers, a lncRNA H19 will be expressed instead, and Igf2 will not be expressed.</p>
<p>In Wilm's tumour (a kidney cancer), imprinting is lost at the H19/Igf2 gene, and the ICR at the maternal allelle is methylated. As a result, the maternal allele will behave like the paternal allele and lgf2 will be expressed. Hence there will be a double expression of Igf2 which promotes tumour growth. This result of a seemingly double parental allele is called Uniparental Disomy. This person will have the Beckwith-Weidemann Syndrome and is likely to have areas and parts of the body which are abnormally large </p>
<p> </p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine belongs to the class DNMTi that inhibits the activity of DNMT1 (which lays down epigenetic marks). Decitabine targets DNMTs and these inhibitors will bind to DNMTs once they are infused into the DNA. Hence DNMTs cannot lay down methylation marks when the cell undergoes division, and methylations are reduced.</p>
<p>In a tumour, hypermethylation occurs at CpG islands, and tumour supressor genes are often silenced. Cell growth and division is then uncontrolled and the tumour grows. These hypermathylation patterns are often caused by mutated DNMTs which have been found largely in AML and myeloidysplastic syndromes. However, when a DNMTi such as Decitabine is introduced, DNMTs are inhibted and cannot lay down methylations on the daughter cells. Since hypermethylation-induced epigenetic mutations requires the continuous activity of DNMTs during cell division, the tumour growth will then stop as previously silenced tumour supressor genes are now reactivated.</p>
<div> </div></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>DNA methylation is mitotically heritable, and it is not easily removed as there are a few factors which need to be removed simultaneously. </p>
<p>A sensitive period is when the epigenome is most/greatly influenced environmental changes. In this period, epigenetic methylation marks are actively removed, and laid down. These periods are usually when the primordial germ cells are developing (pre-puberty) and when pre-implantation development is peaking. Treating patients during these sensitive periods would be inadvisable, as their epigenome is undergoing epigenetic reprogramming and the drugs will greatly influence it. Imprinted marks may also be disrupted or lost, and offspring may have a mutated epigenome because of this. Thus diseases related to epigenetic mutations all may possibly occur to the person's offspring down the generations!</p></div>
  </body>
</html>