<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1677 from Anon (session_user_id: 307bbfd4f5e70fda2fbadd3fc14d0b9888a13600)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1677 from Anon (session_user_id: 307bbfd4f5e70fda2fbadd3fc14d0b9888a13600)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>CpG islands are made of a large number of CG dinulcleotide sequences located at the promotors of genes. C refers to the cytosine, G to the guanine and p to the bounding between the two. They are usually protected from methylation. When they get methylated it silences the following genes by preventing transcritional factors to bind to the strand, oncogene and growth factor have CpG islands methylated in normal cells. In cancer those CpG islands are hypomethylated this allow transcriptional factors to bind and can lead to the expression of oncogenes or growth factors.</p>
<p>Intergenic and repeat regions are hypermethylated in normal cells. The methylation prevents the overexpression of the repetitive elements, avoids cross over, transposition and recombination. In cancer it was found that these elements were hypomethylated. The consequences are genomic instability (deletion, insertion, disruption of genes located near the modified areas), transcriptional interference, active repetitive elements,  active intergenic/poor CpG promoters leading to oncogene activation. The consequences are epigenetic as well as genetical.</p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine is a DNA demethylating agent.It will remove methyl groupes from the DNA but not from the histones. Its action will then have different impact depending on which part of the genome it has access to: Decitabine could increase the demethylation of the repetitive and intergenic sequences, thereby exacerbate the cancer symtoms further or it could counterbalance the CpG's/ CpG shores's hypermethylation caused by cancer. The latter would for example re establish  the tumor suppressor gene expression. The outcome of Decitabine administration was positive, it can be inferred that hypomethylated region were protected by histones involved in heterochromatin.  The cancer was driven by tumour suppressor hypermethylation not by chromosomal instability. It is not likely that this drug cures the loss of imprinting as it could probably not make a distinciton between the 2 strands. It would be interesting to compare the effect of this drug in different tumour as CIMP have different pattern in each cancer.</p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Methylation is mitotically heritable. Its pattern are conserved through cell divisions. Even if a certain plasticity exists the most important changes in DNA methylation will occur at the sensitive periods, which are the primordial germ cell development, the early development as welll as cell differenciation. During those sensitive periods the DNA is actively and passively demethylated before being remethylated. If a drug remodelling the methylation were to be used at those periods, the induced modification would be persistent and potentially all cells would be impacted. If the demethylation was to be partial due to the drugs the totipotency in germ cell could be jeopardized. If the germ cell were not correctly remethylated it could cause infertility issues similarly to the ones observed in assisted reproductive technologies where the maternal germ cell were taken too early. More importantly it could disturb the imprinting in the germ cells and therefore have a repercussion on several generations. For the individual who received a drug enhancing methylation it could encourage mutation as methylated cytosine are prone to deamination to thymine.</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>The H19/Igf2 is a cluster of imprinted genes controlled by an imprinting control reigion.</p>
<p>The normal imprinting of the H19/lgf2 cluster is thz following:</p>
<p>On the paternal strand the imprinting control region is methylated which prevents the binding of the CTCF insulator protein. The promoter of the H19 gene located near the ICR is also methylated preventing its expression. This causes the enhancers to activate the expression of lgf2 located upstream. On the materna strand the ICR and the H19 promoters are left unmethylated, CTCF binds to the ICR which leads to the reading of the following H19 sequence. This transcription is activated by the enhancers located downstream of H19.</p>
<p>In Wilm's tumour both strands are methylated, no CTCF protein can bind to the strands hence lgf2 is doubly expressed. As H19 codes for a ncRNA involved in tumour suppresion and as Igf2 is an insulin-like growth factor, by suppressing the H19 expression and doubling the expression of Igf2 tumours are likely to grow. </p>
<p>Disturbing an imprinted cluster deregulates a fine tuned balance.</p></div>
  </body>
</html>