<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 565 from Anon (session_user_id: 1c5453d850a341a33991e9387faa0c4ca9ee32ee)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 565 from Anon (session_user_id: 1c5453d850a341a33991e9387faa0c4ca9ee32ee)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>DNA methylation is a very stable epigenetic mark. It is found in approximately 70-80% of CpG dinucleotides in somatic mammalian cells. DNA methylation is required for proper embryogenesis and development, for sustaining chromosomal stability, telomere length and predetermined gene expression states. Generally, DNA methylation at the CpG islands usually inversely correlates with gene expression, i.e., if methylated silences the expression. In the genome these sites are usually hypomethylated. In contrast, intergenic regions and repetitive elements are usually hypermethylated which is associated with greater genomic stability as these regions easily “jump” around de DNA. In cancerous cells, the epigenome displays changes such as genome-wide loss and a regional gain of DNA methylation, leading to genomic instability and deregulation of tissue specific and imprinted genes. A reverse pattern of methylation is observed in cancer cells for instance, at CpG islands there is hypermethylation marks at the promoter sites of tumor suppressor genes that silences the expression of those genes that results in loss control of the cell cycle, apoptosis or DNA repair machinery which promotes cell growth and tumour progression. Moreover, DNA hypomethylation at the intergenic regions and repetitive elements is associated with greater genome instability and copying errors such as deletions and insertions, etc. Both effects promote growth and poorly or undifferentiated cells.</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>In human cells the H19/Igf2 cluster is found in chromosome 11 and its epigenetically controlled by enhancer blocking. This cluster has mono-allelic specific gene expression at imprint control region (ICR), i.e., this region is methylated at the paternal allele (paternally imprinted), but un-methylated at the maternal allele. Physiologically, the maternal allele at the ICR is unmethylated and its bound by an insulator protein CTCF, which insulates Igf2 gene from the enhancers down-stream, secondarily by chromatin looping this enhancers act on the H19 gene which expresses a long non-coding RNA (lncRNA). On the paternal allele as the ICR its methylated, the CTCF its unbound so the preferred chromatin looping binds on Igf2 gene promoting its expression, secondarily H19 is inactive due to heterochromatin spreading (also known as DNA methylation spreading).  Thus, in a mammalian cell there is normal gene dose of both H19 from maternal strand and Igf2 from paternal strand. In Wilms tumour both alleles act as a paternal allele from various abnormalities (e.g. mutation/deletion or paternal uniparental disomy), therefore there is an upregulation of Igf2 and as it is an oncogene it promotes cell growth, leading to cancer in these patients.</p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine is an antimetabolite that acts as a nucleotide analogue of cytosine, therefore it is an anti-neoplastic agent.  It is believed to have a dual mechanism of action that is dose-dependant. Nevertheless, this drug incorporates into the DNA and at higher doses it forms a nonreversible covalent bond with DNA methyl-transferases (DNMTs). The mechanism responsible for the anti-tumor effect, at the epigenetic level, is related to its reversal of epigenetic silencing by aberrant DNA methylation of genes that suppress neoplastic growth. Moreover, by inhibiting DNMTs results in hypomethylation of the DNA in specific loci (such as hypermethylation at the promoters of tumor suppressor genes) and the subsequent re-expression of these genes, thereby promoting cell differentiation, stimulation of immune mechanisms, and suppression of tumor growth (Leone et al 2002; Mund et al 2005).</p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>DNA methylation is a very stable epigenetic mark secondary to the activity of DNA methyl-transferases (DNMTs) which ensure that is mitotically heritably. Pharmacological interventions at this level will be passed on from cancerous cell to its daughters cells, removing possible methyl groups from tumor supressor promoters or genes and making DNA more accesible to other antineoplastic agents. However, as any potential drug that will have a global effect can be detrimental in sensitive periods of epigenome reprogramming, where a cell is susceptible to environmental signals. In mammals this period, comprises from gametes through the fertilised egg, blastocyst and epyblast.  Subsequently treating patients during these periods, i.e. a pregnant woman would be inadvisable as a lot of reprogramming happens in early embryonic development, as well as during the formation of gametes (both male and female) in utero and might be cause of posterior altered phenotypes observed in the fetus/newborn or in subsequent generations, or even abortions (as genome wide hypomethylation leads to genomic instability).</p></div>
  </body>
</html>