<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 346 from Anon (session_user_id: e6fd0f09d8a5ce4774974a89eb2b8284b6c02775)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 346 from Anon (session_user_id: e6fd0f09d8a5ce4774974a89eb2b8284b6c02775)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>DNA methylation is an epigenetic mechanism that involves the addition of a methyl-group at cytocines in CG dinucleotides. GC nucleotides are either found in tandem, in CpG islands, or dispersed throughout the genome. CpG island are DNA areas that have many CGs and are most commonly not methylated in normal cells. If they are methylated they result in silencing of neighbouring genes and heterochromatin formation. That mechanism is commonly used to regulate imprinted genes. DNA methylation also occurs in CGs of intergenic regions, mostly to prevent cryptic transcription start sites and cryptic splice sites and in repetitive DNA elements to prevent transposition, recombination and induction of transcription from repetitive DNA elements promoters (mostly Long Terminal Repeats-LTRs).</p>
<p> There are two ways that DNA methylation is altered during cancer:</p>
<p>First, a DNA hypermethylation at CpG islands occurs some CGIs are hypermethylated, particularly in tumour suppressors (locus-specific hypermentylation) leading to their silencing. These epimutations can be rapidly selected for, providing a competitive proliferative advantage and contribute to the cancer phenotype.</p>
<p>Second, a DNA hypomethylation to global methylation (non CpG) levels also occurs. In that way, genomic instability is induced from activation of repetitive DNA elements and the induction of transcription and splicing at intergenic regions.</p>
<p>   Both alterations are dynamic and associated with the type and stage of tumor.</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>The IGF2/H19 locus is paternally imprinted, which means that the imprint control region (ICR) is unmethylated on the maternal and methylated on the paternal allele.</p>
<p> </p>
<p>On the maternal allele ICR is unmethylated and bound by the CTCF insulator protein which insulates Igf2 from the downstream enhancers which act on H19 lncRNA and enhance H19 expression</p>
<p> </p>
<p>On the paternal allele ICR is methylated and CTCF doesn't bind to the insulator, so the enhancers act on Igf2 and promote Igf2's expression only from the paternal allele. H19 is not active on the paternal allele because of heterochromatin at the ICR downstream into the H19 promoter.</p>
<p> </p>
<p>In cancer tissues, like Wilm's tumour, a childhood kidney cancer ICR methylation occurs in both alleles and imprinting is lost. In both alleles Igf2 is expressed, resulting in double transcripts compared to a normal cell, promoting tumor growth.</p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine is a DNA methyltransferase inhibitor (DNMTi) that inhibits DNA  methyltransferase 1 (DNMT1), resulting in DNA-demethylation.</p>
<p>DNMTi's are nucleoside analogs that bound irreversibly DNMT1, upon DNA replication.</p>
<p> The action of decitabine is division-dependent, which means that cells should be actively replicating. Because cancer cells are replicating faster than most normal cells, they will be more affected by decitabine. Decitabine effectively kills cancer cells by acting on hypermethylated CGIs, perhaps by activating tumour suppressor genes.</p>
<p> </p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>DNA methylation is altered during cancer in two ways:</p>
<p>1) DNA hypermethylation at CpG islands occurs some CGIs are hypermethylated, particularly in tumour suppressors (locus-specific hypermentylation) leading to their silencing. </p>
<p>2) DNA hypomethylation to global methylation (non CpG) levels of repetitive DNA and intergenic regions also occurs. In that way, genomic instability is induced.</p>
<p>DNA demethylating agents can either supress tumourigenicity (if driven by CpG hypermethylation) or induce genomic instability (driven by DNA hypomethylation of intergenic regions and repetitive DNA). Thus, the action of DNA demethylating agents is context specific and is also dependent on the stage of the tumor.</p>
<p> Periods of epigenetic reprogramming are sensitive periods, predominantly during establishment of epigenetic marks. There are two waves of epigenetic reprogramming occurring during primordial and germ cell development, and early embryonic development.</p>
<p>DNA demethylating agents can affect the establishment of epigenetic marks during sensitive periods and:</p>
<p>1) lead to loss of imprinting marks,</p>
<p>2) lead to activation of repetitive DNA and induction of genomic instabilty, and</p>
<p>3) alter the epigenetic state of fate-specific genes and essential transcription factors for development.</p>
<p>So epigenetic therapy of younger patients with developing germ cells in sensitive periods should be considered with much caution.</p></div>
  </body>
</html>