<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 442 from Anon (session_user_id: 1da9b7d86b6d5637da00d9bf723a9396247d794f)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 442 from Anon (session_user_id: 1da9b7d86b6d5637da00d9bf723a9396247d794f)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>Alteration of DNA methylation patterns is a hallmark of cancer [16]. Numerous studies describe repression of tumor suppressor genes (TSG) involved in various cellular pathways (cell cycle, apoptosis or genome maintenance) during carcinogenesis by DNA hypermethylation of their promoters. Paradoxically, cancer cells exhibit a global genome hypomethylation that leads to genomic instability and re-expression of silenced genes [16,17]. Mechanisms underlying this paradox are still not clearly explained. Wild and Flanagan depict current knowledge on genome wide DNA hypomethylation associated with cancer [18]. Briefly, two competing theories of “passive” vs. “active” demethylation processes could explain this phenomenon. The former implies a disruption of the link between histone modifications and DNA methylation establishment, an aberrant localization of DNMT1 to DNA damage sites or a metabolic imbalance favoring a decrease in the methyl group donor, S-adenosyl-methionine. Conversely, the latter theory relies on a class of enzymes harboring a demethylase activity. The TET protein family (Ten Eleven Translocation proteins) is described to actively demethylate methyl-cytosines by their oxidization and elimination through different mechanisms in physiological conditions [19]. Briefly, the TET enzyme family facilitates passive DNA demethylation by oxidizing methyl-cytosines to 5-hydroxyl-methylcytosines (5 hmC) leading to a considerable reductions in UHRF1 binding (ubiquitin-like containing PHD and RING finger domains) and in DNMT1 methyltransferase activity at the replication fork [20,21]. A second mechanism involves the DNA repair pathway. Hydroxy-methylcytosines are converted either by further oxidization or by deamination that leads to a nucleotide mismatch, which will be excised and replaced<br />by a cytosine [22,23]. Last, DNMT3a demonstrates methyltransferase activity in reducing conditions and conversely, dehydroxymethylation in oxidizing conditions that converts 5 hmC in cytosines [23].</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p><span>The </span><em>H19/IGFf2</em><span> locus belongs to a large imprinted domain located on human chromosome 11p15.5 (homologue to mouse distal chromosome 7). The </span><em>H19</em><span> gene is expressed from the maternal allele, while </span><em>IGF2</em><span> is paternally expressed. Natural antisense transcripts and intergenic transcription have been involved in many aspects of </span><span class="adtext">eukaryotic gene expression</span><span>, including genomic imprinting and RNA interference. However, apart from the identification of some </span><em>IGF2</em><span> antisense transcripts, few data are available on that topic at the </span><em>H19/IGF2</em><span> locus. We identify here a novel transcriptional activity at both the human and the mouse </span><em>H19/IGF2</em><span>imprinted loci. This activity occurs antisense to the </span><em>H19</em><span> gene and has the potential to produce a single 120-kb transcript that we called the </span><em>91H</em><span> RNA. This nuclear and short-lived RNA is not imprinted in mouse but is expressed predominantly from the maternal allele in both mice and humans within the </span><em>H19</em><span>gene region. Moreover, the transcript is stabilized in </span><span class="adtext">breast cancer cells</span><span> and overexpressed in human breast tumors. Finally, knockdown experiments showed that, in humans, </span><em>91H</em><span>, rather than affecting </span><em>H19</em><span> expression, regulates </span><em>IGF2</em><span>expression in </span><em>trans</em><span>.</span></p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p><span>5-Aza-2′-deoxycytidine (5-AZA-CdR, decitabine), an epigenetic drug that inhibits DNA methylation, is currently used to treat myelodysplastic syndrome (MDS), and is under investigation for treating acute </span><span class="adtext">myeloid leukemia</span><span> (AML) and other malignancies. 5-AZA-CdR can reactivate tumor suppressor genes silenced by aberrant DNA methylation, a frequent event in all types of cancer. Because this epigenetic change is reversible, it is a good target for 5-AZA-CdR therapy. We have reviewed the preclinical data of 5-AZA-CdR to analyze the concentrations and exposure times required to eradicate cancer stem cells. We analyzed the dose-schedules used in animal models that show potent antineoplastic activity of 5-AZA-CdR. We attempted to correlate the preclinical data with the responses obtained in clinical trials of 5-AZA-CdR in patients with cancer. </span></p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p><span>The drugs appear to alter gene activity in stem cell-like cancer cells—cells that are capable of self-renewal and that studies have suggested are inherently resistant to most current therapies. It is <span>found that low doses of the drugs had antitumor effects in cell lines</span><span> and in mouse models</span><span> of different cancer types—including leukemia </span><span>and breast </span><span>and colon</span><span> cancer. Post-treatment analyses of treated cells showed decreased DNA methylation and the reactivation of genes that can affect tumor growth and cell death. <span>The drugs may be having other effects. Epigenetic therapies may affect the structure of chromatin,</span><span> the complex of DNA and proteins that forms chromosomes, which could cause side effects over a long period. In patients with advanced cancer, who have few treatment options, this may not be a significant concern. But as we push these agents forward into earlier disease, it’s something we’ll need to look at very carefully.</span></span></span></p></div>
  </body>
</html>