<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 781 from Anon (session_user_id: a6d0d2a812122b5b3bc1277b84b547b870b1c218)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 781 from Anon (session_user_id: a6d0d2a812122b5b3bc1277b84b547b870b1c218)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>In normal cells, <span>CpG islands <span>have a high CpG density and are usually kept free of methylation independent of their activity state. <span>However, in cancer cells promoter CpG islands tend to become hypermethylated, which then causes silencing of the underlying gene. Hypermethylation of CpG islands silences tumour suppressor genes in cancer. D<span>isruption of DNA methylation at CpG islands <span>contributes to disease by silencing of tumour suppressor genes. I<span>ntergenic regions and repetitive elements are <span>methylated in normal cells whereas they are hypomethylated in cancer cells. Low levels of methylation in repetitive elements causes genomic instability resulting in deletions, reciprocal translocations and insertions which leads to disease.</span></span></span></span></span></span></span></p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p><span>Decitabine belongs to the class of <span>DNA-demethylating agents. It belongs to a class of subsatnce known as <span>JQ1, which inhibits an epigenetic regulator called BRD4 and blocks the activity of a gene by the name of </span><em class="Italic">Myc</em><span>. Thus Decitabine blocks the activity of the gene <em class="Italic">Myc indirectly. </em>Decitabine is used to treat myelodysplastic syndromes, the precursors of acute myelogenous leukaemia. Since Decitabine is a demethylating agent it demethylates CPG islands thus having an anti-tumour effect.</span></span></span></p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>In normal cells  the imprint control region is<span style="font-size:14px;"> methylated on the paternal </span><span style="font-size:14px;">allele, and it's unmethylated on the </span><span style="font-size:14px;">maternal allele. </span><span style="font-size:14px;">When it's unmethylated in the maternal allele CTCF 4 bind the </span><span style="font-size:14px;">insulator element, and it means that the </span><span style="font-size:14px;">enhancers in this case will act on H19. </span><span style="font-size:14px;">But IGF2 will be silent for the maternal </span><span style="font-size:14px;">allele, so we don't see expression. </span><span style="font-size:14px;">On the paternal allele, because the imprint control region is </span><span style="font-size:14px;">methylated, the enhancers c</span><span style="font-size:14px;">an act on IGF2, because CTCF is not </span><span style="font-size:14px;">binding to insulate, and IGF2 is </span><span style="font-size:14px;">expressed from the pattern allele. Now, with </span><span style="font-size:14px;">hypermethylation </span><span style="font-size:14px;">of the imprint control region on the </span><span style="font-size:14px;">maternal allele as well, we have expression of IGF2 on the maternal allele as well. </span><span style="font-size:14px;">So now we have a double dose of IGF2 from both the maternal and the paternal allele in comparison to a normal cell. </span><span style="font-size:14px;">This is associated with Wilm's tumour. </span><span style="font-size:14px;">This is a particular childhood kidney </span><span style="font-size:14px;">genome that is observed. Hence disrupting imprinting at the H19/IGF2 cluster contributes the Wilm's tumor.</span></p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p><span>Epigenetic changes, by altering DNA <span>methylation,</span>  are passed on during cell division to daughter and granddaughter cells until they are actively erased. Once erased, though, they do not return. </span><span style="font-size:14px;">Sensitive periods are </span><span style="font-size:14px;">when the environment may be able to </span><span style="font-size:14px;">influence epigenetic makeup. </span><span style="font-size:14px;">Sensitive periods are </span><span style="font-size:14px;">primordial germ cell </span><span style="font-size:14px;">development all the way through to the </span><span style="font-size:14px;">production of mature eggs and sperms. </span><span style="font-size:14px;">The second sensitive period is the pre</span><span style="font-size:14px;">implantation and early postimplantation </span><span style="font-size:14px;">period. Treating patients during sensitive periods is inadvisable since the treatment is not perfect and may affect genetic regions which we do not want to affect and also evry type of cell may be affected. Since during sensitive periods the cells are susceptible to intervention, and we do not have perfect knowledge of the total genome as yet and methods are not yet fully developed to be able to affect particular sections of the gernome, it is not advisable to treat patients during sensitive periods.</span></p>
<p> </p></div>
  </body>
</html>