<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 2033 from Anon (session_user_id: cb056b65e0a5cc934e01a71bd07d70f1a3e9686b)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 2033 from Anon (session_user_id: cb056b65e0a5cc934e01a71bd07d70f1a3e9686b)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p><span>DNA methylation serves primarily as a defense mechanism to suppress a significant portion of the genome of foreign origin, such as moving the replicated elements, viral sequences and other repetitive sequences. <span>Intragenic methylation is found at repetitive sequences such as satellite repeats and remnants of retroviral insertions such as LINEs and SINEs in human DNA. <span>Although DNA methylation is a very stable epigenetic mark, which is passed on to subsequent cell generations, reprogramming of DNA methylation occurs during gametogenesis and after fertilization or after artificial reprogramming of somatic cells into induced pluripotent stem cells (iPSC).</span></span></span></p>
<p><span><span><span><span>In cancer cells promoter CpG islands tend to become hypermethylated, which then causes silencing of the underlying gene. <span>Unusually high frequency of DNA methylation at CpG rich sites was termed CIMP and was first identified in colorectal cancer. <span>Another target of aberrant DNA methylation in cancer are CpG island shores, defined as approximately 2 kb regions surrounding CpG islands.</span></span></span></span></span></span></p>
<p><span><span><span><span><span><span><span>Overexpression of satellite repeats was also identified in pancreatic and other epithelial cancers lately, indicating that disruption of heterochromatin may result in genomic instability in a variety of human cancers.</span></span></span></span></span></span></span></p>
<p><span><span><span><span><span><span><span><span>Long-range silencing of repetitive sequences and formation of silent heterochromatin but also DNA access for transcription in euchromatin or DNA replication are dependent on epigenetic mechanisms including DNA methylation, histone modification and remodeling, and non-coding RNA. <span>Intragenic methylation is found at repetitive sequences such as satellite repeats and remnants of retroviral insertions such as LINEs and SINEs in human DNA.</span></span></span></span></span></span></span></span></span></p>
<p><span><span><span><span><span><span><span><span><span><span>Another class of non-coding RNAs constitute the recently discovered transcribed ultraconserved regions (T-UCRs)</span><span>, which are transcribed from evolutionary ultraconserved regions found in the human, rat and mouse genome</span><span>. They show aberrant expression in several cancers, including adult chronic lymphocytic leukemia, colorectal and hepatocellular carcinomas and neuroblastomas.</span></span></span></span></span></span></span></span></span></span></p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p><span>Other lncRNAs involved in imprinting processes are the paternally expressed lncRNA AIR, which is required for silencing the maternally expressed protein-coding gene Igf2r <span>and inhibits expression by targeting the H3K9 histone methyltransferase G9a to the Slc22a3 promoter</span><span>, and paternally expressed lncRNA Kcnq1ot1, which mediates lineage-specific silencing of the Kcnq1 locus by interaction with the PRC2 complex and G9a HMT in placenta, but not in fetal liver. <span>Other examples for lncRNAs in the p53 network are the p53-induced lncRNA PANDA and the maternally expressed gene 3 (MEG3).</span></span></span></p>
<p><span>In Wilms tumour and colorectal cancer, biallelic expression of </span><em>IGF2</em><span> has been observed in association with loss of methylation at a DMR in </span><em>IGF2.</em><span> This DMR, known as DMR0, has been shown to be methylated on the silent maternal </span><em>IGF2</em><span> allele presumably with a role in repression.</span></p>
<p><span><em>nsulin-like growth factor 2 (Igf2)</em><span> is a paternally expressed imprinted gene regulating fetal growth, playing an integral role in the development of many tissues including the brain. The parent-of-origin specific expression of </span><em>Igf2</em><span>is largely controlled by allele-specific DNA methylation at CTCF-binding sites in the imprinting control region (ICR), located immediately upstream of the neighboring </span><em>H19</em><span> gene. Previously we reported evidence of a negative correlation between DNA methylation in this region and cerebellum weight in humans.</span></span></p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p><span>Histone-deacetylase inhibitors, made by Celgene and by Merck, another New Jersey-based firm, are being used to treat a rare illness called cutaneous T-cell lymphoma. <span>More recently, researchers led by Stephen Baylin at Johns Hopkins School of Medicine, in Baltimore, have shown that a combination of a histone-deacetylase inhibitor and azacitidine slowed tumour growth in some people with advanced lung cancer.</span></span></p>
<p><span><span><span>It was the first time epigenetic drugs had been deployed successfully against a solid tumour, rather than a leukaemia or a lymphoma. <span>Unlike other forms of gene regulation (those involving transcription factors, for example), epigenetic changes are passed on during cell division to daughter and granddaughter cells until they are actively erased. Once erased, though, they do not return. It might therefore be that epigenetic therapies can effect changes which stop a cancer growing without having to kill all its cells.</span></span></span></span></p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p><span>Histone modifications are a consequence and mere reflection of dynamic processes altering DNA accessibility such as transcription factor or RNA polymerase II (RNAPII) binding or chromatin remodeling. <span>Generally, certain histone modifications such as acetylation or phosphorylation are thought to change chromatin structure by altering the net positive charge of the histone proteins, thereby rendering the underlying DNA sequence information more accessible.</span></span></p>
<p><span>Genetic studies have attempted to elucidate causal mechanisms for the development of complex disease, but genome-wide associations have been largely unsuccessful in establishing these links. As an alternative link between genes and disease, recent efforts have focused on mechanisms that alter the function of genes without altering the underlying DNA sequence. Known as epigenetic mechanisms, these include DNA methylation, chromatin conformational changes through histone modifications, ncRNAs and, most recently, 5-hydroxymethylcytosine. Although DNA methylation is involved in normal development, aging and gene regulation, altered methylation patterns have been associated with disease. It is generally believed that early life constitutes a period during which there is increased sensitivity to the regulatory effects of epigenetic mechanisms. The purpose of this review is to outline the contribution of epigenetic mechanisms to genomic function, particularly in the development of complex behavioral phenotypes, focusing on the sensitive periods.</span></p></div>
  </body>
</html>