<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1875 from Anon (session_user_id: c286e9accd8c660f0afdc21e32f5f6e368a61206)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1875 from Anon (session_user_id: c286e9accd8c660f0afdc21e32f5f6e368a61206)</h1>
    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p style="text-align:justify;"> </p>
<pre class="hg-tinymce-code" style="text-align:justify;">H19/Igf2 is an mechanism described for mediating imprinting in clusters.The maternally-expressed H19 gene and paternally-expressed Igf2 gene share enhancers and their reciprocal imprinting is governed by a CTCF-dependent insulator that is located      between the genes.CTCF insulates Igf2 from downstream enhancers. DNA methylation at ICR blocks binding of CTCF binding sites. Without CTCF, DNA methylation spreads to H19 promoter to silence and enhancers can access Igf2 to activate.  On the maternal   allele in mouse, CTCF binds to 4 binding sites within the ICR, generating an insulator that prevents Igf2 from accessing the  shared enhancers that are located on the H19 side of the insulator. Thus, the insulator effectively acts as an enhancer-blocker. On the paternal chromosome, methylation at the ICR not only prevents CTCF from binding, allowing Igf2 to engage the        enhancers, but it is also required for methylation at the H19 promoter and silencing of H19. Alteration in DNA methylation at ICRs (hypo or hypermethylation) can results in loss of expression of growth restricting genes, overexpresion of growth        promoting - e.g. hypermethylation of ICR results in Igf2 overexpression in Wilm's tumour.</pre></div>

    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><pre class="hg-tinymce-code">DNA methylation is a biochemical process involving the addition of a methyl group to the cytosine DNA nucleotides.            <span>DNA ethylation is the basis for epigenetic processes such as imprinting, X chromosome inactivation or the formation of heterochromatin.</span> DNA methylation is associated with gene silencing when found at promoters.In adult somatic cells, DNA methylation   typically occurs at CpG dinucleotides. <span>The modification of the C5 p</span><span>osition of the cytosine base (5 mC) is found in approximately 70–80% of CpG dinucleotides in somatic mammalian cells and to some extend in non-CpG sequences in embryonic stem cells. The global level of methylation decrease progressive from normal to metastatic tissue. Methylation of CpG islands become more dense because of change from normal to metastatic. In normal cells, CpG islands are hypomethylated and the genome is methylated in the repetitive elements, the intergenic regions and genic introns. In cancer cells, CpG islands are more likely to be methylated and the rest of genome (repetitive elements, intreganic regions and introns) are hypometilated. DNA methylation is an alternative for genetic mutation, to silence tumour suppressor genes in cancer. DNA methylation at intergenic regions maintain genomic integrity (for instance, Dnmt1 null cells display genomic instability. Also, play important role to silence cryptic trancription start sites or cryptic slice sites. DNA methylation at repetitive elements maintain genomic integrity also. Silencing of repeats and mutation of the repeats (meC to T) are important to prevent transposition. Silencing of these repeats avoid transcriptional interference from strong promoters. Methylation of the repeats may prevent illegitimate recombination. Role of DNAmethylation is context dependent. Deletion of Dnmt1 can enhance or suppress tumorigenesis. Different tumors have different dependencies. Effect of DNA methylation depletion is stage specific (implications for effectiveness of epigenetic therapy). DNA methylation can be removed pasively or actively (involved TET proteins). </span></pre></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><pre class="hg-tinymce-code">Decitabine is an DNA-demethylating agent (DNMTi) used to treat myelodysplastic syndromes, the precursors of acute myelogenous leukaemia. DNA methylation is laid down by the novo methyltransferases (DNMT). DNMT are incorporated into DNA. After this,    nucleoside analogues irreversibily bind DNMT incorporated previously into DNA. Therefore, the process is replication          dependent. For this reason, cancer cells which are divided more rapidly, will be more severily affected because they          replicated more. This fact can explain the anti-tumour effect of Dacitabine. </pre></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><pre class="hg-tinymce-code">Abnormal DNA methylation has been demonstrated in human cancers cells. In malignant tumors, genome-wide DNA hypomethylation is observed, concurrent with hypermethylation at normally unmethylated CpG islands. Most cancers show aberrant methylation of    some gene promoter regions. Other tumors are characterized by CIMP, a phenotype involving concurrent methylation of multiple  gene promoter regions. Other environmental factors have also been associated with aberrant DNA methylation in some human      studies (e.g. folate and other one-carbon nutrients). Sensitive periods predominantly occur during establishment of epigenetic marks, rather than maintenance. Epigenetic marks may be transmitted transgenerationally, so any environmental effects may have multi-generational effects. Periods of particular vulnerability include the early stages of embryonic development. Childhood  is also proposed as a period of vulnerability, especially in the germline of females, since oocytes remain in a haploid       de-methylated state until puberty, so environmental influences may potentially disturb the epigenetic state of the oocyte for    many years, with potential implications for both fertility and initial epigenetic state of offspring of an exposed female.    Somatic changes to DNA methylation may also result from environmental exposures in adults, as have been reported in aging and disease processes such as cancer.</pre></div>
  </body>
</html>