<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 824 from Anon (session_user_id: 814be0cfb1645921d80c695adec601a90b0f26b9)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 824 from Anon (session_user_id: 814be0cfb1645921d80c695adec601a90b0f26b9)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>CpGs are a stable epigenetic mark, and tend to be clustered into CpG islands, often at gene promoters. These CpG islands tend to be protected from methylation but when there <span style="text-decoration:underline;">is</span> methylation at CpG islands it results in <span style="text-decoration:underline;">silencing</span> of gene expression because the methylation is associated with the formation of a repressive chromatin structure. </p>
<p>In cancer the DNA methylation of CpG islands is disrupted - an <strong>increase</strong> in the methylation of <strong>specific</strong> CpG islands is observed. ie as cancer develops there is a change from hypomethylation to hypermethylation of certain CpG islands. </p>
<p>This disruption of DNA methylation at CpG islands contributes to disease because it silences tumour suppressor genes. With the suppression from these genes negated there can be cell proliferation, leading to cancer. CGI hypermethylation occurs frequently in tumours, and increases with tumorigenicity. The identity of hypermethylated CGIs varies by tumour type. CIMP hypermethylation tends to have a favourable prognosis whereas high methylation in other illnesses can have a poor prognosis (eg in myelodysplastic syndrome and in lung cancer).</p>
<p>Intergenic regions and repetitive elements are usually methylated. The function of this DNA methylation is to maintain genomic integrity. In cancer, historically, the earliest epigenetic aberration found was a genome-wide lack of methylation, ie hypomethylation at intergenic regions/repeats, leading to genomic instability. This feature has been found in all cancers ever studied, and progresses with tumorigenicity.</p>
<p>Disruption of DNA methylation in intergenic regions and repetitive elements contributes to disease because it leads to genomic instability (an important factor in cancer). This leads to illegitimate recombination between repeats (thus causing reciprocal translocations), activation of repeats and transposition, activation of cryptic promoters and disruption to neighbouring genes (c.f. <em>A<sup>vy</sup></em> and <em>Axin<sup>fu</sup></em> alleles).</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>In imprinted clusters of genes each cluster has at least one imprint control region (ICR). These display differential methylation between the parental alleles. Alterations in DNA methylation at these ICRs can result in loss of expression of growth restricting genes, or overexpression of growth promoting genes. </p>
<p>The <em>H19/ Igf2</em> cluster works by enhancer blocking. In the paternal chromosome DNA methylation at the ICR blocks binding by the insulator protein CTCF. Without CTCF, DNA methylation spreads to the <em>H19</em> promoter and silences it. The enhancers can then access <em>Igf2</em> and activate that. </p>
<p>The opposite is true in the normal maternal allele. The ICR is unmethylated, so CTCF can bind to it. This insulates <em>Igf2</em> from downstream enhancers, which are then free to enhance <em>H19</em>.</p>
<p>In Wilm’s tumour this imprinting pattern is disrupted. Hypermethylation of the maternal ICR causes the maternal allele to behave like the paternal allele. This may be triggered by mutation/deletion causing loss of imprinting, uniparental disomy - paternal UPD in this case, or epigenetic disruption (rare).</p>
<p>Disease results because the altered imprinting pattern leads to overexpression of <em>Igf2</em> (an oncogene, ie a growth promoting gene), leading to overgrowth and hence cancer; such as Wilm’s tumour, which affects the kidneys.</p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine is a DNA-demethylating agent, and works by inhibiting DNA methyltransferases. It is used to treat myelodysplastic syndromes, the precursors of acute myelogenous (or myeloid) leukaemia. </p>
<p>Decitabine causes hypomethylation. It is a nucleoside analogue, which becomes incorporated into DNA and then irreversibly binds DNMTs. Its action is therefore replication dependant. A lack of specificity is inherent to its mechanism of action, so there could also be long term effects on normal cells.</p>
<p>The role of DNA methylation is context dependent, different tumours have different dependencies. If a cancer is driven by <strong>tumour suppressor gene hypermethylation</strong>, then depletion of DNA methylation appears to <strong>suppress</strong> tumorigenesis. This is true of many haematological malignancies, and explains how decitabine is having an effect – by removing methyl marks and thus reactivating tumour suppression genes.</p>
<p>It therefore works by helping the bone marrow to produce normal blood cells and by killing abnormal cells in the bone marrow.</p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Altering DNA methylation can have enduring effects on the epigenome, because epigenetic alterations are heritable through cell division and in some cases are able to behave similarly to mutations in terms of their stability. Importantly epimutations are reversible, unlike mutations, offering the possibility of reversing them through treatment regimes. Epigenetic changes are passed on during cell division until they are actively erased. However, once erased they do not return.</p>
<p>Sensitive periods are times when the epigenome is reset. Many epigenetic marks are cleared during these periods, and new marks are laid down. There are two such principal phases of epigenetic reprogramming:-</p>
<p>1) during pre-implantation development (ie early embryonic development) - removal of much of the DNA methylation and the histone marks across the genome reaches a global low at about the blastocyst stage (i.e. just before implantation). Resetting of epigenetic marks then occurs in a cell-lineage specific manner.</p>
<p>2) during primordial germ cell development - germ cells develop from somatic cells, so obviously the somatic marks need to be removed.</p>
<p>Treating patients during such sensitive periods would be inadvisable, as treatments are not targeted sufficiently to only “attack” the epigenetic marks linked to disease. Other marks could be affected, leading to severe developmental problems.   </p></div>
  </body>
</html>