<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1034 from Anon (session_user_id: 6666ed17b2ccbd3f999f85b8297fda4b3b3a0bab)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1034 from Anon (session_user_id: 6666ed17b2ccbd3f999f85b8297fda4b3b3a0bab)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>CpG islands are non-randomly distributed throughout the genome, concentrated around promoters of genes. Their epigenetic state, determined by the methylation of their cytosine residues dictates the expression of genes. Their methylation is universally associated with gene silencing. Being largely unmethylated, some are dynamically methylated at different times and between different cell types.</p>
<p>Cancer affects the epigenetic state of CpG islands. Such epimutations are reversible. There is hypermethylation at CpG islands and shores of regulatory genes in cancer, whose progression is characterized by the increase of such eg in CIMP.  Hypomethylation occurs in CpG poor promoters.</p>
<p>Hypermethylation in GpG islands of tumour suppressor genes affects their downregulation and aids tumour progression. These deviations are heritable and quickly selected due to the rapid division of cancer cells. Different cancers may result from hypermethylation of different single or sets of genes. Hypomethylation of CpG poor promoters causes activation of tumour promoting genes. </p>
<p>DNA methylation at intergenic regions block cryptic transcription start and splice sites. Methylation at repetitive elements prevent transposition, prevent interference from their own (strong) promoters and illegitimate recombination. Both phenomena function in maintaining genetic stability by keeping these regions heterochromatinized.</p>
<p>In contrast to normal cells, cancer cells have hypomethylation of intergenic regions and most importantly of repetitive elements. Tumour progression is marked by progressive genome wide hypomethylation.</p>
<p>Hypomethylation and subsequent expression of repeats results in: illegitimate recombonations, effecting translocations among similar elements in different chromosomes; transposition which disrupt functional genes or activate silent genes close enough to be affected by the strong cryptic promoters of repeats.</p>
<p>Thus DNA hypermethylation and hypomethylation as drivers of tumour formation are dependent on their occurrence in tumour suppressor genes or repetitive regions respectively. These dependencies can change with the stages of cancer. Epigenetic states can have diagnostic implications and, being reversible, affect prognostic outcomes.</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>The H19/IGF2 cluster is paternally imprinted. The methylation of the paternal ICR prevents binding to the insulator protein CTCF. In this configuration, downstream enhancers act on the IGF2 gene, since the chromatin preferentially loops in this fashion, thus driving its expression. The H19 element on the paternal chromosome is not active because of spreading  of DNA methylation of the ICR.</p>
<p>The unmethylated maternal H19/IGF2 ICR, in contrast, allows the binding of CTCF, blocking the IGF2 gene from the action of the enhancers. This allows them to act on the less preferred and also unmethylated H19, a micro RNA reservoir.</p>
<p>In Wilms tumour, hypermethylation of H19/IGF2 ICR on maternal alleles causes IGF2 to be expressed from it as well as the paternal allele.</p>
<p>Cancer is often associated with loss of imprinting since many imprinted regions are associated with growth. IGF2, being one such gene is tightly regulated in normal cells. In the case of Wilms tumour, the double dose of IGF2 facilitates its progression to the formation of childhood kidney tumours.</p>
<p>Disruption of imprinting can be a consequence of hypermethylation and hypomethylation of alleles in question. Its occurrence is common, and found early, even in pre neoplastic tissue.</p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Drugs that alter (commonly enzymatically) the epigenetic state are used to improve the quality of life and survival of cancer patients. Of the several classes of drugs that act on different facets of the epigenetic machinery, DNMT1 inhibitors target cancers that are driven by hypermethylation of CpG islands of different genes. Decitabine is one such drug.</p>
<p>Chemically decitabine is a nucleoside analog that is incorporated into DNA, being hence replication dependent. It irreversibly binds DNMT1, which then cannot subsequently copy the rest of the methylation of the parent to the daughter strand.</p>
<p>At low doses decitabine inhibits DNA hypermethylation on the controlling elements of tumour suppressor genes. Higher doses are ineffective and prove toxic, primarily due to its nonspecific targeting of cancer and healthy cells. This is particularly well tolerated in MDS and has a good prognostic outcome for older patients, but long term consequences are still to be assessed. </p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>A crucial property of epigenetic marks is somatic maintenance. Once established they are maintained throughout life, because of their mitotic heritability. DNA methylation once established is stably maintained by DNMT1, 3a and 3b in subsequent generations by copying the methylation marks to the daughter strand on hemimethylated DNA.</p>
<p>Sensitive periods are periods of genome wide epigenetic reprogramming, where epigenetic marks are erased completely in an active (cell division independent) or  passive (division dependent) manner. Here, the epigenetic state is particularly vulnerable to environmental influences.</p>
<p>There are two sensitive periods of development. The first is during early embryonic development, when the paternal and maternal pronuclei undergo demethylation. This continues until the blastocyst stage where the marks are reset in a lineage specific fashion. The second is during primordial germ cell development in mid gestation, when oocytes and sperm are formed and the genomes again undergo epigenetic clearing.</p>
<p>Modifying epigenetic signatures in cancer during these periods is not recommended. The non- selectivity of the drugs may confer aberrant epigenetic marks to normal cells, which would disrupt their function and be maintained in subsequent generations mitotically after the marks have been reset. This could be averted by their use in non-sensitive periods. </p></div>
  </body>
</html>