<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 552 from Anon (session_user_id: 55c02d92120e53ad772a69238c2e19170e009624)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 552 from Anon (session_user_id: 55c02d92120e53ad772a69238c2e19170e009624)</h1>
    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>    Decitabine is a DNA-demethylating agent is
used to treat myelodysplastic syndromes, the precursors of acute myelogenous
leukemia. It prevents DNA methylation by inhibiting methyltransferase activity.
This turns on tumor suppressor genes that prevent cancer cells from growing and
dividing.</p><p>References:</p><p>(2012, 7th April). <a href="http://www.economist.com/node/21552168" title="Link: http://www.economist.com/node/21552168">Cancer's epicentre</a>. The Economist.</p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>    Altering DNA methylation can have enduring
effects on the epigenome because it can alter epigenetic marks crucial for
normal patterns of growth and development (1). Pre-implantation
development and germ cell development are sensitive periods in epigenetics,
meaning that these two periods are crucial for the establishment of proper
epigenetic patterns (2).<span> Treating patients during these periods would be
inadvisable because of the possibility of altering epigenetic marks, leading to
abnormal patterns of growth and development.</span></p><p>References:</p><p>1.) (2012, 7th April). <a href="http://www.economist.com/node/21552168" title="Link: http://www.economist.com/node/21552168">Cancer's epicentre</a>. The Economist.<br /></p><p>2.) Waterland RA, Jirtle
RL. (2003). Transposable elements: targets for early nutritional effects on
epigenetic gene regulation. Mol Cell Biol 23(15):5293-300.</p><p><br /></p></div>

    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>    In general, DNA methylation of promoter regions
decreases gene expression, but the opposite is true for CpG islands. Normal functioning of DNA methylation at CpG islands correlates
with increased gene expression. The particular pattern of CpG island methylation
is responsible for active transcription. CpG islands have a high CpG density and
are kept free of methylation that is not a part of their activity state.</p><p>    Changes in DNA methylation cause losses of
genome-wide DNA methylation and regional gain of DNA methylation. The CpG
islands of cancer cells become
hypermethylated, silencing the underlying tumor suppressor genes. These genes control
the cell cycle, apoptosis (cell death), or DNA repair. Disruption of these
genes can lead to uncontrolled cell growth, lack of apoptosis, and faulty DNA
repair genes.</p><p>     Non-coding RNAs, transcribed from intergenic regions,
are important modulators of chromatin regulation and gene expression. Functional
lncRNAs (long non-coding RNAs) fulfill important regulatory roles in gene
expression. Many of  lincRNAs are needed for
pluripotency and differentiation. Cancer can cause the abberant
expression of genes involved in regulatory roles, loss of pluripotency, and inappropriate
differentiation.<br /></p>     It has been speculated that DNA methylation in
repetitive sequences, such as satellite repeats, are present in highly
expressed genes to prevent transcription from alternative start sites, inhibit
antisense transcription, direct RNA splicing, and is related to the timing of
replication. Increased transcription of satellite repeats has been found to be
associated with impaired heterochromatin structure, DNA damage, and instability
in BRCA1 deficient mouse and human breast cancers. Hypomethylation at satellite repeats was
found in malignant peripheral nerve sheath tumors. Hypermethylation and
hypomethylation can lead to cancer through the creation of alternative start
sites, causing abberant RNA splicing, promoting antisense transcription, and
changing the timing of replication.<br /><br />References:<br /><br />Hassler MR, Egger G. (2012). <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3480634/" title="Link: http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3480634/">Epigenomics of cancer - emerging new concepts</a><span>. Biochimie, 94(11):2219-30.</span><br /><p><span> <br /></span></p><p><br /></p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value">    lncRNA AIR is involved in imprinting and is required for the silencing of maternally expressed protein-coding genes Igf2r/Slc22a2/Slc22a3. T<span>he paternally expressed lncRNA Kcnq1ot1 mediates lineage-specific silencing of the Kcnq1 locus by interacting with the PRC2 complex and G9a HMT in placenta and is also involved in imprinting. Igf2 (insulin-like growth factor) imprinting is lost in Wilm's tumor, leading to abberrant growth and predisposition to tumor formation. <br /><br />References:<br /><br /></span>Hassler MR, Egger G. (2012). <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3480634/" title="Link: http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3480634/">Epigenomics of cancer - emerging new concepts</a>. Biochimie, 94(11):2219-30.<br /></div>
  </body>
</html>