<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 2024 from Anon (session_user_id: 1053ddf29c444483565b2872332cdfef9848aec3)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 2024 from Anon (session_user_id: 1053ddf29c444483565b2872332cdfef9848aec3)</h1>
    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><ul><li><strong>Identify the class of epigenetic inhibitors that Decitabine belongs to.   </strong>          </li>
</ul><p>Decitabine is a DNA-demethylating agent or a hypomethylating agent.</p>
<ul><li><strong>Describe the impact of Decitabine on DNA methylation.  </strong>           </li>
</ul><p title="DNA methyltransferase">It hypomethylates DNA by inhibiting DNA-methyltransferase. That means that Decitabine is a chemical analogue of the cytosine nucleoside used in DNA.</p>
<ul><li><strong>Describe how Decitabine can have an anti-tumour effect.</strong></li>
</ul><p>The mechanisms used by this molecule are two: inhibition of DNA methyltransferase at low doses, causing hypomethylation of DNA, and direct cytotoxicity in abnormal hematopoietic cells in the bone marrow through its incorporation into DNA at high doses, resulting in cell death.             </p>
<p> </p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><ul><li><strong>Describe how altering DNA methylation can have enduring effects on the epigenome.</strong></li>
</ul><p>Its incorporation into DNA leads to a covalent binding with DNA methyltransferases, which prevents DNA synthesis and subsequent cytotoxicity    </p>
<ul><li><strong>Define what is meant by a sensitive period.  </strong>    </li>
</ul><p>Sensitive periods  when the environment may be able to  influence epigenetic makeup.  So, it's not necessarily true that  altered environment at any stage of  development could actually result in a  different phenotype or a different  epigenetic outcome.  </p>
<ul><li><strong>Identify sensitive periods of development.    </strong>    </li>
</ul><p>  the first is  the  production of mature eggs and sperms  seems to be one sensitive period, the second one is the pre implanted and early post implantation period.</p>
<ul><li><strong>Explain why treating patients during sensitive periods would be inadvisable. </strong> </li>
</ul><p>  These are unadvisable because the effects are stable, and persist into adulthood. And that would significate important errors on  genomic DNA.</p></div>

    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><ul><li><strong>Describe the normal function of DNA methylation at CpG islands.</strong></li>
</ul><p>CpG islands tend to be protected from methylation. Methylation at CpG island ≡ silencing of gene expression.</p>
<ul><li><strong>Describe how DNA methylation of CpG islands is disrupted in cancer.</strong></li>
</ul><p>CpG island shores hypermethylated in cancer, correlates well with gene expression, as it does in normal tissue differentiation. </p>
<ul><li><strong>Explain how disruption of DNA methylation at CpG islands contributes to disease.</strong></li>
</ul><p>CpG island methylator phenotype (CIMP) - cancers where CpG islands of a set of genes are frequently methylated. Recognised DNA methylation can be one of the hits in the Knudson hypothesis (cancer is the result of multiple hits to DNA)</p>
<ul><li><strong>Describe the normal function of DNA methylation in intergenic regions and repetitive elements.</strong></li>
</ul><p>The genome in general  is methylated in the repetitive elements  through the intergenic regions and  indeed, even in the introns of genes.</p>
<ul><li><strong>Describe how DNA methylation in intergenic regions and repetitive elements is disrupted in cancer.</strong></li>
</ul><p>In cancer cells They're not all methylated, but their  more likely to be methylated than in a  normal cell.  And, the rest of the genome in general  including the repetitive elements, and  these intergenic elements and indeed the  introns are hypomethylated</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><ul><li><strong>Describe the methylation pattern of the paternal allele and how this determines Igf2 expression status.</strong>          </li>
</ul><p>On the patternal allele when this imprint  control region is methlyated CTCF can now  no longer bind.</p>
<ul><li>  <strong>Describe the methylation pattern of the maternal allele and how this determines Igf2 expression status. </strong> </li>
</ul><p>Igf2 is expressed only  from the paternal allele and not the maternal allele.        </p>
<ul><li><strong>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.   </strong>       </li>
</ul><p> So, while I'm showing you these for these one specific example, loss of imprinting is observed in a very wide array of tumor  types.  And in fact, occurs as a very early  event, so it's often seen in  preneoplastic tissues.   And so this suggests that maybe these events at the imprint control regions,  these hypermethylation, or  hypomethylation events. Might be very early events in tumor  regenesis.  </p>
<ul><li><strong>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to disease.</strong> </li>
</ul><p>Imprinted disorder, results from various abnormalities in linked Kcnq1 and H19/Igf2 clusters at 11q15.5 (1Mb). Eg. upregulation of Igf2 (oncogene, growth promoting). --&gt; What's then the phenotype of these  patients.  Well, as you might expect they, they  display both fetal, and post-natal  overgrowth.</p></div>
  </body>
</html>