<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 644 from Anon (session_user_id: f2234bdfbd700579319eec17d3a92af36d07845d)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 644 from Anon (session_user_id: f2234bdfbd700579319eec17d3a92af36d07845d)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value">The normal function of DNA methalation  at CpG islands is to not methlate. In cancer these CpG islands are hypermethlated. This causes silencing of the underlying gene. This seems to promote oncogenes, and silence TNFs. DNA methylation in intergenic regions and repetitive elements is thought to "repress transcriptional noise from alternative start sites, inhibit 
antisense transcription or direct RNA splicing and relates to 
replication timing"[1]. The intergenic regions and repetitive elements are not silenced leading to transcriptional 'noise'. This may contribute to mutagenesis by having more noise than signal, onco-flaws may be more easily introduced. <br /><br /><br /><br /><br /><br /><br /><br /><br />[1] <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3480634/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3480634/</a> <br /><br /><br /></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value">IncRNA AIR is paternally expressed to silent the maternally expression of genes which code for Igf2. And vise-versa with the maternal expression of Igf2. In Wilm’s tumour, the paternally expressed IncRNA Kcnq1ot1 "mediates [the] lineage-specific silencing of the Kcnq1 locus by interaction 
with the PRC2 complex and G9a HMT in placenta, but not in fetal liver"[2] resulting in the H19 promotor being silenced, and the Igf2 being expressed. The combination of this is increased cell growth which can be a precursor to cancer. This can happen when the maternal chromosome does not imprint properly and switches to a paternal epigenetic state[3] . This triggers the cascade which results in cancer. <br /><br /><br /><br /><br /><br /><br /><br /><br /><br /><br /><br />[2] <br /><div>Pandey RR, Mondal T, Mohammad F, Enroth S, Redrup L, Komorowski J, Nagano T, Mancini-Dinardo D, Kanduri C</div><div>Mol Cell. 2008 Oct 24; 32(2):232-46.<br /><br /></div>[3] Steenman, M. <i>et al.</i> Loss of imprinting of IGF2 is linked to reduced expression and abnormal methylation of H19 in Wilms' tumor. <i>Nature Genetics</i> <b>7,</b> 437 (1994) <br /><br /><br /><br /><br /></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value">Decitabine is a DNA methalation agent class of drug. Decitabine interferes with methalation by inhibiting DNA methyltransferase resulting in a sequence demethylation. By interfering with methalation, the resulting hypomethlation can result in apoptosis -- especially in prolific, rapidly dividing cells[4] . In this way, an anti-tumor effect is had through its 'emulated' regulatory effect[5]. <br /><br /><br /><br /><br />[4] <a href="http://www.drugbank.ca/drugs/DB01262">http://www.drugbank.ca/drugs/DB01262</a><br /><br />[5] <a title="Link: http://www.nature.com/onc/journal/v21/n35/full/1205699a.html" href="http://www.nature.com/onc/journal/v21/n35/full/1205699a.html">http://www.nature.com/onc/journal/v21/n35/full/1205699a.html</a><br /><br /></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value">Where the DNA methylation is altered at specific critical moments in the development of new cells (ie proliferation), this effect could be long lasting as those cells go from a 'development' state to a 'maintenance' state -- with the alterations carried forward and maintained. A sensitive period would be any time where the organism is developing and not merely maintaining the DNA/genetic information. Examples of such periods would be when the reproductive 'bits' were germ cell development, during gestation, and early life (to a lesser degree)[6].  Treating patients during sensitive periods is thought to have adverse side effects on fertility[7]; although less likely, they may even exhibit delayed cancer development[8]<br /><br /><br /><br />[6]   <a title="Link: http://www.ncbi.nlm.nih.gov/pubmed/22920183" href="http://www.ncbi.nlm.nih.gov/pubmed/22920183">http://www.ncbi.nlm.nih.gov/pubmed/22920183</a> <br /><br />[7] <a title="Link: http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0024156" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0024156">http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0024156</a> <br /><br />[8] <a title="Link: http://www.ncbi.nlm.nih.gov/pubmed/12724221" href="http://www.ncbi.nlm.nih.gov/pubmed/12724221">http://www.ncbi.nlm.nih.gov/pubmed/12724221</a> <br /><br /><br /><br /><br /></div>
  </body>
</html>