<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 725 from Anon (session_user_id: a5fa63c626c7a194ce1faf0abecbd81c3bf2fa36)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 725 from Anon (session_user_id: a5fa63c626c7a194ce1faf0abecbd81c3bf2fa36)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>Normally CpG islands are hypomethylated, thus allowing genes to be transcribed (provided other mechanisms are not restricting it). Cancer cells on the contrary are hypermethylated at CpG islands. It blocks transcription of the corresponding genes, and they become silenced. If silenced genes are tumour suppressor genes, this gives an additional 'hit' on the way to cancer, because the cell have no working mechanism to limit cell growth.</p>
<p>Intergenic regions and repetitive elements have opposite methylation status. They are hypermethylated in normal cells and hypomethylated in cancers. These repetitive regions when unmethylated can lead to:</p>
<ol><li>genomic instability because of increasing chances of illegitimate recombination between repeats (in normal cells they would be heavily methylated and chromatinized), leading to additional deletions/insertions/reciprocal translocations, which can either disrupt working genes (tumour suppressors) or turn on previously silent oncogenes.</li>
<li>activation of transposons, which can jump around, disrupting working genes (so tumour suppressor genes could be broken)</li>
<li>activation of cryptic promoters, so turning on transcription of neighbouring genes (i.e. oncogenes)</li>
</ol></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>H19/Igf2 cluster consists of Igf2 gene, imprinting control region (ICR) to which binds CTCF protein, H19 gene and enhancers.</p>
<p>Normally Igf2 is maternally imprinted. In this case paternal ICR is methylated, so it prevents binding of CTCF insulator protein. In this case enhancers can act on Igf2 and it expresses from the paternal allele. Maternal ICR is unmethylated, so CTCF can bind to it, so insulating Igf2 from enhancers. Enhancers cannot reach Igf2 and so activating nearby H19 expression. Igf2 is not expressing from the maternal allele.</p>
<p>In Wilm's tumour ICR is hypermethylated, so both maternal and paternal ICRs are methylated, CTCF cannot bind to it, and enhancers activate both alleles of Igf2. This leads to Igf2 overexpression, which are growth-promoting genes leading to Wilm's tumour.</p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine is a DNA-demethylating agent<span>, sold as Dacogen by Eisai. Iе is used to treat myelodysplastic syndromes, which are the precursors of acute myelogenous leukaemia. </span>Decitabine is a DNA Methyltransferase (DNMT) inhibitor. </p>
<p>Decitabine is a nucleoside analogue. It is incorporated into DNA strands and irreversibly binds to DNMT, so blocking it activity. It is replication dependent, so cells that are actively dividing are more severely affected (which are typically tumour cells).</p>
<p>Because of inhibiting DNMT activity, the effect of Decitabine should be the hypomethylation of DNA. If the tumour resulted from hypermethylation of tumour suppressor genes promoters, or from excessive methylation of ICR and disrupted imprinting, then reduced methylation after Decitabine treatment can decrease tumorigenic activity.</p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Altering DNA methylation can have lasting effects:</p>
<ul><li>It could affect chromatin remodelling, leading to opening of closing of specific parts of chromatin. Therefore some genes could become active of silenced. Also opened chromatin can become available for interaction with enzymes, transctiption factors of drugs.</li>
<li>Changes in DNA methylation can allow or disallow transcription factor binding, leading to activation/deactivation of different genes. It could also affect poor promoters, or non-coding RNA expression and binding. This could dramatically change the transcriptome and cell´s response to the drug.</li>
<li>It could affect recombination between repeats, which can lead to reduced illegitimate recombination</li>
<li>It could affect imprinting by changing ICR methylation status, which is also can dramatically change situation in the long-term.</li>
</ul><p>These effects are long-lasting, so they persist even after treatment by epigenetic drugs are finished.</p>
<p>Such drugs shound be used with caution in sensitive periods, when epigenetic markup are established and is very sensitive to environmental changes. There are two large such periods:</p>
<ol><li>Pre-implantation period during early development</li>
<li>Primordial germ cell development.</li>
</ol><p>So, treating patients during pregnancy or in young ages where epigenetic markup can easily be changed should be avoided if possible. Otherwise it could dramatically change epigenetic status and cause unpredictable effects.</p></div>
  </body>
</html>