<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1580 from Anon (session_user_id: 636a246c4f1a7c9831cf32ff3a1f8ae0d7a27b40)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1580 from Anon (session_user_id: 636a246c4f1a7c9831cf32ff3a1f8ae0d7a27b40)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><ul><li>In normal cells, CpG Islands are hypomethylated. Introns, intergenic regions and repetitive elements are likely to be hypermethylated.</li>
<li>In cancerous cells, CpG Islands and CpG Island shores of tumor suppressor genes, ICRs-loss of imprinting are hypermethyaled. Repetitive regions, CpG poor promotors, Intergenic regions are likely to be hypomethyaled.</li>
<li>CpG island hypermethylation silences tumor suppressor genes in cancer.</li>
<li>In normal cells, intergenic regions and repetitive elements are likely to be hypermethylated.</li>
<li>In cancerous cells, hypometylation of intergenic regions and repetitive elements leads to genomic instability. </li>
<li> Genomic instability leads to (1). Illegitimate recombination between repeats<br />(2). Activation of repeats and transposition<br />(3). Activation of cryptic promoters and disruption to<br />neighbouring genes (c.f. Avy and Axinfu alleles)</li>
<li>
<p><strong>Evidence hypomethylation leads to genomic</strong></p>
<p><strong>instability</strong></p>
<p>• Mouse models - hypomethylation due to deletion of <em>Dnmt1</em></p>
<p>in specific tissues increases genomic instability</p>
<p>• Human disease - mutations in <em>DNMT3B </em>result in ICF</p>
<p>syndrome, where genomic instability is a characteristic</p>
<p>features</p>
</li>
</ul></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><ul><li>In paternal allele, CTCF can not bind to ICRs(since it is methyaled) so enhancer activates the expression of Igf2.</li>
<li>In maternal allele, CTCF binds to ICRs (since ICRs is  not methylated) so enhancer activates the expression of H19 and Igf2 remain inactive.</li>
<li>In Wilm’s tumor, maternal and paternal alleles ICRs are hypermethylated  which results in the overexpression of  Igf2.</li>
<li>Common, early event, often seen in pre-neoplastic tissue</li>
<li>e.g. Hypermethylation of ICR, Igf2 overexpression in Wilm’s tumour.</li>
</ul></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><ul><li>The class of epigenetic inhibitors that Decitabine belongs to DNMTi. Decitabine is FDA approved.</li>
<li>
<p>Nucleoside analogues, irreversibly bind DNMTs after they are incorporated into DNA, therefore replication dependent.</p>
</li>
<li>
<p>Low dose of Decitabine has anti-tumor effect. It helps  in DNA demethylation.Myelodysplastic syndrome is tolerant to DNMT     inhibitors.</p>
</li>
</ul></div>
  </body>
</html>