<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1503 from Anon (session_user_id: 85b7f7e333232360cb89abc3f6e4cff0c3e67944)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1503 from Anon (session_user_id: 85b7f7e333232360cb89abc3f6e4cff0c3e67944)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>DNA methylation within promoters is common throughout the genome, increasing the expression of underlying genes . However this is not always the case, as seen with CpG island-associated promoters. CpG islands are usually kept free from methylation, regardless of their activity. In cancer this changes, with CpG islands becoming hypermethylated within the promoter regions of tumor supperssor genes, including retinoblastoma gene (Rb1), gluatione S-transferase pi (GSTP1), and E-cadherin (CDH1). As a result of this hypermethylation these regulatory genes are transcriptionally silenced, resulting in a loss of function.</p>
<p>Similar to its presence amoung promoters DNA methylation may be found at repetitive sequences, including satellite repeats. These regions are found in stretches of non-coding DNA known as intergenic regions or exons. DNA methylation within these regions is thought to reduce erroneous transcription, by reducing the number of transposable elements, including additional start codons within a gene. A typical cancer cell will demonstrate hypomethylation within these intergenic regions, increasing genomic instability. This genomic instability is facilitated through an increase in the number of transposable elements available to transcription. Genomic instability within the cells leaves them vulnerable, particularly to the formation of oncogenes. This can occur through insertions within a gene, which has been documented previously in mouse models with the Notch1 gene becoming oncogenic (Howard, et al. 2008).</p>
<p>DNA methylation can greatly alter the expression of genes within the genome. It can be seen that the effects of both hyper- (DNA methylation at CpG islands) and hypo- (DNA methylation within intergenic regions) methylation can contribute to disease states, including cancer.</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>In the paternal allele the imprinted control region, located in the middle of the gene is methylated. This is in contrast to the maternal allele, which is unmethylated. In the unmethylated maternal allele there is a bound insulating protein (CTCF) which leads to downstream enhancers acting on H19 rather than Igf2.   This leads to the cluster being paternally imprinted as the DNA methylation imprint is on the paternal chromosome. In Wilm's tumor the maternal allele behaves like the paternal allele, this can occur through various mechanisms, including mutation/ deletion, causing a loss of imprinting, and uniparental disomy (inheritance of two copies of one parental chromosome) In Wilm's tumor this is usually due to the hypermethylation of the maternal allele, preventing the binding of the CTCF insulator protein, increasing Igf2 production and therefore cell growth. This uncontrolled cell growth contributes to the disease state through the increased proliferation of cells, leading to cancerous growth.</p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine is a hypomethylating agent acting to hypomethylate DNA, through the inhibitation of DNA methyltransferase. Inhibitors of DNA methylation, such as Decitabine, can epigenetically reactivate silenced genes, restoring normal gene function. In cancerous cells this reactivation would occur within tumor suppressing genes including retinoblastoma gene (Rb1), gluatione S-transferase pi (GSTP1), and E-cadherin (CDH1). The reactivation of these tumor suppressants would lead to the halt of cancerous cell growth and apoptosis of affected cells. </p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>DNA methylation can have enduring effects on the genome as methylation patterns are mitotically heribtable. Mitotic heritability of epigenetic state leads to the same set of genes being expressed in daughter cells. </p>
<p>A sensitive period a defined time within development during which the epigenome is highly susceptable to change through environmental factors. Such sensitive periods exist during times of rapid development and epigenetic reprogramming, such as during the formation of the blastocyst within the womb, the production of primordial germ cells, and the formation the gamates, ova and spermtazoa in females and males respectively. Treating patients during sensitive periods is inadvisable as it it can lead to disruption of epigenetic reprogramming, leading to deterimental effects which may be cross-generational. </p></div>
  </body>
</html>