<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 782 from Anon (session_user_id: d11c1dd96b7a4871856fd28e661a56523b9d2fc6)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 782 from Anon (session_user_id: d11c1dd96b7a4871856fd28e661a56523b9d2fc6)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>Cancer cells exhibit epigenetic abnormalities. Specifically, cancer cells have been shown to uniformly exhibit aberrant DNA methylation patterns.</p><p>In normal, noncancerous cells, 70-80% of the CpG dinucleotide pairs carry the epigenetic marker C5 methylation on the cytosine base. Although CpG pairs are preferentially methylated, 5-methylcytosine is mutagenic and CpG pairs in the DNA are generally rare. Of the CpG pairs present in the genome, the methylated CpG pairs overwhelmingly reside in the intergenetic regions and in the repetitive elements. The CpG islands, present in 60% of promoters, are generally free of methylation in these normal cells.</p><p>In cancerous cells, the methylation pattern is reversed. There is genome wide hypomethylation caused by the demethylation of the CpG pairs in the intergenic regions and the repetitive elements. The local hypermethylation characteristic of cancer cells is caused by methylation of specific CpG islands and their adjacent CpG island shores. The set of methylated CpG islands is diagnostic for the particular type of cancer..</p><p>The cancer promoting activities that the altered DNA methylation patterns enable are: genetic instability and mutations; activating oncogenes; and inactivating tumour suppressors. Hypomethylation of the repetitive elements transforms previously condensed inactive chromatin into euchromatin. This frees the repetitive elements to produce mismatched chromosomal recombinations. In addition, the transposons can now make copies of themselves and jump around in the genome, disrupting other regions of the DNA. Hypomethylation of intergenic regions containing CpG poor promoters, which were previously silenced by methylation, results in transcription of downstream DNA that promotes cancer growth by coding for an oncogene, an oncogene stimulating factor, or a control element that inhibits a tumour suppressor. And hypermethylation of CpG islands can directly silence a tumour suppressor gene downstream or silence a control factor involved in stimulating tumour suppressors or inhibiting  oncogenes.</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><div>Imprinting regions frequently code for growth promoting genes. The H19/Igf2 cluster at chromosome region 11q15.5 is controlled by an ICR, imprint control region, that functions as an insulator and controls the expression of an interferon-like growth factor that acts like an oncogene when over expressed. The ICR is located between Igf2 upstream and H19 downstream with enhancers located further downstream from H19. <br />When the ICR is unmethylated in the unimprinted maternal allele, the transcription factor CTCF binds to the ICR. The resulting structure is thought to act via a looping mechanism that causes the enhancers to act on H19 instead of Igf2. The methylation of the ICR in the imprinted paternal H19/Igf2 cluster prevents binding by the CTCF. In addition, the methylation is extended downstream to the H19 promoter to further silence the downstream gene. Thus the downstream enhancers act on their preferred target, the Igf2 gene and with one copy of the Igf2 gene active, the cell has a normal amount of growth factor.</div><div>In Wilm's tumour, the imprinted region exhibits hypermethylation. Both the paternal and maternal alleles are methylated and both transcribe Igf2. Produced at double the normal dose, Igf2 is now overexpressed and functions as an oncogene.</div><div><br /></div></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><div><div>Decibitabine is the cytosine analog 5-aza-2'-deoxycytidine that functions as a DNA methyltransferase inhibitor, DNMTi. As a cytosine analog, it is incorporated into the DNA where it irreversibly binds to DNMT1, the DNA methyltransferase responsible for methylating the matching CpG pair of a hemi-methylated dividing DNA strand. This mechanism prevents the cell from passing its epigenetic markers to its daughter cells. </div><div>DNMT1 minus cells have also been shown to have genomic instability, probably due to the demethylated repetitive elements. So a perfect DNMTi would only target the CpG islands which are the regions that are abnormally methylated in cancer cells. This would reduce any side effects. <br />Decibitabine cannot target specific regions of the genome, but it does target specific cells by being incorporated into the DNA of rapidly dividing cancer cells over non cancer cells. By localizing the toxic DNA demethylation, decibitabine produces an anti neoplastic effect while limiting side effects.</div></div><div><br /></div><div><br /></div></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value">In mammalian cells, the epigenetic DNA methylation markers have been shown to be set up 'de novo' by DNA methyltransferase3a and 3b and passed to all daughter cells by DNA methyltransferase1. Any chemical modification that produces a cytosine methylated CpG pair in the genome is passed faithfully to all subsequent cell generations. Therefore any changes in DNA methylation produced by the epigenetic drugs will be permanent in the cell and all future daughter cells.<br /><br />There are two major sensitive periods in an organism's life cycle when the epigenetic makeup of the cells needs to be reestablished. In primordial germ cell development, somatic cell epigenetic markers, including imprinting markers and excluding repetitive elements, must be removed and sperm or egg specific epigenetic markers are laid down for the germ cells. After fertilization, the early embryo has to lose the epigenetic markers of the germ cells, excluding both the imprinting markers and repetitive elements, and lay down a new set of cell type specific epigenetic markers.<br /><br />During sensitive periods, the genome is cleared of most epigenetic markers prior to reprogramming. An epigenetic drug that adds markers at this point would produce much more disruptive effects in the germ cell or embryo.<br /></div>
  </body>
</html>