<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 203 from Anon (session_user_id: c647533bc22a3f9b218c60ac0c5a51f93cbcd24a)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 203 from Anon (session_user_id: c647533bc22a3f9b218c60ac0c5a51f93cbcd24a)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>DNA
methylation is laid down by methyltransferases and occurs at CpG dinucleotides in
intergenic regions and at repetitive elements across the genome and maintains
genomic stability. CpGs found in clusters are called CpG islands and are
unmethylated allowing the respective genes to be expressed. </p>

<p>However, compared to a normal genome
as described above, cancer is associated with a genome-wide loss of methylation
and a gain of hypermethylation in locus-specific regions. </p>

<p>Genome-wide
loss of methylation occurs when amine groups from 5-methyl cytosines are lost or null Dnmt1 cells (unmethylated) arise in intergenic regions and
repetitive elements. These areas become susceptible to genomic instability and
mutagenic processes can occur.</p>

<p>When the
methylation process is altered genetically or epigenetically, the CpG islands
can become hypomethylated or hypermethylated. The immediate areas surrounding
hypermethylated CpG islands are known as CpG island shores and are also
associated with genetic expression of some types of cancer. If the CpG island
of an oncogene becomes hypomethylated, the cancer gene can become expressed. If
there is hypermethylation at the promoter regions of tumor suppressor genes,
the genes will be silenced thereby allowing expression of cancerous genes, in a
locus-specific manner.</p>

<p>While both
intergenic regions and repetitive elements are noncoding and thus don't provide
direct gene silencing, DNA methylation in these areas does silence indirectly
via cryptic silencing at transcription or splicing sites in intergenic regions
and via silencing of long interval repeats in repetitive elements when found in
the promoter regions. When the areas are de-methylated or hypomethylated, silencing
is disrupted and transcriptional interference, deleterious transpositions,
recombinations or mutations arise contributing to the development of cancer. </p>

<p>Epigenetic
marks can be removed either passively or actively and are maintained mitotically
not only through cell division to their daughter cells but also meiotically to all
subsequent generations.</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>IGF2-H19 is an imprinted cluster of
genes involving the IGF2 and H19 genes, the CTCF insulator protein, the ICR and
some distal enhancers. IGF2 is a growth promoting gene. H19 is a tumor suppressor
gene. The enhancers promote IGF2 expression. CTCF works to insulate IGF2 by
blocking enhancers. This is the primary preference pathway for this chromatin
loop.</p>

<p>The primary preference loop is found
in the paternal allele: the methylated ICR cannot be bound by CTCF which allows
the distal enhancers to promote IGF2 expression. Since CTCF cannot bind to the
ICR, DNA methylation spreads to the H19 promoter, silencing this suppressor gene.
</p>

<p>The maternal allele shows the secondary
preference chromatin loop: CTCF blocks the enhancers from accessing the Igf2
promoter and they instead promote H19 expression. </p>

<p><span>The H19/IGF2 cluster is disrupted by
loss of maternal imprinting on the IGF2 gene; and expression of IGF2 and suppression
of the H19 tumor suppressor gene in the paternal allele. Without maternal imprinting
the maternal allele is not expressed and the IGF2 gene becomes over-expressed
in the paternal allele, often causing
cancerous tumors like Wilm's tumor.</span></p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine belongs
to the class of drugs called DNA Methyltransferase Inhibitors (DNMTi) which are
enzymatic epigenetic regulators. Decitabine is an FDA approved drug that is
used to de-methylate DNA because the process of DNA methylation is mutagenic. </p>

<p>Decitabine
disrupts DNA synthesis by being directly incorporated into the DNA and bound by
DNA methyltranserase 1 (DNMT1) during replication. Instead of a methyl group
binding to a cystosine base and methylating it, the methyl group binds to
Decitabine, causing a passive loss of methylation at all cytosine bases. Since DNMT1
binds irrevocably to the Decitabine neucleotide, DNMT1 will never be released
and therefore is maintained throughout cell division always passively
de-methylating the cytosine base.</p>

<p>Reducing the
hypermethylation at cytosine bases in CpG islands, that occurs with carcinogenesis,
reverses hypermethylation gene silencing and thus reduces carcinogenesis.
Decitabine may also restore suppressor gene expression, again, by de-methylation
in the same manner described above.</p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Epigentic marks that are altered by
epigenetic drugs that alter DNA methylation can have effects that last beyond
the period of drug treatment because epigenetic marks are mitotically heritable
and are transmitted to and maintained in daughter cells following cell division
and continue for many generations of cell division. </p>

<p><span>Experiences during sensitive periods of development influence DNA
methylation patterns of several genes, particularly imprinting genes
and other metabolic genes. Two key<span> sensitive periods that occur during epigenetic
reprogramming are during early development of the fetus <i>in utero</i> and the other during germ cell development.</span></span></p>

<p>The effects of treatment with epigenetic drugs produce stable
epigenetic modifications, by design to enable passive de-methylation, that are
not only mitotically heritable for the life of the organism but also
transgenerationally heritable through the gametes (meiotically heritable). For
these reasons, it would be ill advised to treat a pregnant patient at either of
these sensitive periods.</p>

<p><span>Another period that is slightly risky for disruption of epigenetic
reprogramming occurs when assisted reproductive technologies (ART) are
employed: when oocytes are harvested and when embryos that are cultured <i>in vitro</i> are harvested, both increasing
the risk of imprinting disorders and other possible disruptions. </span></p></div>
  </body>
</html>