<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 603 from Anon (session_user_id: 7351fa279e4c52079efcd0376f3db97c35d0dfbf)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 603 from Anon (session_user_id: 7351fa279e4c52079efcd0376f3db97c35d0dfbf)</h1>
    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><div><br /></div><div><br /></div><div>And hypomethylation in other imprint </div><div>control regions. </div><div>So if you think back to the IGF2 H19 </div><div>cluster that we spoke about in week 4, we </div><div>know here, here is the imprint control </div><div>region shown as the oval. </div><div>And it's methylated on the paternal </div><div>allele, and it's unmethylated on the </div><div>maternal allele. </div><div>When it's unmethylated CTCF binds its </div><div>insulator element, and it means that the </div><div>enhancers in this case will act on H19. </div><div>But Igf2 will be silent for the maternal </div><div>allele, so we don't see expression. </div><div>On the paternal allele, because this is </div><div>methylated, now the enhancers. </div><div>Can act on IGF2, because CTCF is not </div><div>binding to inhibit this, and IGF2 is </div><div>expressed from the pattern allele. </div><div>However, with loss of imprinting and what </div><div>happened is you have hypermethylation </div><div>of the imprint control region on the </div><div>maternal allele as well. </div><div>Now, on the maternal allele, you also </div><div>have expression of Igf2. </div><div>So now you have a double dose of Igf2 in </div><div>comparison to what you saw in a normal </div><div>cell. </div><div>And Igf2 is both growth promoting, and </div><div>this is associated with Wilm's tumour. </div><div>This is this particular childhood kidney </div><div>genome that is observed. </div><div><br /></div><div><br /></div><div><br /></div></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><div> remove an epigenetic mark or the Swi/Snf type </div><div>class of chromatin remodelers, any of the </div><div>chromatin remodelers.  </div><div> a few of the drugs that are currently either in clinical </div><div>trial or FDA approved or the ones that </div><div>are in pre-clinical trial for various </div><div>classes of drugs. .  </div><div>DNA methyltransferase inhibitors, histone </div><div>deacetylase inhibitors, histone </div><div>methyltransferase inhibitors, histone </div><div>acetyltransferase inhibitors,histone demethylase inhibitors. </div><div> they're targeting the DNA </div><div>demethylases the TET proteins or the </div><div>chromatin remodelers. </div><div>There's an EZH2 inhibitor that's </div><div>currently in pre-clinical trials. </div><div>it's a very effective EZH2 inhibitor called </div><div>GSK126. </div><div>These drugs, Vidaza and decitabine and </div><div>also any of the other that are in </div><div>pre-clinical trial are kind of very </div><div>similar to 5-Azacytidine. </div><div> when the DNA methyltransferase comes along to bind it, </div><div>the DNMT1 comes along to bind that </div><div>nucleotide to then copy the methylation </div><div>to the daughter strand, that DNA </div><div>methyltransferase is bound irreversibly </div><div>and it can no longer be released. </div><div><br /></div></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value">. We are now able to look at the cancer epigenome from a bird's eyes view and we are just beginning to understand how alterations in nuclear architecture and global chromatin, as observed by pathologists since more than 150 years, are related to epigenetic mechanisms<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3480634/#bib169">[169]</a>. Based on latest literature we can envision different pathways that might cause epigenetic alterations in cancer cells as observed by genome-wide epigenomic profiling (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3480634/figure/fig3/?report=objectonly">Fig. 3</a>). (I) Mutations in chromatin related enzymes such as DNMTs, histone modifiers or chromatin remodelers might induce stochastic changes in the epigenome causing global changes in chromatin. (II) Faulty targeting (e.g. by lncRNAs or transcription factors) or altered composition of epigenetic complexes can lead to global and gene specific changes in the <span>epigenetic signature. (III) Changes in the spatio–temporal organization of nuclear architecture or loss of boundaries might cause altered nuclear territories, disrupt ordered epigenetic patterns and induce altered gene expression programs. All three pathways could result from altered external signaling due to permanent stress and act in concert to lock in the cancer epigenome.</span></div>

    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3480634/#bib1">[1,2]</a><span>. DNA methylation is the basis for different epigenetic phenomena such as imprinting, X chromosome inactivation or the formation of heterochromatin </span><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3480634/#bib3">[3]</a>. Generally, DNA methylation of promoter regions inversely correlates with gene expression (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3480634/figure/fig1/?report=objectonly" title="Link: http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3480634/figure/fig1/?report=objectonly">Fig. 1</a><span>). Exceptions are CpG islands, which are found in about 60% of promoters </span><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3480634/#bib4">[4]</a><span>. They have a high CpG density and are usually kept free of methylation independent of their activity state </span><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3480634/#bib5">[5,6]</a><span>. However, in cancer cells promoter CpG islands tend to become hypermethylated, which then causes silencing of the underlying gene </span><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3480634/#bib7">[7]</a><span>. Intragenic methylation is found at repetitive sequences such as satellite repeats and remnants of retroviral insertions such as LINEs and SINEs in human DNA </span><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3480634/#bib1">[1,8]</a>. Gene body methylation is present in <span>highly expressed genes and it has been speculated that this may repress transcriptional noise from alternative start sites, inhibit antisense transcription or direct RNA splicing and relates to replication timing </span><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3480634/#bib9">[9–12]</a><span>. A direct role for alternative splicing has recently been attested to methylation-sensitive CTCF binding and polymerase II pausing at CTCF bound exon boundaries </span><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3480634/#bib13">[13]</a><span>. Comparison of the methylome of different hematopoietic lineages has identified a large number of hypomethylated regions (HMRs), which can be constitutive or lineage-specific and colocalize with transcription factor binding sites </span><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3480634/#bib14">[14]</a><span>. Furthermore, complex patterns of methylation were detected in progenitor cells in HMRs, which resolved in a lineage-specific fashion to methylated or demethylated regions, reminiscent of a bivalent histone methylation state found in progenitor cells </span><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3480634/#bib14">[14]</a><span>. Similarly, the deciphering of the mouse methylome revealed regions of low methylation at distal regulatory elements, which was linked to binding of tissue specific transcription factors </span><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3480634/#bib15">[15]</a><span> (</span><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3480634/figure/fig1/?report=objectonly">Fig. 1</a>). Thus, recent reports have identified genomic regions, which are associated with <span>distinct DNA methylation patterns regulating gene expression profiles and chromatin compartmentalization.</span></div>
  </body>
</html>