<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 524 from Anon (session_user_id: 14a5f5e287f2a8ff59fe723d9b5db9f2805d9f3a)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 524 from Anon (session_user_id: 14a5f5e287f2a8ff59fe723d9b5db9f2805d9f3a)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>DNA
methylation found in  70-80% of somatic cells and embryonic stem cells. This process forms basis of epigenetic
phenomena such as imprinting,X-chromosome inactivation and heterochromatin
formation.</p><p><b><u>Hypermethylation at CpG islands and
shores:</u></b></p><p></p><ul><li>Usually, highly
methylated promoter implies lower expression of that gene. The exception are CpG islands, regions that remain free of methylation
irrespective of gene expression profile in normal cells. These CpG islands are hypermethylated in cancer cells causing silencing of tumor suppressor genes.<br /></li><li>The 2kb region
around the CpG island, known as CpG shore, normally shows tissue specific
methylation. But in cancer cells (colorectal cancer, different tumor cell
lines, nerve sheath tumors) this region is hypermethylated.<br /></li></ul><p><b><u>Changes in Intragenic Methylation:</u></b></p><p></p><ul><li>Intragenic
methylation patterns are found at repeat regions like satellite repeats, LINES
and SINES in normal human DNA. Gene body methylation is a characteristic of
highly expressed genes. These methylation patterns are thought to play
important role in : repression of
transcriptional noise or transcription from alternative start site; inhibition of RNA
splicing; relates to replication
timing. These distinctive
gene methylation pattern regulate gene expression and chromatin
compartmentalization.<br /></li><li>In cancer cells,
there is genome wide loss and site-specific increase in methylation that results
into genomic instability and deregulation of gene expression profiles. There is directed
hypomethylation at satellite regions in cancer cells. Increased hypomethylation (observed in primary colon cancer) at genes that express late
replication associated and nuclear lamina associated domains. There is also
stochastic methylation variability in cancer specific differentially methylated
regions (cDMRs) that results in tumor heterogeneity. The mimicking of
stem-cell like methylation marks is also seen in cancer cells. <br /></li></ul><p><b><u>Mutations in enzymes associated with DNA
methylation:</u></b></p>

<p></p><ul><li>DNMT3a
and TET mutations have been associated with T cell lymphoma, leukemia.
Mutations in IDH1(in glioma) and IDH2 genes cause aberrant TET function
and thus impacting the DNA methylation.<br /></li></ul></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><ul><li>Decitabine is a cytosine analogue and <strong>DNA hypomethylating agent</strong>. It belongs to the class : <strong>DNMT inhibitors</strong>.<br /></li><li>It hypomethylates DNA by inhibiting DNA methyl transferase enzyme.<br /></li><li>As part of normal development, certain genes may be silenced through methylation of CpG islands. However, in cancer DNA hypermethylation can inactivate genes essential for the control of normal cell growth, differentiation, or apoptosis. DNA methyltransferases (DNMTs) catalyze the methylation of cytosine residues in newly synthesized DNA, thus replicating the methylation signal. Dicitabine targets this mechanism by inhibiting DNMT, resulting in <strong>hypomethylation of the DNA and re-expression of tumor suppressor genes</strong>. <br /></li><li>At both lower and higher doses, decitabine incorporates into DNA; however, at higher doses, decitabine inhibits cell proliferation through nonreversible covalent linking with DNA methyltransferase and blocking of DNA synthesis. At lower doses, decitabine induces hypomethylation, thereby promoting cell differentiation, re-expression of tumor suppressor genes, stimulation of immune mechanisms, and suppression of tumor growth <br /></li></ul></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><ul><li>DNA methylation marks are more <strong>permanent</strong> as compared to histone modifications and can be <strong>transgenerationally and mitotically inherited</strong>. Thus, the drugs that change DNA methylation patterns are more prone to cause permanent effects on the epigenome and eventually phenotype of an individual.<br /></li><li>A sensitive period is a time-frame during which epigenetic reprogramming of the cells occurs. During this period cells are more susceptible to the environmental changes that in turn affect the epigenome of an organism. Epigenetic changes caused during this period may not be altered in later life.<br /></li><li>These sensitive periods are :Pre-implantation development and Primordial germ cell development</li><li>Administration of DNA methylation altering drugs can result into improper resetting of epigenetic marks. This can cause skewed X-chromosome inactivation, abberant gene imprinting, changes in gene expression profiles. Thus, Treating patients during these sensitive periods may pose permanent health hazards to the patients<br /></li></ul><br /></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><ul><li>H19/IGF2 cluster consists of Igf2 gene, an ICR (Imprint Control Region) and H19 gene. Enhancers are present downstream of H19. Igf2 plays important role in promoting growth. These two genes are located 90 kb apart and are oppositely imprinted: H19 is maternally expressed and Igf2 paternally expressed.<br /></li><li>The ICR of maternal allele is unmethylated and it can bind to CTCF. CTCF is an insulator that blocks the action of enhancers on Igf2 and thereby preventing its expression.<br /></li><li>Whereas on paternal allele, the ICR is methylated and methylation spreads upto H19 . CTCF cannot bind to ICR. Enhancers act on Igf2 causing its expression. Thus only one of the alleles of H19 and Igf2 is active in normal cells.<br /></li><li>In Wilm's tumor, there is failure of imprinting due to genetic mutation or deletion and both the ICRs are methylated. This is called paternal uniparental disomy (both the alleles resemble paternal allele).There's over-expression of Igf2 causing tumors.<br /></li><li>Hyper-methylation at ICR may cause the spreading of methylation upto H19 gene causing loss of its expression. Since Igf2 is a growth promoter, its over-expression can cause uncontrolled growth. H19 is known to be a tumor suppressor gene, loss of its function can lead to cancer.<br /></li></ul></div>
  </body>
</html>