<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 359 from Anon (session_user_id: f6f741757e511f387a8b3872eff962f684cd48ce)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 359 from Anon (session_user_id: f6f741757e511f387a8b3872eff962f684cd48ce)</h1>
    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p><span>Decitabine is a small molecule used as a drug against cancer that acts as an epigenetic inhibitor and is sorted in the category of DNA Methyl-Transferase inhibitors (DNMTi).</span></p>
<p><span>Decitabine decreases the rates of DNA methylation. Specifically, as other DNMT-inhibitors, it is a noucleotide analogue that is incorporated in the DNA's daughter strand during replication. Then the enzyme DNMT1 has to bind to this noucleotide analogue in order to methylate it and sustain the methylation pattern of DNA. However, once bound to decitabine, DNMT1 is never again released, thus it is inactivated. This results in a semi-methylated double-stranded DNA molecule and methylation fades as cells continue to divide.</span></p>
<p>In this way, Decitabine can have an anti-tumor effect, but only if used in small doses. Large doses of this drug affect both healthy- and tumour-cells while on division since it lacks specificity.</p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>DNA methylation is a heritable epigenetic alteration which is sustained during cell division in the daughter cells . However, an epigenetic mark can be removed and its absence is also preserved in the daughter cells. By this means, drugs that target the epigenetic machinery, in order to prevent the inherit of an epigenetic mark, have an effect on the epigenome that lasts long after the end of the treatment.</p>
<p>Alterations in the epigenome take place during sensitive periods, mostly (alteration may also happen during the maintenance period of somatic cells). Sensitive are those periods when epigenetic reprogramming occurs, e.g.pre-implantation period and primodial germ cells  development(gametogenesis), and cells are prone to environmental effects that may affect the reestablishment of epigenetic marks.</p>
<p>Especially, during development sensitive periods are considered those when germ cell maturation occurs: 9-12 years in males ,8-10 years in females.</p>
<p>Taking into consideration all the above, the treatment of patients during sensitive periods has to be carefully planned so as to avoid side-effects, which may include perpetuation of aberrant epigenetic alterations (caused by the drugs) in healthy cells after the end of treatment or establishment of aberrant epigenetic alterations in germ cells.</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>Aberrations in  DNA methylation patterns of preneoplastic cells may occur in imprint control regions (ICRs) resulting in disruption of imprinting in clusters of imprinted genes (e.g. H19/lgf2 cluster), thus overexpression of genes that promote growth or loss-of-expression of genes that inhibit growth, contributing to tumorigenesis. </p>
<p>Considering H19/lgf2 cluster, in normal cells the paternal allele is methylated on the cluster's ICR and methylation spreads to the promoter of H19 locus inhibiting H19 expression. However, lgf2 is paternally imprinted; the downstream enhancers are allowed (in 3D DNA structure) to interact with the promoter of lgf2 locus resulting in its expression.</p>
<p>The maternal allele is not methylated on its ICR. An insulating protein, called CTCF, binds to this unmethylated ICR  inhibiting interactions between enhancers and lgf2 locus (no maternal lgf2 expression)  and promoting interactions between enhancers and H19 locus, thus maternal H19 expression.</p>
<p>In Wilm's tumour, the maternal ICR is hypermethylated, so CTCF can no longer bind to it; enhancers now interact with the promoter of maternal lgf2 locus leading to lgf2 overexpression.</p>
<p>lgf2 is an oncogene, so -in "cooperation" with other "hits" in the DNA- its overexpression can result in cancer, according to Knudson Hypothesis.</p>
<p><br /><img src="https://coursera-uploads.s3.amazonaws.com/user-e2ecd8421c8a5a7a96b265de/970238/asst-5/970238-5206e15e088ec3.68694597.png" alt="" /></p></div>

    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>DNA methylation is a heritable epigenetic alteration which <strong>usually</strong> occurs at CpG dinucleotides. CpG dinucleotides are spread throughout the genome, but are more dense  in some regions forming clusters called CpG islands (CGIs). <strong>CGIs are often located at genes' promoters</strong> and usually are unmethylated. However,<strong> when methylated</strong>, <strong>they inhibit the expression of that</strong> specific<strong> gene</strong> locus in which they are found, by condensing chromatin or not allowing activating-transcription-factors to bind to the promoter of the gene.</p>
<p><strong>In cancer cells, CGIs</strong> of genes' promoters<strong> are hypermethylated. </strong>Thus, the expression of some genes is not allowed anymore.</p>
<p>When the genes that are silenced are considered as <strong>tumor supressor genes, </strong>aberrant cell growth is not effectively inhibited resulting in tumourigenesis (in cooperation with some other hits in the DNA -according to Knudson Hypothesis).</p>
<p>Furthermore, DNA methylation marks in normal cells are spotted intergenic regions and repetitive elements leading to genomic stability by the following means:</p>
<ul><li>preventing karyotype abnormalities (e.g. deletions, insertions, translocations of chromosomal parts or ever chromosome duplication) from happening during cell division</li>
<li>deactivating cryptic promoters so as to avoid any transcriptional interference and the production of truncated transcripts</li>
<li>deactivating cryptic splicing sites in order to produce only correctly spliced transcripts.</li>
<li>in the case of some repetitive elements called transposons (=DNA sequences, which contain a small number of genes - and sometimes strong cryptic promoters- and are able to copy and paste themselves into another genomic region): stopping their spreading around the genome.</li>
</ul><p>In neoplastic cells these regions are hypomethylated resulting in genomic instability. The chromatin structure is less dense since it lacks methylation marks allowing:</p>
<ul><li>mis-matching of some chromosomal regions which they recombinate illigitimately or are deleted/ inserted in another region, ect. </li>
<li>activation of cryptic promoters, splice sites and transposons</li>
</ul><p>These cause irregular gene expression which intrigues tumourigenesis.</p></div>
  </body>
</html>