<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1631 from Anon (session_user_id: 823877d13fb88351222b056b6296caeca94702cf)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1631 from Anon (session_user_id: 823877d13fb88351222b056b6296caeca94702cf)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>CpG islands are often found in or near promoters and in noncancerous cells these CpG islands are predominantly free of methylation regardless of their activity state.  In cancer, the CpG islands tend to be hypermethylated effectively silencing the nearby genes.  If these silenced genes are involved in tumor suppression this can contribute to the maintenance and progression of the cancer cell if accompanied by other genetic or epigenetic changes as in the Knudson hypothesis.  The hypermethylation of CpG islands is often found in tumors as this is a mitotically heritable change and these epimutations are selected for.  In contrast to the methylation status of CpG islands, intergenic regions and repetitive elements in noncancerous cells are usually methylated.  It is thought that methylation of intergenic regions and repetitive sequences may direct RNA splicing, reduce transcriptional noise from alternative start sites and prevent antisense transcription (Hassler et al 2012).  Generally, in cancer, the intergenic regions and repetitive sequences become hypomethylated.  The consequence of hypomethylation is that the genome becomes less stable and gene deletions, reciprocal translocations and insertions are more likely to occur.  The loss of methylation in repetitive elements causes the chromatin to become less densely compacted, exposing regions of highly similar, repetitive sequences available for reciprocal translocation.  In addition, hypomethylated repeats can become activated and transpose to other regions which can lead to aberrant transcriptional outcomes in surrounding regions. </p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p> In cancer, the imprint control regions can become hyper or hypomethylated which can disrupt expression of many growth restricting and/or growth promoting genes.  In the case of the imprint control region for the H19/Igf2 cluster, the imprint control region on the paternal allele is methylated in noncancerous cells, which enables binding and acting of the enhancers on Igf2 to facilitate expression.  The methylation on and near H19 prevents expression of this gene from the paternal allele.  On the maternal allele in a noncancerous cell, the imprint control region is unmethylated which allows the CTFC insulator element to bind, blocking the Igf2 from the enhancers.  The enhancers instead bind H19 and allow expression from the maternal allele.  In Wilm’s tumor, there is hypermethylation of the imprint control regions on maternal allele which results in expression of Igf2 from both the maternal as well as the paternal allele instead of the monoallelic parent of origin specific expression normally observed.  As Igf2 is a growth promoting gene, it is associated with tumorigenesis and usually manifests as a kidney tumor in children.</p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine is one of the anti-cancer drugs that specifically targets the epigenetic machinery.  Decitabine acts to inhibit DNA methyltransferase.  It is a nucleoside analogue that is incorporated into the DNA upon cell replication and will bind irreversibly to DNA methyltransferase, blocking it’s function and reducing methylation in subsequent cell generations.  Cancer cells, which are usually dividing more rapidly than normal cells are prime targets for this drug as it is replication dependent.  When used at low doses the DNA methyltransferase inhibitors can have antineoplastic effects likely due to the DNA demethylation but the exact mechanisms are still unclear.</p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Epigenetic changes like removing DNA methylation is passed on to daughter cells and to their daughter cells during cell division.  DNA methylation won’t return once removed thus these DNA methyltransferase inhibitors have enduring effects on the epigenome.  Sensitive periods are those in which the epigenetic marks are cleared between generations to restore totipotency.  This epigenetic reprogramming occurs during primordial germ cell development and early development.  Taking drugs that alter the epigenetic machinery that is required during these reprogramming phases could have broad undesired consequences affecting gene expression on many more cells than just the cancer cells.</p></div>
  </body>
</html>