<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 141 from Anon (session_user_id: ece5692518e768f6838b67fb22f5a0a638e4c32b)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 141 from Anon (session_user_id: ece5692518e768f6838b67fb22f5a0a638e4c32b)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p><strong>Approximately 70% of genes possess promoter-associated CpG islands that mostly remain unmethylated in normal cells.</strong></p>
<p><strong>DNA methylation plays a crucial role in the regulation of gene expression and chromatin organization within normal eukaryotic cells.</strong></p>
<p><strong>DNA methylation at CpG islands</strong><strong> plays essential roles in X- chromosome inactivation, genomic imprinting and genome stabilization.</strong><strong> </strong><strong></strong></p>
<p><strong>DNA methylation of CpG islands is disrupted in cancer.</strong></p>
<p><strong>Cancer cells exhibit widespread loss of intergenic region and repetitive element DNA methylation with gain of methylation at many gene-associated CpG islands.</strong><strong></strong></p>
<p><strong>DNA methylation of CpG islands in cancer cells are more likely to be hypermethylated.</strong></p>
<p><strong>DNA methylation of intergenic region and repetitive element in cancer cells are hypomethylation.</strong></p>
<p><strong>CpG islands are found, they're in the promoters of tumor suppressor genes. </strong></p>
<p><strong>CpG island hypermethylation or CGI hypermethylation in cancer cells can cause</strong> <strong>epigenetic silencing of tumor suppressor genes.</strong><strong> </strong></p>
<p><strong>DNA methylation can be the second hit, Knudson proposed that you would have to have multiple hits in order to have a cancer to occur.</strong><strong></strong></p>
<p><strong>DNA methylation in intergenic regions and repetitive elements through </strong><strong>DNA methyltransferases establish and maintain genomic methylation patterns which are of critical importance in various biological processes, including development, genomic imprinting, silencing of parasitic sequence elements and tumorigenesis.</strong></p>
<p><strong>DNA methylation in intergenic regions and repetitive elements is disrupted in cancer.</strong><strong> </strong></p>
<p><strong>DNA methylation in intergenic regions and repetitive elements in cancer become hypomethylated.</strong></p>
<p><strong>Hypomethylation of intergenic regions and repetitive elements cause the genomic instability and chromosomal instability and ICF syndrome.</strong></p>
<p><strong>Depletion of DNA methylation in intergenic regions and repetitive elements appears to enhance tumorigenesis. </strong></p>
<p><strong>Hypomethylation of CpG poor promoters can result in activation of oncogene.</strong></p>
<p><strong>A cancer cell is able to inactivate tumor suppressor genes and activate the oncogene.</strong><strong></strong></p>
<p> </p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p><strong> <em>H19</em> is expressed from the maternal chromosome and <em>Igf2</em> from the paternal.</strong></p>
<p><strong>the maternally expressed <em>H19</em> gene, which encodes a ncRNA, and the paternally expressed <em>Igf2</em> gene, which encodes a fetal mitogen, with their shared enhancers</strong><strong></strong></p>
<p><strong>It's methylated on the paternal allele, and it's unmethylated on the maternal allele. </strong></p>
<p><strong>On the paternal allele, because this is methylated, now the enhancers. </strong></p>
<p><strong>Hypermethylation of ICRs - loss of imprinting</strong></p>
<p><strong>And Igf2 is both growth promoting, and this is associated with Wilm's tumour. </strong></p>
<p><strong>Hypermethylation of ICR, Igf2 overexpression in Wilm’s tumour</strong></p>
<p><strong><em>IGF2</em></strong><strong> is normally transcribed from the paternal allele while <em>H19</em> is transcribed from the maternal .</strong></p>
<p><strong>Imprinted expression in both genes is regulated by an imprinting center in which the maternal allele is unmethylated and the paternal is methylated. </strong></p>
<p><strong>The imprinting center contains CTCF binding sites to which CTCF binds only if they are unmethylated. Binding of CTCF blocks the activity of the enhancer element located downstream of <em>H19</em> and restricts activity to <em>H19</em> expression. </strong></p>
<p><strong>Methylation of the CTCF binding sites prevents the binding of CTCF and allows the enhancer to activate <em>IGF2</em>. Biallelic expression of the CTCF binding sites in cancer results in <em>IGF2</em> overexpression.</strong></p>
<p><strong> </strong></p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p><strong>Decitabine is used in treatment of myelodysplastic syndrome (MDS).</strong></p>
<p><strong>Decitabine is a member of a new class of drugs known as DNA "demethylating" agents. Decitabine also belongs to the category of chemotherapy called antimetabolites..</strong></p>
<p><strong>Decitabine's anticancer effects are believed to be twofold. One way that it works is by demethylation or interfering with the methylation of DNA. By this process of demethylation, normal function to the tumor suppressor genes is restored. Decitabine</strong><strong> are the most widely used inhibitors of DNA methylation which trigger demethylation leading to a consecutive reactivation of epigenetically silenced tumor suppressor genes in vitro and in vivo.</strong><strong></strong></p>
<p><strong>Decitabine-induced hypomethylation in neoplastic cells may restore normal function to genes that are critical for the control of cellular differentiation and proliferation. The cytotoxicity of decitabine may also be attributed to the formation of covalent adducts between DNA methyltransferase . </strong></p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p><strong>Dr Baylin speculates that his epigenetic drugs altered the tumour cells in some lasting way that made them more susceptible to standard chemotherapy.</strong></p>
<p><strong>Unlike other forms of gene regulation epigenetic changes are passed on during cell division to daughter and granddaughter cells until they are actively erased. Once erased, though, they do not return. It might therefore be that epigenetic therapies can effect changes which stop a cancer growing without having to kill all its cells.</strong><strong></strong></p>
<p><strong>The epigenome is dynamic and responsive to environmental signals not only during development, but also throughout life; and it is becoming increasingly apparent that chemicals can cause changes in gene expression that persist long after exposure has ceased.</strong></p>
<p><strong>Premordial germ cell development and Preimplantation period early development are the sensitive period of development.</strong></p>
<p><strong>Treating patient during the sensitive period affect the clearing and resetting of epigenetic marks in PGC development and early embryonic development.</strong></p>
<p><strong>Drugs may alter epigenetic homeostasis by direct or indirect mechanisms. Direct effects may be caused by drugs which affect chromatin architecture or DNA methylation. </strong><strong>Hypomethylation can play a significant role in the carcinogenic process by increasing genomic instability.</strong></p></div>
  </body>
</html>