<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1487 from Anon (session_user_id: 745af99739b97c7890c05be6062d94659ed0eb78)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1487 from Anon (session_user_id: 745af99739b97c7890c05be6062d94659ed0eb78)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>The normal function of DNA methylation at CpG islands is to silence the expression of that DNA. CpG islands are found in promoter regions of genes. If the CpG islands in the promoter region are methylated then the gene is not expressed. Hypermethylation is found in cancer cells. If CpP islands that are normally not methylated become methylated then that gene can no longer be expressed. If this gene is involved in controlling cell growth or cell death then the loss of expression of this gene can be one of the steps (Knudson hypothesis) that leads to cancer.  As methylation marks are mitotically heritable the loss of expression is passed on as the cells divide and the effect is amplified as the number of cell divisions increase.  In other parts of the genome such as intragenic regions and repetitive elements methylation is also used to silence the expression of this DNA. Methylation in these regions stops cryptic transcription start sites, transposable elements from replicating and inserting/disrupting other genes and also prevents illegitimate recombination. Loss of methylation at these sites can lead to increased genomic instability and may result in the activation of or increased expression of genes involved in cell growth or other types of oncogenes. DNA hypermethylation in cancer is locus specific affects CpG islands or CpG island shores silencing tumour suppressor genes or resulting in loss of methylation at ICRs.  DNA hypomethylation is found throughout the genome and is more likely to be found at repetitive elements, intergenic regions, some CpG islands that are poor promoters and also at ICRs. Thus changes in DNA methylation found in cancer can be either a gain in methylation or a loss of methylation and the effect of the gain or loss depends on where in the genome these changes occur.</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>Disruption of the imprinting of the H19/Igf2 cluster contributes to cancer. The methylation pattern of the paternal allele is different to the methylation pattern of the maternal allele and results in the paternal allele being expressed. In the H19/Igf2 cluster methylation of the ICR results in an insulator protein CTCF being prevented from binding and the binding of CTCF prevents Igf2 from being expressed. The paternal allele is methylated at the ICR preventing CTCF from binding and this methylation spreads downstream to H19 promoter which is then silenced. This means that downstream enhancers can access Igf2 to activate expression. In the maternal allele the ICR is not methylated so CTCF can bind to it. This means that the enhancers that activate Igf2 in the paternal allele now activate H19 (lncRNA)  in the maternal allele. In Wilms tumour there is an alteration in methylation the maternal ICR. The ICR becomes hypermethylated which means that CTCF can no longer bind to the maternal ICR. The methylation also spreads to H19 resulting in the Igf2 maternal allele also being expressed. Igf2 is involved in growth and the cell now receives a double dose of a growth promoting factor leading to cancer.</p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine is a DNMT inhibitor. It is a nucleoside analogue which gets incorporated into a nascent DNA strand. DNMT binds to this nucleoside analogue irreversibly which means that not only will the methylation mark not be passed on the amount of DNMT available within the cell will be reduced. As DNMT methylates hemimethylated DNA the new strand will not be methylated as the strand that should have been methylated will have DNMT bound to the nucleoside analogue and can no longer be used as a template for methylation. Decitabine will reduce the overall amount of DNA methylation. If for example a tumour suppressor gene has been silenced by hypermethylation at CpG islands in the promoter region, the use of decitabine could reduce the amount of methylation over subsequent cell divisions resulting in the reactivation of that tumour suppressor gene/s which could reduce the cancer.</p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>DNA methylation is mitotically heritable so altering DNA methylation marks will have enduring effects on the genome. DNMT maintains methylation marks as it uses hemi methylated DNA as its template so as the DNA is replicated the methylation marks are conserved. Drugs that alter DNA methylation patterns are not gene specific. This means that while a drug that for example decreases methylation which could result in reactivation of expression of a tumour suppressor gene may also allow expression of other genes not normally expressed which may cause unwanted side effects and these changes would be mitotically inherited and the long term consequences unknown. A sensitive period is a period of epigenetic reprogramming where epigenetic marks are cleared. They are cleared between generations in early embryonic development: the paternal genome is actively demethylated via hydroxylation and the maternal genome is passive as dependent on cell division. The other sensitive period is during primordial germ cell development. To treat a patient with epigenetic drugs during these periods could result in major disruptions to the expression of other genes with severe consequences eg death of embryo or infertility.</p></div>
  </body>
</html>