<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1842 from Anon (session_user_id: 9553d98a1eb706cc8e4ed589d8239c1fbed50874)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1842 from Anon (session_user_id: 9553d98a1eb706cc8e4ed589d8239c1fbed50874)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>5-methylation of cytosine within cytosine-guanine dinucleotides (CpGs) is a very well known epigenetic mark spread over the genome with mitotic heritability. Even thought that its immediate consequence is silencing in terms of gene expression, its biological behavior is placement-dependent. CpG methylation can be found either in CpG islands (CGIs), intra- and intergenic regions, imprinted control regions (ICRs) or repetitive elements.</p>
<p>CGIs are found near the promoter of a gene and there is far enough evidence that its methylation state inversely correlates to the downstream gene expression. In a normal cell we usually find this CGIs hypomethylated, but the CGIs methylation pattern differs among tissues (the unmethylated CGIs of a cell’s genome could be considered as a guide of its transcriptome). Examples of genes that are likely to be unmethylated (transcribed) in a normal cell are the tumour suppressor genes (such as RB, MLH1 or BRCA1). In contrast, what can be found in cancer cells is specific hypermethylation of these CGIs (each cancer has a particular CGI hypermethylation pattern). This results in silencing the expression of these tumour suppressor genes and thus leading to the inability to arrest the cell cycle or to response to stress conditions.</p>
<p>Intergenic regions and repetitive elements can roughly be described as DNA motifs (including transposons, tandem repeats or IAPs between others) whose transcription leads to genomic instability. In normal cells these motifs are spread all over the genome and they tend to be methylated (and so, repressed). In cancer cells we find genome-wide hypomethylation, meaning that these elements are more accessible for RNApolII. The transcription of these elements, sensitizes DNA to mutation.</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>H19/Igf2 cluster has imprinted regulation, which means that paternal and maternal allele have different epigenetic marks (DNA methylation or unmethylation at the ICR) leading to different expression patterns.</p>
<p>In this cluster, paternal ICR is methylated. This methylation (or its expansion) silences the expression of H19 (a lncRNA) and, more related to the imprinted behavior, this methylation prevents CTCF (an isolator DNA binding protein) binding to the ICR itself. This fact let downstream enhancer reach to Igf2 (Insulin-like growth factor 2) gene and thus activates its transcription.</p>
<p>On the other hand, the maternal ICR is unmethylated and so CTCF’s zinc fingers are able to reach DNA. This binding forces a different chromatin loop that makes downstream enhancers promote H19 expression and therefore prevents them for reaching Igf2 gene which remains isolated and silent.</p>
<p>In Wilm’s tumour is common to find loss of imprinting by hypermethylation of the maternal allele. This epigenotype stimulates the overexpression of Igf2 which is, by the fact, an oncogene</p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine is a DNMTi (DNA Methyl Transferase inhibitor). Chemically it is a cytidine analog that incorporates into the DNA after replication. It is able to kidnap (by irreversible covalent bonds) DNMT when it arrives to copy the methylation pattern on the new synthesized strand. This results in mitotic-dependent genome demethylation.</p>
<p>The way that this demethylation can have an anti-tumour effect is not yet well known, but it might be (as evidence highlights in some leukemias) because of tumour suppressor genes’ CGIs demethylation (and therefore activation).</p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>One of the most interesting aspects of targeting DNA methylation is its mitotic heritability. That means that any change achieved in the methylome of a cancer cell would be maintained throughout its descendents.</p>
<p>Physiological great changes in methylome (including addition and depletion of methyl groups to DNA) only takes place in two situations: early embryonic development and primordial germ cell development. These two periods of active epigenetic remodeling (including erasure and reestablishment of epigenetic marks) are called sensitive periods. Its name is not fortuity, as it express the possibility to alter the epigenetic state of an organism by external agents (as changes in the methylome in these periods would be transmitted to every daughter cell).</p>
<p>So we may be able to deduce that treating patients with drugs interfering DNA methylation during sensitive periods might change the methylome of stem cells that will be maintained through all of its descendent cell types. And we can’t forget that an altered epigenotype is very likely to provoke disease.</p></div>
  </body>
</html>