<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 773 from Anon (session_user_id: 908d5445b871f43cc7d1bdf23e1d235a128e3ccb)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 773 from Anon (session_user_id: 908d5445b871f43cc7d1bdf23e1d235a128e3ccb)</h1>
    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Any methylation event that escapes the clearance-&amp;-reprogramming functions of the two major embryonic check-points during which epigenetic marks are typically ‘re-set’ (early embryogenesis, the 4-to-16-cell stage, and/or the primordial germ cell [PGC] stage) are termed ‘imprinting’.  These become traits that are heritable from that one PGC to its daughter cell.  Possibly heritable from the organism to its progeny, if this imprinting event takes place in a germ cell rather than a somatic cell.  The two embryonic check-points are considered to be the <em>“<span style="text-decoration:underline;">sensitive periods</span>”</em> when a cell may be especially sensitive to environmental epigenetic mutagens.  So that as a consequence, administering any sort of treatment, especially treatment with regard to cancer(s) that may often be more or less bioactive (and possibly intrusively so), might be highly ill advised when applied at a time of these two epigenetic developmental checkpoints.  Which, since they take place at a time of embryonic development, may seem like the old adage “shipping coals to Newcastle”, but understanding some of the dynamics of those risks (that are very recently elucidated), are not at all out-of-line.  No one would argue with that for a single moment.</p></div>

    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p class="Default">CpG islands are often found (upstream) near promoter regions of many genes.  Methylated DNA (of almost any sort) tends to be closed, and otherwise unavailable for transcription.  The CpG islands associated with gene promoter regions are usually <strong><em><span style="text-decoration:underline;">un</span></em></strong>menthylated.  If they become methylated, that methylation can (and usually does) spread to the gene’s promoter region, which silences the gene.  CpG islands can become methylated in many cancers.  If the CpG island for a tumor suppressor protein becomes methylated, then that gene becomes silenced, and the result is would be silenced, and the result would be a lack of the protein that helped to reduce tumors, leading to an increase in a cancerous state.</p>
<p class="Default"> </p>
<p class="Default">In those portions of DNA inbetween that which actively codes for functioning genes, there can frequently be found regions in which there are the same series of bases repeated over and over again, often quite lengthy regions.  <em>In vivo</em>, these regions are usually methylated, which under normative conditions keeps these repetitive elements closed and otherwise unavailable.  However, in many cancers these repetitive elements become <strong><em><span style="text-decoration:underline;">un</span></em></strong>methylated.  As a result, large chromosomal recombination events can take place at these now open areas of same-base-pair repetitions, if similar repetitions of that base pair combination exist on some other (nearby!) chromosome, as a result of the cancer.  And these long terminal repeat (LTR) destabilizing mutations are not small point-mutation-level events such as frameshift, nonsense, or missense mutations;  these are great big translocations of entire sections of chromomes.  (<em>e.g.:</em>  translocations, deletions, insertions, <em>etc.</em>)</p>
<p class="Default"> </p>
<p>So that there’s sort of a reversal of methylation status:  CpG islands that are normatively non-methylated become methylated, and repetitive elements that are normally methylated lose their methylation status, in many transformative neoplasias.</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p class="Default">Inbetween Igf2 (insulin-like growth factor 2) and H19 (a repeating element; lnRNA), is the imprint control region (ICR) regulating expression of both genes (dependent on maternal or paternal allele).  <em>In trans</em> enhancer regions are capable of looping back and attaching to either H19 or Igf2, initiating expression of either gene.  If the ICR is <strong><em><span style="text-decoration:underline;">un</span></em></strong>methylated, then the enhancers bind to H19 preferentially.  When the H19 gene is <em><span style="text-decoration:underline;">in</span></em>active because of methylation, then the enhancers bind further upstream to Igf2.</p>
<p class="Default"> </p>
<p class="Default">An ‘insulator’ protein, CTCF, binds to the ICR in the maternal allele only and prevents ICR methylation, which prevents H19 methylation, assuring H19 expression (and <em><span style="text-decoration:underline;">not</span></em> Igf2).  Only in the maternal allele.</p>
<p class="Default"> </p>
<p class="Default">CTCF, is not present in the paternal allele, and the ICR <em><span style="text-decoration:underline;">is</span></em> methylated, so the <em>in trans</em> enhancers instead bind to the Igf2 gene, promoting expression of Igf2.  In the paternal allele.</p>
<p class="Default"> </p>
<p>In “Wilm’s tumor”, there is no maternal CTCF, so the maternal ICR <em><span style="text-decoration:underline;">is</span></em> methylated.  As such Igf2 is expressed, as if the maternal allele was a paternal one.  Resulting in twice as much Igf2 protein; a growth protein.  Leading to tumor growth.  Tumors of the kidney, oddly enough, in very young children (not adults).</p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p class="Default">The article calls decitabine a DNA <em><span style="text-decoration:underline;">de-</span></em>methylating agent.  <em>If</em> DNA methylation preceeds acute myelogenous leukemia, one presumes (since DNA methylation silences gene expression) some tumor suppressor gene(s) are being <em>hyper</em>methylated, causing leukocyte dysplasia, and eventual neoplasia.  A drug <em>de-</em>methylating such sites would cause increased tumor <em>suppression</em>, and a <em>decrease</em> in pathology.</p>
<p class="Default"> </p>
<p class="Default">REFERENCE:</p>
<p class="Default"> </p>
<p>"Decitabine is believed to exert its antineoplastic effects after phosphorylation and direct incorporation into DNA and inhibition of DNA methyltransferase, causing hypomethylation of DNA and cellular differentiation or apoptosis.  Decitabine inhibits DNA methylation in vitro, which is achieved at concentrations that do not cause major suppression of DNA synthesis.  Decitabine-induced hypomethylation in neoplastic cells may restore normal function to genes that are critical for the control of cellular differentiation and proliferation…"</p>
<p> </p>
<p class="Default"><strong><em><span style="text-decoration:underline;">DailyMed</span></em></strong> :  US NIM/NIH drug web site:</p>
<p><em>http://dailymed.nlm.nih.gov/dailymed/lookup.cfm?setid=4e1f4eb4-19b2-eee8-3402-bd2ccf622791#nlm34090-1</em></p></div>
  </body>
</html>