<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 177 from Anon (session_user_id: 1592418a92adaf40b06e3e1608389dcfeedde3ed)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 177 from Anon (session_user_id: 1592418a92adaf40b06e3e1608389dcfeedde3ed)</h1>
    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>Genomic imprinting occurs when one of the two copies of inherited alleles of a gene is silent and the other <span style="font-size:14px;line-height:21px;">is expressed dependent on the parent of origin. Transcriptional repressor CTCF, a chromatin remodeler, </span><span style="font-size:14px;line-height:21px;">binds to DNA on chromosome 11. It binds to the maternal allele upstream of H19 long non-coding RNA  </span><span style="font-size:14px;line-height:21px;">region resulting in expression of H19 lncRNA and insulation of  the Igf2 gene from downstream enhancers. </span><span style="font-size:14px;line-height:21px;">CTCF does not bind to the paternal allele as the  binding site is methylated, i.e., the Imprint Control Region</span><span style="font-size:14px;line-height:21px;">(ICR)  is paternally imprinted.  On the paternal allele, Igf2 expression will occur due to the direct action of </span><span style="font-size:14px;line-height:21px;">the enhancers and H19 lncRNA is not expressed.</span></p>
<p><span style="font-size:14px;line-height:21px;">Normally, H19 lncRNA would be maternally expressed and Igf2 paternally expressed.  However, when </span><span style="font-size:14px;line-height:21px;">DNA hypermethylation of the ICR upstream of H19 occurs, binding of CTCF to the maternal allele is inhibited </span><span style="font-size:14px;line-height:21px;">and chromatin remodeling occurs. This disruption of imprinting causes activation and overexpression </span><span style="font-size:14px;line-height:21px;">of Igf2 in Wilm’s tumor.  Igf2 acts, therefore, as an oncogene in Wilms’ tumor.  H19, normally a </span><span style="font-size:14px;line-height:21px;">tumor suppressor, becomes methylated on the maternal allele in Wilms’ tumor due to methylation spreading </span><span style="font-size:14px;line-height:21px;">during chromatin remodeling and its normal function is disrupted.</span></p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine acts as a DNMT (DNA methyltransferase) inhibitor by binding irreversibly to DNA strands.  Specifically, the <span style="font-size:14px;line-height:21px;">methyl group found at position five of the Cytosine base is removed resulting in (DNA) hypomethylation.  The modified </span><span style="font-size:14px;line-height:21px;">Cytosine base (i.e., “nucleoside analog”) without the methyl group is incorporated into the DNA resulting in </span><span style="font-size:14px;line-height:21px;">inhibition of both DNA replication and DNMT activity.  As Decitabine has been demonstrated to reverse DNA </span><span style="font-size:14px;line-height:21px;">methylation in regions of (Myelodysplastic Syndrome) gene promoters, silenced tumor suppressor genes are turned </span><span style="font-size:14px;line-height:21px;">on again. Because DNA methylation itself  is an epigenetic modification, Decitabine, which results in DNA </span><span style="font-size:14px;line-height:21px;">hypomethylation, can be considered to have an epigenetic effect on the patient’s genome.</span></p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Drugs with an epigenetic mode of action include ones in which epigenetic marks including methyl groups incorporated in <span style="font-size:14px;line-height:21px;">(or removed from) DNA are passed to daughter cells during cell division. These alterations will not change the primary DNA sequence </span><span style="font-size:14px;line-height:21px;">but may affect gene function.  The marks will be maintained (and will result in cell differentiation) until they are actively removed </span><span style="font-size:14px;line-height:21px;">through enzymatic or other actions.</span></p>
<p><span style="font-size:14px;line-height:21px;">In light of this mechanism of action, it would be prudent not to administer these drugs during those periods where epigenetic marks are </span><span style="font-size:14px;line-height:21px;">normally established and/or reprogrammed, i.e., sensitive periods. These periods occurs in primordial germ cell development and </span><span style="font-size:14px;line-height:21px;">early embryonic development, i.e., pre-implantation of the embryo.  During these periods of genome-wide epigenetic reprogramming, </span><span style="font-size:14px;line-height:21px;">imprinting, X-chromosome inactivation and other genomic activities occur. Administration of epigenetic drugs during sensitive periods may therefore result in imprinting disorders, X-chromosome inactivation issues and other </span><span style="font-size:14px;line-height:21px;">unforeseen consequences in specific tissues or cells resulting from disruptions of epigenetic machinery .</span></p></div>

    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>The normal function of DNA methylation at CpG islands is gene expression silencing. As these islands <span style="font-size:14px;line-height:21px;">are present in the 5’ promoter regions of the genes, the methylation serves to turn off transcription </span><span style="font-size:14px;line-height:21px;">of the gene(s).</span></p>
<p> <span style="font-size:14px;line-height:21px;">In cancer, these islands are frequently hyper-methylated resulting in modifications to epigenetic control </span><span style="font-size:14px;line-height:21px;">mechanisms.  Methylation can occur at the CpG islands or in neighboring regions (“shores”) and is </span><span style="font-size:14px;line-height:21px;">mitotically heritable. Hyper-methylation in CpG islands on gene promoters can result in a loss of imprinting </span><span style="font-size:14px;line-height:21px;">leading to overexpression of tumor promoter genes or it can have a loss of function effect on tumor </span><span style="font-size:14px;line-height:21px;">suppressors.  There may also be recruitment of other proteins (e.g. PCR2) that impact remodeling of </span><span style="font-size:14px;line-height:21px;">chromatin and a role for microRNA in CpG island hyper-methylation. It ishould be noted that loss of </span><span style="font-size:14px;line-height:21px;">methylation in CpG islands (hypo-methylation) can activate oncogenes. Therefore, disruption of DNA </span><span style="font-size:14px;line-height:21px;">methylation at CpG islands including both hyper- and hypo-methylation can result in disease and/or cancer.</span></p>
<p><span style="font-size:14px;line-height:21px;">DNA methylation in intergenic regions and in repetitive elements allows for maintenance of normal </span><span style="font-size:14px;line-height:21px;">chromosome structure and maintenance of gene regulation through transcriptional silencing. Loss of DNA </span><span style="font-size:14px;line-height:21px;">methylation in these regions and in repeats can lead to genomic instability through reciprocal translocation, </span><span style="font-size:14px;line-height:21px;">deletions and insertions in chromosomes. This occurs due to activation of transposons and cryptic promoters.</span></p>
<p><span style="font-size:14px;line-height:21px;">Genomic instability is associated with tumor growth and the effect of loss of DNA methylation can be </span><span style="font-size:14px;line-height:21px;">related to the stage of the cancer, i.e., it progresses with the development of the tumors.  In addition, </span><span style="font-size:14px;line-height:21px;">mutations in DNMT enzymes required for DNA methylation in intergenic regions and repeats have been </span><span style="font-size:14px;line-height:21px;">associated with genomic instability and disease.</span></p>
<p> </p></div>
  </body>
</html>