<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 217 from Anon (session_user_id: 469fd6f87b6d1d75103a4bb245a0776461b1461c)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 217 from Anon (session_user_id: 469fd6f87b6d1d75103a4bb245a0776461b1461c)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>It is well known that DNA methylation at CpG islands correlates heavily with gene expression; the methylated promoters are silenced while the unmethylated ones are expressed. Cancer cells frequently show overall hypermethylation of CpG islands, which is thought to promote the disease by deactivating the tumour suppressor genes. The intergenic regions and repeats, meanwhile, are normally hypermethylated; this prevents sporadic transcription of transposon DNA and nearby areas as well as erroneous crossing-over at similar but non-homologous parts of the chromosome during meiosis (as methylated DNA tends not to form chiasms). Cancer cells, however, are characterized by genome-wide hypomethylation, which decreases genomic stability (leading to more mutations) and enabling the transposon transcription (which, in turn, destabilizes the genome even more and leads to dysregulated transcription of surrounding genes).</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>H19/Igf2 cluster is one of the better studied imprinted gene clusters. Its expression pattern is governed by the methylation of its imprint control region (ICR). On the paternal allele, the ICR is normally methylated; this blocks the binding of insulator protein CTCF, allowing expression of the growth factor Igf2. On the maternal allele, the ICR is normally unmethylated; this allows the binding of CTCF, thus blocking the expression of Igf2. In the absence of Igf2, H19 is transcribed, while on the paternal allele the methylation spreads from the ICR into the H19 promoter, silencing the gene. Overall, H19 is expressed only from the maternal allele, while Igf2 expression is constricted to the paternal one.</p>
<p>Having Wilm's tumour, however, changes the epigenetic layout. This particular tumour is associated with the aberrant hypermethylation of both maternal and paternal ICRs. Thus both alleles show expression pattern identical to the normal paternal allele, namely Igf2 is expressed and H19 is not. The double dose of Igf2, a growth factor, stimulates the cell proliferation, so the affected cells reproduce more readily, which is the characterizing feature of any tumour. Studies show that H19 acts as a growth suppressor, so the silencing of H19 in Wilm's tumour contributes to the growth promotion even further.</p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine belongs to the class of DNMT inhibitors; it thus opposes the various DNA methyltransferases that lay down DNA methylation. Decitabine contributes to the passive methylation loss; if DNMT is stopped from maintaining the methylation through mitosis, the methylated DNA will dilute over cell divisions. Moreover, no 'de novo' DNA methylation will be laid by malfunctioning epigenetic mechanisms while Decitabine is present in the system. As we know, the hypermethylation of CpG islands is generally characteristic of cancer, so stopping or reverting it might be a good way to treat the disease, for example, by reactivating the tumour suppressor genes which have been silenced.</p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>The organism has several mechanisms which propagate the epigenetic state through mitosis. Some of them deal with the DNA methylation; for example, DNMT1 acts on hemimethylated DNA and thus restores the methylation pattern during mitosis (more precisely, during DNA synthesis). Any changes to the DNA methylation which are not actively reverted will therefore remain present in all of the cell's descendants. This may be problematic during sensitive periods when the epigenetic marks are removed and laid down anew; these periods include early embryonic development and primary germ cell growth. Any disruption of finely tuned epigenetic machinery during these periods may well permeate the whole lifespan of an individual (if the impact occurs during early embryonic period) or their offspring (if it occurs during PGC development); the effects of this persistent change are hard to predict but highly unlikely to be beneficial. Thus epigenetic drugs should not be administered during these stages.</p></div>
  </body>
</html>