<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 216 from Anon (session_user_id: 064d26351bb8475251d2058ef34b8d757f3d49f4)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 216 from Anon (session_user_id: 064d26351bb8475251d2058ef34b8d757f3d49f4)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>DNA methylation is an epigenetic mechanism
which is found in a large number of CpG dinucleotides in cell genome, but not
in CpG islands (found in about 60% of promoters). DNA methylation of promoter
regions usually causes silencing of the underlying gene. Instead of that, CpG
islands, which are characterized for having a high CpG density, are usually
kept free of methylations independently of their activity state. In cancer
cells, however, CpG islands are found to be hypermethylated, which means that
the underlying gene is not expressed. DNA methylation in this case is an
alternative to genetic mutation, to silence tumour suppressor genes. These
genes are involved in mechanisms which could stop or slow down cancer
progression; so, by silencing them, cancer cells can grow without control. It
occurs frequently in different kinds of tumours and it is mitotically heritable
and rapidly selected. </p>

<p><span>Methylation in normal cells is also generally
present at repetitive sequences and intergenic regions. It is a way to repress
transcriptional noise in highly expressed genes, to inhibit antisense
transcription or to direct RNA splicing. It is also useful to repress repetitive
sequences such as satellite repeats or retroviral insertions (LINEs and SINEs),
to get more stability in the genome. But cancer cells usually show to be
hypomethylated in these regions, and it can result in a high genomic
instability, which is one of the hallmarks of cancer. Another less common
alteration in cancer cell is hypomethylation of CpG poor promoters, which can
lead to the activation of oncogenes.</span></p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>Disruption of imprinting means that genes, which
should be displaying monoallelic expression (parent-of-origin-specific), become
either expressed or silent from both parental alleles. In cancer, this happens
because a lot of imprinted genes are related to growth. </p>

<p>An example is the H19/Igf2 cluster. In normal
conditions, maternal allele has an unmethylated imprint control region and
paternal allele shows methylation in the same region. When it is unmethylated,
an insulator element called CTCF can bind to this region. CTCF insulates Igf2
from downstream enhancers, but the enhancers will act on H19. So, Igf2 will be
silent for the maternal allele. On the paternal allele, because of the
methylation, CTCF cannot bind and the enhancers are not blocked to act on Igf2
and Igf2 is expressed.</p>

<p><span>In Wilm’s tumour, it is found to be hypermethylation
of the control region on the maternal allele, too. So, there is also expression
of Igf2 on the maternal allele. The quantity of Igf2, which is related to
growth promotion, is increased compared to normal cells. That is why cancer can
make progress and tumours are generated. </span></p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine is a DNA-demethylated agent. It is a nucleotide analogue which inhibits DNMTs, by binding them irreversibly after they are incorporated into DNA. There is a lack of specificity and its mechanism of action is still unclear.</p><p>As it
is known, hypermethylation of CpG islands of promoters can lead to inhibition
of some tumour suppression genes in cancer cells. Besides, sometimes, there is
a loss of imprinting in cancer cells because of hypermethylation. The use of a
drug which has an effect demethylating DNA in patients will affect to the
methylation pattern in their cancer cells. That means that tumour suppression
genes could be expressed again, for example. They usually have effect on growth
control or apoptosis, so cancer cells will not be as much uncontrolled as they
were, and they will disappear or will become more susceptible to other drugs
against cancer.</p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Epigenetic alterations
are mitotically heritable changes in gene expression, which is why DNA
methylation can have enduring effects on the epigenome, by transmitting from
cell to cell.  So, once the drug treatment has had effect, its
influence will remain for long periods. There are some periods which there is
increased sensitivity to regulatory effects of epigenetic mechanisms, that is
called a sensitive period. </p>

<p><span>During development, there
are some sensitive periods: germ cell development and early embryonic
development. In these periods, epigenetics marks are removed and there is an
epigenetic reprogramming. So, it would be inadvisable to treat patients during
these episodes, because it could alter epigenetic information of the cell and
unexpected effects could appear. It is better to use drugs when epigenetic
reprogramming is done.</span></p></div>
  </body>
</html>