<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 870 from Anon (session_user_id: 095ac3a8f64ccea3159c9be84c3d927e89a27634)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 870 from Anon (session_user_id: 095ac3a8f64ccea3159c9be84c3d927e89a27634)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><div>Methylation at CpG islands is associated with gene silencing as CpG islands are common in gene promotors - the methylation prevents promotion. This methylation is strongly related to imprinting. Normally, promotors are unmethylated and hence carry out their normal duty. In cancer, CpG islands (and shores) become hypermethylated and this results in the silencing of tumour suppressor genes. Unfortunately, this methylation is mitotically heritable and the epimutation is rapidly selected leading to uncontrolled growth and reproduction in, for example, retinoblastoma and breast cancer. This effect is one of the required multiple hits in Knudson hypothesis. </div><div><br /></div><div>Normally, intergenic regions and repetitive elements are methylated to stabilize the genome by maintaining imprinting and silencing transposable elements. HOwever, in cancer, these regions may become hypomethylated (for example, through mutations in DNMT3B) leading to genomic instability.</div><div><br /></div><div>Instability is a result of several mechanisms. The activation of repeats can lead to transpositions. The presence of similar repeats on multiple chromosomes can alse lead to incorrect recombinations. The outcome may be chromosomal translocations, deletions and insertions. Finally, hypomethylation may lead to activation of cryptic promotors with concommitant gene disruption.</div></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><div>The H19/Igf2 cluster consists of the Igf2 gene, an imprinting control region (ICR), the H19 gene and Igf2 enhancers. In the paternal allele, methylation of the ICR prevents binding of the CTCF insulator protein. When CTCF does not bind, methylation spreads along the cluster to the H19 promoter silencing the H19 gene. This allows the enhancers to activate the Igf2 gene.</div><div><br /></div><div>In the maternal allele the ICR is not hypermethylated and so the CTCF protein can bind to it. Following CTCF binding, methylation of the H19 promoter does not occur and the H19 lncRNA is expressed blocking the enhancers. Thus the enhancers cannot activate the Igf2 gene.</div><div><br /></div><div>In Wilm's tumour, there is hypermethylation of the maternal allele's ICR which makes it behave in the same way as the paternal allele described above. This leads to (biallelic) overexpression of Igf2. In turn, there is now poor growth regulation of affected cells (Igf2 is mitigenic) and subsequent tumourigenesis. </div></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><div>Decitabine is a DNA methyltransferase inhibitor (DNMTi) which reduces DNA methylation. It is a cytidine analogue and is thought to function in two ways. In high doses, its incorporation into DNA is cytotoxic and, while it kills tumour cells, there are associated side effects. At lower doses, Decitabine covalently binds DNMT blocking its methyltransferase activity. Hence, methylation is lost during replication.</div><div><br /></div><div>The outcome of this demethylation is the reversal of cancer related methylation. This may include upregulation of growth inhibitors, tumour suppressors and possible apoptosis promoters.</div></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><div>DNA methylation can be passed on to daughter cells mitotically. DNA methylation is mitotically heritable. This is due to the way DNMT1 copies methylation marks from one strand of hemi-methylated DNA to the other. Both daughter cells will have the same methylation marks. Due to this mitotic heritability, the methylation state set in early development would remain for life (metastable epi-alleles). Note that TET does not cause demthylation of the DNA as it is expressed only at specific periods of development. Note also that DNA methylation is unlikely to be the transgenerational epigenetic mark.</div><div><br /></div><div>Epigenetic drugs should be avoided during the sensitive periods - the times at which epigenetic reprogramming (clearing and resetting of epigenetic marks) occurs. These times are during early development of the embryo (fertilisation to epiblast) and during primordial germ cell development. It has also been suggested that there may be an alteration to oogenesis during pre-conception making this another sensitive period.</div><div><br /></div><div>Treating patients during these sensitive periods is not advised due to the risk of modifying a newly established epigenetic marks. It is always a possibility that an uncorrected error could arise due to a methylation change. This uncorreted error could lead to cancer.</div></div>
  </body>
</html>