<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1270 from Anon (session_user_id: d4398e31a96080a367598c8bc9eec5a8eeb4c6a1)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1270 from Anon (session_user_id: d4398e31a96080a367598c8bc9eec5a8eeb4c6a1)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p><strong>DNA methylation</strong> is a specific epigenetic modification that occurs at cytosines that are followed by guanines (CpG dinucleotides). Many CpG are found at the <strong>GpG islands</strong> which are often located at the promoters of genes, that is, at the start of the genes where the transcription machinery binds. However, in <strong>normal cells</strong> CpG islands tend to be protected from methylation, so most of them are <strong>unmethylated</strong>. At particular times methylation of a small subset of CpG islands can be found, which is almost always indicative of silencing of gene expression. In contrast, in <strong>cancer cells</strong> CpG islands are more likely to be <strong>methylated</strong> than in normal cells. This event is called CpG island hypermethylation (<strong>CGI hypermethylation),</strong> it's locus-specific and it tends to occur in the CpG islands of the promoters of <strong>tumor suppressor genes</strong>, leading to their silencing or inactivation. Therefore, cancer cells are more likely to divide more rapidly or to not die as much as normal cells. Since DNA methylation is <strong>mitotically heritable</strong>, epigenetic inactivation of these genes can be transmited to daughter cells and these epimutations can be rapidly selected for, promoting the appearance of <strong>tumorigenesis</strong>.</p>
<p>Other regions of the genes such as <strong>intergenic regions</strong> and <strong>repetitive elements</strong> are <strong>methylated</strong> at CpG dinucleotides in <strong>normal cells</strong>. DNA methylation at these regions makes chromatin be more compacted (<strong>heterochromatin</strong>) preventing from illegitimate recombinations between regions with certain homology that shouldn't occur. So, these DNA methylations <strong>maintain genomic integrity</strong>. In <strong>cancer cells</strong> there is genome-wide <strong>DNA hypomethylation</strong> so that intergenic regions and repetitive elements are <strong>unmethylated</strong>, leading to an open-chromatin structure  (<strong>euchromatin</strong>) which facilitates misaligning of these regions leading to <strong>illegitimate recombinations</strong>. This can result in deletions, insertions, duplications and translocations of/between chromosomes. Repetitive elements can also be activated because of the hypomethylation, so they can make a copy of themselves and jump around the genome (<strong>transposition</strong>), potentially disrupting the coding region of a gene or activating neighboring genes.</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p><strong>H19/Igf2</strong> is a cluster of <strong>imprinted genes</strong> that are <strong>paternally imprinted</strong>, so all cells of the organism are DNA methylated on the paternal <strong>imprint control region</strong> (<strong>ICR</strong>) located between these two genes (Igf2 upstream ICR; ICR upstream H19).</p>
<p>By contrast, this ICR is unmethylated on the <strong>maternal allele</strong>. The unmethylated ICR is bound by an insulator protein called CTCF, which insulates Igf2 promoter from the action of enhancers located downstream H19, so <strong>Igf2 gene remain inactive</strong>. These enhancers then act on H19 as secondary option, which <strong>activates H19 expression</strong> from the maternal allele.</p>
<p>On the <strong>paternal allele</strong>, CTCF cannot bind to methlyated ICR, so that the enhancers are free to act on Igf2 and <strong>promote Igf2's expression</strong>. <strong>H19 gene is inactive </strong>on the paternal allele because the DNA methylation at the ICR can spread downstream into the H19 CPG island promoter, which causes gene silencing.</p>
<p>Imprinting at the H19/Igf2 cluster is disrupted in <strong>Wilm’s tumour</strong> by <strong>hypermethylation</strong> of its ICR on the maternal allele, which leads to expression of Ifg2 also on the maternal allele. So children suffering from this disease have <strong>overexpression of Igf2</strong> in comparison to normal children. As Igf2 is a growth promoting gene, cells with this imprinting disruption are more likely to grow and divide faster than normal cells, leading to <strong>tumorigenesis</strong>.</p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p><strong>Decitabine</strong> is a cytosine nucleoside analog that is incorporated into the DNA upon replication. Once incorporated into DNA, decitabine blocks further DNA methylation by the <strong>inhibition of DNA methyltransferase-1</strong> (<strong>DNMT1</strong>). When DNMT1 binds that nucleotide to copy the methylation to the daughter strand, it is bound irreversibly and it can no longer be released. In this way, decitabine <strong>declines</strong> the general level of <strong>DNA methylation</strong> through cell division.</p>
<p><strong>DNA methylation</strong> is an important epigenetic mechanism for regulation of gene expression and plays a key role in the pathogenesis of hematologic malignancies. <strong>Hypermethylation</strong> in CpG island promoters of tumour suppressor genes has been detected in myelodysplastic syndrome and acute myelogenous leukemia, and it has been also associated with <strong>tumour progression</strong>. Aberrantly silenced genes, including tumour suppressor genes, can be reactivated after <strong>treatment with decitabine</strong> due to its effect on DNA methylation decline. Since the action of this DNMT1 inhibitor is division dependent, cancer cells seem to be more severely affected.</p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Symmetric <strong>DNA methylation</strong> at CpG dinucleotides is <strong>mitotically heritable</strong>. When the DNA is replicated, only the parent strand maintains the methyl group on the CPG dinucleotide, while the daughter strand has an unmethylated cytosine. DNMT1 binds to this hemi-methylated DNA and lays down methylation on the daughter strand, restoring a fully methlayted CpG dinucleotide. So, any alteration caused in DNA methylation status is transmitted after cellular division. That’s how drugs that alter DNA methylation have effects that last beyond the period of treatment.</p>
<p>Epigenetic marks are very important in cell differentiation, in control of gene expression and for chromosome structure. However, there are periods through life’s cycle in which these epigenetic marks must be removed and <em>de novo</em> established by <strong>epigenetic reprogramming</strong>. These are <strong>sensitive periods</strong> because is when environment is more likely to influence epigenetic conformation.</p>
<p>There are two epigenetic reprogramming periods through life’s cycle: during early embryonic development (DNA methylation reprogramming of parents’ genome) and during primordial germ cell development (DNA methylation reprogramming of parents’ genome and imprinted genes).</p>
<p>Treating patients during sensitive periods is inadvisable because is when they are more likely to influence epigenetic makeup.  Potential epigenetic alterations occurred during epigenetic reprogramming process may lead to consequences that can last whole life and could be transgenerationally inherited through the gametes.</p></div>
  </body>
</html>