<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 947 from Anon (session_user_id: 58988267b21f9480fd828a45a48364ccbc5456f5)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 947 from Anon (session_user_id: 58988267b21f9480fd828a45a48364ccbc5456f5)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>The main repository of genetic information comprises the order of base pairs forming DNA. The requirements of germ cells differ from those of somatic cells; germ cells are potentially immortal (capable of indefinite division) and totipotent (within the repertoire of information encoded in the genome) whereas somatic cells are required only for the life of the organism, must cooperate with other somatic cells in the formation of specialised tissues, and must desist from unrestrained division.</p>
<p>DNA is typically organised in: regions coding for proteins ("genes", subdivided by introns into information-rich exons); regions rich in cytosine followed by guanine (CpG), known as CpG islands (CGIs), typically located at promoter regions; and  intergenic regions (some featuring repetitive elements). Transcriptional activity is approximately inversely proportional to methylation. Differential methylation allows differential expression of genes in specialised tissues (including extraembryonic tissues eg placenta), and differential expression of genes by parent of origin ("imprinting"). Reversible methylation allows: totipotency in germ cells; imprinting at key stages of development; establishment and maintenance of specialised somatic cells; and "compensation" for duplicated genes (particularly X chromosome). </p>
<p>CGIs are usually unmethylated; methylation typically suppresses transcription of adjacent genes, so that mitotically inheritable differential expression might be established in specialised tissues. However, some genes are of "tumour suppressor" type so that methylation in these cases risks loss of expression of antioncogenes, with acquisition of behaviours seen in cancer.</p>
<p>Intergenic regions and repetitive elements are usually methylated (typically at diffusely distributed CpG dinucleotides); this protects against genomic instability, suppressing replication errors (during mitosis) and transposition of chromosomal sections (possibly including promoter regions). Cancerous cells typically show generalised demethylation in these regions, associated with genomic instability, risking transposition of active promoter regions to oncogenes, resulting in their overexpression. </p>
<p>Abnormal methylation disrupts imprinting, with abnormal gene expression affecting growth, tumour susceptibility and behaviour. </p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>DNA methylation provides a means to regulate DNA expression in somatic cells but remains reversible to meet ontogenic requirements at particular stages.</p>
<p>Gametic DNA is heavily methylated, protecting the genome and assisting dense packaging (particularly for spermatozoa). Demethylation follows fusion of pronuclei (active for the paternal genome), together with clearance of other epigenetic marks, reestablishing totipotency in the diploid zygote. Passive maternal demethylation (by exclusion of DNMT) follows cell division to the blastocyst stage. The inner cell mass (presumptive embryo) is identified at this stage, followed by differential remethylation, reestablishing tissue specific (and imprinted) epigenetic marks (including extraembryonic trophoblastic tissues). Primordial germ cells (destined for gametogenesis) undergo a second wave of demethylation, followed by a further wave of gonocyte-specific methylation (particularly in respective imprint controlling regions of spermatogonia and oogonia). </p>
<p>Orderly tissue-specific DNA methylation is important in these two periods (immediately following formation of the zygote and following selection of primordial germ cells) after physiological clearance. Consequently, interference with methylation at these particular stages (the "sensitive periods") will have enduring effects on the epigenome. </p>
<p>As administration of DNMT inhibitors during sensitive periods of physiological methylation (young children and pregnant women) risks congenital abnormalities, their use would be imprudent. </p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>Many imprinted genes affect growth, promoting or suppressing cellular division. As cancer includes abnormal cellular division, these genes require tight control. Imprinting provides one mechanism whereby this might be achieved by ensuring monoallelic (rather than biallelic) expression, providing control over the "dose" of gene product.</p>
<p>One such growth promoting gene is IgF2; expression is typically controlled by downstream enhancers. Between IgF2 and its enhancers lie an imprint controlling region (ICR)<span>, and a region (H19) </span>which produces a long noncoding RNA (lncRNA). H19 lncRNA suppresses the IgF2 enhancers.</p>
<p>Methylation of the ICR is normally present in the paternally derived allele. This leads to the chain of events: suppressed binding of CCCTC-Binding factor (CTCF), permitting methylation of H19 promoters, suppressing production of lncRNA, allowing expression of downstream enhancers, fostering expression of paternal IgF2.</p>
<p>In contrast, maternally derived ICR is demethylated, allowing binding of CTCF, suppressing methylation of H19 promoters, allowing production of lncRNA, suppressing downstream enhancers thereby suppressing IgF2.</p>
<p>In Wilm's tumour, additional methylation of maternally derived ICR leads to expression of maternal IgF2, leading to overall overexpression. This may be sporadic.</p>
<p>Constitutional disruption of imprinting may cause Beckwith-Wiedmann syndrome, characterised by macrosomia and predisposition to Wilm's tumour. </p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine is an inhibitor of denovo methyltransferase (DNMT); consequently, the class of chemotherapeutic agents is typically denoted by DNMTi. DNMT is required to methylate DNA, both under physiological conditions, and in cancer.</p>
<p>Abnormal patterns of methylation (particularly at CpG islands) are known associations of cancer. Decitabine binds to DNA, where it blocks the action of DNMT, hence preventing methylation; that is, it does not cause demethylation, rather it prevents mitotically inherited methylation, including the abnormal type seen in a clone of cancer cells. Thus it will not affect cells that are not dividing. Of those that are dividing, it will exert its greatest effect on rapidly dividing cells, such as cancer cells (but it will inevitably affect any dividing cells).</p>
<p>Empirical evidence shows it is particularly useful in acute myeloid leukaemia, when given in carefully controlled doses. It is not known whether its antineoplastic effects rely solely on DNMT inhibition. </p></div>
  </body>
</html>