<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 515 from Anon (session_user_id: 2ccee6e9bf635d6a41288c08e6f8da5b71d18f0c)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 515 from Anon (session_user_id: 2ccee6e9bf635d6a41288c08e6f8da5b71d18f0c)</h1>
    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><br />1. <br />CTCF insulates Igf2 from downstream
enhancers. <br /><ul><li>DNA methylation at ICR blocks binding of CTCF binding.<br /></li><li>Without CTCF, DNA methylation spreads to H19 promoter to
silence and enhancers can access Igf2 to activate .<br /></li><li>The ICR at IGF2 and H19 is normally methylated on the
paternal allele. Methylation is excluded from the maternal
allele by CTCF binding. The presence of CTCF confers an
insulator function upon the ICR, which blocks access of the
IGF2 promoters to the enhancers downstream of the H19
gene.</li></ul>1+2.<br /><ul><li>Paternal allele: Enhancers help express igf-2 as ICR is methylated and CTCF can't bind.<br /></li><li>Maternal allele: No methylation at ICR, CTCF binds ICR and blocks transcription.</li></ul>3.<br /><ul><li>Disruption of imprinting caused by deletion of the <i>H19</i> gene region in Wilm's tumor</li></ul>4.<br /><ul><li>The imprinted <i>H19</i> gene, which encodes an untranslated RNA, lies at the end of a cluster of imprinted genes in the mouse. Imprinting of the insulin-2 and insulin-like growth factor 2 genes, can be disrupted by maternal inheritance of a targeted deletion of the <i>H19</i> gene and its flanking sequence.Paternal inheritance of the disruption has no effect, which presumably reflects the normally silent state of the paternal gene.</li></ul><br /></div>

    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value">A.) <br /><ul><li>Genetic imprinting</li><li> X chromosome inactivation </li><li>Suppression of repetitive elements.</li><li> formation of hetero-chromatin </li><li>In promoter region, inhibition of gene expression</li></ul>B.) <br /><ul><li>Tumor suppressor genes (TSG's) are hyper-methylated and hence can not produce products that suppress tumor --&gt; Cancer</li><li>Proto-oncogenes (POG's) under-methylated such that they are over-expressed and  lead to production of oncogenic proteins leading to uncontrolled cellular proliferation --&gt; Cancer</li><li>Regulatory regions (intergenic regions) are also over or under-methylated which I'll discuss in subsequent question later.</li></ul>C.)<br /><ul><li>LOH- LOSS OF FUNCTION of TSG's produce products that keep the cellular division in check. If TSG's regulatory proteins are missing due to hypermethylation, mitosis would not be restrained and result will be undifferentiated pre-mature cells. e.g. TSG's like (<span> retinoblastoma gene (</span><i>Rb1</i>), glutatione S-transferase pi (<i>GSTP1</i>), and E-cadherin (<i>CDH1</i>) </li><li>GAIN OF FUNCTION of POG's , themselves, are not oncogenic. Hypomethylation makes them so. Mitosis exceeds it's normal rate. Result = Cancer. e.g. myc gene.<br /></li></ul>D.)<br /><ul><li>Transposition of transposable elements</li><li>lncRNA formation (regulatory sequences)</li></ul>E.)<br /><ul><li>Hypomethylation of intergenic regions</li><li>Widespread loss of intergenic DNA methylation<br /></li></ul>F.)<br /><ul><li>Transposable elements may become active and contribute to the genomic instability observed in cancer cells<br /></li><li>Loss of biologically active lncRNAs  associated with chromatin remodeling complexes such as PCR2 (needed for pluripotency and differentiation) <br /></li></ul><br /></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value">1.<br />Hypomethylating agent/ cytidine analog<br />2.<br />It hypomethylates DNA by inhibiting DNA methyltransferase.It can only be incorporated into DNA strands but not RNA chains.<br />3.<br />a.) Hypomethylating agents block the activity of DNA methyltransferase<span> (DNA methyltransferase inhibitors / DNMT inhibitors). DNA methylation affects cellular function through successive generations of cells without changing the underlying DNA sequence.</span><br />b.) DNA methylation is the modification of DNA nucleotides by addition of a methyl group. These methyl groups are associated with changes in the ability of the corresponding DNA to be used. Patterns of DNA methylation are stable during cellular division. Excessive methylation of TSG'<span>s in some cancers contributes to the growth and survival of the cancer. Hypomethylating agents decrease the amount of cellular DNA methylation and disrupt these patterns.<br /></span>c.)  Its incorporation into DNA leads to a covalent binding with DNA methyltransferases, which prevents DNA synthesis and subsequent cytotoxicity<br /><br /></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><span>1. <br />Epigenetic changes are passed on during cell division to daughter and granddaughter cells until they are actively erased. Once erased, though, they do not return. It might therefore be that epigenetic therapies can effect changes which stop a cancer growing without having to kill all its cells.<br />2. <br /></span>Sensitive periods are those in which treatment of epigenome can alter the genome of same or different individual (mother-child)<span><br />3. <br /></span><ul><li>prenatal and early-life development<br /></li><li>Pregnancy<br /></li></ul><span>4.<br /></span><ul><li> Developing fetus and neonate
are extremely sensitive to perturbation by chemicals,
and we know that the developmental period is the most
sensitive to epigenetic alterations<br /></li><li>Epigenetic dysregulation that may underlie various psychiatric  disorders.<br /></li><li>Risk for developing
certain mental disorder<br /></li></ul></div>
  </body>
</html>