<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1235 from Anon (session_user_id: 3dca63ca5798f65718a9bcc937ac64c0af3e4160)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1235 from Anon (session_user_id: 3dca63ca5798f65718a9bcc937ac64c0af3e4160)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>DNA methylation can occur at CpG islands, intergenic regions(IRs),and repetitive elements.</p>
<p> </p>
<p>CpG islands are mostly  found at the promoters of the genes, usually unmethylated.Methylation at CpGs of the promoters means silencing of the correspondent genes. Although CpGs are usually unmethylated, methylated ones are found depending on cell type or in X inactivation. Hence, methylation at CpGs under normal conditions is important for cell identity(cell-lineage specific gene expression)and dosage compensation.</p>
<p> </p>
<p>In cancer, CpGs in the promoters of tumour suppressor genes are hypermethylated and genes are silenced.Methylation promotes tumorigenesis since these genes cannot suppress abnormal cell growth. Hypomethylation of CpGs of oncogenes contributes to tumorigenesis since these genes are now active.Finally, in cancer a generalized deregulation of methylation of CpGs of different genes is observed leading to deregulation in the expression of tissue specific genes.</p>
<p>IRs are normally methylated. Methylation silences cryptic transcription start sites  preventing transcriptional interference. Moreover,methylation at IRs is necessary for maintaining genomic integrity(cells lacking Dnmt1 display genomic instability).</p>
<p> </p>
<p>Repetitive elements can jump out of the genome and place themselves somewhere else in it.This is mutagenic since it is unknown which gene they will disrupt.Normally, repeats are methylated. Methylation maintains genomic integrity because silencing prevents transposition and transcriptional interference. Methylation also makes them more prone to mutations(meC to T)preventing transposition.Finally, methylation may prevent illegitimate recombination since chromosomes are more compacted.</p>
<p> </p>
<p>In cancer, IRs and repeats are hypomethylated. Hypomethylation in IRs results in transcriptional interference(tumor suppressor genes not be transcribed) due to absence of silencing of cryptic transcription sites. It can also lead to loss of genomic integrity which is a characteristic of cancer.Hypomethylation of repeats leads to a generalized genomic instability leading to abnormal karyotype.Instability is caused by illegitimate recombination between repeats.Recombination requires genetic identity and euchromatin(available due to hypomethylation).Moreover, hypomethylation of repeats activates them leading to their transposition (genes play a role in cancer might be disrupted).</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>Imprinting means monoallelic gene expression. DNA methylation at ICRs does not necessarily correlates with silencing. This is the case for Igf2 gene in H19/Igf2 cluster.</p>
<p>                                            </p>
<p>Igf2 gene is expressed only from the paternal allele. Paternal ICR is methylated blocking CTCF (insulator protein) to bind. Therefore, downstream enhancers can bind to Igf2 which is expressed.</p>
<p> </p>
<p>Maternal ICR is unmethylated. CTCF binds to ICR and insulates Igf2. Hence, downstream enhancers cannot bind to Igf2. They bind to their alternative choice H19 (lncRNA) which is expressed while Igf2 is not.</p>
<p> </p>
<p>H19 is only maternally expressed because methylation of paternal ICR spreads to H19 coding region (this methylation=silencing).</p>
<p> </p>
<p>In Wilm's tumour imprinting at the H19/Igf2 cluster is disrupted. Although no change is observed on the paternal allele, expression of Igf2 is aberrant on the maternal allele. Maternal ICR is methylated instead of unmethylated. Methylation prevents CTCF  to bind and hence downstream enhancers are free to bind to Igf2. Therefore, Igf2 is overexpressed. Furthermore, methylation of maternal ICR spreads to H19 which hence is expressed from neither alleles. Disruption of imprinting at this cluster contributes to cancer because Igf2 is an oncogene (growth promoting gene). Overexpression of Igf2 leads to abnormal overgrowth contributing to tumorigenesis.</p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine is a DNA demethylating agent, more specifically a DNA methylatransferase inhibitor (DNMTi).</p>
<p> </p>
<p>DNA methyltransferases (DNMTs)  are enzymes that lay down DNA methylation. Since Decitabine blocks these enzymes it causes a reduction in DNA methylation.</p>
<p> </p>
<p>Decitabine like other DNMTi is a nucleoside analogue (incorporated to the DNA) where DNMTs bind irreversibly to it upon replication. Hence, since DNMTs are not released again, they cannot methylate DNA. Moreover, since the function of DNMTi is cell-division dependent, cancer cells (divide more rapidly) are more affected than normal cells. Decitabine acts by reducing DNA methylation. Therefore,  it must work on hypermethylated tumor suppressor genes rather than hypomethylated intergenic regions or repeats. Since hypermethylation in this case means silencing of tumor suppressor genes Decitabine by reducing DNA methylation, it allows them to act again. These genes are responsible for stopping the uncontrolled cell growth that causes cancer. Therefore, Decitabine has an anti-tumour effect.</p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>DNA methylation is a stable epigenetic mark because it is mitotically heritable. Any change in DNA methylation in the parent cell will be inherited to the 2 daughter cells, and so on. Therefore, alterations in DNA methylation can have enduring effects on the epigenome.</p>
<p>A sensitive period  is when active remodeling of the epigenome takes place. There are two main ones: (a) during pre-implantation development and (b) during primordial germ cell development. An additional one is after artificial reprogramming of somatic cells into iPSCs. There are also tissue specific sensitive periods. Finally, in the Overkalix epidemiological study the 9-12 yrs for grandfathers (germ cell maturation period), 8-10  yrs and infant/fetal  life for grandmothers were identified as sensitive periods.</p>
<p>Drugs targeting epigenetic machinery have been developed to treat cancer. However, they  lack specificity in their mechanism of action. Scientists know that a drug works in a specific way affecting the abnormal epigenetic machinery of cancer cells but it may also act in another way too (unknown) affecting normal cells too(long-term effects still unknown). Hence, it is inadvisable to treat patients during sensitive periods since  normal epigenetic reprogramming not only of themselves but also of subsequent generations may be affected.</p></div>
  </body>
</html>