<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1452 from Anon (session_user_id: af152ff6b8f735bfa99eb1aedb5500b1482f910a)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1452 from Anon (session_user_id: af152ff6b8f735bfa99eb1aedb5500b1482f910a)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>CpG islands are often found at the promoter of genes. They are mostly unmethylated and protected from DNA methylation in normal cells. Methylation of these sites in general correlates with gene silencing and allows tissue specific, dynamic gene regulation.</p>
<p>CpG islands are often found to be hypermethylated at specific sites in cancer, corresponding to tumor suppressor genes. These genes would normally keep aberrations in cell cycle and cell proliferation in check. Thus silencing of tumor suppressor genes contributes to cancer by silencing negative regulators of cell growth.</p>
<p>The extensive methylation at intergenic regions and repetitive elements in normal cells has a protective function. Both can act as contributing factor to genome instability. In Dnmt1 null mutants one finds deletions, insertions and translocations as consequence of failed silencing of these regions.</p>
<p>Intergenic regions often contain regulatory elements. Hypomethylation of these regions in cancer might reactivate cryptic splice sites and cryptic transcription start sites, thus causing deregulation of transcription and translation in disease.</p>
<p>Repetitive elements often stem from mobile genetic elements that are able to act mutagenic by transposition or contribute to genomic instability by illegitimate recombination (whereby repeats serve as homologue sites). In addition to that, mobile genetic elements often are under the regulation of strong promoters. If active, these promoters can deregulate other genes. This is why repetitive elements are methylated in normal cells. The meCytosin is susceptible to mutation (C -&gt; T). On the long run, mutations in repetitive elements might disrupt elements essential for transposition of mobile genetic elements.</p>
<p>In cancer, repetitive elements are found hypomethylated, thus possibly causing reactivation of mobile genetic elements (and their transposition), reactivation of cryptic strong promoters and allowing illegitimate recombination between repeats. This contributes to genomic instability, deregulation of gene expression and accumulation of mutations, all of which are hallmarks of cancer.</p>
<p> </p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>The imprinting control region (ICR) at the H19/Igf2 cluster serves as a potential binding site for the insulator protein CTCF, which can restrict the influence of enhancers within the locus on gene expression of Igf2 (insulin like growth factor 2). There are two enhancers downstream of the H19 promoter. Those enhancers can either act on Igf2 or on the H19 promoter.</p>
<p>In normal cells, the ICR on the paternal allele is methylated, thus CTCF binding is impaired. In addition to that, the DNA methylation on the paternal allele spreads to the H19 promoter. Thus the enhancers can act now solely on promoting the expression of Igf2.</p>
<p>On the maternal allele the ICR is unmethylated, the insulator protein CTCF is able to bind to the ICR and restricts the influence of the enhancers on the H19 promoter. Igf2 is thus not expressed from the maternal allele.</p>
<p>In Wilm’s tumor, hypermethlyation at the Igf2/H19 locus causes both alleles to be methylated at the ICR. After the loss of imprinting, both alleles now show a paternal epigenotype. This results in an overexpression of Igf2. Igf2 is a growth promoting gene and contributes to the aberrant proliferation of tumor tissue.</p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine belongs to the class of DNA methyltransferase (Dnmt) inhibitors. Decitabine is a cytidine analogue. Once incorporated in the DNA, Decitabine binds irreversibly to Dnmt. It therefore acts as a demethylating agent by inhibiting Dnmt in a replication specific manner.</p>
<p>Reduction of active Dnmts makes full and correct maintenance of methylation on the new DNA strand unlikely. Progressively, DNA methylation is “diluted” with each replication.</p>
<p>A hallmark of cancer is the hypermethylation at specific CpG islands upstream of tumor suppressor genes, especially in lymphomas. Methylation at these sites facilitates tumor growth by silencing those factors that control cell cycle and cell proliferation. Inhibition of DNA methyltransferases could lead to progressive demethylation at these sites and thus potential re-expression of tumor suppressor genes. Re-expressed tumor suppressor genes could facilitate cancer treatment by keeping cancer growth in check and thereby display an anti-tumor effect.</p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>DNA Methylation in cells is maintained during DNA replication by the action of Dnmt1, who acts preferably on hemi-methylated DNA. By the action of DNA demethylating agents, DNA methylation marks are progressively lost. If treatment with demethylating agents is stopped at some time point, cells will now have an altered, hypomethylated epigenome and the information about DNA methylation sites is lost, due to failure of maintenance during treatment. This is also likely to occur in normal, healthy and dividing cells. Thus long lasting effects on all cells of the body are to be expected, when treating patients with DNA demethlyating agents.</p>
<p>Periods in the development during which existing epigenetic marks, especially DNA methylation, are being erased and new marks are being established on a genome wide scale are called sensitive periods.</p>
<p>During primordial germ cell development and during early embryogenesis (fertilized cell to epiblast) such vast resetting is observed.</p>
<p>DNA demethylating drugs would probably impair correct establishment and maintenance of the germ cell specific/embryo specific epigenome. In addition to that, interference with DNA methylation at imprinted genes is likely to occur. Treating patients during such sensitive periods with DNA demethylating agents poses a high risk for their progeny.</p></div>
  </body>
</html>