<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1060 from Anon (session_user_id: c517bc5328c0b80c487e9f4b6f38a107c41126a7)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1060 from Anon (session_user_id: c517bc5328c0b80c487e9f4b6f38a107c41126a7)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p><strong>DNA methylation</strong> is an epigenetic phenomenon carried out at cytosine residues by de novo methyltransferases (DNMTs), almost exclusively at CpG dinucleotides. While some CpG clusters (known as islands), often found at gene promoters, are dynamically methylated between cell types, most are unmethylated in normal cells: methylation results in gene silencing. <strong>In cancer, </strong>as tumour development progresses, global genome methylation is decreased in a non-uniform manner. While repetitive elements, intergenic regions and introns become demethylated, CpG islands show increasing methylation at promoters of specific genes, resulting in their silencing. <strong>Disruption </strong>of DNA methylation at CpG islands in tumour supressor promoter regions (hypermethylation) leads to their silencing and uncontrolled cellular proliferation. This effect is context-dependent, varying with the tumour type and stage. It can link to genetic effects through mutation of the epigenetic enzymes such as the DNMTs.</p>
<p><strong>Intergenic regions and repetitive elements </strong>tend to be methylated in normal cells. Intergenic region methylation is thought to maintein genomic stability by minimising chromosomal deletions, reciprocal translocations and insertions. Methylation at repetitive elements is thought to silence repeats that could be transposed and lead to mutations, transcriptional interference and illegitimate recombination.<strong> In cancer</strong>, gradual hypomethylation of these regions leads to illegitimate recombination between repeats, activation of transposition and disruption of neighbouring genes through activation of cryptic promoters. This can results in the deletions, reciprocal translocations and insertions mentioned. <strong>Disease </strong>is promoted by this global hypomethylation, which is context-dependent as in CpG methylation. The increased genomic instability worsens the numerous mutagenic and epimutagenic effects seen in cancer. The effects result in a loss of proliferative control, whether it be due to silencing of tumour suppressors, activation of oncogenes or the general changes in ploidy, nuclear organisation and chromosome structure and expression that hallmark the disease.</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>In healthy cells, the paternal Igf2 allele is expressed while the maternal allele is silenced through imprinting. The <strong>paternal allele</strong> shows methylation at the imprinting control region (ICR) and H19 region that codes for a long no-coding RNA, preventing the binding of CTCF to the ICR. Chromatin looping of enhancers thus gives expression of Igf2. On the <strong>maternal allele</strong>, this methylation is not seen, allowing binding of CTCF to the ICR and transcription of H19 with help of the enhancers. Chromatin looping of the enhancers to Igf2 is prevented by CTCF insulation and so Igf2 is silenced. </p>
<p>In <strong>Wilm's tumour</strong>, hypermethylation of the ICR leads to the same effect being seen at the maternal allele as at the paternal: CTCF can't bind to either ICR and H19 is silenced at both, so the enhancers give expresson of Igf2 from both maternal and paternal alleles. </p>
<p><strong>Disrupted imprinting</strong> at the H19/Igf2 cluster is seen in several diseases including cancer and Beckwith Wiedemann syndrome. The overexpression of Igf2 causes problems as it is a growth promoter. Changes in H19 transcription are also seen to be involved in cancer mechanisms and it has been suggested as an oncogene.</p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p><strong>Decitabine</strong> belongs to the DNA-demethylating class of epigenetic inhibitors, and is used in treatment of <span>myelodysplastic syndromes such as those that have progressed to acute myeloid leukemia (AML). As a nucleoside analogue, it acts through integration into the chromosome during replication and irreversibly binds de novo methyl transferase (DNMTs) that attempt to methylate it. Hence, its action is dependent on the cells dividing - this is, of course, more prevalent in the highly proliferative cancer cells than normal tissue, resulting in a greater effect on tumour cells as desired. As the DNMTs are irreversibly inhibited in this way, DNA methylation that is normally found at CpG islands in cancer is decreased. While there were concerns about the effects in normal cells where methylation is necessary for general cellular survival, it seems that, at low doses, the drug is effective in suppressing tumour development with relatively few side-effects. </span></p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p><strong>DNA methylation</strong> is an epigenetic effect that is replicated through cell divisions. DNA methyltransferases (DNMT3a/b) lay down de novo methyl groups, which are symmetrical at CpG dinucleotides. Replication gives parental hemi-methylated DNA which is recognised by DNMT1, resulting in completed methylation of the new replica strand. While methylation can be removed enzymatically, changes in methylation generally endure through cell life and division. In the <strong>sensitive periods</strong>, global methylation reprogramming occurs in order to allow for totipotency in the cells of new organisms. These have been identified as embryonic pre-implantation development and primordial germ cell development, meaning that epigenetic marks are low in abundance at the blastocyst and in gamete formation. After demethylation of paternal and maternal chomosomes, methylation occurs in a cell type-specific manner, hence determining levels of celular gene expression and imprinting. This whole process removes the epigenetic markers that were required for the sperm/oocyte or developing germ cell in order for totipotency to be restored and new roles assigned. Treatment with epigenetic drugs in these periods could, therefore, have abberant effects on the reprogramming, leading to differences in the epigenome with major impacts on the chances or developing diseases or other abnormalities.</p></div>
  </body>
</html>