<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 894 from Anon (session_user_id: 97120ac37324566926900a9d4efa953423a19e1e)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 894 from Anon (session_user_id: 97120ac37324566926900a9d4efa953423a19e1e)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>CpG islands (CGIs) are found in approximately 60% of promoters. In normal cells, CGI-associated promoter regions are usually unmethylated, independent of their activity.</p>
<p>In cancer, CGI-associated promoter regions are frequently methylated, causing epigenetic silencing of underlying gene expression. DNA methylation levels increase with tumour progression. Subsets of some tumour types including colorectal cancers (CRC), breast cancers, and gliomas have the CGI methylation phenotype (CIMP), exhibiting unusually high levels of tumour-specific CGI methylation in sets of genes, with elevated epimutation rates. Some tumour types including CRC demonstrate hypermethylation of CGI shores (2kb regions surrounding CGIs), which is correlated with gene expression.</p>
<p>Methylation of CGI-associated promoter regions of tumour suppressor genes (TSGs) resulting in epigenetic silencing can act as one of the ‘hits’ (as an alternative to gene mutation) in the Knudson hypothesis of tumourigenesis, whereby multiple hits lead to cancer developing. Cells with epimutations are rapidly selected for, with the advantage of rapid proliferation and reduced cell death. TSGs which demonstrate single ‘hits’ from CGI hypermethylation and epigenetic silencing include <em>RB</em> (retinoblastoma),<em> MLH1</em> (CRC), <em>BRCA1</em> (breast cancer) and <em>MGMT</em> (gliomas and CRC).</p>
<p>Normally, there is DNA methylation in the intergenic regions and repetitive elements.  Methylation of gene bodies is found in genes with high expression levels, and may function to repress transcriptional noise through regulation of alternative transcriptional start sites, inhibit expression of antisense strand mRNA, and direct splicing of RNA. Intragenic methylation has also been associated with replication timing.</p>
<p>In cancer, there is hypomethylation of the intergenic regions and repetitive elements (which also increases with tumourigenicity), leading to genomic instability.</p>
<p>Genomic instability may occur through illegitimate recombination between repeats resulting in reciprocal translocations, as well as activation and transposition of repeats and activation of cryptic promoters leading to aberrant expression of neighbouring genes, deletions or insertions.</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>The H19/IGF2 cluster comprises two imprinted genes at chromosome 11p15.5, a region frequently implicated in cancer. At this locus, on the paternal allele, methylation of the imprinting control region (ICR) prevents binding of the insulator protein CTCF (which functions to insulate the<em> IGF2</em> gene from downstream enhancers). Therefore, downstream enhancers can access and preferentially activate<em> IGF2</em>, promoting paternal<em> IGF2</em> expression. Heterochromatin spreading results in <em>H19</em> promoter methylation and silencing of paternal <em>H19 </em>expression (ie. <em>H19</em> is paternally imprinted).</p>
<p>On the maternal allele, CTCF and its cofactors (cohesins, p68/SRA) are able to bind the ICR, which is unmethylated. Downstream enhancers are therefore blocked from accessing <em>IGF2</em>, and <em>IGF2</em> is not maternally expressed (ie. <em>IGF2</em> is maternally imprinted). Downstream enhancers can then act on <em>H19</em>, promoting maternal <em>H19</em> expression.</p>
<p>In Wilms tumours, abnormal hypermethylation of the ICR on the maternal<em> </em>allele results in biallelic <em>IGF2</em> expression (and biallelic <em>H19 </em>silencing). Therefore, <em>IGF2</em> is overexpressed compared to in a normal cell. This is an early event which is observed in pre-neoplastic tissue.</p>
<p>The <em>IGF2</em> gene codes for insulin-like growth factor 2 protein, which promotes cell growth and proliferation during development. Hence, <em>IGF2</em> overexpression leads to Wilms tumour formation.</p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine, an azanucleoside, belongs to the DNA methyltransferase (DNMT) inhibitor class of epigenetic inhibitors.</p>
<p>Decitabine irreversibly binds and inhibits DNMTs after they are incorporated into DNA, leading to replication-dependent DNA demethylation.</p>
<p>Activity of DNMTs during cell division is required for the maintenance of hypermethylation-induced epimutations. Decitabine-induced DNA demethylation can therefore reverse hypermethylation-induced epimutations, resulting in reactivation of epigenetically silenced TSGs and, consequently, inhibition of tumour growth. As Decitabine causes replication-dependent<em> </em>DNA methylation, cells with a higher replication rate, such as cancer cells, are more affected by Decitabine therapy. Collectively, this forms the basis for the use of Decitabine as an FDA approved epigenetic therapy for Myelodysplastic Syndrome (a pre-malignant condition that can progress to Acute Myeloid Leukemia), where TSG silencing through promoter methylation frequently occurs. </p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Epigenetic changes in gene expression are mitotically heritable (passed down to daughter cells and subsequent generations of cells through mitosis), and persist until they are erased. Hence, drugs that alter DNA methylation and act to reverse the abnormal epigenetic states in cancer cells can have effects which are sustained beyond withdrawal of the drug treatment.</p>
<p>Sensitive periods are periods of development when epigenetic marks are cleared from the genome to achieve developmental potency, through epigenetic reprogramming.</p>
<p>The sensitive periods are during (i) pre-implantation and early development and (ii) primordial germ cell (PGC) development. The epigenetic reprogramming that occurs during pre-implantation is important for removing gametic marks; this permits somatic marks to be laid down in a lineage-specific manner after implantation, restoring totipotency to the zygote and enabling formation of the complete organism. In PGC development, epigenetic reprogramming is important for removing somatic marks, permitting the laying down of germ cell epigenetic marks including sex-specific imprints.</p>
<p>Treating patients with epigenetic modifying drugs during the sensitive periods would be inadvisable, as this may disrupt the epigenetic reprogramming that occurs during these periods and potentially lead to the patients’ babies having disorders resulting from epigenetic abnormalities (eg. imprinting disorders).</p></div>
  </body>
</html>