<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1091 from Anon (session_user_id: ceb2d9b3e9a9d7b984d9e555db222928d577b06d)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1091 from Anon (session_user_id: ceb2d9b3e9a9d7b984d9e555db222928d577b06d)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>In normal, healthy cells, DNA methylation helps to regulate the activity of the cell, and is often associated with silencing of gene expression.  The underlying mechanism is considered to be the formation of repressive structures like MeCP 1 &amp; 2 [1].  DNA methylation occurs almost exclusively at the CpG dinucleotides.  These DNA dinucleotides can be found in intergenic regions, repetitive elements and CpG Islands. </p>
<p> </p>
<p>CpG Islands are locations with a high concentration of CpG dinucleotides; often found in promoters [2].  In normal cells, CpG islands are mostly unmethylated.  In the cases that CpG islands have been methylated, it is associated with silencing of gene expression.  This makes sense as promoters must be able to express genes, and should thus not be silenced, which explains why most of CpG islands are not methylated. </p>
<p> </p>
<p>Intergenic regions and repetitive elements however can lead to genomic instability if they become active, through transcription, cryptic splicing, transposition and recombination [3].  In normal cells these regions are thus methylated to silence them. </p>
<p> </p>
<p>In cancer cells however, the situation is reversed.  CpG islands become mostly methylated, preventing the normal gene expression of the promotors, while intergenic regions and repeats are consistently hypomethylated, thus giving rise to the disruptive effects mentioned above, undermining genomic integrity [3]. </p>
<p> </p>
<p>The (hyper)methylation of the CpG islands specifically prevents the expression of tumour suppressor genes.  As DNA methylation is mitotically heritable, this undermines the defense mechanism of the cell against oncogenic elements [4].  Also the neighbouring regions, called CpG island shores, become hypermethylated and are a strong predictor of oncogenic presence [5]. </p>
<p> </p>
<p>Hypomethylation of repeats and intergenic intervals activates cryptic promoters and thus disrupts the neighboring genes.  Likewise, recombinations and transpositions occur [6].</p>
<p> </p>
<p><strong></strong><br /><a href="https://coursera-uploads.s3.amazonaws.com/user-16eb0a787e92ef09910057b3/970238/asst-5/970238-520fc67527b654.50056133.pdf">Epigenetics writing assignment_dd.pdf</a></p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>In the H19/Igf2 cluster, downstream enhancers work on the H19 promoter or the Igf2 promoter.    In the absence of the CTCF insulator protein, the enhancers preferably work on Igf2. </p>
<p> </p>
<p>However, when the H19 imprinting control region (ICR) is not methylated, as is the case in the maternal allele, the CTCF insulator protein binds to the ICR.  The Igf2 promoter is then isolated from the enhancers.  Instead the enhancers work on the H19 promoter, activating its expression.</p>
<p> </p>
<p>On the paternal allele, DNA methylation of the ICR occurs, preventing the insulator protein from binding.  Therefore, enhancers will work on the Igf2 promoter, increasing Igf2 expression.  Simultaneously, DNA methylation spreads towards the H19 promoter, thus silencing expression [7]. </p>
<p> </p>
<p>In Wilm’s tumour, the ICR is hypermethylated in the maternal allele as well, leading to a silencing of H19 expression and overexpression of Igf2 [5].  Overgrowth is often found to coincide with overexpression of Igf2.  This matches the pattern of an overexpression of oncogenic, growth promoting cells (Igf2).  Silencing of tumour suppressing genes might also reinforce Igf2 overexpression in cancer growth in the absence of the Cdkn1c gene of the neighboring Kcnq1 cluster, as with the Beckwith Wiedemann syndrome [7].<br /><a href="https://coursera-uploads.s3.amazonaws.com/user-16eb0a787e92ef09910057b3/970238/asst-5/970238-520fc67a176197.84724089.pdf">Epigenetics writing assignment_dd.pdf</a></p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine works by demethylating DNA [8].  It belongs to the group of DNMT inhibitors which are nucleoside analogues.  They bind DNMTs irreversibly to DNA, which leads to a decreased capacity for methylation over time [9], effectively demethylating DNA. </p>
<p> </p>
<p>In cancer, intergenic regions and repetitive elements are already hypomethylated, and CpG islands are in a state of hypermethylation.  In the first answer it was already observed that hypermethylation leads to a silencing/suppression of the tumour suppressing genes. </p>
<p> </p>
<p>Decitabine can hence have a selective effect in oncogenic cells, as the CpG islands will be demethylated by the administration of the drug.  This will impact cancer by reactivating the tumour suppressing genes.  This should at least slow down the growth of the oncogenes. </p>
<p> </p>
<p>In low doses the drug appears effective, but the danger remains that demethylation will also start affecting healthy cells, in which intergenic regions and repetitive elements could become demethylated [9]. <br /><a href="https://coursera-uploads.s3.amazonaws.com/user-16eb0a787e92ef09910057b3/970238/asst-5/970238-520fc686b3e9c1.47992486.pdf">Epigenetics writing assignment_dd.pdf</a></p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p><br /><a href="https://coursera-uploads.s3.amazonaws.com/user-16eb0a787e92ef09910057b3/970238/asst-5/970238-520fc787cccd10.98037160.pdf">Epigenetics writing assignment_dd.pdf</a><br /><br />The irreversible binding of DNMT leads to (mitotically) heritable silencing of gene expression in the affected cells.  This is why DNMT inhibitors would be toxic in high doses; <br />demethylation would be forced upon the epigenetic machinery even when not desirable. </p>
<p> </p>
<p>Sensitive periods are the times when there is an extensive amount of epigenetic reprogramming.  During these periods, the epigenetic marks are laid down that will become permanent through the mitotical division of cells, with DNMT working to lay down methylation on daughter strands. </p>
<p> </p>
<p>Early (pre-implantation) embryonic development and primordial germ cell development are the main sensitive periods.   During the former, epigenetic marks specific to the germ cells are first removed, after which the stem cells are epigenetically reprogrammed for their later destination in the body, becoming somatic cells of various tissues.  These somatic cells then keep a stable epigenetic imprint.  During the latter sensitive period, primordial germ cells undergo another round of epigenetic reprogramming, as they need to pass on the epigenetic signature to later offspring. </p>
<p> </p>
<p>If patients would receive treatment affecting the epigenetic marking (like DNA demethylation) taking place during this period, their stem cells would be affected and they could have unviable tissue formation. </p></div>
  </body>
</html>