<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 495 from Anon (session_user_id: 9d9efdc039974c14f785ab2d5972e1e89220589e)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 495 from Anon (session_user_id: 9d9efdc039974c14f785ab2d5972e1e89220589e)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>DNA methylation at CpG islands leads to silencing of gene expression. In cancer, there is locus specific DNA hypermethylation of CpG islands of tumour supressor genes, and genome-wide hypomethylation of CpG poor promoters. Furthermore, cancers often exhibit a CpG island methylator phenotype, whereby CpG islands of a set of genes are frequently methylated. CpG island hypermethylation of tumour supressor genes silences these genes, perhaps one of the two 'hits' as per the Knudson Hypothesis. Tumour supressor genes, in normal function, repress regulation of the cell cycle, or promote apoptosis, and their silencing allows uncontrolled proliferation of tumours. This DNA methylation is mitotically heritable, and thus this silencing of tumour suppressor gene expression is inherited by the epimutated cell's progeny, with rapid selection of the epimutation. Hypomethylation of CpG poor promoters can result in activation of oncogenes ,a potential biomarker of tumorigenesis. DNA methylation maintains genomic integrity at both intergenic regions and repetitive elements, preventing an abnormal karyotype with deletions, reciprocal translocations, and insertions. In intergenic regions, it largely serves to silence cryptic transcription start sites or cryptic splice sites, whilst in repetitive elements, it silences repeats to prevent transposition (protecting the genome from transposable elements) and transcriptional interference from strong promoters, mutates repeats to prevent transposition (a methylated cytosine mutates to a thymine), and methylation of repeats may prevent illegitimate recombination. These functions fall under the Genome Defense Model. In cancer, there is genome-wide hypomethylation at intergenic regions and repetitive elements, thus leading to genomic instability. This progresses with tumorogenicity, leading to increased illegitimate recombination between repeats, activation of repeats,transposition, activation of cryptic promoters, and disruption to neighbouring genes. Again driven by chromosomal instability, this hypomethylation enhances tumorigensis in disease e.g. mutations in DNMT3B result in ICF syndrome, where genomic instability is a characteristic feature.</p>
<p style="text-align:left;"> </p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>At the paternal allele, DNA methylation at the ICR blocks the binding of CTCF. Without CTCF, DNA methylation spreads to the H19 promoter to silence, and enhancers can access lgf2 to activate, and so lgf2 is expressed. At the maternal allele, CTCF, the insulator protein, binds to the ICR, insulating lgf2 from downstream enhancers, preventing expression of lgf2. In Wilm's tumour, imprinting is disrupted due to various abnormalities at H19/lgf2 clusters at 11q15.5 (1Mb). This can lead to upregulation of lgf2, which is a growth promoting oncogene. The disrupted imprinting leads to the maternal allele behaving like the paternal allele - a mutation or deletion can cause the loss of imprinting at the maternal allele, leading to paternal uniparental disomy, and thus epigenetically disrupting imprinting ( a rare occurance). This results in tumour, as lgf2 is expressed both maternally and paternally, and thus, as it is an oncogene, this results in a predisposition to embryonic or childhood tumorigenesis. The overdose of lgf2 can also result in fetal or post-natal overgrowth, and macroglossia. Loss of imprinting, and thus maternal silencing of the growth promoter results in excessive growth, and thus tumorigenesis (parent-of-origin expression at imprinted allele disrupted).</p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine is a DNA-demethylating agent. Decitabine can inhibit DNA methyltransferases that lay down methylation on DNA, and thus can be <span>used to treat myelodysplastic syndromes, the precursors of acute myelogenous leukaemia. It is thus classed as a DNMT inhibitor (DNMTi)  - these are nucleoside analogues that irreversibly bind DNMTs after they are incorporated into DNA, and Decitabine is therefore replication-dependent. At low doses, Decitabine results in a reduction in DNA methylation, giving rise to an anti-neoplastic effect. Its mechanism of action is still unclear, but it may be that Decitabine results in a loss of hypermethylation at tumour supressor genes, a common feature of haemotological malignanies, and thus has an anti-tumour effect. Decitabine has a lack of specificity inherent to its mechanism of action, and thus, at high doses, it is toxic and non-specific in its action, with little or no anti-neoplastic effect. </span></p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Altering DNA methylation can have enduring effects on the epigenome due to the mitotic heritability of the epigenetic mark, allowing both epigenetic homogeneity within a tissue and transgenerational epigenetic inheritance. A sensitive period is a period of development whereby an altered environment has an effect on epigenetic control i.e. a period of high removal of epigenetic marks and epigenetic reprogramming. They are predominantly during establishment of epigenetic marks, rather than maintenance. In mammals, sensitive periods are those of embryogenesis and gametogensis. Although DNA methylation is a very stable epigenetic mark, which is passed on to subsequent cell generations, reprogramming of DNA methylation occurs during gametogenesis and after fertilization or after artificial reprogramming of somatic cells into induced pluripotent stem cells (iPSC). Treating patients during sensitive periods would be inadvisable, as, due to the high epigenetic reprogramming in many tissues at these sensitive periods, this may also lead to healthy tissues becoming more susceptible to chemotherapeutic treatments. These changes would then be mitotically heritable, with alterations caused by epigenetic drugs affecting healthy cells for many generations, as opposed to targeting the unhealthy tumour cells only. These epigenetic alterations that affect tumorigenesis can also be inherited transgenerationally.</p></div>
  </body>
</html>