<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1095 from Anon (session_user_id: b7fc2a58d168df97bd4bad08c13e89ec5abc61a7)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1095 from Anon (session_user_id: b7fc2a58d168df97bd4bad08c13e89ec5abc61a7)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>Cancer cells display a number of altered epigenetic
marks when compared with normal cells, particularly locus-specific DNA
hypermethylation and genome-wide loss of DNA methylation. DNA methylation of
CpG dinucleotides functions to regulate gene expression. In normal cells, high
concentrations of CpG dinucleotides (CpG islands) present in <span>the regulatory regions of many genes</span>
remain unmethylated, although a small proportion become methylated in a
tissue-specific manner during development. In
cancer, CpG hypermethylation within gene promotors leads to transcriptional silencing
of neighbouring genes, associated with a closed chromatin configuration. When these
hypermethylation events occur within tumor suppressor genes, silencing provides
the cell with a growth advantage as a result of aberrant cell proliferation and
a loss of appropriate DNA repair and cell death. Secondarily, CpG island
shores, normally methylated in a tissue-specific manner in normal cells, are also
hypermethylated in cancer cells, leading to silencing of gene expression.
Notably, as a heritable epigenetic mark, DNA methylation changes are passed on
through cell division.</p>

<p>Global
loss of methylation is another distinguishing features of cancer cells. In
normal cells repetitive elements, including heterochromatic DNA repeats and retrotransposons, and
repeat-rich intergenic regions are methylated, resulting in gene silencing. Hypomethylation
of these repetitive elements and intergenic regions in cancer cells leads to
chromosomal instability and loss of imprinting. In particular, the repetitive
nature of these regions enables illegitimate recombination events to occur. Loss
of DNA methylation also enables activation of retrotransposons, normally
silenced within the genome, leading to their ability to move within the genome and
disrupt gene expression, either by inserting into genes creating alternative
splice variants/stopping epxression, or by acting as gene promotors to activate
transcription. Insertions, deletions and
transpositions, along with inactivation of tumor suprressors and activation of
oncogenes, promotes rapid cell proliferation, favouring oncogensis.</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>Maintenance of imprinted, parental allele-specific
expression of specific genes depends upon reading ofthe  DNA methylation
status of Imprinting Control Regions (ICRs) which are regulated by chromatin insulators
or non-coding RNAs, and ensure that only the maternal or paternal allele is expressed.</p>

<p>In normal cells, Igf2 expression is restricted to the
paternal allele. DNA methylation of the ICR on the paternal allele blocks the
binding of CTCF, an insulator protein, leading to silencing the H19 gene. Downstream enhancers are then free to access Igf2
and activate its transcription. Conversely, on the maternal allele, CTCF binds
to the unmethylated ICR, insulating Igf2 from the enhancers and activating H19
expression.</p>

<p>In Wilm’s tumour, loss of maternal
imprinting through DNA hypermethylation at the H19/Igf2 ICR
leads to the biallelic expression of Igf2. This results from the inability of CTCF is bind the
ICR in the maternal allele, enabling the downstream enhancers to bind and
activate expression. The resultant overexpression of Igf2, a growth promoting gene, is one event that contributes
to tumorigenesis. Secondly, loss of H19 expression may act as a tumor
suprressor, further contributing to tumorigenesis. Other epigenetic ‘hits’ (the
Knudsen hypothesis), likely contribute to tumorigenesis, in addition to loss of
imprinting (LOI).</p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value">Drugs that inhibit or interfere with DNA
methylation can be used to reactivate silenced gene expression and re-establish a normal
chromatin state in cancer cells. Unlike the genome, changes in the epigenome
can be reversed. For example, decitabine, a DNA
methyltransferase (DNMT) inhibitor, has been used to treat patients with
myelodysplastic syndrome. As a DNMT inhibitor,
decitabine irreversibly binds to DNMTs and is incorporated into DNA, blocking their
enzymatic activity. As a result, decitabine induces DNA hypomethylation, and
thereby re-expression of silenced tumor suppressor genes. The action of
DNMTis requires that cells are proliferating. In this way, reactivation of
tumor suppressors would lead to r<span>epression of cell proliferation or the promotion
of apoptosis. Alternatively, these inhibitors may make cancer cells more
susceptible to chemotherapy drugs.</span></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Unlike the relatively stable genome, the epigenome can
change in response to environmental cues throughout the life of an individual. The
establishment of epigenetic marks, particularly DNA methylation, is mitotically
heritable and thereby maintains cell identity. It is clear that epigenetic processes
are more susceptible to environmental exposures during the periods of
development when extensive reprogramming of DNA methylation and histone
modification patterns take place (‘sensitive periods’).</p>

<p>The first sensitive period corresponds
to the time of early embryo development following fertilization through the
blastocyst stage and is required to erase oocyte and sperm
specific epigenetic marks and enable the re-establishment of totipotency. The second sensitive period encompasses primordial germ cell
development within the fetus, characterized by almost complete demethylation then remethylation of the genome, and resetting of imprinting in germ cells.</p>



















<p>Treatment of patients during sensitive
periods would not be advisable as epigenetic marks are being laid down during
these times. Inhibition of the enzymatic activities of epigenetic regulators
(e.g. DNMTs) during these periods<span> of development could result in life-long
effects on the structure of chromatin, and aberrant activation/silencing of
genes. For example, use of a demethylating drug could result in the activation
of IAPs and loss of imprinting.</span></p></div>
  </body>
</html>