<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1735 from Anon (session_user_id: 7bc7b84309b0dd7723222395becf793f8d2a5d7c)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1735 from Anon (session_user_id: 7bc7b84309b0dd7723222395becf793f8d2a5d7c)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>DNA methylation involves the addition of a methyl (-CH<sub>3</sub>) group to the 5’ carbon of cytosines by methyltransferases such as DNMT3a and DNMT3b. Groups of CpGs, called CpG islands are found primarily in promoter regions and are normally unmethylated. Methylation at the CpG islands causes gene silencing either by changing the open euchromatin to compacted heterochromatin thereby impeding the transcription process, or by preventing the promoter transcription factor from binding – which in turn prevents the binding of subsequent factors and enzymes.  In cancer, the DNA methylation of CpG islands is disrupted…</p>
<p>In cancer, CpG islands are found to be hypermethylated, the epigenetic equivalent to gene mutation, in silencing the tumour repressor genes although it can be reversed, unlike gene mutations. These hypermethylations accumulate with aging which can contribute to the development of cancer (as per Knudson’s Hypothesis). The inactivation of tumour repressor genes improves conditions for tumour growth; these hypermethylations are found in <em>RB</em> in retinoblastoma, <em>BRCA1</em> in breast cancer among others.</p>
<p>Panels of CpG islands have been found to be uniquely hypermethylated (e.g. colorectal cancer) and the hypermethylation of CpG Island Shores (found about 2kb from CpG island tumours) mayindicate of transcription problems.</p>
<p>The normal function of DNA methylation in intergenic regions is to maintain genomic integrity by preventing RNA Pol II from binding to inappropriate regions (cryptic splice/transcription sites) which could cause transcriptional interference and incomplete transcription with concomitant loss of protein function.</p>
<p>Methylation in repetitive elements also serves to maintain genomic integrity, but by silencing the repeats to prevent transposition, illegitimate recombination and transcriptional interference from strong promoter regions found within the repetitive elements. Conditions of hypomethylation which are common in cancer destabilize the genome by allowing for activation of cryptic promoters, illegitimate reciprocal translocations and subsequently possible transposons in this opened euchromatin</p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><div>
<p>Decitabine, a demethylating agent, is in the class of epigenetic inhibitors called DNA methyltransferase inhibitors. </p>
<p>While the absolute mechanism of action is still unclear, it is known that Decitabine is a nucleoside analogue; it contains a sugar-base combination that allows them to be incorporated into the DNA during the replication process as if they were one of the four DNA nucleosides. They differ in that when DNA methyltransferase binds to them, in order the methylate the daughter strand, the binding is irreversible. The result is DNA demethylation. </p>
<p>Since this DNA nucleoside analogue requires incorporation into the DNA of the cell itself, it acts in dividing cells.  This explains their anti-tumour effect in the rapidly dividing cancerous tumours. In the “Cancer’s epicentre” article, the potential for this type of drug to enhance chemotherapeutic effects is noted.  It almost postulates a possible reduction in heritable cancer in that germline.</p>
</div></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Altering DNA methylation can result in long-term effects on the epigenome by either silencing (or activating) genes involved with tumour repression, DNA expression, DNA transcription, histone-protein modifications and chromatin packing. Since a number of the epigenetic marks can be inherited, these effects can pass to subsequent generations.</p>
<p>Sensitive periods are the times when epigenetic reprogramming occurs in a mammal's life cycle: prenatal development and primordial germ cell development.</p>
<p>In the pre-implantation period of early fetal development, between fertilization and blastocyst formation, the DNA methylation and histone marks are removed from the maternal and paternal pronuclei at different rates. This allows epigenetic marks to be reset after implantation.  During primordial germ cell development, somatic cells in both the male and female are cleared and reprogrammed to be germ cells; the rate of clearing and reprogramming differs between oogenesis and spermatogenesis.</p>
<p>It would be inadvisable to treat a patient with epigenetic drugs such as Decitabine during either of these sensitive periods since DNA methylation is an integral component of setting epigenetic marks. Disruption of this can result in loss of imprinting, permanent damage (since the incorporated nucleoside analogues will keep regulators such as DNMTi bound) which is heritable.</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><div>
<p>H19 and Igf2 are found close together on chromosome 11.  Architecturally, the <em>H19</em> and <em>Igf2</em> genesare separated by an ICR region with the enhancers found upstream of both.</p>
<p>When the cluster is paternally imprinted, the paternal ICR is methylated preventing the insulator protein CTCF from binding to it. The DNA methylation spreads to the H19 promoter region – causing heterochromatin spreading and reducing <em>H19</em> expression. The action of the enhancers on the opened chromatin increases (upregulates) the expression of  <em>Igf2</em>  – postulated to be through a looping mechanism.  </p>
<p>The CTCF protein binds to the unmethylated maternal ICR preventing the enhancers from acting on the downstream <em>Igf2</em> gene.  With the CTCF binding, the enhancers loop to the H19 gene causing its overexpression and the production of additional long non-coding RNA H19…increasing  material for an miRNA production.</p>
<p>The <em>Kcnq1 </em>cluster is also found on chromosome 11 (along with H19./Igf2).<em>W</em>hen the growth suppressing gene <em>Kcnq1 </em>is downregulated (via mutation, uniparental disomyor loss of epigenetic imprinting) concomitant with the upregulation of the growth promoting <em>Igf2</em> cluster, Beckwith Wiedermann disorder occurs. This can also cause Wilm’s tumour,an embryonic kidney tumour found in about 5% - 10% of children with Beckwith Wiedemann disorder.  </p>
</div></div>
  </body>
</html>