<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 99 from Anon (session_user_id: efb487a9c5a8463493f72cfe3c13156a20238acb)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 99 from Anon (session_user_id: efb487a9c5a8463493f72cfe3c13156a20238acb)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>In noncancerous cells,
CpG-islands (CGIs) tend not to be methylated; however, when DNA methylation occurs
at CGIs, it’s almost always associated with gene silencing at particular times
in a lifetime, for example, X-chromosome inactivation. In contrast, in all
types of cancerous cells, there’s likely to be more methylation at CGIs. In
such cases, DNA methylation at CGIs is causing silencing of tumor suppressor
genes, which—per the Knudson hypothesis—can be one of “multiple hits” necessary
for cancer to occur.</p>

<p>Unlike CGIs, intergenic
regions and repetitive elements tend to be methylated in normal cells; this DNA
methylation maintains genomic stability, for example, by silencing inappropriate
repeats and their illegitimate recombination. However, with the aberrant DNA
methylation that occurs in cancer, intergenic regions and their repetitive
elements tend to be unmethylated, resulting in genome-wide hypomethylation and
genomic instability, eg, inappropriate deletions, copies, insertions, and
translocations, <span>which can result in inappropriate gene activation and transcription, in
turn possibly resulting in growth promotion and decreased programmed cell death (Taby,
Issa. <i>CA Cancer J Clin</i>
2010;60:376-392)</span>. The specific effects of such genomic instability will vary,
depending on exactly which genes are affected.</p>

The relative dependence on CGI
hypermethylation versus genome-wide hypomethylation varies between tumors. Also,
this relative dependence can change within a given tumor’s lifespan.<br /></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>In noncancerous cells, imprinted genes often regulate growth (turning growth promoters off and/or tumor suppressors on). In <span>the H19/Igf2 gene cluster, the maternal allele is unmethylated
and its imprint control region (ICR) can bind the insulator protein CTCF,
preventing the cluster's enhancers from activating the growth promoter (oncogene) Igf2. Unable to
bind Igf2, the enhancers bind and activate the “second choice,” H19, which produces a long noncoding RNA and (per <span>Riccio et al [<i>Endocr
Dev</i>. 2009;14:1-9</span>]) might exert tumor-suppressing activity. However, this cluster's paternal allele is methylated and therefore its ICR cannot bind with CTCF. Without this binding,
enhancers in this cluster bind Igf2 and activate its expression. Also, paternal methylation can
spread from the ICR to H19, silencing H19.</span></p><p>In cancer, loss of imprinting--whether by gene mutation/deletion, uniparental disomy, or (rarely) DNA methylation disruption (ie, direct disruption of the imprint itself)--can turn growth promoters on and tumor suppressors off (ie, opposite of previous paragraph). With imprint disruption in cancer, eg, Wilm's tumor, maternal H19/Igf2 methylation occurs and Igf2 expresses on that allele. Since paternal allele methylation is already occurring, there's twice the normal Igf2 expression. If Knudson's hypothesis applies here, this can be among "multiple hits" necessary for carcinogenesis.</p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value">Decitabine is a DNA-demethylating agent. These agents affect DNA
methylation by their (1) incorporation into the cells’ DNA during cell replication, and
(2) irreversibly binding to the enzyme DNA methyltransferase
1 (DNMT1) in the cells so the DNMT1 becomes unable to implement its function of
copying DNA methylation to the daughter DNA strands. The occurrence of cell
replication is a requirement for these agents to work. Because tumor cells tend
to replicate more rapidly than normal cells, the tumor cells will be more
vulnerable to decitabine than most normal cells,
providing that an appropriate dose (ie, not so high that it frequently kills normal
cells) is used.<br /></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value">Two FDA-approved drugs--decitabine and Vidaza--modify DNA methylation, as both are DNA-demethylating agents. (No DNA-methylating agents are approved.) Replication of demethylated DNA during mitosis initially generates two unmethylated "parent" strands. Then, two unmethylated "daughter" strands are generated and pair with the "parents," resulting in two double unmethylated strands. (Although I'm undoubtedly oversimplifying things, this seems reasonable because there's no hemi-methylated DNA for the enzyme DNA methyltransferase-1 to recognize and methylate). In this way, this unmethylated state is mitotically-heritable and--barring effects of additional factors--may well continue for further mitoses throughout a patient's lifetime. <br />  <br />Sensitive periods are when epigenetic marks are actively reprogrammed (ie, old marks are removed/new ones are established) and the epigenotype is most vulnerable to the environment. Sensitive periods include <i>primordial germ cell formation </i><i>to mature gamete production</i> and <i>pre-implantation </i><i>to early
post-implantation</i>. During these periods of maximal germ cell and embryo vulnerability to environmental factors (including DNA-demethylating drugs), unexpected/undesirable effects can result. Therefore, groups I'd avoid treating include pregnant women (because of <em>primordial germ cell</em> production in the developing offspring), females/males who eventually might become parents (because they have various stages of <em>germ cells</em>), and women who might be pregnant (because <em>implantation of the embryo</em> might be occurring).<br /></div>
  </body>
</html>