<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1891 from Anon (session_user_id: cede740326c578702a028618764a6cc20b957e9d)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1891 from Anon (session_user_id: cede740326c578702a028618764a6cc20b957e9d)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p><span style="font-size:14px;line-height:21px;">Methylation of the C5 position of the cytosine base is found in approximately 70-80% of CpG dinucleotides in somatic mammalian cells [3]. About 50-60% of the genes in the mammalian genome have CpG-rich promoters [5, 3]. The CpG-rich sequences are called CpG islands and they are unmethylated in normal cells [5, 3]. However, in cancer, CpG islands of the promoters are aberrantly hypermethylated and can result in the silencing of the relevant tumour-suppressor or other cancer-relevant genes [1], such as DNA repair genes [5]. Methylation at CpG sites modifies the cytosine residue into 5-methylcytosine. The deamination of 5-methylcitosine, and its subsequent mutation into thymine is also carcinogenic [5]. For human, the two-hit model proposed by Alfred Knudson in 1971 was extended to the epigenetic pathway [4]. Hence, two genetic or epigenetic ‘hits’ are required to cause the loss of function of tumour-suppressor genes [5]. Therefore, double loss of functions by silencing and/or mutation are sufficient to cause cancer.</span></p>
<p><span style="line-height:21px;font-size:14px;">Intergenic regions between two genes can transcribe long intergenic non-coding RNAs (lincRNAs) [3]. Many lincRNAs ca</span><span style="line-height:21px;font-size:14px;">n play a role in chromatin modification, pluripotency and differentiation [3]. Functional l</span><span style="line-height:21px;font-size:14px;">ncRNAs fulfill important regulatory roles in gene expression and regulation by assembling protein complexes and localizing them to their genomic target DNA sequence [2]. Altered epigenetic patterns in cancer cells are associated with the deregulation of lncRNAs and subsequent re-positioning of chromatin modifying complexes [3]. Actions of methylation and noncoding RNAs in subnuclear structures can achieve coordinated programs of regulated gene expression [6]. In cancer however, pathological changes in nuclear architecture are related to epigenomic alterations [3]. </span></p>
<p class="western" style="text-align:justify;"> <span style="font-size:14px;line-height:21px;color:inherit;font-family:inherit;font-weight:bold;">References</span></p>
<div><ol><li>Egger <em>et al.</em> 2004</li>
<li><span style="line-height:21px;font-size:14px;">Guttman </span><em style="line-height:21px;font-size:14px;">et al.</em><span style="line-height:21px;font-size:14px;"> 2012</span></li>
<li><span style="line-height:21px;font-size:14px;">Hassler </span><em style="line-height:21px;font-size:14px;">et al.</em><span style="line-height:21px;font-size:14px;"> 2012</span></li>
<li><span style="line-height:21px;font-size:14px;">Jones </span><em style="line-height:21px;font-size:14px;">et al.</em><span style="line-height:21px;font-size:14px;"> 1999</span></li>
<li>Jones <em>et al.</em> 2002</li>
<li>Yang <em>et al.</em> 2011</li>
</ol></div>
<p><span><span><br /></span></span></p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine treats myelodysplastic syndromes that are the precursors of acute myelogenous leukemia. Decitabine is an epigenetic inhibitor for treating cancer. It belongs to the class of DNA-demethylating agents (The Economist, Cancer's Epicentre 2012). <span> Decitabine indirectly depletes methylcytosine and causes hypomethylation of target gene promoters </span>(Kantarjian <em>et al.</em> 2006). In normal cells, the promoters are kept free from methylation at the CpG islands. In cancer, the cytosines in the CpG dinucleotides are aberrantly hypermethylated into methylcitosine. The methylation of the promoters causes the tumour-supressor genes to be silent (Egger et al. 2004). Incorporated into DNA strands (Wikipedia), decitabine acts as a counter agent against the DNA methyltransferase enzyme responsible for the pathologically aberrant methylation.</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p><span>Long non-coding RNAs (lncRNAs) involved in imprinting processes are the paternally expressed lncRNA AIR, which is required for silencing the maternally expressed protein-coding genes Igf2r/Slc22a2/Slc22a3 (</span><span style="line-height:21px;font-size:14px;">Hassler </span><em style="line-height:21px;font-size:14px;">et al.</em><span style="line-height:21px;font-size:14px;"> 2012)</span><span style="font-size:14px;line-height:21px;">.</span></p>
<p><span><span>The Igf2-H19 imprinted cluster of genes contains the imprint control region (ICR), the Igf2 and H19 genes and the distal enhancers. The ICR and the H19 gene can be methylated or unmethylated. The enhancers normally promote Igf2 expression. The insulator protein CTCF blocks the enhancers from accessing the Igf2 promoter and they instead promote H19 expression (Marnie Blewit in Epigenetic Control of Gene Expression, Coursera 2013).</span></span></p>
<p><span><span>The methylation pattern of the paternal allele comprises a methylated ICR and a methylated H19 gene promoter. Therefore, <span>H19 expression is silent, Igf2 expression is active. The methylation pattern of the maternal allele comprises an unmethylated ICR and an unmethylated promoter region of the H19 gene, hence CTCF blocks the enhancers from accessing the Igf2 promoter and they instead promote H19 expression (Marnie Blewit in Epigenetic Control of Gene Expression, Coursera 2013).</span></span></span></p>
<p><span><span><span>In Wilm's tumour, hypermethylation of ICR in the maternal allele blocks CTCF from binding the ICR, therefore leading to the overexpression of Igf2, and silencing of H19.</span></span></span></p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>It is quite understandable that drugs that alter DNA methylation can have lasting effects beyond the period of drug treatment. The enduring effects is certainely the fact of epigenomic patterns that are mitotically heritable down to several generations. After treatment with an epigenetic drug, the new epigenomic status of a cancer cell should also be transferred to the daughter cells through cell division. </p>
<p>Treatment of certain patients, i.e. pregnant women and young children before puberty, should be avoided because they are sensitive to extensive reprogramming of epigenetic marks. These are sensitive periods because they are periods of development that are susceptible to environmental insults, such as the embryonic development period and early childhood (Marnie Blewit in Epigenetic Control of Gene Expression, Coursera 2013). During these periods, epigenetic marks are not established yet. Treatment with epigenetic drugs which inhibit methylation will surely interfere with the normal methylation periods require for the normal developments of the embryo and the gametes of mammals.</p></div>
  </body>
</html>