<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1421 from Anon (session_user_id: e900124593039b1058b7859de3d33a38ac80e2ed)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1421 from Anon (session_user_id: e900124593039b1058b7859de3d33a38ac80e2ed)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>In a normal cell, CpG islands are typically unmethylated, which usually leads to gene expression. In cancer, CpG Islands are hypermethylated, typically in promoter of tumour-supressor genes. This causes the gene expression of tumour-supressors to be silenced, and leads to cells not having a 'barrier' for their continuous growth, hence cancer. In intergenic regions and repetitive elements, DNA methylation is usually hypermethylated in a normal cell.  In cancer, these regions are hypomethyated which leads to genomic instability. Genomic instability contributes to disease by causing increased recombination between repeats (leading to transolcations/deletions/duplications) or activation of repeats and transpositions, or disruption of genes. </p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine belongs to the class of DNMT inhibitors. It is a nucleoside analogue and becomes incorporated into the DNA during replication (and thus is replication dependent). When DNMT binds to the decitabine region, it cannot be released and therefore cannot go on to lay down the methylated epigenetic mark in daughter cells.  As discovered, cancer cells typically have hypermethylated CpG Islands in promoters of tumour-supressor genes (which causes silencing of that gene and hence the proliferation of tumours). Myelodysplastic syndrome has poor outccome associated with increased methylation (known as CIMP (CpG Island methylator phenotype)).  Therefore, if cells are not methylated because of DNMT inefficiency, this effectively leads to an active tumour suppressor gene and better outcomes for CIMPs.  </p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Drugs that alter DNA methylation are targeting the epigenome, which by basic definition (by Art Riggs et al, 1996), the epigenome is mitotically inheritable. In this way, when cells replicate, the daughter cells will have the same set of epigenetic marks.  This ensures tissue homogeneity as well as the continued set of modified epigenetic marks from one cancer cell to its progeny, hence treatments may have an 'enduring' effect.  A sensitive period is when environmental factors my influence the epigenetic marks.  During development, sensitive periods are during the development of primordial germ cells and the preimplantation period.  Treating a patient during these sensitive periods would be inadvisable because it could disrupt epigenetic reprogramming (the natural 're-setting' of epigenetic marks). This would lead to disruptions in gene expression (overexpression or silencing or mistiming of expression) and could make cells prediposed to imprinted conditions. Specifically, these disruptive predispositions could carry into the next generation (for example, if treatment affected the primordial germ cells), and have implications for offspring. </p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>Usually, we inherit one maternal allele and one paternal allele of the H19/Igf2 cluster.  The maternal allele has an unmethylated imprinted control region, which enables CTCF to bind to it.  CTCF can then protect the Igf2 gene (which is an oncogene) from the effects of the enhancers and is not expressed.  Instead, H19, a long non-coding RNA, is expressed.  In the paternal allele, the imprinted control region (ICR) is methylated (therefore CTCF does not bind to it) and the Igf2 gene is then activated by downstream enhancers. On top of this, H19 is methylated, leading to its own silencing.  In imprinting conditions such as Beckwith Wiedemann syndrome, there are two paternal alleles. This causes an increased dosage of Igf2 expression (overall hypermethylation of ICR), and because Igf2 is an oncogene, leads to a predisposition of childhood tumours such as Wilm's tumour. In addition, on the same chromosome 11, is the neighbouring Kcnq1 cluster.  On the paternal allele, this is usually unmethylated and leads to in cis silencing of neighbouring genes, such as Cdkn1c, which is a tumour-suppressor.  Therefore the loss of CdKn1c and upregulation of Igf2 leads to overgrowth and predisposition of tumours in Beckwith-Wiedemann. </p></div>
  </body>
</html>