<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 547 from Anon (session_user_id: 7431e36df5de5d939ea96751a2aeca8304101e01)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 547 from Anon (session_user_id: 7431e36df5de5d939ea96751a2aeca8304101e01)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>Cancer alters DNA methylation in a number of ways. In a normal cell, DNA methylation is usually found at the gene introns, intergenic regions and repetitive elements and the CpG islands remain mostly unmethylated. CpG islands are most often found within the promotors of genes. Methylation to these CpG islands can lead to gene silencing. </p>
<p>In cancer cells, CpG islands can undergo hypermethylation. This results in the silencing of various genes, for example tumour suppresser genes. As a result, tumerogenesis will not be suppressed if it occurs. This can contribute to cancer. CpG islands can also undergo hypomethylation, this has been associated with the over-expression of oncogenes within cancer cells.</p>
<p>In normal cells, intergenic regions and repetitive elements are spread out along the gene and are usually methylated. This methylation helps keep the chromatin stable and closed.</p>
<p>In cancer cells, these regions undergo hypomethylation and become less stable with more open chromatin. This can cause deletions, insertions, illegitimate recombination between repeats, reciprocal translocations or even activating neighbouring genes (via promotor activation e.g. Avy).</p>
<p>Hypomethylation of CpG poor promotors also occurs but with less frequency and is usually associated with the activation of genes e.g. oncogenes.</p>
<p>Both DNA hypermethylation and DNA hypomethylation are altered in cancer. Hypermethylation can lead to tumerogenesis as the tumour suppresser genes are silenced and hypomethylation can lead to genome wide disruptions such as deletions, insertions and activations.</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>Imprinting occurs monoallelically, one allele is silenced and the other active in certain genes. This is the case for H19/IGF2 cluster which uses DNA methylation as its mechanism of silencing.</p>
<p>In the paternal allele of this cluster, the hypermethylation results in inactive H19. This then allows enhancer proteins to target the downstream IGF2 and allow its expression.</p>
<p>In the maternal allele of this cluster, the H19 remains unmethylated allowing its transcription to occur as normal. When H19 is expressed, it binds to CTCF which insulates the IGF2 from the downstream enhancers.</p><p>Disruptions to this imprint control region is seen in cases of WIlm's tumour which show an epigenotype of in active IGF2 and inactive H19. This can happen through hypermethylation of H19 DNA or through loss of heterozygosity. This epigenetic change is thought to affect other genes as well and can occur at many stages throughout tumour progression.</p><p>Loss of imprinting at this loci is associated with overexpression of IGF2.</p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Dectabine is a DNA methyltransferase inhibitor (DNMTi). When used, it binds to the nucleosomes and inhibits the enzyme DNA methyltransferase from binding to the nucelosomes when the cell is undergoing transcription, preventing the DNA methylation being copied to the daughter cell. This drug is effective in cancer cells as they replicate more frequently, the drug would target these cells containing DNMT.</p>
<p>After using this drug, the levels of DNA methylation would decrease in these cells because the methylation marks would not be copied on to the daughter cell.</p>
<p>Dectabine can therefore have an anti-tumour effect by reversing the epigenetic changes, enabling more cells to become unmethylated again. For example, if tumour suppresser genes were silenced in cancer cells, Dectabine could result in their demethylation and possible activation. This could slow down tumerogenesis and result in an anti-tumour effect.</p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Epigenetic effects are reversible and mitotically heritable, therefore if DNA methylation is altered in some way, daughter cells that divide from that cell onwards will inherit the same epigenetic marks. Epigenetic drugs reverse the DNA methylation in some cells and their effects can be felt enduringly because of this.</p>
<p>Sensitive periods refer to periods where epigenetic marks are being laid down in the cells of the next generation. This occurs in gametogenesis and again in primordial germ cell (PGC) development.</p>
<p>Administering Dectabine and other epigenetic drugs during these sensitive periods is highly inadvisable as it is not certain what mechanisms these inhibitors use to make changes. These drugs are not selected for cancer cells, so if the drugs act globally around the body, inhibiting DNA methylation for example, the cells developing these and other epigenetic marks could be affected. </p></div>
  </body>
</html>