<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 374 from Anon (session_user_id: 064d7d6b17561113db3c1e2daa335415ae579c48)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 374 from Anon (session_user_id: 064d7d6b17561113db3c1e2daa335415ae579c48)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>In normal cells CpG islands associated with some gene promoters are typically not methylated.  By contrast 80-90% of cell CpG dinucleotides are methylated but these are mostly not in CpGIs but rather scattered throughout the genome associated with highly repetitive elements (e.g. LINES, SINES, IAPs), introns, and inter-genomic sequence regions where methylation is correlated with suppressed transcription of repetitive DNA and possible down regulation of splice signals.</p>
<p>In cancer cells patterns of methylation are substantially opposite of the above.  CpG islands at promoters, especially tumour suppressors, are methylated while CpG dinucleotides at repetitive, intronic, and inter-genic regions tend to be relatively less demethylated resulting in an increased transription of repetitive sequence.  </p>
<p>Methylation at suppressor gene promoters frequently acts to down-regulate transcription of cancer suppressors resulting lack of suppression and more rapid progress to cancer.</p>
<p>Demethylation of repetitive sequence or "jumping genes" tends to increase the rate at which the repetitive sequences transcribe themselves and "jump" and then insert themselves into random places in the genome.  It's easy to see how such insertions may "break" otherwise functional genes by disruption of DNA sequence or by moving potent promoters (somethime embedded in the jumping gene) upstream of normally "silent" oncogenes. </p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine is a nucleoside analogue which, when incorporated into DNA, tends to bind irreversibly to DNMT inhibiting its action.  Decitabine is in a class of drugs called DNA methyl-transferase inhibitors, DNMTi.  They work by inhibiting the copying of epigenetic marks from the old to the new strand during DNA replication.  They are effect anti-cancer because they act on dividing cells, eg the hall mark of cancer is its rapid replication.</p>
<p>In many cancers hypermethylation of DNA tends to downregulate "tumour suppressors" by hypermethylating promoter related CpG islands.  Decitabine would tend to slow this methylation possibly increasing the transcription of "tumour suppressors" which would likely act in an anti-tumour way to improve patient prognosis.</p>
<p> </p>
<p> </p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Altering DNA methylation by hypo- or hyper-methylation can alter not only cancer but also non-intended targets that are non-cancerous.  Demethylation may remove epigenetic marks from normal cells that effect that cell and possible its entire line of derived cells in a tissue for the life of the patient.  Failure to hemi-methylate with a defective methyltransferase could lead to abnormal regulation of not just translated genes in a cell line but to de-regulation of LINES, SINES, ICR's, and many other repetitive DNA any of which by "jumping" into the sequence of another gene can disrupt any gene in the genome.  If the methylation changes occur in embryonic stem cells then the disruption can extend across multiple cell types.</p>
<p>Since imprinting is dependent on methylation, disruption can be catastrophic.</p>
<p>Disruption during some periods may be more disruptive than others.  Early embryonic development when stem cells are most active and DNA marks are being set is one of the two most sensitive periods.  Treating pregnant women should be avoided for this reason.</p>
<p>Another sensitive period is primordial germ development.  Changes to methylation at this stage may impact future generations of offspring from the affected individual.</p>
<p> </p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>The H19/Igf2 cluster located on chromosome 11 contains an Inprint Control Region, ICR, that is methylated in the parternal chromosome but not the maternal chromosome.  The unmethylated maternal ICR readily binds a CTCF protein which blocks transcription of the growth stimulating gene Igf2 also on the maternal chromosome.  </p>
<p>In contrast, the paternal ICR is heavily methylated which inhibits CTCF binding and allowing cis Enhancers to activate transcription from paternal Igf2.  In normal function only the paternal chromosome 11 transcribes Igf2 growth stimulating gene, the maternal Igf2 is silenced.</p>
<p>Another nearby ICR locus on chromosome 11 transcribes acyclin dependent kinase, Cdkn1c, that tends to tightly regulate cell growth.  It's pattern of expression is opposite to that of Igf2: Cdkn1c is transcribed from maternal chromosome but silent on the paternal chromosome.</p>
<p>Any disruption that caused the maternal chromosome to look like the paternal chromosome would result in 2 x Igf2 expression and 0 x Cdkn1c expression.  This appears to be the primary cause of Wilm's tumor which is an overgrowth of kidney tumors in childhood.  Disruptions may be to multiple copies of the paternal chromosome 11 or to mutations that affect epigenetic marking enzymes.</p></div>
  </body>
</html>