<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1908 from Anon (session_user_id: f6a321ce582a7cb00feb8f0a0e6a8ba84371b269)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1908 from Anon (session_user_id: f6a321ce582a7cb00feb8f0a0e6a8ba84371b269)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>Methylation of cytosine is a covalent modification of DNA, in which hydrogen H5 of cytosine is replaced by a methyl group. It almost exclusively occurs at CpG dinucleotides in mammals. Cytosine hydroxymethylation is another important epigenetic modification on DNA. </p>
<p>CpGs are clustered into CpG islands, often at promoters of genes. Their methylation leads to <strong>gene silencing</strong>, therefore <span style="text-decoration:underline;">CpG islands are <em>protected from methylation in normal cells, thus allowing gene expression</em></span><em>.  </em></p>
<p>The <em>recruitment of MeCP2 to methylated CpG dinucleotides</em> represents a major mechanism by which DNA methylation <em>can repress transcription</em>. MeCP2 silences gene expression partly by recruiting histone deacetylase (HDAC) activity, resulting in chromatin remodeling.</p>
<p><span style="text-decoration:underline;">In cancer cells promoter CpG islands tend to become <em>hypermethylated</em></span>, which then causes <em>silencing of the underlying genes</em> such as growth-regulatory genes which control cellular proliferation. <em><span style="text-decoration:underline;">Silencing of tumor suppressor genes</span></em> – controlling cell cycle, apoptosis or DNA repair - can be one of the multiple hits to DNA in the Knudson hypothesis regarding carcinogenesis.</p>
<p><em><span style="text-decoration:underline;">Intergenic regions are usually methylated in normal cells</span></em> and contribute to the maintenance of genomic integrity and stability. Cryptic transcription start sites or cryptic splice sites are kept silent.</p>
<p><em>Cancer-associated loss of DNA methylation from intergenic enhancers, promoter regions, silencers, and chromatin boundary elements may <span style="text-decoration:underline;">alter transcription rates</span></em>.</p>
<p><em><span style="text-decoration:underline;">Repetitive elements are methylated in normal cells</span></em>. This leads to silencing of repeats and/or mutation of the repeats (meCytosine to Thymine) to prevent transposition. Silencing of repeats also avoids transcriptional interference from strong promoters. Methylation of repeats may suppress homologous recombination between repeats.</p>
<p><em><span style="text-decoration:underline;">Hypomethylation of the repeats in cancer  induces genomic instability</span></em><em> through reciprocal translocations between two different chromosomes, transcriptional aberrations due to transposition (deletion/insertion) or due to promoter activation</em>.</p>
<p>A feature of common tumor cells is the massive overexpression of certain DNA sequences that do not code for proteins called satellite repeats.</p>
<p> </p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>Genomic imprinting is an epigenetic process that involves DNA-methylation and histone modulation in order to achieve monoallelic gene expression in a <em>parent-of-origin-specific manner</em>. <span style="text-decoration:underline;">Imprinted alleles are silenced</span>.</p>
<p>Two imprinted genes - insulin-like growth factor 2 (<em>Igf2</em>), encoding an embryonic mitogen, and <em>H19</em>, producing a noncoding RNA are located in a defined region on the short arm of chromosome 11 (11q15) and are expressed <strong><em>only</em></strong> from the paternal and maternal alleles, respectively.</p>
<p>On the <strong><em>paternal</em></strong> chromosome, the <em><span style="text-decoration:underline;">ICR (imprint control region) is methylated</span></em>. CTCF binding is inhibited.The methylated paternal ICR exhibits no insulator activity and the <em><span style="text-decoration:underline;">paternal <em>Igf2</em> promoter interacts with the downstream enhancers leading to Igf2 expression</span></em>. The hypermethylated paternal ICR directs silencing of the <em>H19</em><em> </em>promoter in <em>cis (spreading of DNA methylation)</em><em>.</em> The promoter becomes hypermethylated and H19 is not expressed.</p>
<p>The <em><span style="text-decoration:underline;">hypomethylated</span> <strong>maternally</strong> inherited ICR</em> binds the insulator protein CTCF and <em>blocks activity of the proximal <em>Igf2</em>promoter </em>by insulating it from its distal enhancers. Igf2 is <em>repressed</em>. Downstream enhancers activate the H19promoter in cis and allow H19 expression.</p>
<p>In Wilms tumors (nephroblastoma in young children) the maternal <strong><em>Igf2</em></strong> imprint is relaxed and the gene is <strong><em>expressed</em></strong> <strong><em>biallelically (</em></strong><em>paternal</em><strong><em> and </em></strong><em>maternal allele</em><em>)</em>. This phenomenon is known as <em><span style="text-decoration:underline;">loss of imprinting</span></em> (LOI). <em>The increased amount of Igf2 may stimulate cell proliferation and induce tumorigenesis. </em></p>
<p> </p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Cancer is considered to be both a genetic and an epigenetic disease. Epigenetic abnormalities are reversible and as a result novel therapies that work by reversing epigenetic modifications are being increasingly explored.</p>
<p><strong>Decitabine</strong> - 5-Aza-2′-deoxycytidine (5-AZA-CdR), an epigenetic drug that <em><span style="text-decoration:underline;">inhibits DNA methylation</span></em>, is currently used to treat myelodysplastic syndrome (MDS), acute myeloid leukemia (AML) and other malignancies.</p>
<p>Decitabine is a <strong>DNA Methyltransferase Inhibitor (DNMTi)</strong> or <strong>Demethylating (Hypomethylating)</strong> <strong>Agent</strong>. It hypomethylates DNA by inhibiting DNA methyltransferase (DNMT). After being incorporated into DNA decitabine, a <em><span style="text-decoration:underline;">nucleoside analogue</span></em>, irreversibly binds DNMTs. It is <em>replication dependent</em>.</p>
<p>Decitabine <strong>can</strong> <strong>reactivate tumor suppressor genes</strong> silenced by aberrant DNA methylation which is a reversible epigenetic change.</p>
<p>The efficacy of DNMT inhibitors and HDAC inhibitors in blood cancers proves that therapeutic modification of the cancer cell epigenome can produce clinical benefit. Unfortunately, clinical trials with DNMTi in solid tumors did not have the same results.</p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Epigenetic marks are dynamic and respond to environmental signals not only during development, but also throughout life. It has been shown that <em><span style="text-decoration:underline;">chemicals can cause changes in gene expression that persist long after exposure has ceased</span></em>. Direct effects on epigenetic homeostasis can be caused by drugs which <em>affect DNA methylation</em>.</p>
<p><span style="text-decoration:underline;">Epigenetic DNA modiﬁcations  persist from one cell division to the next</span>. Changed DNA-methylation patterns are <span style="text-decoration:underline;">mitotically or even meiotically heritable</span></p>
<p>and therefore can enforce long-term changes in tissue identity or ‘‘cellular memory’’.</p>
<p>There are <em><span style="text-decoration:underline;">sensitive periods</span></em>, <em>when altered environments may have an effect on epigenetic control:</em> the <em>period of </em><em>PGC (primordial germ cell) development </em>and also the <em>preimplantation and early postimplantation period in embryonic development</em>. <em><span style="text-decoration:underline;">Active remodeling of the epigenome takes place in these intervals (reprogramming of epigenetic marks)</span></em>. Demethylating agents should therefore <em><span style="text-decoration:underline;">not</span></em> be given to <em><span style="text-decoration:underline;">pregnant women</span></em> because of serious side effects to the fetus. <em><span style="text-decoration:underline;">Persons who wish to conceive</span></em> should <em><span style="text-decoration:underline;">not</span></em> have been exposed to DNMT inhibitors neither during <em>PGC development (weeks 2 to 6 of embryonic development) nor during gametogenesis and maturation of gametes. </em>Hypomethylation<em> could induce </em>inappropriate expression of certain genes in the zygote. <em><span style="text-decoration:underline;">Young children</span></em> should not be treated with demethylating agents.</p>
<p> </p>
<p> </p></div>
  </body>
</html>