<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 2229 from Anon (session_user_id: 6c98c71372c10c78bf70bd5bd89a4bf5008f428f)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 2229 from Anon (session_user_id: 6c98c71372c10c78bf70bd5bd89a4bf5008f428f)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><ol><li>Cytosine C5 position methylation of CpG dinucleotides concentrated as shored islands in exonic promotor regions of mammalian DNA serves to prevent loss of these pairs to deamination (Cohen,<em> et al. Cell</em>. 145:773-86, 2011), may assist in proofreading of newly synthesized DNA (Saxonov, <em>et al. PNAS</em>. 103:1412-7, 2006), and inversely correlates with gene expression (Hassler and Egger, <em>Biochimie</em>. 94:2219-30, 2012).</li>
<li>CpG islands are typically hypermethylated in cancer causing gene silencing of the downstream promoted gene (Hassler).</li>
<li>Disrupted CpG methylation may be one of several hits that contribute to DNA-mediated disease by altering baseline imprinting, X chromosome inactivation, heterochromatin formation, and gene silencing from balanced health to disease (Hassler).</li>
<li>Intergenic and repetitive element DNA methylation serves to prevent transcription from starting at nearby incorrect start sites, inhibit antisense transcription, inhibit direct RNA splicing, regulate transcription timing, regulate tissue specific transcription factor binding, regulate spatial chromatin organiziation, and regulate gene expression patterns (Hassler) (specifically reducing transposable element activity).</li>
<li>Intergenic hypomethylation typically occurs in cancer cells allowing greater transposable elememnt activity to increase genomic instability thereby aggravating or initiating other DNA hits to perpetuate malignancy (McCabe, <em>et al</em> <em>Clin Cancer Res</em>. 15:3927-37, 2009).</li>
<li>Intergenic hypomethylation increases genetic instability via greater transposable element activity (McCabe). Variation in intergenic methylation may also regulate noncoding HOTAIR RNA expression so affecting dependent transcription factors (Lu, <em>et al. Breast Cancer Treatment Res.</em> 136: 875-93, 2012). Genomic wide hypomethylation may disrupt normal DNA demethyltransferase activity resulting in abnormal mitoses (Esteller. <em>Ann Rev Pharm Tox</em>. 45: 629-56, 2005). These genetic disruptions favour carcinogenesis, inflammation, and poor healing.</li>
</ol></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>1.) Methylation of the intergenic region between the active upstream IgF2 gene and the silenced H19 downstream gene results in stimulated cell growth in the normal paternal allele.</p>
<p>2.) The unmethylated intergenic region between the inactive upstream IgF2 gene and the active H19 downstream gene results in inhibited cell growth in the normal maternal allele.</p>
<p>3.) In Wilm's tumour imprinting results in the methylation of both maternal and paternal intergenic regions causing H19 silencing in both parental DNA and so duplicate IgF2 activation enhancing neoplastic cell growth.</p>
<p>4.) Two copies of expressed IgF2 gene cause at lest duplicate cell growth stimulation that favour neoplastic conversion.</p>
<p><em>source</em>: http://nature.com/citable/content/model-of-loss-of-imprinting-of-igf2-38079 (last accessed 2013.08.20)</p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>1.) Decitabine is a DNA-demethylating agent (<em>Economist</em>, April 7, 2012).</p>
<p>2.) Decitabine incorporates into DNA srands and so positioned inhibits DNA methyltansferase an ezyme that methylates DNA (<em>http://en.wikipedia.org/wiki/Decitabine</em>)</p>
<p>3.) Through the above mechanism methylcytosine depletion and resulting tumour supressor gene promotor hypomethylation (and so transcription, translation, and anti-tumour product effect) occur.</p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>1.) Altered DNA methylation can alter gene imprinting, X chromosome inactivation, heterochromatin structure, gene promotor region activity, replication factor binding, transcription timing in a persistent mitotically inheritable manner. These multiple mechanisms of persistent changes in gene expression effect local to genome wide changes in functional epigenetic marking.</p>
<p>2.) A sensitive period is a time of development were DNA is sensitive to epigenetic influences such as changes in DNA methylation, histone regulation, and other influences.</p>
<p>3.) Gametogenesis, early blastocyst devevelopment, and pluripotent stem cell differentiation are sensitive periods.</p>
<p>4.) Treating patients during sensitive periods could have negative epigenetic consequences (tumour, epigenetic diease) to the patient and any progeny beyond treating the intended medical condition.</p></div>
  </body>
</html>