<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 431 from Anon (session_user_id: c2894ed53592c274720fa64269519535929c46fe)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 431 from Anon (session_user_id: c2894ed53592c274720fa64269519535929c46fe)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>Unmethylated cytosine-guanine (GC) dinucleotides (called CpG islands) are DNA regions of more than 500 base pairs that have a GC content  greater than 55%. CpG islands are commonly located at 5’ regulatory regions of ~44% of mammalian genes that can “promote” gene transcription in the 5’ to 3’ direction of a gene. Methylation of CpG islands via DNA-methyl transferases (DNMTs) maintains genomic DNA structure (chromatin) by controlling S-cycle of DNA replication. 5-Methylated cytosine spontaneously deaminates to thymine. Thymine mutations remain stable and downregulate (silence) gene transcription of transposable elements by physically preventing the binding of transcriptional proteins to genes or allowing binding of methyl-CpG-binding domain proteins that promotes histone deacetylases and chromatin remodeling into inactive heterochromatin.</p>
<p>Human repetitive elements (REs) contain about 52% of all CpG dinucleotides in the human genome. DNA methylation is removed in the zygote and preserved in somatic cells. Thus, DNA methylation plays a role in X-chromosome inactivation and genetic imprinting. The 5’ nucleotide sequence regulates gene transcription by either silencing or promoting transcription. Thereby, each daughter cell receives DNA that codes the “correct” proteins in the “correct” quantities. Both increased (hyper-) or decreased (hypo-) methylation can lead to changes in altered gene expression and function that lead to cancerous cells.</p>
<p>Repressed tumor suppressor genes are silenced changing cell cycle, apoptosis or genome maintenance. Hypermethylation can increase or activate an alternative promoter and is associate dwith most tumors.</p>
<p>Hypomethylation is also linked to cancer. Unmethylated cytosine spontaneously deaminates to become uracil. Cells have the enzymes to demethylate uracil back to cytosine or respond to mismatch nucleotides by excising and replacing with a cytosine. If hypomethylation occurs, cells can upregulate or overproduce proteins leading to cell proliferation or decrease proteins signaling cell death (apoptosis).</p>
<p> </p>
<p><span></span></p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>In healthy somatic cells, maternal H19 and paternal IGF2 genes on chromosome 11p15.5 are expressed. Several sites upstream of H19 are normally methylated on the paternal allele and unmethylated on the maternal allele.</p>
<p>H19 is a non-coding RNA that does not produce a protein. H19 expression is necessary for cell entry into S-phase and is negatively regulated by p53. H19 modulates angiogenesis and blood vessels development which are needed by tumors.</p>
<p>IGF2 has four promoters with CPG islands that are normally methylated by DNA methyltransferase on silenced maternal allele with paternal allele expression.</p>
<p>Loss of imprinting due to inactive H19 leads to maternally transmitted translocations. The maternal chromosome reverses to a paternal epigenotype, with a paternal pattern of methylation of the H19 promoter, IGF2 turned on, and H19 turned off, causing increased cell growth. LOI of H19 on the maternal chromosome, when it occurs, could occur independently or could be influenced by events in the paternal chromosome.</p>
<p>The maternal IGF2 is often deleted in embryonic and childhood related Wilms tumors of the kidney. In age-related Wilms tumor, hypermethylated promotors of IGF2 lead to expression of both maternal and paternal alleles.</p>
<p> </p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine (5-aza-2'-deoxycytidine) is a analogue of nucleoside 2’-deoxycytidine that acts a nucleoside inhibitor. Decitabine is converted to deoxynucleotide triphosphates that are phosphorylated and incorporated in place of unmethylated cytosine into S-phase replicating DNA but not RNA. Decitabine can covalently bind  DNA methyltransferases which targets degradation. Decitabine is a hypomethylating agent that inhibits DNA methyltransferase 1 (DNMT1). Decitabine activates the intrinsic apoptotic pathway in T-lymphoma leukemia in a p53-independent manner by activateing TNF-alpha and downregulating anti-apoptotic, proto-oncogenic Bcl-2. Decitabine reverses hypermethylation of p15INK4B. Normal p15INK4B protein overexpresses p73 that promotes Bax mitochondrial promoter transactivation and cytochrome c release and caspase activation. The resulting mitochondrial dysfunction stops G2-M phases of the cancerous cell leading to apoptosis. Consequently, decitabine restores gene function by controlling epimutations in cellular differentiation and proliferation that induces monocytic differentiation.</p>
<p> </p>
<p> </p>
<p> </p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>The epigenome refers to all chemicals that modify without changing the sequence of DNA code.  Epigenetic changes occur in meiotic cells (sperm &amp; egg) and mitotic cells (somatic cells). Epigenetic drugs cause changes in constitutive and facultative chromatin which alter cellular phenotypes of proteins produced, receptors bound and protein-protein interactions. Constitutive heterochromatin effects genes near the sequence; whereas, facultative heterochromatin results from silencing of genes. Constitutive heterochromatin is poorly expressed. Facultative heterochromatin is more tightly regulated and is related to morphogenesis or differentiation of cells. Methylation of lysine 9 on the N terminus of histone H3 (H3-K9) silencing DNA distributed in centromeres involved in development and telomeres associated with aging. It is also found on inactive X chromosome and at silenced promoters. H3-K9 methylation directs DNA methylation to pericentromeric heterochromatin.</p>
<p>Epigenetic drugs can alter activity leading to improper gene silencing resulting in developmental X-chromosome inactivation and genetic imprinting and dysfunctions of brain memory plasticity (called “chemobrain”) .<em> </em>Epigenetic changes can be inherited. Each daughter cell receives DNA that codes the “correct” proteins in the “correct” quantities. Both increased (hyper-) or decreased (hypo-) methylation can lead to changes in altered gene expression and function that lead to cancerous cells.</p>
<p> </p>
<p> </p></div>
  </body>
</html>