<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1148 from Anon (session_user_id: d01e644eedade11d656cf7016895f1ff5304a6a3)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1148 from Anon (session_user_id: d01e644eedade11d656cf7016895f1ff5304a6a3)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p><span lang="en-us" xml:lang="en-us">DNA methylation aberrations in cancer are best characterized by locus specific DNA hypermethylations and a genome-wide DNA hypomethylation. The first phenomenon may be seen at CpG islands of tumor suppressor genes that tend to be found non-methylated in normal cells. DNA methylation at CpG islands is associated with gene silencing though the binding of methyl CpG binding proteins (MeCP) that possess a transcriptional repression domain and recruit heterochromatin formation factors. By these locus specific DNA methylations, cells are unable to express tumor suppressor genes that protect them against becoming a cancer cell. The DNA hypomethylation aberration in cancer is seen throughout the genome at intergenic regions where repetitive sequences are located. These sequences are prone to genomic instability through illegitimate recombination, transposition, and disruption of neighboring genes. The accumulation of these mutations may lead to cancer. Evidence to this phenomenon has been studied through the deletion of <em>Dnmt1</em> which is in charge of setting DNA methylation marks after cell division. In normal cells these regions are commonly methylated as a mechanism to prevent their activation or recombination. This may occur by the silencing of repeats since methylated cytosines can easily mutate to thymines and thus avoid homologous recombination. It is unknown when genetic mutations cause epigenetic changes or vice versa, but definitely there is an interesting crosstalk.</span></p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p><span lang="en-us" xml:lang="en-us">Alterations in DNA methylation at imprinted control regions (ICRs) result in loss of imprinting. These alterations may appear as hypomethylations or hypermethylations causing a lower expression of growth restricting genes or the overexpression of growth promoting ones. Normally at the imprinted H19/Igf2 cluster, the paternal allele possesses a methylated ICR that </span>inhibits the binding of the CTCF insulator protein to this region. Enhancers located downstream to the ICR are able to promote the expression of <em>IGF2</em>, which is found upstream to the ICR. Furthermore, the methylation of the ICR invades the <em>H19</em> locus, which is located next to it, and silences its expression. At the maternal allele, where the ICR is non-methylated, the insulator protein binds and does not allow <em>IGF</em>2 expression activation by the enhancers. As a consequence, <em>H19</em> is expressed and <em>IGF2 </em>is silenced. In Wilm's tumour, methylation occurs at the ICR of the maternal and results in an expression pattern identical to the paternal allele. The overexpression of the Igf2 protein results in overgrowth and tumor development.</p>
<p> </p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p><span>The Economist article "Cancer's epicentre" </span>classifies Decitabine as a <span>DNA-demethylating agent and is used <span>to treat myelodysplastic syndromes. This means that the epigenetic mark that the drug inhibits is DNA methylation. The use of the drug would result on DNA hypomethylation or, in other words, the inhibition of DNA methylation on CpG islands of tumor suppressor genes or ICRs that induce the expression  growth promoting genes when methylated. The expression of tumor suppresor genes and the silencing of growth promoting genes would provoke the anti-tumor effect. </span></span></p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Since DNA methylation is inherited to daughter cells, the effect of a drug that alters this epigenetic mark would last even after treatment. This means that if a cell changes its DNA methylation state at a specific locus by the use of a drug, the daughter cells originated from that cell would keep the same change even if they are not treated. We must remember that during cell division the two daughter cells keep one strand of the mother cell. That strand would carry the epigenetic information. If it was methylated the new complementary strand would become methylated by Dnmt1, but if it was not, then no methylation would take place.</p>
<p>There are specific periods that are susceptible to epigenetic changes not inherited by mother cells. These periods are called sensitive and occur when DNA methylation is being reset. Sensitive periods are early development and germ cell development (spermatogenesis and oogenesis). Treating patients during sensitive periods is inadvisable since it will result in DNA methylation aberrations of their progeny or in the case of a pregnant woman of her early embryo since it would prevent the correct resetting of DNA methylation.</p></div>
  </body>
</html>