<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1215 from Anon (session_user_id: b21ac22d2947c60f7f24948bf4e02711f2cde09e)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1215 from Anon (session_user_id: b21ac22d2947c60f7f24948bf4e02711f2cde09e)</h1>
    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><ul><li>Decitabine is called according The Economist article “<a href="http://www.economist.com/node/21552168">Cancer’s epicentre</a>” DNA-demethylating agent and belongs to<strong> </strong>DNA methyltransferase inhibitors (DNMTi).  </li>
</ul><p> </p>
<ul><li>The small-molecule drugs target the active site of particular enzyme. The DNMTi are very similar and function as nucleoside analogues; they (e. g. Decitabine) incorporate into the DNA upon replication. DNA methyltransferase binds with such a nucleotide in irreversible way and cannot be released. The action of DNA methyltransferases are division depended. </li>
</ul><p> </p>
<ul><li>Since the action of DNA methyltransferases are replication dependent and the cancer cells replicate much more than normal cells, therefore the DNMTi (Decitabine) effects the cancer cells more. It seems to be most effective in treatment of myelodysplastic syndromes.</li>
</ul><p> </p>
<p>DNMTi used in high doses in 70s, resulted in some toxicity and side effects. They were used not in a range, which causes DNA methylation. Nowadays, in much more lower doses they have very good anti-neoplastic effect</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><ul><li>DNA methylation in imprint control region (ICR) does not correlate with silencing of genes expression because of different mechanism of action. DNA methylation imprint is on paternal chromosome. Paternal ICR is DNA methylated, which blocks CTCF protein binding. DNA methylation on ICR spreads downstream to H19 promoter. There is no insulating action and enhancers promote the lgf2 expression. Thus, lgf2 is only expressed from paternal allele.</li>
</ul><p> </p>
<ul><li>DNA methylation does not occur on ICR of maternal allele. This ICR is bound by CTCF protein. CTFC is an insulator and insulates <em>Igf2 </em>from downstream enhancers. Because it is insulating lgf2, these enhancers are free to act on H19 and enhance H19 expression. The preferred chromatin loop is between enhancers and lgf2. This is not happening because CTCF is bound, enhancers enhance H19 expression instead.</li>
</ul><p> </p>
<ul><li>Common feature in cancer cells is loss of imprinting. They no longer show imprinted expression. Wilm’s tumour is associated with hypermethylation of ICR. There is methylation also on maternal alleles. Enhancers promote the lgf2 expression of lgf2. This results in double dose of lgf2.</li>
</ul><p> </p>
<ul><li>Wilm’s tumour observed early, in preneoplastic tissues. lgf2 (oncogene) is promoting growth therefore overexpression of lgf2 contributes to this childhood kidney tumor. </li>
</ul></div>

    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p> </p>
<ul><li>The CpG islands at the normal (healthy cells) are hypomethylated. Generally, DNA methylation of promoter regions inversely correlates with gene expression CpG islands, which are found in about 60% of promoter’s density and are usually kept free of methylation independent of their activity state.</li>
</ul><p> </p>
<ul><li>Hypermethylation of CpG islands is usually found at tumor suppressors genes. In the cancer tumor, suppressors genes can be silenced genetically or epigenetically. Hypermethylation of CpG islands in the promoters of tumor suppressors’s genes causes their silencing.</li>
</ul><p> </p>
<ul><li>While the global level of methylation decreases in normal cells towards progression of cancer, the methylation of CpG islands increases towards the cancer. The silenced tumor suppressor’s genes (because of hypermethylation of CpG islands in their promoters) are mitotically heritable and epimutations are rapidly selected, which give them competitive advantage over others in surrounding tissues. Cells with epimuations divide more rapidly and die slower.</li>
</ul><p> </p>
<ul><li>DNA methylation of repetitive elements helps maintain genomic integrity by preventing illegitimate recombinations, transposition [by silencing of the repeats or by mutation of the repeats (meC to T) The C--T is a mutation, the renders the intergenic region, or sequence unable to move around, copy and paste, or cut and paste itself.  Therefore, the transposon cant move, and cause mutations elsewhere]. The silencing of repeats allows to avoid transcriptional interference from strong promoters.</li>
</ul><p> </p>
<ul><li>In cancer, intergenic regions and repetitive elements are hypomethylated and no longer densely packaged into heterochromatin: most common repetitive elements, occurs in almost every tumor already in early tumor genesis and progress with time. Hypomethylated repetitive elements recombine illegitimately. The repeats are activated, make copy of them and jump around the genome. It activates of cryptic promoters and disrupts the coding region of neighboring genes.</li>
</ul><p> </p>
<ul><li>It leads to deletions, insertions, reciprocal translocations, transcriptional aberrations in the surrounding genes.</li>
</ul></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><ul><li>Epigenetic marks need to be cleared between generations to restore totipotency. Altering DNA methylation in sensitive periods may lead to misexpression of genes within specific imprinted clusters or changes in epigenetic basis of disruption to imprinted gene cluster, because genomic Imprinting concerns parent-of-origin specific gene expression. It can also be affected by environmental factors.<strong></strong></li>
</ul><p> </p>
<ul><li>Sensitive periods are the periods of reprogramming and active remodeling of epigenetic marks.</li>
</ul><p> <span style="font-size:14px;line-height:21px;"> </span><span style="font-size:14px;line-height:21px;"> </span></p>
<ul><li>1) from primordial germ cells development to the production of gametes (Differential resetting of DNA methylation in spermatogenesis vs oogenesis) and 2) preimplantation in early postimplantation period (rapid and active demethylation of paternal genome, Passive demethylation of maternal genome). There are brief windows in time during tissue specific differentiation reprogramming, although this is less sensitive for whole organism.</li>
</ul><p> <span style="font-size:14px;line-height:21px;"> </span></p>
<ul><li>During the sensitive period offspring may inherit phenotypes or gene expression patterns from parent through the germ cells, not explained by genetic differences, called “Transgenerational epigenetic inheritance through the gametes”. Hence, threatening patients with remodeling enzyme in this period could establish epigenetic marks, which will be inherited into next generation. It might also cause incomplete epigenetic reprogramming. Messenger molecule can be transmitted in the gametes <span style="font-size:14px;line-height:21px;">causing altered re-establishment of epigenetic marks.</span></li>
</ul></div>
  </body>
</html>