<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 8 from Anon (session_user_id: b7c646d08c61c070125e0dc580df8b33f6782be0)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 8 from Anon (session_user_id: b7c646d08c61c070125e0dc580df8b33f6782be0)</h1>
    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value">Decitabine is DNA-demethylating agent, that is sold as Dacogen by Eisai for treatment of myelodysplastic syndrome, which is precursor of acute myelogenous leukaemia (<a href="http://www.economist.com/node/21552168">http://www.economist.com/node/21552168</a>). As hypomethylising agent, it functions as inhibitor of DNA methyltransferase (<a href="https://www.ncbi.nlm.nih.gov/pubmed/12879469">https://www.ncbi.nlm.nih.gov/pubmed/12879469</a>). It is a chemical analogue of cytidine, a nucleoside present in DNA and RNA. Cells in the presence of Decitabine incorporate it into DNA during replication and RNA during transcription. The incorporation of Decitabine into DNA or RNA
 inhibits methyltransferase thereby causing demethylation in that 
sequence. This adversely affects the way that cell regulatory proteins 
are able to bind to the DNA/RNA substrate (<a href="http://www.drugbank.ca/drugs/DB01262">http://www.drugbank.ca/drugs/DB01262</a>).<br /></div>

    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value">CpG islands are parts of DNA which are found in about 60% of promoters (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2802667/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2802667/</a>). They have a high CpG density and are usually kept free of methylation independent of their activity state (<a title="Link: http://www.ncbi.nlm.nih.gov/pmc/articles/PMC122594/" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC122594/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC122594/</a>)<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3480634/#bib5"></a>.
 However, in cancer cells promoter CpG islands tend to become 
hypermethylated, which then causes silencing of the underlying gene. <br />Intragenic methylation is found at repetitive sequences such as 
satellite repeats and remnants of retroviral insertions such as LINEs 
and SINEs in human DNA (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2976721/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2976721/</a>).
 Gene body methylation is present in highly expressed genes and it has 
been speculated that this may repress transcriptional noise from 
alternative start sites, inhibit antisense transcription or direct RNA 
splicing and relates to replication timing  (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3998662/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3998662/</a>; <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3261681/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3261681/</a>).
 A direct role for alternative splicing has recently been attested to 
methylation-sensitive CTCF binding and polymerase II pausing at CTCF 
bound exon boundaries (<a href="http://www.ncbi.nlm.nih.gov/pubmed/21964334">http://www.ncbi.nlm.nih.gov/pubmed/21964334</a>).
 Comparison of the methylome of different hematopoietic lineages has 
identified a large number of hypomethylated regions (HMRs), which can be
 constitutive or lineage-specific and colocalize with transcription 
factor binding sites.
 Furthermore, complex patterns of methylation were detected in 
progenitor cells in HMRs, which resolved in a lineage-specific fashion 
to methylated or demethylated regions, reminiscent of a bivalent histone
 methylation state found in progenitor cells (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3412369/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3412369/</a>).
 Similarly, the deciphering of the mouse methylome revealed regions of 
low methylation at distal regulatory elements, which was linked to 
binding of tissue specific transcription factors (<a href="http://www.ncbi.nlm.nih.gov/pubmed/22170606">http://www.ncbi.nlm.nih.gov/pubmed/22170606</a>).<br /></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value">Two regulatory regions are important for their parental allele-specific 
expression: a differentially methylated region (DMR) upstream of H19 and
 a set of tissue-specific enhancers downstream of H19. The enhancers 
specifically activate Igf2 on the paternal chromosome and H19 on the 
maternal chromosome. The interactions between the enhancers and the 
genes are regulated by the DMR, which works as a selector by exerting 
dual functions: a methylated DMR on the paternal chromosome inactivates 
adjacent H19 and an unmethylated DMR on the maternal chromosome 
insulates Igf2 from the enhancers. These processes appear to involve 
methyl-CpG-binding proteins, histone deacetylases and the formation of 
chromatin insulator complexes (<a href="http://www.ncbi.nlm.nih.gov/pubmed/10788777">http://www.ncbi.nlm.nih.gov/pubmed/10788777</a>).<br /><br /> Wilms' tumor is a type of embryonic kidney cancer that is associated with the <i>IGF2</i>/<i>H19</i> locus on chromosome 11. <i>H19</i> is a noncoding RNA of unknown function with properties that can suppress growth. <i>IGF2</i>
 codes for insulin-like growth factor 2, a growth factor highly 
expressed in many types of tumors. These two genes are both imprinted, 
and normally, only the maternal copy of <i>H19</i> and the paternal copy of <i>IGF2</i> are expressed (Steenman <i>et al.</i>, 1994). In cancer cells, however, both <i>H19</i> and <i>IGF2</i> typically lose their imprinting<i> </i>(<a href="http://www.nature.com/scitable/content/Loss-of-imprinting-of-IGF2-is-linked-14631">http://www.nature.com/scitable/content/Loss-of-imprinting-of-IGF2-is-linked-14631</a>).<br /></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value">Epigenetic changes are passed on during cell division to daughter and 
granddaughter cells until they are actively erased. Once erased, though,
 they do not return. It might therefore be that epigenetic therapies can
 effect changes which stop a cancer growing without having to kill all 
its cell (<a href="http://www.economist.com/node/21552168">http://www.economist.com/node/21552168</a>).<br /><br />Sensitive period was defined (during lectures) as a period, during which the epigenetic marks are being established. This normally happens during reprogramming of DNA 
methylation which occurs during gametogenesis and after fertilization or after
 artificial reprogramming of somatic cells into induced pluripotent stem
 cells (iPSC) (<a title="Link: http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2981432/" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2981432/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2981432/</a>). Treating patients during these would be inadvisable, because the discussed drugs impact the DNA methylation - altering DNA methylation during sensitive periods might cause further harm to the patient.<br /></div>
  </body>
</html>