<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1101 from Anon (session_user_id: cbb61171f696021db7876cd91ddefadafa468a74)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1101 from Anon (session_user_id: cbb61171f696021db7876cd91ddefadafa468a74)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>Under normal circumstances, CpG islands are not methylated (in fact, CpG islands are though to remain conserved because of this lack of methylation, since methylation at CpG has been proved to be mutagenic), whereas intergenic regions and repetitive elements exhibit methylation. However, DNA methylation distribution is abnormal in cancer.</p>
<p>First, CpG islands tend to increase their methylation rate as cancer develops. CpG islands are often placed at the promoters of certain genes, and, since methylation contributes to dense packaging of DNA via diverse manners (the most extended of which involves especific proteins such as MeCP1 or MeCP2) and given that dense packaging of DNA correlates with transcriptional inactivity (because as a result DNA becomes inaccessible to the proteinic transcription machinery), those genes end up inactive. If they are tumour supressors (or maybe are near tumour supressors; we must remember heterochromatin spreading and methylated CpG islands shores), their inactivation will contribute to cancer.</p>
<p>Second,  intergenic regions and repetitive elements lose their methylation, what may bring about disastrous consequences because their methylation is thought to have a fundamental role in genome stability. As far as intergenic regions are concerned, this role is not very well understood yet (at least we have not mentioned it in this course), but for repetitive elements it is known that their lack of methylation leads to the expression of the transposons that they might contain (transposons which are able to interrupt essential genes) and to harmful recombination events, which are favoured by repetitive elements because of their extended distribution throughout the genome.</p>
<p>Furthermore, repetitive elements usually possess strong promoters whose activity can bring about the transcription of neighbour genes that are possibly not supposed to be transcribed at that precise moment, and it is this abnormal transcription along with the increased genomic instability what contributes to cancer.</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>Imprinted genes are usually involved in growth regulation, and that is why loss of imprinting (which may occur by both hypermethylation or hypomethylation) may contribute to cancer, either because a growth restricting gene is less expressed or because a growth promoter gene is more expressed than normal. That is the case of Wilm's tumour (which affects the kidney): Igf2, a growth promoter gene, is overexpressed as a consequence of loss of imprinting at the H19/Igf2 cluster.</p>
<p>Under normal circumstances, Igf2 is only expressed from the paternal allele (and its expression is favoured by certain downstream enhancers), while H19 is not paternally expressed because the imprint control region or ICR is methylated in the paternal chromosome and its heterochromatinization spreads, silencing it. However, the ICR remains unmethylated in the maternal chromosome (from which H19 is expressed and Igf2 is not), being then possible to bind it for an insulator protein called CTCF. The enhancers which I have already mentioned prefer enhancing Igf2 expression but CTCF insulates Igf2 from them, so they have to enhance H19 expression instead.</p>
<p>In Wilm's tumour, it happens that both the paternal and the maternal ICR get abnormally methylated, so Igf2 is overexpressed.</p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine is one of the DNMT (DNA methyltransferases) inhibitors, also called DNA-demethylating agents. They are all nucleoside analogues that function in the same replication dependent manner: once they have been incorporated to the DNA they irreversibly bind the nearby DNMT1 molecules which are going to copy the DNA methylation pattern to the daughter strand. Thus, they contribute to passive demethylation of DNA, presumably acting more strongly against those cells which divide more rapidly and frequently (cancer cells). However, they are supposed to affect every cell which undergoes division, not only cancer cells, so they might present long term effects which remain unknown so far. They have been proved to be more efficient in rather low doses (very high doses are toxic) and against myelodysplastic syndrome progressed to AML, allegedly because it could depend on CpG islands hypermethylation (although they are being tested against other types of cancer too).</p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>The reason why drugs that alter DNA methylation have effects that last beyond the period of drug treatment is because epigenetic changes in general and DNA methylation in particular are heritable from one cell generation to another (thanks in the case of DNA methylation to the activity of DNMT1 or maintenance DNMT). The long term consequences of these artificial changes are utterly unpredictable, and that is why patients should not receive epigenetic treatment during sensitive periods, i. e., during certain periods when the epigenome is most prone to be influenced by environmental circumstances (physical activity, chemicals such as drugs, diet...). As one could intuitively expect, sensitive periods coincide with phases of intense epigenetic reprogramming.</p>
<p>There are only two major sensitive periods in development: the period of pre-implantation and early post-implantation of the embryo and the period of primordial germ cell development, and in both of them there is active removal and then laying down of epigenetic marks. However, it is worth mentioning that we can find more times of epigenetic reprogramming that may therefore be sensitive if we go through tissue specific differentiation for each of the organs that are present in the adult.</p></div>
  </body>
</html>