<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1120 from Anon (session_user_id: 17d13c72a4f71db4888c3064c1f0c09ad3b14b54)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1120 from Anon (session_user_id: 17d13c72a4f71db4888c3064c1f0c09ad3b14b54)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p style="text-align:left;"><span style="font-family:'Times New Roman', serif;"><span style="font-size:small;"><span style="color:#000000;"><span>DNA methylation is established de novo and maintained by specific DNA methyltransferases (DNMT). In normal somatic mammalian cells the majority of CpG sites (70-80%) are methylated. There is a characteristic g</span></span><span style="color:#000000;"><span>enome-wide DNA hypermethylation of intergenic regions and repetitive elements while g</span></span><span style="color:#000000;"><span>ene body methylation is elevated in expressed genes</span></span><span style="color:#000000;"><span>. </span></span><span style="color:#000000;"><span>CpG methylation mainly acts as a silencing epigenetic mark. In mammals, t</span></span><span style="color:#000000;"><span>he normal function of </span></span><span style="color:#000000;"><span> DNA methylation is required for proper embryogenesis and development, for sustaining chromosomal stability, telomere length, and predetermined gene expression states. It is</span></span><span style="color:#000000;"><span> an important regulator of gene transcription </span></span><span style="color:#000000;"><span>and has a role in the genomic imprinting, genomic stability, and X chromosome inactivation. </span></span><span style="color:#000000;"><span>However, since 5-methylcytosine is prone to spontaneous deamination and point mutation to thymine, t</span></span><span style="color:#000000;"><span>he human genome is not methylated uniformly and contains regions of unmethylated sites. These are r</span></span><span style="color:#000000;"><span>egions of DNA which have high density of CpGs grouped in clusters called <em>CpG islands</em> that are present in the 5' regulatory regions (promotors). In normal cells, CPG islands within promotor regions are typically unmethylated, </span></span><span style="color:#000000;"><span>independent of their activity state, </span></span><span style="color:#000000;"><span>which makes them less susceptible to </span></span><span style="color:#000000;"><span>hypermutability </span></span><span style="color:#000000;"><span>of methylated cytosine residues. DNA methylation </span></span><span style="color:#000000;"><span>is an important regulator in many biological processes </span></span><span style="color:#000000;"><span>and defects in methylation cause diseases. </span></span><span style="color:#000000;"><span>In cancer cells, the</span></span><span style="color:#000000;"><span> total amount of methylated cytosine is usually decreased resulting in global hypomethylation. Decreased cytosine methylation typically affects repetitive elements, intergenic regions and CpG sites located in gene bodies (introns and inner exons). </span></span><span style="color:#000000;"><span>However, promoter CpG islands tend to be hypermethylated, which then causes silencing of the underlying gene. This can result in silencing of tumour suppressor genes, subsequently disrupting the control of cell cycle, apoptosis or DNA repair. Regional gain and genome-wide loss of DNA methylation also cause genomic instability and deregulation of tissue specific and imprinted genes.</span></span></span></span></p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p style="text-align:left;"><span style="color:#000000;"><span style="font-family:'Times New Roman', serif;"><span style="font-size:small;">At the H19/Igf2 locus, </span></span></span><span style="color:#000000;"><span style="font-family:'Times New Roman', serif;"><span style="font-size:small;">H19</span></span></span><span style="color:#000000;"><span style="font-family:'Times New Roman', serif;"><span style="font-size:small;"> encodes a ncRNA expressed from the maternal chromosome and the </span></span></span><span style="color:#000000;"><span style="font-family:'Times New Roman', serif;"><span style="font-size:small;">Igf2</span></span></span><span style="color:#000000;"><span style="font-family:'Times New Roman', serif;"><span style="font-size:small;"> gene encodes a fetal growth factor expressed from the paternally inherited chromosome. Both genes compete for shared enhancer elements. Imprinting control region (ICR) containing differentially methylated domains (DMD) is located between the Igf2 and H19 and contains transcriptional repressor CTCF binding sites. On the maternal chromosome, DMD/ICR is unmethylated and binds CTCF protein which acts as an insulator, preventing the enhancers from activating </span></span></span><span style="color:#000000;"><span style="font-family:'Times New Roman', serif;"><span style="font-size:small;">Igf2 while the nearby H19 is activated</span></span></span><span style="color:#000000;"><span style="font-family:'Times New Roman', serif;"><span style="font-size:small;">. On the paternal chromosome DMD/ICR is methylated, therefore CTCF cannot bind so the enhancers activate Igf2 promoter and Igf2 gene is expressed.</span></span></span><span style="font-size:small;"><span style="color:#000000;"><span style="font-family:'Times New Roman', serif;"><span style="font-size:small;"> While Igf2</span></span></span><span style="color:#000000;"><span style="font-family:'Times New Roman', serif;"><span style="font-size:small;"> is normally only transcribed from the paternal allele, the maternal imprinting is lost in many tumours including </span></span></span><span style="color:#000000;"><span style="font-family:'Times New Roman', serif;"><span style="font-size:small;">Wilm’s tumour. Both maternal and paternal ICRs are methylated </span></span></span><span style="color:#000000;"><span style="font-family:'Times New Roman', serif;"><span style="font-size:small;">in Wilm's tumour and this </span></span></span><span style="color:#000000;"><span style="font-family:'Times New Roman', serif;"><span style="font-size:small;">h</span></span></span><span style="color:#000000;"><span style="font-family:'Times New Roman', serif;"><span style="font-size:small;">ypermethylation </span></span></span><span style="color:#000000;"><span style="font-family:'Times New Roman', serif;"><span style="font-size:small;">has the effect of biallelic silencing of </span></span></span><span style="color:#000000;"><span style="font-family:'Times New Roman', serif;"><span style="font-size:small;">H19</span></span></span><span style="color:#000000;"><span style="font-family:'Times New Roman', serif;"><span style="font-size:small;"> and</span></span></span><span style="color:#000000;"><span style="font-family:'Times New Roman', serif;"><span style="font-size:small;"> results in overexpression of Igf2</span></span></span><span style="color:#000000;"><span style="font-family:'Times New Roman', serif;"><span style="font-size:small;"> and disrupted imprinting. </span></span></span></span><span><span style="font-size:small;"><span style="color:#000000;"><span style="font-family:'Times New Roman', serif;"><span style="font-size:small;"><span>Researchers suggest that H19 </span></span></span></span><span style="color:#000000;"><span style="font-family:'Times New Roman', serif;"><span style="font-size:small;"><span>may act as a tumour suppressor gene which, being silenced in Wilm's tumour, can lead to cancer.</span></span></span></span></span></span><span style="font-size:small;"><span style="color:#000000;"><span style="font-family:'Times New Roman', serif;"><span style="font-size:small;"> Also, overproduction of insulin-like growth factor 2 (Igf2) may lead to growth abnormalities.</span></span></span></span></p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p style="text-align:left;"><span style="color:#000000;"><span style="font-family:'Times New Roman', serif;"><span style="font-size:small;">Decitabine belongs to a class of epigenetic inhibitors called DNA-demethylating or hypomethylating agents. Decitabine is used to treat myelodysplastic syndrome (MDS). In some hematopoietic neoplasms including MDS, DNA hypermethylation can inactivate genes essential for the control of normal cell growth, differentiation or apoptosis. DNA-demethylating agents target the mechanism of DNA methylation by inhibiting DNA methyl-transferases (DNMTs), resulting in hypomethylation of the DNA and re-expression of tumor suppressor genes. Cytosine analogues such as decitabine have been shown to inhibit DNMT and are being used against MDS, as well as other cancers. It has been shown that decitabine incorporates into DNA and, at higher doses, inhibits cell proliferation through nonreversible covalent linking with DNA methyltransferase and blocking of DNA synthesis. Decitabine also induces hypomethylation, thereby promoting cell differentiation, re-expression of tumor suppressor genes, stimulation of immune mechanisms and suppression of tumor growth.</span></span></span></p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p style="text-align:left;"><span style="color:#000000;"><span style="font-family:'Times New Roman', serif;"><span style="font-size:small;">DNA methylation can have enduring effects on the epigenome because of the ability of DNA methylation patterns to be established and maintained in somatic cells. This mechanism of maintenance depends on semiconservative copying of the parental-strand methylation pattern onto the progeny DNA strand which results in passing epigenetic information between cell generations. </span></span></span></p>
<p style="text-align:left;"><span style="color:#000000;"><span style="font-family:'Times New Roman', serif;"><span style="font-size:small;"><span>For most cell types in the body, epigenetic marks become fixed following differentiation or cell division. However, some cells undergo major epigenetic ‘reprogramming’ which involves the removal of epigenetic marks in the nucleus, followed by the establishment of different set of marks. Reprogramming occurs during several distinct phases of mammalian development, known as sensitive periods of development, which are period between fertilisation and formation of the blastocyst, early embryo development (after the fifth cell cycle and beginning of differentiation), and development of primordial germ cells. </span></span></span></span><span style="color:#000000;"><span style="font-family:'Times New Roman', serif;"><span style="font-size:small;">Genomes are “plastic” and can accommodate some errors in epigenetic reprogramming that can be transmitted to subsequent generation. </span></span></span><span style="color:#000000;"><span style="font-family:'Times New Roman', serif;"><span style="font-size:small;"><span>Interfering with these highly sensitive periods may have significant short-term and long-term effects on the individual and his or her progeny and can result in some diseases and cancer, which is why it would be inadvisable to treat patients during those periods.</span></span></span></span></p></div>
  </body>
</html>