<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 620 from Anon (session_user_id: e53976eaaa0fc96ce55f93a038585ab150e1cfb1)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 620 from Anon (session_user_id: e53976eaaa0fc96ce55f93a038585ab150e1cfb1)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p><a></a><a></a><a></a> DNA methylation is the basis for different epigenetic phenomena such as imprinting, X chromosome inactivation or the formation of heterochromatin. Generally, DNA methylation of promoter regions inversely correlates with gene expression . Exceptions are CpG islands, which are found in about 60% of promoters . They have a high CpG density and are usually kept free of methylation independent of their activity state. However, in cancer cells promoter CpG islands tend to become hypermethylated, which then causes silencing of the underlying gene .</p>
<p>Changes in DNA methylation include a genome-wide loss and a regional gain of DNA methylation. This causes on one hand genomic instability and deregulation of tissue specific and imprinted genes and on the other hand silencing of tumor suppressor genes – controlling cell cycle, apoptosis or DNA repair – by hypermethylation of their promoter CpG islands</p>
<p>Intragenic methylation is found at repetitive sequences such as satellite repeats and remnants of retroviral insertions such as LINEs and SINEs in human DNA. Gene body methylation is present in highly expressed genes and it has been speculated that this may repress transcriptional noise from alternative start sites, inhibit antisense transcription or direct RNA splicing and relates to replication timing . A direct role for alternative splicing has recently been attested to methylation-sensitive CTCF binding and polymerase II pausing at CTCF bound exon boundaries</p>
<p>The epigenome of cancer cells displays numerous alterations in comparison to the epigenome of their normal counterpart. Changes in DNA methylation include a genome-wide loss and a regional gain of DNA methylation. This causes on one hand genomic instability and deregulation of tissue specific and imprinted genes and on the other hand silencing of tumor suppressor genes – controlling cell cycle, apoptosis or DNA repair – by hypermethylation of their promoter CpG islands. Mutations in chromatin related enzymes such as DNMTs, histone modifiers or chromatin remodelers might induce stochastic changes in the epigenome causing global changes in chromatin.</p>
<p> </p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>The genes encoding <em>IGF2</em> and <em>H19</em> are located ∼120 kb apart on human chromosome 11p15.5 and are reciprocally imprinted, leading to paternal expression of <em>IGF2</em> and maternal expression of <em>H19</em> in most tissues . Parental allelic expression is determined by an imprinting control region (ICR) located between the two genes, upstream of the <em>H19</em> promoter. A critical role has CTCF, which binds to the unmethylated maternal ICR and serves as a strategic protein that implements DNA loops (<a class="bibr popnode tag_hotlink tag_tooltip" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2816615/#DDP558C7">7</a>,<a class="bibr popnode tag_hotlink tag_tooltip" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2816615/#DDP558C8">8</a>) and helps silence DNA regions by binding and recruiting the polycomb repressive complex 2. The <em>Igf2</em> differentially methylated region-2 (DMR2) loops out to interact with the distant methylated <em>Igf2</em>/<em>H19</em> ICR, thereby pushing the <em>Igf2</em> promoter into close contact with the H19-enhancer, which lies ∼100 kb downstream. This interaction results in <em>Igf2</em> transcription from this allele. In contrast, on the maternal chromosome, DMR1 interacts with the unmethylated ICR via CTCF binding, partitioning the <em>Igf2</em> promoter into a silent loop, inhibiting <em>Igf2</em> and promoting <em>H19</em> transcription</p>
<p>Wilms tumor is the commonest childhood kidney cancer, Two principal somatic abnormalities occur at 11p15: paternal uniparental disomy (pUPD 11p15) and <em>H19</em> epimutation, also known as <em>IGF2</em> loss of imprinting. These defects result then in <em>H19</em> hypermethylation and biallelic <em>IGF2</em> expression. I</p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Changes in DNA methylation include a genome-wide loss and a regional gain of DNA methylation. This causes on one hand genomic instability and deregulation of tissue specific and imprinted genes and on the other hand silencing of tumor suppressor genes – controlling cell cycle, apoptosis or DNA repair – by hypermethylation of their promoter CpG island.</p>
<p>Unlike other forms of gene regulation (those involving transcription factors, for example), epigenetic changes are passed on during cell division to daughter and granddaughter cells until they are actively erased. Once erased, though, they do not return. It might therefore be that epigenetic therapies can effect changes which stop a cancer growing without having to kill all its cells.</p>
<p>A Sensitive period is a period of epigenetic reprogramming, of active remodelling of the epigenome: the clearing and resetting of epigenetic marks, and laying down of epigenetic marks at different parts in the genome. so changes in the environment may be able to influence epigenetic makeup. The period of primordial germ cell development  to the production of mature eggs and sperms seems to be one sensitive period. The second sensitive period is the pre implanted and early post implantation period. These are the periods most sensitive to changes in the environment. All changes in the environment  can influence these critical time periods.<br /><br /><br /></p>
<p> </p>
<p> </p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine is a DNA methyltransferase inhibitor that gets incorporated into the DNA upon replication, and then, when the DNA <br />methyltransferase comes along to bind it, the DNMT1 binds that nucleotide to then copy the methylation to the daughter strand, so that DNA methyltransferase is bound irreversibly and it can no longer be released. <br />And this means that the action of these DNA methyltransferase inhibitors is division dependent so you have to have the cell replicating. <br />This means that cancer cells which are dividing much more rapidly than most other cells in the body will be more severely affected because they are replicating more. Such demethylating agents have been proposed to reactivate tumor suppressor genes aberrantly methylated in tumor cells, leading to inhibition of tumor growth. An important consequence of this is that, unlike conventional cytotoxic agents, it may be best to use such drugs at concentrations lower than the maximum tolerated dose and in a manner dependent on their demethylating activity.</p></div>
  </body>
</html>