<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Evaluation 3377 by Anon (session_user_id: 711b29d86d7e0729db60efb24f499664057d8faa)</title>
    <link href="../../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Evaluation 3377 by Anon (session_user_id: 711b29d86d7e0729db60efb24f499664057d8faa)</h1>
    <div class="field-name" id="5f5b2768bd30c421">Describes the normal function of DNA methylation at CpG islands:
<i>DNA hypermethylation at CpG islands is normally associated with gene silencing.</i><i><br></i></div>
    <div class="field-value" id="594880b636567c49">2</div>

    <div class="field-name" id="6f94202e1e3f40f">Describes how DNA methylation of CpG islands is disrupted in cancer:&nbsp;<i>Cancer cells display locus-specific DNA hypermethylation. Locus-specific DNA hypermethylation in cancer occurs at CpG islands.</i><br></div>
    <div class="field-value" id="968f93b33a0fe448">2</div>

    <div class="field-name" id="5b88505d3b18f1fd">Explains how disruption of DNA methylation at CpG islands contributes to cancer:&nbsp;<i>Locus-specific DNA hypermethylation can contribute to cancer by silencing tumor suppressor genes.</i><br></div>
    <div class="field-value" id="f05cac633c0a2ef3">2</div>

    <div class="field-name" id="d181df912dc9667">Describes the normal function of DNA methylation in intergenic regions and repetitive elements:&nbsp;<i>Methylation in intergenic regions and repetitive elements normally acts to protect genome stability.</i><br></div>
    <div class="field-value" id="2a4b4500d5a6fc8c">2</div>

    <div class="field-name" id="c1c936f92be2e4b7">Describe how DNA methylation in intergenic regions and repetitive elements is disrupted in cancer:&nbsp;<i>Cancer cells display genome-wide hypomethylation; hypomethylation in cancer is found in intergenic regions and repetitive elements.</i><br></div>
    <div class="field-value" id="86cb4376132b41db">2</div>

    <div class="field-name" id="856c686f766d1391">Explain how disruption of DNA methylation in intergenic regions and repetitive elements contributes to cancer:&nbsp;<i>Hypomethylation of intergenic regions and repetitive elements promotes genome instability.</i><br></div>
    <div class="field-value" id="6334bb14589c3da1">2</div>

    <div class="field-name" id="a09d84bfa75542cf">Describes the methylation pattern of the paternal allele and how this determines Igf2 expression status:&nbsp;<i>On the paternal allele, methylation of the ICR prevents binding of CTCF. The enhancers can access the Igf2 promoter, leading to Igf2 expression.</i><br></div>
    <div class="field-value" id="ac5b1f189902f0d9">0</div>

    <div class="field-name" id="29a15b29eb68c18">Describes the methylation pattern of the maternal allele and how this determines Igf2 expression status:&nbsp;<i>On the maternal allele, the ICR on the maternal allele is normally not methylated. CTCF binds to the ICR and blocks the enhancers from accessing the Igf2 promoter, leading to no Igf2 expression.</i><br></div>
    <div class="field-value" id="7277eddae68977c1">0</div>

    <div class="field-name" id="6f1135763f6df663">Describes how imprinting at the Igf2 locus is disrupted in Wilm’s tumour:&nbsp;<i>In Wilm’s tumour, the ICR on the maternal allele is methylated. Methylation of the maternal ICR prevents CTCF from binding, thereby allowing the enhancers to access the Igf2 promoter.</i><br></div>
    <div class="field-value" id="1f6b511d38daa79e">0</div>

    <div class="field-name" id="f240901da3fb2b03">Explains how disrupting imprinting at the Igf2 locus contributes to cancer:&nbsp;<i>As a result of disrupted imprinting at the Igf2 locus, the growth-promoting factor Igf2 is overexpressed and contributes to tumour growth.&nbsp;</i><br></div>
    <div class="field-value" id="89e0f252c3bb3ad0">2</div>

    <div class="field-name" id="3f01b7bd773c6b52">Identifies the class of epigenetic inhibitors that decitabine belongs to:&nbsp;<i>Decitabine is a DNA methyltransferase inhibitor (DNMTi).&nbsp;</i><br></div>
    <div class="field-value" id="4e9a30727ede1720">2</div>

    <div class="field-name" id="d20fb9a2ee256b4">Describes the impact of Decitabine on DNA methylation: <i>DNA methyltransferase inhibitors such as Decitabine trigger demethylation of DNA.&nbsp;</i><br></div>
    <div class="field-value" id="e03849bba7ae2e80">2</div>

    <div class="field-name" id="7b3c742dea5bde0">Describe how Decitabine can have an anti-tumour effect: <i>DNMTi such as Decitabine can trigger demethylation of CpG islands in tumor suppressor genes, leading to reactivation of genes that inhibit tumor growth.</i><br></div>
    <div class="field-value" id="7aefa1a74346f571">2</div>

    <div class="field-name" id="69bef9aa7171b08">Describes how altering DNA methylation can have enduring effects on the epigenome:<i>&nbsp;DNA methylation is mitotically heritable, so changing the DNA methylation pattern can have effects that last beyond mitosis.</i><br></div>
    <div class="field-value" id="82a8ceef6afe42c3">2</div>

    <div class="field-name" id="529f51fd4d70e088">Defines what is meant by a sensitive period:&nbsp;<i>Sensitive periods refer to particular periods when epigenetic marks are cleared and reset, i.e. reprogrammed.</i><br></div>
    <div class="field-value" id="de8b3748a6b0f34a">2</div>

    <div class="field-name" id="3054b412bc5b110b">Identifies sensitive periods of development:&nbsp;<i>Sensitive periods include germ cell development and early embryonic development.&nbsp;</i><br></div>
    <div class="field-value" id="76b05e6db9affbb">1</div>

    <div class="field-name" id="792b4bc39bec7d37">Explains why treating patients during sensitive periods would be inadvisable: <i>It would be preferable to avoid treating patients during sensitive periods as epigenetic marks are being established and inhibiting DNA methylation could have more pronounced effects on the genome at these times.&nbsp;</i><br></div>
    <div class="field-value" id="d4e36f0fb678e50c">2</div>

    <div class="field-name" id="e576a4ddbe0a2940">Please make any overall comments here.</div>
    <div class="field-value">You did a fine work on question #1. I gave you 2 points for each section because you really made the effort of defining certain key terms in order to answer the question. You also had a good sense of organization. My comments for your written responses for question #1 are: Thanks for defining what inter-genic regions are in the human genome. I think that by doing this, it helped me further understand, as the reader, (and your evaluator), why it is important for those regions to be methylated, and what exactly goes wrong in cancer. Secondly, great job on clearly identifying that gene silencing is associated with methylation, and for your information on where exactly these CpG islands are located in the human genome. Thanks for emphasizing that there is unregulated cell growth as a result of silencing of tumor suppressor genes from CpG island hyper methylation. Out of all of the papers that I’ve evaluated, yours, by far, was the most well thought-out and best response. And so I gladly scored you 2 points for each section.

But then when I start reading your response to question 2, I got kind of disappointed --- you just got a little bit rusty. I expected the same valor and sense of persuasion that I saw for your responses to question 1. Just like you integrated definitive/key terms with your responses to question #1, you failed to do this with question #2. I think that most of your time was spent on the description of the H19/IGf2 locus, rather than (as the question poses), describing the methylation pattern of the maternal allele versus the paternal allele. Question #2 is wanting us to focus on the methylation pattern of both the maternal and paternal allele so that it will allow one to see exactly what goes wrong in imprinting. Because you lacked a sense of describing the methylation pattern of each parental allele, you couldn’t paint the big picture as to what happens in loss of imprinting. You were able to recognize how loss of imprinting contributes to childhood cancer (this is where I only awarded you points; your only 2 points), but, again, you could not correctly identify exactly how an alteration in DNA methylation can affect something as vital as imprinting. 
Because I loved your responses to question #1, and the much effort that you put into the first question, I am going to give you a few helpful hints as to how you could have answered question #2. The maternal allele has an unmethylated ICR (or DMR; imprint control region) which allows a CTCF insulator protein to bind, so that downstream enhancers do not ‘act on’ the maternal IGf2 to turn it on. This is why the maternal IGf2 is always silenced. So, instead, the preferable route of activation is H19. But on the paternal allele, the ICR is methylated. And so because it is methylated, that CTCF insulator protein is not able to bind, and now IGf2 is ‘turned on’ by downstream enhancers, and thus IGf2 is always expressed. The DNA methylation mark on the paternal ICR allows heterochromatin to spread to the H19 promoter, and so it too, becomes methylated. This explains why H19 is always silenced on the paternal allele. In conclusion, IGf2 is an imprinted gene that is always expressed from the paternal allele.
This is why when a normal embryo inherits their imprinted genes, IGf2 is never expressed from the maternal allele, but is instead always expressed from the paternal allele. 

So then what exactly goes wrong in Wilm’s tumor? Why is the IGf2 imprinted gene disrupted in Wilm’s tumor? Just like I described above for the paternal allele is what happens for the maternal allele. In Wilm’s tumor, the maternal ICR becomes methylated just like the paternal allele. As a result of this methylation, CTCF is no longer  able to bind; and what happens when CTCF can’t bind is that IGf2 is expressed. So, a double dose of a growth-promoting hormone being expressed means an overexpression of growth-promoting genes (in this case, IGf2) and a loss of whatever tumor suppressor genes are associated with controlling cell growth at that imprint control region. A loss of imprinting at the H19/IGf2 locus (coupled with the fact that IGf2 is growth-promoting) leads to cancer. And as Dr. Blewitt had stated in her lectures, loss of imprinting (overgrowth) is a common feature in cancer.

Okay, on now to question 3. You did a great job here, but I was specifically looking for one simple word --- demethylation. That’s the function of Decitabine. How does Decitabine affect DNA methyltransferase activity? By demethylation….when introduced into DNA during replication, DNMT1 binds to Decitabine. And once it binds to Decitabine, it is irreversible (this is another term which you failed to mention). How does Decitabine affect the cancer cells? Well, this drug works great on cancer cells because these replicate much more than normal cells! Since you need inactivity of tumor suppressor genes for unregulated cell growth, and in cancer, tumor suppressor genes are methylated, Decitabine will adversely affect the way that tumor suppressor genes will bind to DNA methyltransferase. Since DNA methyltransferase becomes irreversible in the presence of Decitabine, the result is that the cancer cells stop growing, tumor suppressor genes which were silent are epigenetically reactivated, and normal cell growth is restored (this latter part, you clearly explained). So I gave scored you all of the necessary points for question 3. Again, good job!

Now unto the final question. You got rusty here again, and lacked a fine sense of organization. This time, I think the problem is that you repeated yourself. You have a sense of understanding how a drug which targets an enzymatic epigenetic regulator works in altering the individual’s epigenome for an enduring period (into sensitive periods; periods of epigenetic reprogramming), but I would have loved to see specifics. Like for example, you could have taken into consideration that these type of epigenetic inhibitors (such as Decitabine) are administered to older patients, and so are not advisable for younger patients (as stated in Dr. Blewitt’s lecture for week 7). You really only mention one period of epigenetic reprogramming in your response as a sensitive period of development. That’s why I only gave you a score of 1 for this section.
So but by specifics, I mean that you could have mentioned another sensitive period such as primordial germ cell development and expounded on that. Why this is important is that the embryo (who will be male or female); that primordial germ cell development is very sensitive, and cannot be altered because this is the pathway to gametogenesis. And so it is not advisable to administer such drugs to (for example), a younger female patient who plans on reproducing in the future (both pre-conception and conception). Why? Considering the female germ line, and that it takes a longer amount of time following primordial germ cell development for oocytes to fully mature, delivering this drug to a younger female patient would be absolutely detrimental. It would not only further de methylate the oocytes (oocytes remain in a haploid demethylated state until puberty), but even if given after puberty, it would alter the maternal affect proteins which are necessary for imprinted genes to maintain their methylation during early embryonic development. Like you clearly stated in your beginning sentence, altering epigenetic marks is a bad idea ---- not only for the patient, but also for the next generation of cells.
Overall, you did a great job, and you gave a good thought-process to your paper. I enjoyed reading your paper. I hope that I offered some good suggestions to you. Farewell. Thanks again for a good paper.
</div>
  </body>
</html>