<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 514 from Anon (session_user_id: affc1986451be2af5416955cbce80e853fd190a4)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 514 from Anon (session_user_id: affc1986451be2af5416955cbce80e853fd190a4)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>In
cancer cells, DNA methylation is reduced across the genome as a
whole, but increased at CpG islands.  
</p>

<p>CpG
islands are often promoters of genes.  In normal cells, where a
particular gene is expressed, any preceding CpG island is generally
unmethylated and acts as a promotoer for the gene.  When in cancer
CpG islands are hypermethylated the expression of the associated
genes are reduced.  When the genes whose expression is reduced are
tumour suppressors, this increases the chances that the cell will
become cancerous.  
</p>

<p>In
normal cells, the genome as a whole (including intergenic regions and
repetitive elements) is more heavily methylated than in cancer.  In
the case of repetitive elements heavy methylation prevents
transposons from copying themselves throughout the genome.  It also
reduces the chances of a transposon from one chromosome attaching to
an identical transposon on another chromosome during mitosis and
causing undesirable swaps of genes between chromosomes.  For
intergenic regions, the methylation keeps these from being
transcribed themselves or from 'accidentally' acting as a promoter
for other genes.  
</p>

<p>In
cancer the intergenic regions and repetitive elements become
hypomethylated. In terms of repetitive elements, this means that
transposons are more able to copy themselves throughout the genome. 
They can therefore disrupt normal gene expression and could both
inhibit expression of tumour suppressor genes, by disrupting their
promoters or the genes themselves, and increase the expression of
oncogenes by acting as a strong promoter for them.  In terms of
intergenic regions, hypomethylation makes it more likely that there
can be undesirable transcription of the intergenic region or that
expression of genes (oncogenes or tumour suppressor genes) can be
increased or decreased. 
</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p style="margin-bottom:0cm;line-height:100%;"><font face="Arial, sans-serif">Disruption
to imprinting can lead to either overexpression or underexpression of
the imprinted genes.  An over-expressed oncogene or an
under-expressed tumour suppressor gene could lead to cancer.</font></p><p style="margin-bottom:0cm;line-height:100%;"><font face="Arial, sans-serif">The
H19.Igf2 cluster has a region where the CTCF insulator protein binds.
 In the maternal allele, CTCF is allowed to bind, as the binding area
(the imprint control region) is unmethylated.  This means that local
enhancers cannot act on their preferred downstream target, the IgF2
gene, but act instead on the closer H19 gene, ensuring that it is
expressed.</font></p><p style="margin-bottom:0cm;line-height:100%;"><font face="Arial, sans-serif">In
the paternal allele, the CTCF cannot bind.  This is because the
binding area is methylated.  The local enhancers act on the preferred
IgF2 gene (which is not insulated as there is no CTCF) and so IgF2 is
expressed. The binding area methylation spreads to the H19 gene
meaning that this cannot be expressed.</font></p><p style="margin-bottom:0cm;line-height:100%;"><font face="Arial, sans-serif">In
Wilm's tumour, the ICR on the maternal allele is methylated, as well
as the ICR on the paternal allele.  This means that IgF2 is
overexpressed which leads to cell overgrowth.  When combined to loss
of tumour suppressor activity, this can lead to cancer.  This often
occurs through associated loss of the upstream tumour suppressor gene
Cdkn1c.</font></p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine
is a DNA Methyl Transferase inhibitor used to treat myelodysplastic
syndromes which develop into acute myelogenous leukaemia.  
</p>

<p>Decitabine
is incorporated into a replicating DNA strand due to its similarity
with the material from which nucleotides are made.  It's sufficiently
different though to ensure that once DNMT1 binds to it to attempt to
methylate it, it doesn't let go of the DNMT1. This means that
methylation is stopped at this point and that the DNMT1 is not free
to methylated other parts of the chromosome.  As this only happens
during cell division, it will inhibit methylation in cells that
replicate more frequently, i.e. cancer cells.  
</p>
<p> 
</p>
<p>Decitabine
has been found to help in some cancers caused by hypermethylation
such as  myelodysplastic syndromes.  This may relate to errantly
methylated tumour suppressor genes becoming expressed again.  Or it
may be that mutated DNMT1 (with a tendency to overmethylate) is
'captured' by the nucleosides.</p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Altering
DNA methylation can have enduring effects on the epigenome as DNA
methylation is mitotically heritable.  This means that any changes to
the epigenome (and therefore the genes expressed) will be passed on to daughter cells.  This is because
when DNA is divided, when the new strand is created, DNMT1 can
recognise that one strand is methylated and the other not, and
methylate the new strand to maintain methylation marks.  
</p>
<p>DNA
methylation cannot yet be directed therapeutically at a specific
region of the genome. For this reason it is not advisable to use such
treatment during sensitive periods .  
</p>
<p>A
sensitive period is one where the epigenome is being extensively
reprogrammed and there is therefore a higher risk that more of the
genome will be available for unwanted methylation and therefore produce side
effects.  During less sensitive times, much of the genome is less
accessible reducing the risk of side effects.  
</p>
<p>Such
sensitive periods occur during primordial germ cell development and
early embryo development.  
</p></div>
  </body>
</html>