<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 366 from Anon (session_user_id: 458d5e361f45f048bd4a9e0eb329b10a09ad77ed)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 366 from Anon (session_user_id: 458d5e361f45f048bd4a9e0eb329b10a09ad77ed)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><br /><br /><p><br /></p>



<p> The reason of methylation at CpG
islands is primarily to silence a gene.  Some times are cyclically methylated and demethylated at a specific
gene. Epigenetic control uses methylation to turn off a gene permanently;
particularly genes that only play roles during the development of an organism. Once
DNA methylation happens this typically sets the stage for the methylated portion
to be wrapped up around histones, zipping it, to completing the silencing of
the gene.</p>







<p> Cancer origin is always epigenetic
lesions. The presence of CpG island-promoter hypermethylation affects genes
involved in cell cycle, DNA repair and carcinogen metabolism. One explanation
is that the hypermethylated genes are those that are more exposed to external
carcinogen agents. <br /></p>









<p> <span>Methylation in CpG  shores is related to gene expression,
but hypermethilation of CpG islands has also been detected in different tumor
cell lines and nerve sheath tumors, in these regions were initially identified
hypermethylated in cancer and represent regions of tissue specific methylation
in normal tissues. <br /></span></p>





<p> <span>DNA methylation occurs at cytosines
in any context of the genome, more than 98% of DNA methylation occurs in a CpG
dinucleotide context in somatic cells. DNA methylation is regulated by a family
of DNMTs: DNMT1, DNMT2, DNMT3A, DNMT3B, and DNMT3L. DNMT1 preferentially
methylates hemimethylated DNA<br /></span></p>



<p> <span>Most CG dinucleotides are methylated
on cytosine residues, where as CG dinucleotides  tend to be protected from methylation. Defects in DNA methylation are
closely associated with cancer, although no mutation or deficiency in any DNMT
has been identified as causally linked to tumor development, most likely
because of their critical role during embryogenesis. <br /></span></p>







<p> <span>Overexpression
of EZH2 increases CpG methylation, while RNA interference knockdown of EZH2
reduces H3K27 methylation and DNA methylation at known EZH2 target genes. G9A,
another histone methylase, catalyzes monomethylation and dimethylation of H3K9
and  H3K27. <br /></span></p>

<p> <a title="Link: http://www.quora.com/What-is-the-purpose-of-DNA-methylation-at-CpG-islands" href="http://www.quora.com/What-is-the-purpose-of-DNA-methylation-at-CpG-islands">http://www.quora.com/What-is-the-purpose-of-DNA-methylation-at-CpG-islands</a></p>

<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/18179966">http://www.ncbi.nlm.nih.gov/pubmed/18179966</a></p>





<br /></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><br /><br /><p><span>At the H19/Igf2
locus, a region 5' to the H19
gene is methylated on the paternal allele. Deletion of 2 kbp of this region did
not perturb establishment of methylation of the remaining
differentially-methylated sequences, however, established methylation patterns
were not efficiently maintained. This highlights potentially important
differences between regulation of methylation establishment in the germ line
and maintenance of established patterns in somatic tissue.</span></p>

<span>Loss of imprinting (LOI) of <i>IGF2</i> and the
tightly-linked H19 locus has
been associated with tumorigenesis in a variety of patient s</span>tatus.



<p><span> Imprinted loci other than H19/IGF2 have also been implicated in cancer. In neuroblastomas
without N-MYC amplification,
maternal-specific deletions of 1p36 have been reported.<br /></span><br /></p>



<p> <span>In Wilms' tumour,
imprinted expression of IGF2 is
commonly relaxed resulting in maternal allele expression. This is associated
with increased methylation and reduced expression from the maternal copy of the
tightly linked <i>H19</i> locus.</span> </p>



<p>              
<span>Imprinted
loci other than H19/IGF2 have also been implicated in
cancer. In neuroblastomas     without N-MYC amplification, maternal-specific deletions of 1p36 have
been reported. This region was later shown to contain the p73 tumor suppressor, a p53 homologue
which is maternally-expressed. <br /></span></p>

<p>               
<a title="Link: http://www.nature.com/ejhg/journal/v10/n1/full/5200768a.html" href="http://www.nature.com/ejhg/journal/v10/n1/full/5200768a.html">http://www.nature.com/ejhg/journal/v10/n1/full/5200768a.html</a></p>

<p><span>             
<a title="Link: http://www.ncbi.nlm.nih.gov/pubmed/20231622" href="http://www.ncbi.nlm.nih.gov/pubmed/20231622">http://www.ncbi.nlm.nih.gov/pubmed/20231622</a></span></p>





<br /></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><br />



Decitabine is a DNA-demethylating agent, is used to treat myelodysplastic
syndromes. It has a dual mechanism of action depending on dose. At both
lower and higher doses, decitabine incorporates into DNA; however, at higher
doses, decitabine inhibits cell proliferation through nonreversible covalent
linking with DNA methyltransferase and blocking of DNA synthesis at lower
doses, decitabine induces hypomethylation, thereby promoting cell
differentiation, re-expression of tumor suppressor genes, stimulation of immune
mechanisms, and suppression of tumor growth. <br /><p><span>Decitabine treatment has been shown to reverse
hypermethylation of p15<i>INK4B</i>, allowing for re-establishment
of normal p15<i>INK4B</i> protein expression in addition,
hypomethylation of p15INK4B has been associated with hematologic
response, supporting pharmacologic demethylation as a possible mechanism for
clinical response. Decitabine exhibits antitumoral properties in ACC cells at right doses may be an effective adjuvant therapy in
patients with advanced disease. Decitabine recovers expression of silenced
genes on 11q13, which suggests a epigenetic gene silencing.</span></p>

<p><a title="Link: http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2376088/" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2376088/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2376088/</a></p>

<p><br /></p>





<br /></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>The DNA methyltransferase inhibitors 5AC and DAC are analogs of the
naturally occurring pyrimidine nucleoside cytidine. Their mechanism of action
is concentration-dependent, inducing cytotoxicity presumably as a consequence
of incorporation into RNA (5AC) and/or DNA (5AC and DAC) at high
concentrations, and inducing hypomethylation at lower doses through depletion
of cellular DNMTs. <br /></p>



<p> The sensitive
periods represent heightened epochs of epigenoma development, where the early
environment is able to shape genes and thus determine structural and functional
aspects of genoma and behavior of them for the lifespan. It is through this
so-called programming effect on the epigenoma mechanism that early-life
experiences are thought to confer either risk in DNA configuration.</p>



<p>There are a number of environmental
factors which, working alone or in combination, can alter   methylation states during
sensitive periods, such as embryogenesis and early life. </p>





<p>Evidence that early life may be a
sensitive period for epigenetic changes comes from studies investigating
maternal care in rodents and nonhuman primates. These studies have shown that gestation
and early-life environment can influence stress responses in adult rodents.<br /></p>

<p>Any treatment  with Azacitadine or Dacogen during
sensitive periods methylation or demethylation on histones maybe cause unexpected
reaction on DNA replication.</p>

<p><br /></p>





<br /></div>
  </body>
</html>