<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1219 from Anon (session_user_id: 38cc8b92896b63cb289209b7149a9cdcadc13f22)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1219 from Anon (session_user_id: 38cc8b92896b63cb289209b7149a9cdcadc13f22)</h1>
    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine is an inhibitor of DNA methylation.</p>
<p>Normally DNMT forms a covalent bond with the cytosine ring and transfers methyl group from S-adenosylmethionine to cytosine ring. Release of DNMT from cytosine is dependent on the addition of the methyl group. When decitabine is incorporated into DNA, DNMT forms a covalent bond with the decitabine ring, but the N group at 5′ position prohibits the addition of the methyl group, thereby preventing methylation and covalently trapping DNMT.<br /><br />Decitabine can form complexes with DNA-DNMTs that induce DNA damage and cytotoxicity, suggesting that decitabine may exert its anti-cancer activity by actions that are dependent and independent of its demethylating activity.</p></div>

    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>DNA methylation is controlled at several different levels in normal and tumor cells. The addition of methyl groups is carried out by a family of enzymes, DNA methyltransferases (DNMTs). Chromatin structure in the vicinity of gene promoters also affects DNA methylation and transcriptional activity. These are in turn regulated by various factors such as nucleosome spacing and histone acetylases, which affect access to transcriptional factors.</p>
<p>As compared with normal cells, the malignant cells show major disruptions in their DNA methylation patterns. In the normal cell, CpG island tend to become hypomethylated but methylated repetitive element and intergenic region. In cáncer cell, hypomethylation usually involves repeated DNA sequences, such as long interspersed nuclear elements, whereas hypermethylation involves CpG islands.</p>
<p>In tumour cells, <em>hypermethylation of CpG islands</em> also occurs in cancer cells, and can result in the transcriptional silencing of growth-regulatory genes. These changes in methylation are early events in tumorigenesis. There are several protective mechanisms that prevent the hypermethylation of the CpG islands. These include active transcription, active demethylation, replication timing, and local chromatin structure preventing access to the DNA methyltransferase. The genes that are susceptible are the genes involved in cell cycle regulation (<em>p16<sup>INK4a</sup>, p15<sup>INK4a</sup>, Rb, p<sup>14ARF</sup></em>) genes associated with DNA repair (<em>BRCA1, MGMT</em>), apoptosis (<em>DAPK, TMS1</em>), drug resistance, detoxification, differentiation, angiogenesis, and metástasis. There are a number of tumor types, particularly hematopoietic malignancies such as myelodysplastic syndromes (MDSs), in which hypermethylation occurs and results in leukemic transformation.</p>
<p><em>Repetitive element and intergenic region</em><em> hypomethylated</em> activates transcription of protooncogenes, retrotransposons, as well as genes encoding proteins involved, contributes to genomic instability, a hallmark of tumour cells, through increased mitotic recombination events. The methylation of genomic DNA in malignant cells is catalyzed by DNA methyltransferases DNMT1 and DNMT3B, revealing significantly elevated expression in different types of cancers.</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>The paternal allele is methylated and the maternal allele is unmethylated. The transcription of the <em>IGF2</em> gene is regulated by a differentially methylated region (DMR) upstream of the <em>H19</em> promoter and by an <em>H19</em> downstream enhancer. The <em>H19</em> DMR contains binding sites for CTCF, preventing <em>IGF2</em> transcription and enabling <em>H19</em> expression. On the paternal allele, methylation of the DMR prevents CTCF binding, leading to <em>IGF2</em> expression and <em>H19</em> silencing.</p>
<p> </p>
<p>In Wilms' tumors occur the reactivation of the silent maternal IGF2 allele (loss of imprinting, LOI), resulting in IGF2 expression from two active copies (biallelic expression) similar to paternal IGF2 heterodisomy. IGF2 is a potent mitogen with possible autocrine and paracrine effects. Biallelic expression, resulting hypothetically in double the growth factor dosage, may explain not only the somatic, but may also play a role in tumour development.</p>
<p>Other mechanism of altered imprinting in cancer is hypermethylation and hence silencing of one copy of a tumour suppressor gene. This could result from mutations in, or inappropriate activation of, imprintor genes. Hypermethylation as an alternative pathway for tumour suppressor gene inactivation has been demonstrated for the pl6, the von Hippel Lindau (VHL) syndrome and the retinoblastoma (Rbl) suppressor genes.</p>
<p> </p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>the epigenome is particularly dynamic during embryogenesis because of extensive DNA synthesis, and the elaborate DNA methylation patterning required for normal tissue development is established during early development. <span lang="en" xml:lang="en"><span class="hps">DNA methylation</span> <span class="hps">in</span> <span class="hps">the promoter regions of</span> <span class="hps">genes</span> <span class="hps">results</span> <span class="hps">in</span> <span class="hps">gene silencing</span> <span class="hps">probably due</span> <span class="hps">to inhibition of</span> <span class="hps">transcription complexes</span> <span class="hps">DNA binding</span> <span class="hps">regulator.</span> <span class="hps">If</span> <span class="hps">these processes are not</span> <span class="hps">regulated</span> <span class="hps">in</span> <span class="hps">sensitive</span> <span class="hps">or critical</span> <span class="hps">periods</span> <span class="hps">as</span> <span class="hps">gametogenesis and</span> <span class="hps">early embryogenesis</span><span>, epigenetic changes</span> <span class="hps">are transmitted during</span> <span class="hps">cell division</span> <span class="hps">daughter</span> <span class="hps">cells</span> <span class="hps">and so on.</span> <span class="hps">Therefore</span> <span class="hps">DNA methylation</span> <span class="hps">is a</span> <span class="hps">process</span> <span class="hps">which must be</span> <span class="hps">correctly</span> <span class="hps">set</span> <span class="hps">in the germline</span> <span class="hps">and</span> <span class="hps">is maintained</span> <span class="hps">throughout life</span><span>,</span> <span class="hps">being deleted</span> <span class="hps">and</span> <span class="hps">restored to</span> <span class="hps">the germline of</span> <span class="hps">the next generation.</span> <span class="hps">Therefore if</span> <span class="hps">treatment is performed</span> <span class="hps">to alter</span> <span class="hps">DNA methylation</span> <span class="hps">sensitive</span> <span class="hps">periods</span> <span class="hps">could lead to</span> <span class="hps">the transmission</span> <span class="hps">of changes in</span> <span class="hps">the epigenome</span> <span class="hps">in</span> <span class="hps">the following cell lines</span><span>.</span></span></p></div>
  </body>
</html>