<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1510 from Anon (session_user_id: 71a03d3528a3759e38964bedd59f1d88522c1b49)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1510 from Anon (session_user_id: 71a03d3528a3759e38964bedd59f1d88522c1b49)</h1>
    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>Under normal conditions:</p>
<p>The H19/Igf2 genes are clustered as part of an ICR that is <em>paternally</em> imprinted.   Methylation of <em>paternal</em> ICR leads to methylation of  nearby H19 , silencing the paternal copy.   It is the paternal methylation of the ICR that allows the CTCF insulator protein to bind at the <em>un-methylated maternal ICR</em>, and downstream enhancers can act upon the H19 gene maternally, enhancing transcription.  CTCF blocks the action of these enhancers, and therefore blocks transcription of Igf2 on the maternal strand, which lies upstream of the ICR.  On the paternal strand, the enhancers are free to act upon Igf2, and so expression of Igf2 comes from the <em>paternal</em> allele.  </p>
<p>In Willm's tumour:</p>
<p>There is <em>hypermethylation</em> at the ICR for this gene cluster on the <em>maternal</em> strand, leading to a loss of imprinting.  Both the maternal and paternal ICRs are methylated (should be paternal only), and this causes silencing of both parental copies of the H19 gene by methylation.  Because both ICRs are methylated (blocked), enhancers are free to promote transcription of the upstream Igf2 gene on <em>both</em> parental strands.  Therefore, overall, there are two copies of Igf2 being expresesed, which is tumour-causing as Igf2 is a growth-promoting gene.</p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>The small-molecule drug Decitabine acts as a DNA-demethylating agent used to treat myelodysplastic syndromes, which is a precursor to acute myelogenous leukemia (AML)--a type of blood cancer.  It is a nucleoside analog that gets incorporated into newly-replicated DNA and acts as DNA methyltransferase (Dnmt) inhibitor , meaning that it can block the methylating activity of  Dnmt1 on daughter strands of DNA during the cell-division cycle--which is highly active in tumour cells.   In AML, CpG Island <em>hypermethylation</em> by Dnmt1 is the suspected cause of tumour development by way of silencing AML-related tumour suppressor genes.  If Decitabine is administered in the earlier neoplastic stages of AML, it can reduce the silencing of these tumour supressor genes, reversing neoplastic development.</p>
<p> </p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>DNA methylation is mitotically heritable until actively reset during certain periods of development, and can therefore have enduring effects on the epigenome through the normal cell-division cycle. </p>
<p>The sensitive periods are those in which epigenetic marks are cleared, and re-set.  Imprinted genes (parent-of-origin specification) are maintained from gamete formation (pre-fertilization) throughout the life span until primordial germ cell (PGC) development (post-fertilization).  The maternal and paternal genomes are both cleared (at different rates)  until early embryonic development and then reset until cleared again for PGC formation in the embryo.  In both females and males, these marks will be reset for gamete formation when puberty hits and gametogenesis begins.    </p>
<p>Treating patients with chemotherapeutics that inhibit DNMTs during these sensitive periods could have a genome-wide effect, resulting in hypomethylation throughout the genome.  As discussed previously, there are many potential negative outcomes to this, such as inappropriate silencing of genes after epigenetic marks are reset, or lack of silencing of transposable elements leading to abnormal karyotypes, or loss of parental imprinting at ICRs in the early embryo for example.</p></div>

    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>DNA methylation is a ubiquitous method for epigenic control of gene regulation.  DNA methylation is found at CpG islands within the promoters of genes to maintain a silent state.   In cancer, this regulation may be  disrupted in two ways: <em>hypo</em>methylation--resulting in expression of oncogenes; or <em>hyper</em>methylation--resulting in silencing of tumour suppressor genes.  </p>
<p>Intergenic regions normally possess methylation, however <em>hypo</em>methylation in these areas is observed in cancer, as is <em>hypo</em>methylation of repetitive elements, which are normally methylated for silencing.  When this happens, and repetitive elements are not silenced, transposable elements are activated and may be inserted randomly into other regions of DNA; potentially giving cryptic  promoters for other genes or deleting/inserting genes altogether by creating false regions of homology in chromosomes leading to inappropriate recombinations that give abnormal karyotyptes.</p>
<p>Changes in methylation at ICRs result in a loss of imprinting, which can result in overexpression of a growth gene and silencing of another that is linked to it in a cluster (<em>eg</em>. Igf2/H19 system).  This is commonly seen in early neoplastic tissues that lead to tumour development.</p>
<p>There is evidence in humans and mice that show that mutation of DNMTs and the resulting DNA-hypomethylation causes genomic instability.</p></div>
  </body>
</html>