<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 380 from Anon (session_user_id: 504495e3e9928cfcf06757a9d0d694c6b4472ca9)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 380 from Anon (session_user_id: 504495e3e9928cfcf06757a9d0d694c6b4472ca9)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>DNA methylation is an epigenetic process occuring almost exclusively at regions dense in cytosine-guanine dinucleotides at gene promoters, called CpG islands. These regions are normally unmethylated. Methylation occurs when a methyl group attaches to the 5’ position on the cytosine. The normal function is to silence gene expression. In cancer, DNA methylation at CpG islands is disrupted so that the promoters of tumour suppressor genes become hypermethylated – more than expected - reducing their expression. For example, hypermethylation has been found in the tumour suppressor genes RB (retinoblastoma), MLH1 and MGMT (colorectal cancer) and BRCA1 (breast cancer).  This higher-than-expected methylation pattern is a recognised phenotype in cancer – the CpG island methylator phenotype (CIMP). Hypermethylation of the two kilobase regions upstream and downstream of CpG islands, called CpG island shores, has also been associated with many cancers.</p>
<p>Intergenic regions and repetitive elements of the genome are usually methylated, to prevent gene expression. This promotes genomic stability by regulating the expression of transposable elements, which are mutagenic. DNA methylation is disrupted at intergenic regions and repetitive elements in that these regions are hypomethylated, leading to over-expression. This genomic instability can contribute to cancer in two ways. If an already mutated oncogene becomes hypomethylated, it will express more, which in concert with other genomic and epigenomic ‘hits’ (as per the Knudson hypothesis) will contribute to the development of cancer. In addition, if a transposable element is hypomethylated, thereby allowed to express more, and goes on to mutate a proto-oncogene (i.e. make it an oncogene) it can also contribute to the disease. An example of an element that is hypomethylated and therefore overexpressed in cancer is the HOTAIR lncRNA, which regulates the methylation of histone. The over-expression leads to hypermethylation at H3K27 and H3K4 unmethylation, tightening the chromatin (and lessening the expression) of tumour suppressor genes.</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>Normal genetic imprinting results in parent-of-origin monoallelic expression of imprinted genes. The process is controlled by patterns of methylation at imprinting control regions (ICRs) of target genes (e.g. the H19/Igf2 cluster). Normally, the ICR of the cluster on the maternal allele is unmethylated. This allows a transcription factor (CTCF) to bind to it. As a result, H19 is expressed and so Igf2 is suppressed. Additionally, the normal ICR of the cluster on the paternal allele is methylated. CTCF therefore cannot bind, and so H19 is suppressed, while Igf2 is expressed. In Wilm’s tumor, normal imprinting is disrupted in that there is hypermethylation of both parental alleles. This essentially makes both clusters express as if they were the paternal allele – biallelic (over)expression of Igf2 and non-expression of H19. This contributes to the disease in that the product of Igf2 is a growth factor, and so is a candidate oncogene (that is overexpressed due to the faulty imprinting) and H19 is a candidate tumour suppressor gene (that is not expressed at all due to the faulty imprinting). Overexpression of a growth factor and suppression of tumour suppressors are, at the general level, how cancers proliferate and metastasise.</p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine is in the class of drugs called DNA methyltransferase inhibitors (DNMTis). DNA methyltransferases are the group of enzymes that adds methyl groups to cytosine bases in DNA. Decitabine is a nucleoside analogue. It inhibits methylation by being incorporated into the DNA and forming a covalent bond with the enzyme DNA methyltransferase 1 (DNMT1). This happens when the DNA is replicating – the mechanism of Decitabine is replication dependent. The strong covalent bond binds DNMT1 irreversibly to the DNA. Consequently, the enzyme cannot fulfil its role of transporting and attaching methyl groups and so the region becomes hypomethylated over time. Decitabine has an anti-tumour effect by reversing the hypermethylation that is often found in the promoters of tumour suppressor genes for many cancers. This allows the genes to be reactivated and continue to express their product. Decitabine is also cytotoxic at his doses, which may further function to kill rapidly dividing oncogenes (unfortunately also healthy cells).</p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>DNMTIs like Decitabine get incorporated into DNA and covalently and irreversibly bind DNMT1 – permanent effects that therefore endure beyond the treatment period. More generally, DNA methylation patterns are mitotically heritable. This means that the gradual hypomethylation due to the effect of DNMTis is likely to be passed on in subsequent cell divisions. Caution must be taken when promising epigenetic treatments like these are to be administered in epigenetic sensitive periods. These periods are when a change in the environment (including changes in the body itself) can affect epigenetic control. These sensitive periods are during embryogenesis and during gameotgenesis. Altered DNA methylation patterns during this period can have enduring effects on an organism and perhaps their offspring as well. In plants and mice, for example, altering methylation patterns during gametogenesis has been advanced as an explanation for transgenerational inheritance of epigenetic marks. The same has not been shown as strongly in humans. However, during embryogenesis, epigenetic alterations have been advanced as the explanation for later disease (as per the Barker hypothesis). If this is true, cancer treatment that alters epigenetic marks is inadvisable during these sensitive periods because it may predispose individuals and maybe their offspring to disease and other aberrations.</p></div>
  </body>
</html>