<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Evaluation 496 by Anon (session_user_id: 7f5a5cd64f4604ff02d6b8e7c7ba6ea3f2024cab)</title>
    <link href="../../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Evaluation 496 by Anon (session_user_id: 7f5a5cd64f4604ff02d6b8e7c7ba6ea3f2024cab)</h1>
    <div class="field-name" id="5f5b2768bd30c421">Describes the normal function of DNA methylation at CpG islands:
<i>DNA hypermethylation at CpG islands is normally associated with gene silencing.</i><i><br></i></div>
    <div class="field-value" id="594880b636567c49">2</div>

    <div class="field-name" id="6f94202e1e3f40f">Describes how DNA methylation of CpG islands is disrupted in cancer:&nbsp;<i>Cancer cells display locus-specific DNA hypermethylation. Locus-specific DNA hypermethylation in cancer occurs at CpG islands.</i><br></div>
    <div class="field-value" id="1631d5297774e442">1</div>

    <div class="field-name" id="5b88505d3b18f1fd">Explains how disruption of DNA methylation at CpG islands contributes to cancer:&nbsp;<i>Locus-specific DNA hypermethylation can contribute to cancer by silencing tumor suppressor genes.</i><br></div>
    <div class="field-value" id="f05cac633c0a2ef3">2</div>

    <div class="field-name" id="d181df912dc9667">Describes the normal function of DNA methylation in intergenic regions and repetitive elements:&nbsp;<i>Methylation in intergenic regions and repetitive elements normally acts to protect genome stability.</i><br></div>
    <div class="field-value" id="2a4b4500d5a6fc8c">2</div>

    <div class="field-name" id="c1c936f92be2e4b7">Describe how DNA methylation in intergenic regions and repetitive elements is disrupted in cancer:&nbsp;<i>Cancer cells display genome-wide hypomethylation; hypomethylation in cancer is found in intergenic regions and repetitive elements.</i><br></div>
    <div class="field-value" id="86cb4376132b41db">2</div>

    <div class="field-name" id="856c686f766d1391">Explain how disruption of DNA methylation in intergenic regions and repetitive elements contributes to cancer:&nbsp;<i>Hypomethylation of intergenic regions and repetitive elements promotes genome instability.</i><br></div>
    <div class="field-value" id="6334bb14589c3da1">2</div>

    <div class="field-name" id="a09d84bfa75542cf">Describes the methylation pattern of the paternal allele and how this determines Igf2 expression status:&nbsp;<i>On the paternal allele, methylation of the ICR prevents binding of CTCF. The enhancers can access the Igf2 promoter, leading to Igf2 expression.</i><br></div>
    <div class="field-value" id="77a6c81550ee5b9b">2</div>

    <div class="field-name" id="29a15b29eb68c18">Describes the methylation pattern of the maternal allele and how this determines Igf2 expression status:&nbsp;<i>On the maternal allele, the ICR on the maternal allele is normally not methylated. CTCF binds to the ICR and blocks the enhancers from accessing the Igf2 promoter, leading to no Igf2 expression.</i><br></div>
    <div class="field-value" id="c22ac22fb94191e4">1</div>

    <div class="field-name" id="6f1135763f6df663">Describes how imprinting at the Igf2 locus is disrupted in Wilm’s tumour:&nbsp;<i>In Wilm’s tumour, the ICR on the maternal allele is methylated. Methylation of the maternal ICR prevents CTCF from binding, thereby allowing the enhancers to access the Igf2 promoter.</i><br></div>
    <div class="field-value" id="711cacfd4a3ee767">1</div>

    <div class="field-name" id="f240901da3fb2b03">Explains how disrupting imprinting at the Igf2 locus contributes to cancer:&nbsp;<i>As a result of disrupted imprinting at the Igf2 locus, the growth-promoting factor Igf2 is overexpressed and contributes to tumour growth.&nbsp;</i><br></div>
    <div class="field-value" id="89e0f252c3bb3ad0">2</div>

    <div class="field-name" id="3f01b7bd773c6b52">Identifies the class of epigenetic inhibitors that decitabine belongs to:&nbsp;<i>Decitabine is a DNA methyltransferase inhibitor (DNMTi).&nbsp;</i><br></div>
    <div class="field-value" id="4e9a30727ede1720">2</div>

    <div class="field-name" id="d20fb9a2ee256b4">Describes the impact of Decitabine on DNA methylation: <i>DNA methyltransferase inhibitors such as Decitabine trigger demethylation of DNA.&nbsp;</i><br></div>
    <div class="field-value" id="e03849bba7ae2e80">2</div>

    <div class="field-name" id="7b3c742dea5bde0">Describe how Decitabine can have an anti-tumour effect: <i>DNMTi such as Decitabine can trigger demethylation of CpG islands in tumor suppressor genes, leading to reactivation of genes that inhibit tumor growth.</i><br></div>
    <div class="field-value" id="57bb6708b3cdc32b">1</div>

    <div class="field-name" id="69bef9aa7171b08">Describes how altering DNA methylation can have enduring effects on the epigenome:<i>&nbsp;DNA methylation is mitotically heritable, so changing the DNA methylation pattern can have effects that last beyond mitosis.</i><br></div>
    <div class="field-value" id="9e3fc953bb9570e4">0</div>

    <div class="field-name" id="529f51fd4d70e088">Defines what is meant by a sensitive period:&nbsp;<i>Sensitive periods refer to particular periods when epigenetic marks are cleared and reset, i.e. reprogrammed.</i><br></div>
    <div class="field-value" id="48733751904ceade">0</div>

    <div class="field-name" id="3054b412bc5b110b">Identifies sensitive periods of development:&nbsp;<i>Sensitive periods include germ cell development and early embryonic development.&nbsp;</i><br></div>
    <div class="field-value" id="95f39fd3c9961ab2">2</div>

    <div class="field-name" id="792b4bc39bec7d37">Explains why treating patients during sensitive periods would be inadvisable: <i>It would be preferable to avoid treating patients during sensitive periods as epigenetic marks are being established and inhibiting DNA methylation could have more pronounced effects on the genome at these times.&nbsp;</i><br></div>
    <div class="field-value" id="2bca734361b5a8c">0</div>

    <div class="field-name" id="e576a4ddbe0a2940">Please make any overall comments here.</div>
    <div class="field-value">The comments and observations below can explain the rating and can also be used for improving the understanding of the material and writing skills.  
Question 1. I liked the concise answers given to the question.  However, more clarity was needed to complete the answer.  For instance, the effects of DNA hypermethylation in CPG islands needed to be separated from those caused by the hypomethylation in intergenic intervals and repeats.  In the answer, the impact of both was listed together, without specifying which change in DNA methylation was specifically responsible for tumor suppression happens and which – for genome instability.  In addition, the answer 1 would benefit from the use of terminology suggested during the course ("locus specific hypermethylation" or “genome wide hypomethylation” etc).  There was a little reduction (1) in the rating for this answer overall.
Question 2.  
There are some discrepancies from the lecture material in the responses to this question.  For instance, there is an incorrect reference to “genome wide hypermethylation in cancer “.  In fact, in cancer there is a genome-wide hypomethylation (including repeats and intergenic regions) and hypermethylation at CpG islands; in the meantime, ICRs can be hypermethylated or hypomethylated in cancer.  In Wilm's tumor, there is ICR hypermethylation, which blocks binding of CTCF and contributes to the expression of Igf2 from maternal allele.  While the answer correctly mentions hypermethylation of ICR in cancer, it is not because of genome wide hypermethylation.   

The hypermethylation of ICR causes an overexpression of igf2 that is expressed from both alleles in cancer – this doubles igf2 expression and promotes growth but doesn’t necessarily imply "twice as much growth".

The answer doesn't specify that the ICR on maternal allele is unmethylated under normal conditions, when imprinting is maintained, and doesn’t mention that ICR’s unmethylated status makes the binding of CTCF and further insulation of igf2 from enhancers possible.  In cancer, methylation of ICR blocks CTCF binding and, therefore, it is not only ICR hypermethylation but also linked with it absence of insulation that changes the expression of igf2 from maternal allele.  

Besides that, the enhancers are not located upstream to igf2 but rather downstream.  Finally, I am not sure lecture materials didn't mention "binding" of enhancers to igf2; rather, "accessing", "enhancing", "acting upon" was mentioned., so I am not sure whether “binding” would be a right word to use here.

Question 3. It is DNMT inhibitor, and not “inhibitor”.  The misspelling didn’t affect the grade.  The rest of grading is self explanatory when comparing with the provided answer.
 Question 4.  Compare the answers with the ones provided  - the difference in answers is self explanatory.  You should also mention "early embryonic development", rather than early development, for clarity (no changes in grading were made for this).

I hope that these comments are useful and explain the approach to grading!</div>
  </body>
</html>