<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1123 from Anon (session_user_id: 5759d2a78e4761f89d8dd8aca602cd2319bb66d6)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1123 from Anon (session_user_id: 5759d2a78e4761f89d8dd8aca602cd2319bb66d6)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p><span>Small regions of DNA are very rich in linked cytosines and guanines, but are unmethylated. These regions consist of  500 to 5,000 base pairs that are unmethylated DNA and are referred to as CpG islands or CGI. These "islands" seen in mostly promotor</span><span> regions of genes and </span><span>are located at the 5′ end of the DNA. About 50 percent of all genes contain a CpG island in their promoter regions. The lack of methylation in CpG islands leads to a less compact chromatin</span><span> structure, and generally allows for active gene expression. The methylation of unmethylated CpG islands leads to the silencing of genes required for proper cell growth control and is a common mechanism in the development of many types of cancer. The</span><span style="font-size:14px;line-height:21px;"> examples of hypermethylation and the resultunt cancers are retinoblastoma (RB), Colorectal cancer (MLH1), breast cancer (BRCA1), and in gliomas and colorectal tumors (MGMT)</span><span style="font-size:14px;line-height:21px;">. </span><span style="font-size:14px;line-height:21px;">DNA methylation such as </span><span style="font-size:14px;line-height:21px;"> locus specific DNA hypermethylation (</span><span style="font-size:14px;line-height:21px;"> CpG islands and CpG island shores of tumour</span><span style="font-size:14px;line-height:21px;">suppressor genes), </span><span style="font-size:14px;line-height:21px;"> ICRs  (loss of imprinting, </span><span style="font-size:14px;line-height:21px;">Genome-wide DNA hypomethylation), r</span><span style="font-size:14px;line-height:21px;">epetitive regions, </span><span style="font-size:14px;line-height:21px;">CpG poor promoters, and </span><span style="font-size:14px;line-height:21px;"> ICRs with loss of imprinting are abberations are the leading causes contributing to disease and thus </span><span style="font-size:14px;line-height:21px;">DNA methylation plays an alternative role to genetic </span><span style="font-size:14px;line-height:21px;">mutation, to silence tumour suppressor genes causing </span><span style="font-size:14px;line-height:21px;">cancer or several associated diseases. </span><span style="font-size:14px;line-height:21px;">In a normal cell the intergenic and the repetitive elements in the chromosomes are generally methylated where as the CpG islands are hypomethylated. Conversely, it the intergenic regions are hypomethylated, hypomethylated in the repetitive elements and in the intron regions as well. This occurs in all tumours (example the tumorigenicity due to CpG hypermethylation) to some degree and depending on the hypomethylated location in genome and causes genomic instability. In addition situations wehre CpG poor promoters are hypomethylated to activate oncogene and all these direct a normal cell toward triggering cancer.<br /><a href="https://coursera-uploads.s3.amazonaws.com/user-fac1e9ac84c6fa163a48dcb8/970238/asst-5/970238-52102b5954b372.54660151.pdf">DNA Methylation and Cancer-work paper.pdf</a><br /></span></p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>Mechanism of action of the Imprint Control Region (ICR)  varies with each cluster. For example  Kcnq1 cluster in the human chromosome 11 uses long noncoding RNA and  H19/Igf2 cluster, human chromosome 11 through enhancer blocking. <span style="font-size:14px;line-height:21px;">The Igf2- (insulin-like growth factor 2) and H19-genes are imprinted and they are expressed unevenly from the two parental alleles. The Igf2 is a growth factor expressed in normal tissues solely from the paternal allele. The H19-gene is transcribed (but not translated to a protein) from the maternal allele. The Igf2-protein is a growth factor particularly important during pregnancy, where its growth inducing effect via the placenta affect its embryo-carrier. </span><span style="font-size:14px;line-height:21px;">Imprinted disorder, results from various abnormalities in linked Kcnq1 and </span><span style="font-size:14px;line-height:21px;">H19/Igf2 clusters at 11q15.5 (1Mb). </span><span style="font-size:14px;line-height:21px;">Loss of Cdkn1c (tumour suppressor, growth suppressing) which u</span><span style="font-size:14px;line-height:21px;">pregulates the Igf2 (oncogene, growth promoting). The abberation is about 1 Mb is size and results in the f</span><span style="font-size:14px;line-height:21px;">etal and post-natal overgrowth (loss of Cdkn1c, overdose Igf2 and/ or </span><span style="font-size:14px;line-height:21px;">other genes), m</span><span style="font-size:14px;line-height:21px;">acroglossia (large toungue), and p</span><span style="font-size:14px;line-height:21px;">redisposition to embryonic/ childhood tumours, but not adult tumours such as the </span><span style="font-size:14px;line-height:21px;">Wilm’s tumour  in the kidney. The disease is caused by </span><span style="font-size:14px;line-height:21px;">maternal allele behaving like paternal allele because of the imprint loss due to mutation, uniperantal disomy or due to epigenetic disruption. </span></p>
<p> </p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p><span>Decitabine is a DNA methyl transferase inhibitor (DNMTi) which can be used to demethylate hypermethylated tumor suppressor genes which are inappropriately silenced in cancer patients. Chemically, it is a cytindine analog and cause DNA methyltransferase I depletion and  leading to DNA hypomethylation in the abberent cells  and enhanced gene expression and thus arresting the tumor growth. Furthermore, the role of hypomethylating agents in epigenetic priming seems to be another interesting approach. Decitabine hypomethylating effect is limited to only the tumor cells and not the healthy cells. It has been found that the side effects appear to be apoptosis and more importantly the Decitabine mainly caused an increase of cells in G2/M unlike the other common analog Azacytidine. Many patients still do not respond to hypomethylating agents such as this oneand it will be of enormous interest to identify molecular predictors that respond to such drugs. May be the biomarker profiles are needed before treatment in these patients  (please see the answer below)</span></p>
<p><span><a href="https://coursera-uploads.s3.amazonaws.com/user-fac1e9ac84c6fa163a48dcb8/970238/asst-5/970238-5210732910c6c7.74165687.pdf">Decitabine.pdf</a><br /><span>  </span><br /></span></p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p><span>The  understanding that some cancers are caused by epigenetic change: tiny chemical tags that accumulate over time and can turn genes on or off has takend the field away from the assumption <span>that cancer was caused by mutations in the DNA. </span></span>Instead  we find some cancers are caused by epigenetic changes; tiny chemical tags that accumulate over time and can turn genes on or off. Unlike genetic damage, epigenetic changes can sometimes be reversed with treatments that are less toxic than conventional chemotherapy. However, the use of these drugs would during the sensitive period where alterations or removal of the normal process result in epigenetic abnormalities and diseases. These periods during the germ cell development and the preimplantation period of embryo development are called "sensitive period". In addition  IVF/ ICSI: germ cell harvest, in vitro culture of early embryo, and reprogramming for iPS generation are also sensitive periods. We can envision different pathways that might cause epigenetic alterations in cancer cells as observed by genome-wide epigenomic profiling. Clinicians now see the ability to scan patient tumors using Next Generation Sequence  for thousands of mutations instead of one or two "biomarkers". Biomarkers are as impoartant as drugs in taming the tumor genetic revolution that depends upon the understanding of the right biomarkers before attempting to treat them</p>
<p><span><span> </span></span></p></div>
  </body>
</html>