<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 927 from Anon (session_user_id: 401d60d95916a093d3b3922c9bdcdfa485098414)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 927 from Anon (session_user_id: 401d60d95916a093d3b3922c9bdcdfa485098414)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value">In general, the normal function of DNA methylation at CpG islands is to silence or significantly reduce associated gene expression. In cancer several CpG island sites are often found to be <em>hyper</em>methylated, leading to inactivation of tumour suppressor genes. This can result in genomic instability and failure to repair DNA damage, contributing to cancer progression. CpG island <em>hyper</em>methylation can also spread to nearby CpG island shores, reinforcing such silencing further.<br /><br />DNA methylation at intergenic regions and repetitive elements is normally present to prevent incorrect recombination during DNA replication, and to stop transposable elements becoming haphazardly activated by silencing their strong promoters. In cancer, DNA <em>hypo</em>methylation at repetitive regions can result in genomic instability through improper recombination and chromosomal translocations. DNA <em>hypo</em>methylation of transposon promoters can result in these elements 'jumping' into random parts of the genome, disrupting normal gene expression, and causing genome-wide instability.<br /></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value">At the H19/Igf2 cluster the CTCF insulator protein binding site is methylated on the paternal allele, this prevents the binding of CTCF, which normally acts by blocking downstream enhances from acting upon Igf2; thus the enhancers are free to increase the expression of Igf2 from the paternal allele. DNA methylation from the CTCF binding site also spreads downstream to cover the H19 lncRNA encoding gene, preventing its expression.<br /><br />On the maternal allele there is no DNA methylation. Therefore, CTCF attaches to its binding site freely and Igf2 is <em>prevented</em> from being expressed. There is, therefore, also no spread of methylation to the H19 gene. Thus, H19 lncRNA is freely expressed from the maternal allele.<br /><br />In Wilm's tumour, the maternal allele of this cluster is defective and expresses Igf2, behaving like the paternal allele. This can be due to mutation (e.g. in the CTCF binding site, preventing Igf2 blocking), or through inappropriate methylation of the maternal allele. As a strong growth factor required in development, the over-expression of Igf2 can lead to overgrowth and rapid cell proliferation. This is why cancers which are the result of such deregulated imprinting at the H19/Igf2 cluster often occur early in childhood.<br /><br /><br /><br /></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value">Decitabine is a DNA methyltransferase (DNMT) inhibitor. Decitabine is a cytidine nucleotide analogue which is incorperated into DNA during DNA replication. It causes DNA<em> hypo</em>methylation by preventing DNMTs from binding. Decitabine can also be incorporated into RNA during transcription.<br /><br />By inhibiting DNMTs, at the right dosage Decitabine has therapeutic effects against some cancer types such as acute myeloid leukemia (AML) and myelodysplastic syndromes (MDS), where systemic DNA <em>hyper</em>methylation is a serious contributor to tumour progression.<br /><br /><br /><br /></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value">Targeted DNA methylation normally plays a key role in gene regulation, cell identity, genomic stability, and silencing of transposable elements. It is also important in development - e.g. in maintenance of parental imprinting. If DNA methylation is increased or reduced stochastically (at random) through the application of drugs such as DNMT inhibitors, off-target effects may cause unpredictable gene deregulation which could have long term consequences. It may, inadvertently, for example, silence a gene vital for DNA repair; which would increase the chance of subsequent cancers.<br /><br />The most sensitive periods of development would be during the genesis of germ cells, or during embryonic development where mass reprogramming is active. During this time any disruption could result in epigenetically derived developmental disorders such as <span>Angelman and Prader-Willi syndromes.</span> However, as DNA methylation is mitotically heritable through DNMT1 activity, any period of rapid growth could be severely affected. It would be a very bad idea to give DNA methylation targeting drugs to pregnant women, or young people who are still developing. <br /><br /></div>
  </body>
</html>