<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 549 from Anon (session_user_id: c219b3ead48ce6b20cb824c50e2dc372ac191716)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 549 from Anon (session_user_id: c219b3ead48ce6b20cb824c50e2dc372ac191716)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>Briefly, the genomic DNA in normal cell consists regions of CpG island, gene, intron, integenic, and repetitive element respectively. In the normal cell, CpG island is usually non-methlyated, for instance, it can then express the tumour suppressor genes which invovled in cell cycle regulation, DNA repair or others events. While the rest of the regions are methlyated to facilitate chromosomal stability.  </p>
<p>In contrast to the genomes of cancer cells, it has shown to be hypomethlyated overall when compared to normal cells, with the exception of hypermethylated at CpG island, which may lead to inappropriate gene silencing particularly, in the promoter regions of tumour suppressor genes. On the other hand, the other regions (ntron, integenic, and repetitive element) are hypomethylated which may contribute to chromosomal instability (a hallmark of cancer) resulting damage such as deletions, amplification or translocations. </p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine is <span>a modified form of a methylation-blocking drug</span> by which it blocks methyl-transferase to reduce overmethylation activity resulting the effect of hypomethylation of DNA in cancer cells.</p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Epigenetic treatment in cancer may have enduring effects because these epigenetic changes can passed on during cell division to daughter and granddaughter cells until they are actively erased. </p>
<p>Epigenetic treatment in cancer patients should be avoided and would be inadvisable during sensitive period of embryogenesis. As this is a period whereby the genome undergoes reprogramming (clearing and resetting of epigenetic marks) in the early development and primordial germ development stage involving global changes in DNA methylation.  </p>
<p> </p>
<p> </p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>The action of imprinting depends on imprint control region (ICR). Using the example of the H19/Igf2 cluster, in normal cell, imprinting is controlled by enhancer blocking. CTCF binds to unmethylated maternal allele of ICR in the igf2/H19 cluster forming a long-range intrachromosomal loop and block the interaction of downstream enhancers with remote Igf2 promoters. On the contrary, ICR of paternal allele is methylated and blocks the binding of CTCF. Without the CTCF, DNA methylation could act on H19 promoter for silencing, and furthermore, enhancers can access Igf2 to activate.  </p>
<p>While in cancer, elevated level of Igf2 is observed. This is because the loss of imprinting in maternal allele leads to biallelic transcription of Igf2. </p></div>
  </body>
</html>