<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1318 from Anon (session_user_id: 5f5cf4c7b84cdb085dc144182356558325c8c1d0)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1318 from Anon (session_user_id: 5f5cf4c7b84cdb085dc144182356558325c8c1d0)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>DNA methylation is considered to be an important epigenetic modification. Methylation mostly occurs at the C-5 position of the cytosine in CpG dinucleotides and plays important roles in gene activity, genomic imprinting, transposon silencing, aging, embryonic development, differentiation and carcinogenesis. CpG islands are short interspersed DNA sequences usually found in the promoter region of genes that are being rich in GC and CpG regions and are predominantly non-methylated in normal cells. They are sites of transcription initiation attracting proteins that create a transcriptionally permissive chromatin state. In cancer, these CpG islands are often hypermethylated at their C-5 position disrupting their normal function and causing silencing of the underlying genes. The genes that are often hypermethylated at the CpG islands in their promoter regions are tumor suppressor genes. Therefore, down-regulation of tumor suppressor genes could contribute to the development and establishment of cancer.</p>
<p>DNA methylation in the intergenic regions and repetitive elements of the genome is usually methylated and the function of the methylation in these instances is mainly to maintain chromosome structure and genomic integrity. In the case of intergenic regions, DNA methylation appears to promote genomic stability and silence cryptic transcription start sites. In the case of REs, DNA methylation appears to maintain genomic integrity by a) silencing of repeats to prevent transposition, b) inserting mutation of the repeats (meC to T) to prevent transposition, c) silencing of repeats to avoid transcriptional interference from strong promoters, and d) preventing illegitimate recombinations. Hypomethylation of REs in cancers is a common finding that can be regarded as a biomarker of various cancers. Cancer is also associated with genome-wide hypomethylation at intergenic regions/repeats and as a consequence genomic instability is quite often observed (chromosomal deletions, reciprocal translocations, insertions).</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>The IGF2 gene encodes the insulin-like growth factor that promotes cell proliferation. Normally, the IGF2 gene undergoes genomic imprinting and only the copy inherited from a person's father is active. In some cancers, both the paternal and the maternal copies of the gene are active, increasing the amount of IGF2 in the cells (loss of imprinting).  IGF2, together with H19, (also involved in growth and development), belong to a cluster of genes that undergo genomic imprinting. The imprinting control region (ICR) that lies between the two genes controls their monoallelic parent-of-origin-specific expression. The CTCF4, an insulator element, binds to the ICR of Igf2 and H19 genes and regulates their expression.  The ICR in the maternal allele is unmethylated and promotes binding of CTCF to the ICR. This binding serves as a chromatin boundary that prevents distant enhancers from activating Igf2 enabling only the expression of H19 gene. The paternal allele is methylated at the ICR. Paternal ICR methylation prevents CTCF binding and enables the enhancers to allow the expression of IGF2 gene. Loss of imprinting of the IGF2 gene has been identified in several types of cancer, including Wilms tumor, a rare kidney cancer form.</p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine (5-aza-2'-deoxycytidine) belongs to the class of DNA methyltransferase (DNMT) inhibitors. The details of its mechanism of action remain unclear. It is a nucleoside (cytosine) analog that in low doses causes DNA hypomethylation by inhibiting DNMT1 after being incorporated into the DNA strands where it binds irreversibly with DNMT1 resulting in rapid loss of its activity. Because decitabine acts as an anti-metabolic agent and gets incorporated into the DNA structure, its function is therefore replication dependent. This means that cancer cells which have more rapid growth rates than most other cells in the body will be more severely affected. At high doses, decitabine exerts toxic, non specific effects, similar to other DNMT inhibitors. Decitabine is used to treat myelodysplastic syndromes that have progressed to acute myelogenous leukaemia. Its efficacy is currently tested towards a large number of malignances in clinical trials.</p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Cancer is a complex disease considered to be the result of summation of both genetic and epigenetic alterations. Unlike genetic mutations, epigenetic alterations associated with cancer phenotype can be reversed and corrected. They may therefore function as potential therapeutic targets; erasing these epigenetic marks may specifically target cancer cells. Epigenetic drugs have the potential to act upon the epigenome and alter specific patterns of DNA methylation and histone marks associated with the disease. Because epigenetic alterations are stable and heritable changes in gene expression they can actually have long lasting effects. Drug sensitivity/resistance to chemotherapy is influenced by alterations in gene expression also governed by epigenetically-mediated events. In this case, epigenetic therapy could erase the epigenetic marks associated with drug resistant phenotype and therefore sensitize tumors to chemotherapy. However, caution should be taken in avoiding treating patients during the “sensitive periods" of pre-implantation (early development) and primordial development as during these periods the epigenetic marks are erased and reset correctly. Exposure to epigenetic modifiers during early development could alter genome activity associated with the differentiation-linked programming of cells/organ systems, whereas during premordial development could reprogram the germ line promoting epigenetic transgenerational inheritance of altered phenotypes. </p></div>
  </body>
</html>