<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 378 from Anon (session_user_id: 369263ce2650968b25b47f087f0bd30f0b0133b8)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 378 from Anon (session_user_id: 369263ce2650968b25b47f087f0bd30f0b0133b8)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value">The normal function of DNA methylation at CpG islands is silencing or down regulate gene expression (not always silencing/downregulating but much of the time).  Most importantly,  gene expression varies with each cluster.  Normally, the island itself is unmethylated and the intergenetic regions and the repeats are methylated.  There are marks laid down i.e DNMT1 which recognize methylation and that recruits PRC2 which is a repressor.  These processes result in a stable genome.  Factors are attracted to make the chromatin tight further increasing stability.<br /><br />In cancer there is hypomethylation at the intergenetic regions and repeats.  The island itself becomes methylated and the intergenetic regions and the repeats become unmethylated.  This is because the marks, DNMT1 which recognizes methylation (and hemi-methylation) is faulty and the repressive marks are not in place resulting in genomic instability especially regarding the repeative elements.  These repetitive elements are able to jump from one gene to another so that when chromosomes line up, not all of them will line up correctly because there is a repetitive element in there that shouldn't be. There could also be a deletion of a section of a chromosome.  Because the epigenetic changes are mitotically heritable, these abnormalities perpetuate contributing to the possibility of developing cancer.  <br /><br /></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value">There is a paternal and maternal allele for  Igf2-H19 imprinted cluster of genes. Both alleles have the same structure, each has the following in same order:<br />an Igf2 gene, followed by==&gt; imprint control region,  ==&gt;  H19 gene, ==&gt; enhancers.  <br /><br /> Igf2 promotes growth, and H19 is  gene for lncRNA.  H19 is only transcribed from the maternally inherited allele.  The paternal is normally methylated and silent.  The maternal is hypomethylated or unmethylated and is expressed. There is inverse relationship between Igf2 and H19.  As H19 expression increases, Igf2 expression decreases.  Also, CTCF (an important insulator protein), binds with the unmethylated ICR on maternal allele, allowing enhancers to further express H19 =  Igf2 expression decreases on the maternal allele.  <br />CTCF does not bind to methylated ICR region on the paternal allele and, the enhncers increase expression of Igf2 from  paternal. <br /><br />In Wilms tumor, Igf2 is expressed from both alleles maybe because inactivation of the H19 gene (maybe because of a defiency of the CTCF). This is also the way disrupting the imprinting can contribute to cancer.  If H19 became over expressed this could also contribute to cancer development.  Loss of imprinting, i.e. loss of methylation at the paternal allele contributes to cancer development. <br /><br /><br /><br /><br /><br /></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value">Decitabine is a Dnmt1 (DNA methyltransferase 1) inhibitor.  Normally, Dnmt1 adds a methyl group to hemimethylated CpG dinucleotides (and maybe unmethylated as well).  The decitabine inhibites the binding of the methol group.  The drug is incorporaed into the DNA as it replicates. so it is genone wide.  In cancer, tumor supressor genes are considered to be hypermethylated and therefore not able to be expressed (need both alleles of a tumor supressor gene to be active).  The drug removes the methylation and allows the expression of the tumor supressor genes.  Tumor supressor genes make cells die and with activation the cancer cells may die.<br /></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value">DNA methylation is an epigenetic mark,  mitotically heritable and is therefore passed onto all daughter cells with each replication.  <br />Sensitive period is when there is a clearing and resetting of epigenetic marks.  <br />There are 2 sensitive periods during development, in embryonic stem cell development (shortly after fertilization), and   in the primordial germ cell development (which go onto become germ cells).  <br />At this stage of our knowledge, treatment during sensitive periods seems inadvisable since there are so many variables that are unknown.  The amount of harm could be quite rapid and irreversible since many of the drugs are genome wide.  <br /></div>
  </body>
</html>