<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 88 from Anon (session_user_id: 3a0ce3064760b610efda6852d55d5be80b889417)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 88 from Anon (session_user_id: 3a0ce3064760b610efda6852d55d5be80b889417)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p><span>CpG islands (regions of a gene rich in CpG dinucleotides)
are found in most promoter regions where transcription factors bind.  These regions are free of methylation in
normal cells, thereby enabling transcription.  But methylation of  CpG islands, catalysed by DNA
methyltransferase, also characterises some normal processes, for example silencing
of the X chromosome that occurs late in the process of X inactivation and determines
mitotic heritability. </span></p><p></p><span></span><p>

</p><p><span>In cancer
cells, CpG islands in promoters of tumour suppressor genes and also CpG island
shores around these genes, become hypermethylated.  This results in silencing of the underlying
genes.  Development of the cancer is
characterised by progressive methylation of these sites. </span></p><p></p><span></span><p>

</p><p>Hypermethylation of CpG islands occurs frequently in cancer.
Epigenetic silencing of tumour suppressor genes gives the cancer cell the
advantage of dividing more rapidly than healthy cells in the tissue, enabling
the cancer to spread. Methylation is mitotically heritable so the disease will
be spread through cells arising from cell division. </p><p>

</p><p><span>CpGs at intergenic regions and repetitive elements are normally
methylated.  Broadly the function of
methylation in these areas is to maintain genomic stability. Methylation is
believed to prevent transposable elements jumping into inappropriate regions on
the genome and protect against inappropriate recombinations of chromosomes.</span></p><p></p><span></span><p><b>

</b>

</p><p><span>In cancer, repetitive elements and intergenic regions tend
to become hypomethylated early in the development of a cancer and progressively
with time.  Hypomethylation occurs
predominately in repetitive elements.  In
normal cells methylation of repeats facilitates dense packing of hyperchromatin.
Hypomethylation converts the chromatin to euchromatin an open structure.</span></p><p><span><span>Hypomethylation
and euchromatisation leaves the 
chromosome vulnerable to 
inappropriate recombinations between repeats and general genomic
instability.  Hypomethylation results in both
activation of the promoters of  repetitive elements, permitting them to copy
themselves, and also jumping of repeats within the genome and disruption to
adjacent coding regions of the gene</span></span></p><p>

</p><p></p><p></p><p>

</p><p></p><p>

</p><p></p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>The imprint control region of the paternal allele is normally
methylated. Methylation prevents binding of the insulator protein, CTCF, to the
imprint region. Downstream enhancers then act on Igf2 upstream of the imprint
region enhancing Igf2 expression. This process is mediated by methylation
spreading from the imprint centre to H19, a CpG island promoter that is thereby
silenced.</p><p></p><span></span><p>

</p><p><span>On the maternal allele the imprint control region is normally
unmethylated and Igf2 is not expressed.  Without
methylation, CTCF binds to the imprint control region insulating Igf2 from
downstream enhancers, blocking Igf2 expression. 
In this situation, the enhancers act instead on H19 increasing its
expression.</span></p><p></p><span></span><p>

</p><p><span>Igf2 is an oncogene, its expression promotes growth.  In Wilms tumour abnormal methylation of  the maternal allele at the Igf2 imprint
region means that mono-allelic gene expression is compromised; parent-of-origin
expression is lost.   Igf2 is produced
from the maternal allele in addition to the paternal allele. Aberrant
methylation seems to occur early in tumour development.</span></p><p></p><span></span><p>

</p><p><span>Methylation of the imprinting regions of both parental<b> </b>alleles makes it likely that double
the normal amount of growth factor will be produced as Igf2 is expressed from
both parental genes.  Cell division will
occur more rapidly encouraging spread of the cancer.</span></p><p>

</p><p></p><p>

</p><p></p><p>

</p><p></p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p><span>Decitabine is a DNA methyltransferase inhibitor used in the
treatment of megalodysplastic syndrome. 
It is effectively a hypomethylation agents acting throughout the genome.</span></p><p></p><span></span><p>

</p><p><span>DNA methyltransferase enzymes methylate cytosine residues in
the new strand of DNA formed during genome replication.  This ensures perpetuation of methyl marks
laid down during development so that the daughter cell DNA  is a true copy of the DNA in the original cell.  Decitabine is a nucleoside analogue that is
incorporated into DNA during, and only during, replication of the genome, when
it will block methylation of that site. </span></p><p></p><span></span><p>

</p><p><span>When DNA methyltransferase encounters decitabine
incorporated into the newly-synthesised daughter strand of DNA it becomes
irreversibly bound and unable to copy the methylation mark to the daughter
strand.  This prevents the genome
replicating and producing cancerous cells. 
In the case Decitabine this treatment is presumably directed at bone
marrow cells.</span></p><p>

</p><p></p><p>

</p><p></p><p></p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p><span>DNA
methylation is genetically heritable, being maintained through mitosis by the action
of DNA methyltransferase (DNMT1). This enzyme recognise hemi-methylation in DNA
as it is replicating and copies the methylation mark from the original strand
of DNA to the daughter strand.  This
process is repeated at every cell division.</span></p><p></p><span></span><p>

</p><p>Sensitive periods are those
periods when changes in the environment have the greatest effects on the epigenetic
features of the genome. During sensitive periods changes in the environment can
affect the reprogramming processes, altering the epigenetic makeup of the
organism.</p><p>

</p><p><span>Primordial germ cell development
and early embryonic development, the periods of active remodelling of the
genome when removal and laying down of epigenetic marks occurs, are the most
sensitive periods.  For these purposes
germ cell development encompasses the period up to maturation of the eggs and
sperm. Early embryonic development includes pre-implantation to early post-implantation.
</span></p><p></p><span></span><p>

</p><p>Drug treatments during the
sensitive periods of development could interfere with removal and laying down
of epigenetic marks, indeed most current cancer interventions are directed at
enzymes involved in epigenetic control. In contrast, during the somatic
maintenance period, for most cell types, epigenetic marks are established, so
drug treatments are less likely to affect the genome. </p><p>

</p><p></p><p>

</p><p>

</p><p></p><p></p></div>
  </body>
</html>