<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 332 from Anon (session_user_id: 1f3b4dd013a6f79bc8c91c2eeb5d290668cbad04)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 332 from Anon (session_user_id: 1f3b4dd013a6f79bc8c91c2eeb5d290668cbad04)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p><span lang="en-us" xml:lang="en-us">Disruption of normal genetic development and cell growth program in cells shifts their phenotype to neoplastic cells, that eventually leads to a whole system disease called “cancer”. Recent advances in understanding the role of epigenetics in normal cell fate gave insight into how the patterns of DNA organization into chromatin and its regulatory mechanisms are disrupted in cancerous cells. </span><span lang="en-us" xml:lang="en-us">Methylation of DNA is one of the regulatory mechanisms and in mammals occurs almost exclusively in short (200 bp) genomic regions rich in Cytosine and Guanine linked by a phosphodiester bond, the so-called “CpG islands” or “CGIs”. </span><span lang="en-us" xml:lang="en-us">Although DNA methylation of promoter regions are negatively correlated with gene expression, CpG islands are kept free of methylation </span><span lang="en-us" xml:lang="en-us">independently</span><span lang="en-us" xml:lang="en-us"> of their activity status. </span><span lang="en-us" xml:lang="en-us">In cancerous cells has been observed an hypermethylation of CGIs, usually at the promoters of tumor suppressor genes: the overall effect is the silencing of onco-suppressors, a survival advantage of these fast-replicating cells and their positive clonal selection. The disruption of normal methylation pattern also spread to intergenic regions and repetitive elements. In normal cells, methylation</span><span lang="en-us" xml:lang="en-us">-driven silencing</span><span lang="en-us" xml:lang="en-us">at these sites is crucial for genomic stability, since it prevents transpositions, mutations, transcriptional interferences from strong promoters and illegitimate recombinations, </span><span lang="en-us" xml:lang="en-us">according to the proposed “</span><span lang="en-us" xml:lang="en-us">G</span><span lang="en-us" xml:lang="en-us">enome </span><span lang="en-us" xml:lang="en-us">D</span><span lang="en-us" xml:lang="en-us">efense </span><span lang="en-us" xml:lang="en-us">M</span><span lang="en-us" xml:lang="en-us">odel” by prof. </span><span lang="en-us" xml:lang="en-us">T</span><span lang="en-us" xml:lang="en-us">i</span><span lang="en-us" xml:lang="en-us">mothy </span><span lang="en-us" xml:lang="en-us">Bestor. </span><span lang="en-us" xml:lang="en-us">The hypomethylation at these sites, that is responsible for the “genome-wide hypomethylation” found in cancer since they represent the largest proportion of total DNA, is an early event in tumorigenesis, progresses with the acquisition of the  neoplastic phenotype and occurs in some extent in all tumors. The consequent genomic instability, also confirmed by the tendency of tissues lacking de-novo methyltransferase-1 (DNMT1) to develop specific cancer, shows how cancer should be considered a genetic and epigenetic disease.</span></p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>The DNA methylation is a mitotically hereditable process and drugs that remove this epigenetic mark could affect the epigenome of the daughter cells. This is especially true in young patients and in pregnants were the gametes and the embryo are more susceptible during “sensitive” periods. These periods are characterized by an extensive reprogramming were epigenetic marks are erased and laid down, such as in the pre- and early post-implantation period and during primordial germ cells development as well. Also, although during somatic maintainace the organism as a whole is less sensitive, the development and maturation of tissues and organs is an ongoing process that spans over many years and specific organs can show peculiar sensitive periods. These are the reasons why treating pregnants and young people in general with drugs that interfere with the epigenetic machinery could lead to the disruption of mitotically hereditable epigenetic marks, bearing consequences beyond our actual knowledge.</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>Growth and development of the embryo is directed by tissue-specific and/or temporal specific pattern of monoallelic parent-of-origin expression of a subset of genes, the “genomic imprinting”. Differential marking occurs during gametogenesis, mostly by DNA methylation. The “insulator model” of imprinting is employed by the H19/Igf2 imprinted locus.  The maternally expressed H19 gene and paternally expressed Igf2 share enhancers and the reciprocal imprinting is governed by CTCF-dependent insulator. If the maternal imprinted control region (ICR) is unmethylated, the CTCF binds to it and “insulate” the downstream Igf2 gene and transcription of H19 gene occurs; in the paternally imprinted allele the ICR is methylated, CTCF can't binds to ICR and enhancers can act on Igf2, leading to its expression. The loss of imprinting can lead the maternal chromosome to reverse to a paternal epigenotype, i.e. ICR methylation, Igf2 turned on and H19 turned off; furthermore a deletion of the maternal allele can lead to a duplication of the paternal allele (uniparental disomy). A doubled production of IGF2 occurs and, since it is an embryonal growth factor, it predisposes to organs overgrowth, macroglossia, hemi-hypertrophy, adrenal cytomegaly and Wilms' tumor.</p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine is an FDA-approved De Novo Methyl-Transferase inhibitor (DNMTi) and is used to treat hematological malignancies such as Myelodysplastic Syndrome (MDS) that progressed to acute myeloyd leukemia (AML). These malignancies are often provoked by CpG island hypermethylation at promoters of oncosuppressors genes. Decitabine acts as nucleoside analogue that irreversibly binds to DNMTs, that can't no longer access DNA for methylation. While developed in the 1970s, it was soon discontinued due to its non-specificity and side effects; nowadays is know that the DNMTi effect is characteristic of low dosage the drug that patients tolerate well. Unfortunately, the lacking of selectivity leads to inhibit the enzyme in both cancerous and normal cells. Further assessment of DNMT inhibition in the long term is required, especially in youger patients, since these drugs are interfering in a key process that is mitotically hereditable.</p></div>
  </body>
</html>