<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 98 from Anon (session_user_id: 5dedbd6a27d5eb149b58130a21e94dba6f9dfd45)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 98 from Anon (session_user_id: 5dedbd6a27d5eb149b58130a21e94dba6f9dfd45)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value">Locus specific DNA hpermethylation is seen is cancer. Tumor suppressor hypermethylation is more frequent than mutations.<br /><ol><li>Normally the CpG islands are unmethylated. Methylation in general is not seen at CpG islands in a normal cell. As methylation leads to silencing of genes.</li><li>In cancer cells, the CpG islands are more likely to be methylated, than a normal cell. </li><li>They are especially hypermethylated at promoters of the tumor suppressor genes, so that the cancer cells are able to inactivate those genes. Epigenetically, the cancer cells locks them in silent inactive state.</li></ol>Genome wide hypomethylation is seen in cancer. Consequence of hypomethylation depends on where it occurs, usually at repeats and CpG poor promoters, especially at oncogenes.<br /><ol><li>Normally, intergenic regions and repetitive elements are methylated. It is thought to provide genomic stability/integrity.</li><li>In cancer cells, the intergenic regions and repetitive elements are more likely to be hypomethylated, than a normal cell.</li><li>Hypomethylation at the intergenic regions and repetitive elements, causes genomic instability, due to illegitimate recombination between repeats, as they are not densely packed into the heterochromatin. Transposing events occurs more frequently. Also could result in transcriptional aberration in surrounding regions where they transpose to. Could lead to activation of cryptic promoters and also disruption of neighboring genes.</li><li>Hypomethylation of CpG poor promoters are less frequent. Results in activation of oncogenes, like R-RAS in gastric cancer.</li></ol><br /><br /></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value">H19 is a gene coding for lncRNA. H19 is only transcribed<span> from the maternally inherited allele</span>; the paternal H19 allele is not expressed (paternal imprinting). CTCF, an insulator protein, prevents Igf2 geneexpression from downstream enhancers (enhancer blocking). Binding of CTCF is blocked by methylation of the <b><i>Imprinting Control Regions(ICR). </i></b>Without CTCF, the methylation spreads from ICR to the CpG islands in the H19 promoter and silences the transcription of H19 gene, as a consequence, the downstream promoters can now access the Igf2 gene and activate it.<br /><ul><li>In paternal allele, the ICR is methylated, so no CTCF binding, H19 promoters are methylated, and H19 expression is silenced and Igf2 is activated. H19 is paternally imprinted.</li><li>In maternal allele, the ICR is unmethylated, so there's CTCF binding, H19 promoters are unmethylated, and H19 expression is activated and Igf2 is silenced. Enhancement of maternal H19 expression. Igf2 is maternally imprinted by enhancer blocking.<br /></li><li>Loss of imprinting of the Igf2 gene is the most common genetic or epigenetic alteration in Wilms tumor in kidney; it involves aberrant activation of the normally repressed maternally inherited allele.<br /></li><li>Hence, loss of imprinting, overexpression of growth promoting genes(Igf2) and loss of tumor suppressor gene (H19) leads to tumor growth.</li></ul></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><ul><li>Decitabine, sold as 'Dacogen' by Eisai, is a <b><i>DNA demethylating agent</i></b>, which is used to treat myelodysplastic syndrome, the precursors of acute myelogenous leukemia. They are <b><i>DNA methyltransferase inhibitors</i></b>.<br /></li><li>Decitabine(2'-deoxy-5-azacytidine) is a hypomethylating agent. It <b><i>hypomethylates DNA by inhibiting DNA methyltransferases</i></b>. </li><li>Decitabines are cytidine nucleoside analogs that become incorporated into newly synthesized DNA where they bind DNA methyltransferases (DNMTs) in an irreversible, covalent manner. The sequestration of DNMTs prevents maintenance of the methylation state of DNA, leading to DNA hypomethylation. </li><li>As a consequence, genes previously silenced by DNA hypermethylation (eg: tumor suppressor genes) can be re-expressed upon treating cancer cell lines with these DNMT inhibitors.</li><li>At higher doses, decitabine inhibits cell proliferation through nonreversible covalent linking with DNMTs and blocking of DNA synthesis. At lower doses, decitabine induces hypomethylation, thereby promoting cell differentiation, re-expression of tumor suppressor genes, stimulation of immune mechanisms, and suppression of tumor growth.<br /></li></ul><br /><i></i></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><ul><li>The cell might be able to adapt the changes to DNA methylation and imprint these changes in the chromatin/nuclear architecture. This could be the reason why altering DNA methylation has enduring effect on the epigenome.</li><li>The sensitive period is a period of undefined duration often associated with development and early life, during which the organism is more susceptible to epigenetic changes through environmental changes. Sensitive periods are periods when environment may be able to influence the epigenetic makeup of an organism, have an effect on epigenetic control -<b> most sensitive to changes in environment</b>.</li><li>Sensitive periods of development are during <b><i>germ cell development </i></b>and<b><i> early embryonic development</i></b>. <i>The period of primordial germ cell development all the way to the production of mature oocytes and sperm</i> is one sensitive period and the second sensitive period is the <i>pre-implantation and post implantation period</i> -<b> periods of active remodelling of the epigenome</b>. </li><li>Treating patients during these sensitive periods is inadvisable, because during active remodelling of epigenome, marks are laid and removed. External influence, eg., medicine, is changing the environment, which could have a negative effect, or make changes to the chromatin, which could alter the normal cell function.</li></ul></div>
  </body>
</html>