<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 517 from Anon (session_user_id: 10cfb3d0524728fed8b5d634a37870ea3692a645)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 517 from Anon (session_user_id: 10cfb3d0524728fed8b5d634a37870ea3692a645)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p><span>     The human genome has a pattern of base modifications using methylation in healthy cells. In cancer we often find that pattern altered. For example, CpG islands are sections that have dense concentrations of repeated cytosine-guanine dinucleotides. They act as promotors of expression for genes nearby. Methylation of the cytosine in these sections however inhibits expression of the nearby genes. In normal cells, methylation is used to turn off certain genes and control which type of tissue a cell becomes. A common feature of cancer is increased methylation of CpG islands (and sometimes their surrounding 'shore' regions). We believe this results in a reduction in tumor suppressor genes being expressed.</span><br /><br /><span>     To have cancer, there must be an increase in oncogene (growth factor) expression and a reduction in tumor suppressor factors. That allows the cancerous cells to grow rapidly.</span><br /><br /><span>     Another common change in cancer is to repetitive elements and intergenic regions. Healthy cells maintain methylation on cytosine in those regions to suppress expression of nearby elements. When unmethylated, these regions tend to cause translocations, deletions, or insertions (copy number variations). In cancer we often see these regions become unmethylated, causing genomic instability and further mutation. That will cause further disease in the cell line by changing gene expression or creating aberrant sequences.</span><br /><br /><span>     The mechanism for how these changes takes place is not completely understood, but we do find mutations often in the epigenetic machinery. For example mutations in the enzymes which maintain methylation, like DNMT1, or mutations in enzymes that control histone methylation or modification. With those impacted, the cells' ability to maintain methylation marks or its regulation of them is impacted. The amount of dis-regulation, and resulting change to methylation, is indicative of the progression of the cancer and prognosis.</span></p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p><span>     In diploid species, one autosome gene is inherited from each parent. In a parent-of-origin, imprinted gene expression of each is regulated based upon the gene's inherited parent. Errors in imprinting impact gene expression and result in disease or cancer. The H19/lgf2 cluster is an example. </span><br /><br /><span>     In the H19/lgf2 cluster we find a promotor region which enhances the expression of nearby genes. When unmethylated, a control region will bind with a CTCF protein. This will cause the promotor to enhance expression of the nearby H19 section. When the control region is methylated, CTCF cannot bind and the promotor instead enhances the further lgf2 section.</span><br /><br /><span>     With normal parental imprinting, the paternal allele is methylated and the maternal allele is unmethylated. So the maternal allele will express H19 and the paternal allele lgf2.</span><br /><br /><span>     H19 and lgf2 balance each other. lgf2 is a growth promoting factor whereas H19 is a growth restricting factor. But if an error occurs with imprinting, for example if the maternal allele is also methylated, then the patient could have double the lgf2 growth factor and no H19 growth limiting factor. The cell will reproduce rapidly; called a Wilm's tumor.</span></p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p><span>     Decitabine is a DNMT inhibitor. This will reduce methylation in cell progeny when cells divide, because DNMT1 is responsible for the mitotic heritability of methylation and DNMT3a/b in de novo methylation. In cancer, hypermethylation is associated with down regulation of tumor suppressor genes. Reversing hypermethylation in those regions may re-activate tumor suppressors, reducing the growth of the tumor or improving outcomes.</span><br /><br /><span>     Certain cancers may be more responsive to DNTM inhibitors like Decitabine. Currently it is used to treat myelodysplastic syndrome that has evolved to be acute myelogenous leukemia. But not all cancers would be good candidates for DNMT inhibitors, as some forms of cancer may relate more to hypomethylation of key regions of the genome and DNMT inhibitors would make that worse. Also, there is a risk of any epigenetic drug that healthy tissue will be impacted. Tumor cells divide more rapidly however and would be more impacted.</span></p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p><span>     Epigenetic marks on chromatin are persistent in cells and some types of marks (DNA methylation) are also passed down from cells to their progeny. Drugs which effect epigenetic machinery may then alter marks that persist in the cell or are mitotically inherited to those cell's progeny.</span><br /><br /><span>     Patients have epigenetic sensitive periods when the epigenetic machinery is removing or establishing healthy epigenetic marks, for example during early embryo development in utero and during germ cell development in young people. Also during organ development in infants and toddlers. Therefore, drugs which impact epigenetic machinery are probably more risky to use with people who may be or who are pregnant as well as in young people. Germ cell development continues in young people through their teen years.</span><br /><br /><span>     If a drug impacts normal epigenetic programming in a sensitive period, the effect could be misregulation of gene expression during embryonic development (leading to birth defects or life long illness), or aberrant development of organ tissue during early development, or impacts to the subsequent generation born to the patient, if administered when their primordial germ cell lines (ie. egg or sperm progenitor cells) are still maturing.</span></p></div>
  </body>
</html>