<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 721 from Anon (session_user_id: d8c614991d1f620811afa02c0f6d3d752bd076a7)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 721 from Anon (session_user_id: d8c614991d1f620811afa02c0f6d3d752bd076a7)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>Cancer develops as a result of the loss of cell growth control and starts because of accumulation of both, genetic mutations and epigenetic marks abnormalities. One of the hallmarks of cancer is the altered pattern of DNA methylation including global gene hypomethylation with decreased level of methylated cytosine and locus-specific hypermethylation at CpG islands and CpG island shores.</p>
<p>DNA methylation occurs at 5’ C atom of cytosine where a methyl group is added. This chemical reaction in mammals is happen extensively when cytosine is followed by guanine in DNA sequence and is catalized by methyltransferases DNMT3a and 3b. During cell division DNA methylation is maintained in dauther cells by other methyltransferase DNMT1 and the protein UHRF1 which recognizes methylated cytosine on a single strand.</p>
<p>The clusters enriched with CpGs, CpG islands are usually found in gene promoter regions and are unmethylated. CpG island methylation results in gene silencing because of formation of repressive remodeling complexes (including MeCP proteins) or of prohibiting the binding of transcription factors to the promoters. In cancer hypermethylation in CpG islands is observed in a specific tumor suppressor gene or in a set of genes creating CpG island metylator phenotype (CIMP). Tumor suppressor gene hypermethylation occurs more often than a mutation and is characteristic for each tumor and disease stage. It can be used as a diagnostic and prognostic marker. </p>
<p>In normal cell intragenic region, introns and repetitive elements are important to sustain genomic stability and thus they are methylated. Methylation in such regions prevents the reciprocal translocations, insertions, deletions, duplications, recombinations and transpositions. It also enables the transcription of cryptic genes that might be affected by gene transposition. In cancer intragenic regions and repetitive elements are hypomethylated increasing genomic instability. Ususally the loss of methylation is related with the loss or alteration of methyltransferases. </p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>Genomic imprinting is an epigenetic process in which a subset of genes related to growth and development of fetus shows a pattern-of-origin specific monoallelic expression. Imprinted genes are controlled by imprint control regions (ICRs).</p>
<p>H19/Igf2 cluster is located in human chromosome 11. The place for binding of a CTCF insulator in ICR is found between Igf2 and H19 genes. The expression of Igf2 depends on the enhancers located after H19. In paternal allele, methylation in the ICR prevents the binding of CTCF allowing Igf2 to engage the enhancers. DNA methylation spreads to the H19 promotor, silences the H19 expression and thus the enhancers activate Igf2 expression. In maternal allele CTCF is bound to the ICR region preventing the enhancers to access the Ifg2 gene. In addition H19 is activated and serves as a reservoir for non-coding RNA altering Igf2 expression.</p>
<p>Wilm’s tumor is a kidney cancer occurred in children. The disease is related with ICRs methylation in both, maternal and paternal alleles. As a consequence Igf2 is over-expressed and cell growth is promoted. In pre-neoplastic tissues the loss of imprinting control leads to overexpression of growth promoting genes or the loss of growth restricting genes.</p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine is a hypomethylating agent used for treatment of myelodysplastic syndrome (MDS). DNA hypermethylation in MDS is related to tumor progression and poor prognosis. Decitabine is a nucleoside analogue of cytosine that incorporates into DNA instead cytosine and binds covalently methyltransferases blocking the latter activation. The drug acts as a DNMT inhibitor and also induces DNA damage and double strand breaks which increase the genotoxic stress and have pro-apoptotic effects. At high doses decitabine is toxic and causes overall depletion of nucleotides in the cells pool and complete cell cycle arrest. At low doses the drug application leads to DNA demethylation. As the action of decitabine is dependent on cell division, it is considered that cancer cells are more sensitive to its action than normal ones. One side effect of the therapy might be severe hypomethylation of DNA that can increase illegitimate transcription events.</p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Dr. Baylin showed that a combination of different epigenetic modifying drugs (HDAC with DNMT inhibitors) and their combination with conventional chemotherapy can be more successful in treating cancer than a monotherapy. The reason for that might be that cancer is a process which is caused by mistakes in the genome and abnormalities in several layers of the epigenetic machinery. These mistakes accumulate with the time in a stochastic manner so ‘several hits” on individual cell are required for its transformation to tumor cell. Thus, targeting multiple parts of epigenetic machinery will be more effective. Moreover, it should be considered that cancer cells show stem cell like behavior and the mistakes in the epigenome can be passed to the dauther cells via altered DNA methylation. For example transgenerational inheritance of epimutations in MLH1 is characteristic for colon cancer and leads to aberrant somatic DNA methylation in each generation. So treated cancer cells might “remember” the epigenetic modifications in DNA methylation induced by the drugs. The administration of DNA demethylating agents however should be avoided during sensitive periods when epigenetic marks are erased and re-established – like in early embryonic cell and primodial germ cell development. DNA demethylation occurs normaly in these sensitivity periods and allows epigenetic marks to be are established and maintained in the generation.     </p></div>
  </body>
</html>