<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 481 from Anon (session_user_id: 94bd4d9c9c4ac06df985ac58133fb580ad0b77db)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 481 from Anon (session_user_id: 94bd4d9c9c4ac06df985ac58133fb580ad0b77db)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>The normal function of DNA  methylation at CpG islands is to be free of methylation,which in the </p>
<p>60% of promoters in which they are found, is independent of their activity state. In the human</p>
<p>genome there are about 13,000 CpG islands that are unmethylated (2) .The general effect</p>
<p>of demethylation is to allow greater access to the DNA, thus allowing regulated gene expression. In</p>
<p>cancer, these islands become hyper-methylated, leading to silencing of tumor suppressor and</p>
<p>other genes (1). This silencing is due to the formation of a repressive chromatin stucture (primary</p>
<p>mechanism) and the prevention of transcription binding (secondary mechanism)  (Lecture 1.5).</p>
<p>Possible effects include inhibition of differentiation, maintenance of proliferation, and repression of</p>
<p>genes needed for repair or apoptosis (2). Examples are the RB,MLH1,BRAC1,and MGMT</p>
<p>genes. Hypermethylation of CpG islands also include GSTP1 in prostate cancer (not benign),</p>
<p>miR-34b/c (associated with metastasis), and MGMT in glioma (suggests patients won't</p>
<p>respond to temozotomide) (Lecture 6.2).</p>
<p> </p>
<p>There is a phenotype in some cancers known as "CpG island methylator phenotype"</p>
<p>(CIMP), with high frequency of CpG island hyper-methylation. CIMP is found in colorectal,</p>
<p>glioma, and breast cancers and is being evaluated for diagnosis, prognosis, and prediction</p>
<p>of therapy response (1). For example, the CIMP phenotype in colorectal cancer is associated</p>
<p>with a good prognostic outcome (Lecture 6.3).</p>
<p> </p>
<p>There can also be hypermethylation in CpG island shores (2 kb areas around CpG islands).</p>
<p>This correlates well with gene expression and may be a better predictor of gene</p>
<p>expression than CpG island methylation (Lecture 6.3).</p>
<p> </p>
<p>The normal function of DNA methylation in intergenic regions and repetitive elements is to be</p>
<p>methylated. In cancer, these areas are hypomethylated, causing genomic instabillity, including</p>
<p>illegitimate recombination between repeats, activation of repeats and transposition, activation of</p>
<p>cryptic promoters and splice sites, and disruption of neighboring genes (Lecture 6.4).</p>
<p> </p>
<p> </p>
<p> </p>
<p>1. Biochimie 94:2219-2230(2012).</p>
<p>2. Nature Structural&amp;MolecularBiology 20:274-281(2013).</p>
<p> </p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>Two mechanisms exists for imprinting in clusters -  the rarer insulator model and the use of a long</p>
<p>non-coding RNA (lncRNA). The H19/Igf2 cluster uses the insulator model (1).</p>
<p> </p>
<p>The methylation pattern of the paternal allele is that the H19 promoter and ICR (imprint control</p>
<p>region) are methylated. The methylation at the ICR blocks binding of CTCF, and without CTCF the</p>
<p>methylation spreads to the H19 promotor to silence it and enhancers can access Igf2 to activate.</p>
<p>The methylation pattern of the maternal allele is that the CTCF insulator protein binds the ICR</p>
<p>because it is unmethylated. This allows the promoter of H19 to remain unmethylated and subject </p>
<p>to the enhancers. Thus Igf2 expression is not activated by the enhancers. In Wilm's tumor</p>
<p>there is hypermethylation of ICR in the maternal allele so that the maternal allele looks like</p>
<p>the paternal allele, and there is overexpression of Igf2 (1,Lectures 4.4,6.3).</p>
<p> </p>
<p>Igf2  (insulin-like growth factor 2 ) is an oncogene whose overexpression plays a key role in the </p>
<p>tumors seen in the Beckwich-Wiedemann Syndrome (including Wilm's tumor) (Lecture 4.4)</p>
<p>and other tumors (pediatric sarcomas, nephroblastoma,rhabdomyosarcoma) (2).</p>
<p> </p>
<p> </p>
<p>1. Current Opinion Genetics&amp;Development 22:72-78(2012).</p>
<p>2. Folia Histochemica&amp;Cytobiologica 50:171(2012).</p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine is a DNMTi - DNA methyltransferase inhibitor - which decreases DNA methylation</p>
<p>by irreversibly binding DNA methyl-transferases after being incorporated into DNA.</p>
<p>Decitabine is thus replication dependent. Mechanism of action against cancer is unclear. There could</p>
<p>be correction of the hypermethylation of the CpG islands described in question 1. Decitabine is</p>
<p>approved by the FDA for myelodysplastic syndrome (MDS), but not acute myeloid leukemia (AML).</p>
<p>Side effects include bone marrow suppression (anemia, neutropenia, and thromocytopenia), but the</p>
<p>drug is suprisingly well tolerated.  Decitabine could be effective in hematological malignancies,</p>
<p>because they are dependent on tumor suppressor gene hypermethylation. Dosing must be adjusted</p>
<p>to avoid toxicity and maximize anti-tumor effect (Lecture 6.9). This class of drugs is also being</p>
<p>evaluated as agents which can make standard chemotherapy more effective. In one study,</p>
<p>decitabine caused a reactivation of  SMAD1 and chemosensitization of tumor cells in diffuse large B-</p>
<p>cell lymphoma (DLBCL) (1).</p>
<p> </p>
<p>1.Cancer Discovery 9/2013.</p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Almost all methylation is erased in the early embryo and a new pattern is reestablished at the time</p>
<p>of implantation. Once set up, this pattern is maintained through cell division, although adjustments</p>
<p>are made. There is a second period of reprogramming in germ cell development. These two </p>
<p>periods are termed sensitive because of the dramatic effects during that time. Imprint control</p>
<p>regions (ICRs) methylation is established in primordial germ cells and is maintained in the</p>
<p>embryo (1,Lectures 4.1-2).</p>
<p> </p>
<p>Treating patients during these sensitive periods with drugs which modify methylation has the</p>
<p>potential to disrupt important stages of embryo and germ cell developmment. The</p>
<p>guidelines for use of decitabine (see question 3) include the following: men should not</p>
<p>father a child during treatment and two months following; women should not become pregnant</p>
<p>during treatment and one month following (2). Treatment during the reprogramming of germ</p>
<p>cell development could affect fertility and the transmission of disease to future generations.</p>
<p>Treatment during these sensitive periods could result in the full range of epigenetic disease,</p>
<p>including early imprinting disorders, early embryonic lethality, placental insufficiency,</p>
<p>germ cell tumors, tumorigenesis, blocks in differentiation, and infertility (Lecture 1.2). </p>
<p> </p>
<p> </p>
<p>1. Nature Structural&amp;MolecularBiology 20:274-281(2013).</p>
<p>2. FDA approved labeling.</p></div>
  </body>
</html>