<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 267 from Anon (session_user_id: 5fe254bb007fc35aab4d98d7bc22aa4fdfe35258)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 267 from Anon (session_user_id: 5fe254bb007fc35aab4d98d7bc22aa4fdfe35258)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value">1. CpG islands are clusters of CG sequences, often at promoters. DNA methyltransferases (DNMTs) catalyse methylation of C in the CpG islands, generally resulting in gene silencing. This occurs by several mechanisms, including binding of methylated CpG binding proteins that can decrease transcription and cause condensation of chromatin.<br />2. In cancer, methylation in CpG islands is often increased (hypermethylation).<br />3. If these CpG islands are located in the promoter of a tumour suppressor gene, the activity of that gene will be reduced. Silencing of that one gene may not be enough to result in cancer. However, it can act together with somatic mutations in a sequence of events ('hits') that accumulate, eventually resulting in cancer by irreversible disruption of mechanisms controlling cell growth.<br />4. At intergenic regions, DNA methylation maintains genomic stability against indels and translocations. At repetitive elements, it maintains genomic stability by (a) silencing of repeats to reduce transpositions and transcriptional interference by strong promoters, (b) mutation of Me-C to T, thus interrupting the repeating sequence and reducing transpositions, and (c) reduction of illegitimate recombination, which requires open chromatin.<br />5. At intergenic regions and repetitive elements, DNA methylation is often reduced in cancers (hypomethylation).<br />6. Hypomethylation in intergenic regions and repetitive elements favours the various kinds of genomic instability listed in item #4.<br /><br />Main sources: Course lectures.</div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value">1. In the paternal allele, the H19 imprinting control region (ICR) is hypermethylated. This prevents transcriptional repressor CTCF (CCCTC-binding factor or 11-zinc finger protein) from binding to the ICR. When CTCF is bound to the ICR, enhancers located downstream from the ICR and the H19 gene are blocked from acting on the insulin-like growth factor 2 (Igf2) gene located upstream from the ICR. Therefore, because CTCF is not bound to the ICR, the enhancers activate the Igf2 gene. (Also, the H19 gene is methylated.)<br />2. In the maternal allele, the ICR is not hypermethylated, so CTCF can bind. Therefore, the enhancers are blocked from acting on Igf2, which remains silent. (Also, the H19 gene is not hypermethylated.)<br />3. In Wilms' tumour, the ICR in the maternal allele is hypermethylated as well. So, the Igf2 gene is expressed in both alleles.<br />4. As Igf2 is a growth factor during gestation, increased production of it because of the abnormal methylation of the maternal ICR presumably makes development of the tumour more likely.<br /><br />Main sources: Course lectures.</div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value">1. Decitabine is a DNA methyltransferase (DNMT) inhibitor.<br />2. By inhibiting DNMT, it will decrease DNA methylation.<br />3. Various ways in which DNA hypermethylation can favour development of cancer have already been described in question 1. For example, hypermethylation of CpG islands in promoters of tumour suppressor genes can decrease activity of those genes. If this is the main mechanism for development of a particular type of tumour, then decitabine might be effective in treatment of that type, although perhaps not effective in treatment of other types arising by different mechanisms.<br /><br />Main sources: Course lectures.</div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value">1. DNA methylation epigenetic marks persist during somatic cell division ('somatic epigenetic memory or maintenance') by dynamic or static mechanisms. Therefore, an effect of an epigenetic drug on tumour cells could persist even though the cells may have divided many times. This could make the cells more susceptible to chemotherapy well after the time of exposure to the epigenetic drug.<br />2. A 'sensitive period' is a time when there is epigenetic reprogramming, i.e., when various epigenetic marks are removed and a new set of marks created.<br />3. The sensitive periods are (a) primordial germ cell development starting in mid-gestation and (b) preimplantation early development.<br />4. For proper germ cell development to occur, somatic cell marks must be removed and germ cell-specific ones laid down. Likewise, for proper somatic cell development, germ cell marks must be removed and somatic cell-specific ones laid down. If a drug affecting removal or generation of epigenetic marks were given, a wide variety of developmental abnormalities might presumably occur.<br /><br />Main sources: Course lectures and Shipony, Z., et al., 'Dynamic and static maintenance of epigenetic memory in pluripotent and somatic cells'. Nature 513: 115 (2014).<br /><a href="http://www.nature.com/nature/journal/v513/n7516/full/nature13458.html">http://www.nature.com/nature/journal/v513/n7516/full/nature13458.html</a>  <br /></div>
  </body>
</html>