<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 2256 from Anon (session_user_id: 60a687244b9a72f989a8f3b22a72d973843a4db4)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 2256 from Anon (session_user_id: 60a687244b9a72f989a8f3b22a72d973843a4db4)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>dna methylation normal function at Cpg islands is to silence genes which are not reuired to be expressed, repeats of genes and hence avoids transcriptional interferances while maintaining genomic stability.In cancer, dna methylation of cpg islands gets hypermethylated or hypomethylated.In any case the normal function of the gene is disrupted leading to either an increase in functions or loss of necessary functions and hence leads to many different epigenetic and genetic diseases.The normal function of dna methylation in intergenic regins and repetive elements is to regulate gene silencing of the repetive  and intergenic gene sequences and hence prevent any  irregular function of genes.In cancer , dna methylation in intergenic regions gets altered resulting in the hypomethylation or hypermethylation of the genetic marks. These hypomethylated marks are therfore not able to silence the nonrequired genes resulting in genomic instability,random recombination b/w repeats, activation of cryptic promoters and disrruption of neighbouring genes.While the hypermethylated marks act by disrupting the tumour supressor zoneresulting in loss of imprinting and loss of expression of various crucial genes.Examples of  diseases caused by such disruptive marks are wilomintin disease due to hpermethylation of icr's leading to inactivation of igf2 gene.Examples of hypomethylation induced diseases are the activation of genes R-RAS gene responsible for gastric cancer and micro rna- miR21 that targets tumour supressor genes of PTEN-GLIOME. OTHER examples include hypermethylation of MLH1 gene in coloreactal cancer.</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>H19/Igf2 cluster is an example of enhancer blocking genomic imprinting. In it, the CTCF ia an insulator protein which insulates Igf2 from downstream enhancer or maternal allele.DNA methylation at upstream or parental srand at ICR region blocks binding of CTCF binding on parental strand hence no regulation of gene is expressed on the parental allele. WIThout CTCF dna methylation spreads to H19 promoter to silence it and enhancers could then acess IGF2 to activate the gene expression in maternal allele. In Wilm's tumour hypermethylation at the ICR region is observed  resulting in loss of imprinting and hence overexpression of IGF2 gene in maternal allele. Irregular imprinting at the H19/IGF2 cluster contributes to disease like wilm's tumour by hypermethylation of the CTCF in parental allele, which then  result in abaerrant activation of the normally repressed maternally activated allele.</p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>The Decitabene epigenetic  drug belongs to the class of DNA demthylating agents used to treat myleosdysaplastic syndrome precusrrors of acute myleogenous luekamia.The decitabine works by reducing dna methylation that is in  hypomethylated  DNA by inhibiting DNA methyl transferase. Decitabine can be incorporated in dna strands during cell division. Decitabine role in treating cancer is by controlling the hypermethylation of dna and overactivation of repeats which occur in cancer cells. Thus, decitabene works by controlling undesired cell growth by reversing  epigenitically silencing of aberrantly methylated genes.It causes reactivation of tumour supressor genes, thus in all leading to inhibition of tumour drug. Apart from these decithin drug should be used in low concenterations as it is highly toxic in high doses leading to cell toxicity, irreversible dna damage and hence prolonged cell inactivation.</p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Altering dna methylation can lead to hypoactivation of genes or hyperactivation of genes depending on the cancer type. Since activation gene marks are passed to daughter and grandaughter cells during division therefore deletion of these marks by active methods can result in controlling the cancer phenotype to a larger extent.  A sensitive period is defined as the period in which reprogramming of epigentic marks take place before being transferred to the next daughter cells.( in next progeny too). This is completly in regulation with germ cells which  undergo series of sensitive periods for reprogramming of genetic marks in GENOMIC IMPRINTING. Treating patients in those sensitive periods where series of reprogramming in their germ  cells is taking place is highly inadvisable as it would lead to irregular development of imprints in their germ cells and hence loss of rquired function or undesirable aftereffects may be observed. As it would lead to disturbance in their germ cells and hence their sexual growth will be disturbed(affected) resulting in the negative action on their phenotype.</p></div>
  </body>
</html>