<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 571 from Anon (session_user_id: ef8e2f56107a108d3ea79e5630d46db72072188e)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 571 from Anon (session_user_id: ef8e2f56107a108d3ea79e5630d46db72072188e)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>DNA methylation is an important epigentic modification playing roles in development as well as disease. It involves addition of a methyl group to 5' cytosines at C5 position on CpG islands. Under normal physiological conditions, DNA methylation works to silence gene expression as CpG islands are most frequently present at the vicinity of gene promoters. The methylated CpG is then bound by proteins like MeCP1 and 2 which have transcription reprssing motifs and DNA binding motifs. These further condense chromatin to form heterochromatin which inhibits transcriptions.</p>
<p>DNA hypermethylation and DNA hypomethylation have both been reported in epigenetic studies of cancer. Hypermethylation of tumour suppressor genes is a mechanism which propagates tumorigenesis by epigenetic silencing of the genes. For example, the genes <em>RB</em> in retinoblastoma and <em>BRCA1/2</em> in breast cancer responsible for DNA repair and cell cycle control have been known to be silenced by mypermethylation. DNA hypomethylation also leads to carcinogenesis by activating transcription of oncogenes.</p>
<p>Hypomethylation can also lead to cancer by activating DNA repeats and intergeneic regions and leading to chromosomal instability and an increased risk of gene translocations. The normlly silenced tandem repeats in a chromosome when activated due to poor methylation leads to illegitimate recombinations between intergenic region and transpositions. This may cause activation of cryptic promoters, disruption of legitimate genes and protein synthesis. For example, <em>PTEN</em> is  tumour suppressor gene which is targetted my <em>miRNA21</em> which is activated in glioma. Hypomethylation may also  cause loss of imprinting by either silencing or activating both parental alleles leading to either a complete inhibition of gene expression or over expression.</p>
<p><span style="text-decoration:underline;"><strong>REFERENCES:</strong></span></p>
<p><strong>[1] Blewitt M, Lecture notes, Coursera</strong></p>
<p><strong>[2] Futscher, 2013, <em>Adv Exp Med Biol</em></strong></p>
<p><strong>[3] You &amp; Jones, 2013, <em>Cancer Cell</em></strong></p>
<p><strong>[4] Brzezianska <em>et al</em>, 2013, <em>Mol Bio Rep </em></strong></p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p><em>Igf2</em> is a gene widely expressed during embryonic development. It is paternally imprinted or has monoallelic expression of the paternal gene. <em>Igf2</em> is present opposite non-coded but imprinted gene <em>H19</em> both of which share enhancers that act on the genes depending on parental origin. On the paternal chromosome, <em>H19</em> is silenced by methylation and <em>Igf2</em> is activated while on the maternal gene <em>Igf2</em> is silenced and <em>H19</em> is activated. In a cell, the CTCF domain or the imprint control region is present a few kilobases upstream of the H19 gene in the  maternal chromosome. The demethylated ICR on the maternal chromosome ensures silencing of the maternal <em>Igf2</em> and activation of the <em>H19</em>. Similarly, methylation of the paternal ICR allows the enhancers to act on the active paternal <em>Igf2</em> gene and silences the paternal <em>H19</em>. </p>
<p>In Wilm's tumor, there is mutations in the WT1 gene which encodes a transcriptional repressor of <em>Igf2</em>; the epigenetic reprogramming of the cells, which results in hypermethylation of the maternal ICR which results in a silencing of the maternal <em>H19</em> gene. This allows the enhancers to act on both parental <em>Igf2</em> regions causing an over expression of the <em>Igf2</em> gene. Loss of imprint control in <em>Igf2</em> locus has been identified in 90% of the cases.<br /><br /><img src="https://coursera-uploads.s3.amazonaws.com/user-4da9fb80d131c474e4e8a544/970238/asst-5/970238-5209b8fd535e86.35438581.png" alt="" /></p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine which is also known as 2'-deoxy-5-azacytidine is a nucleoside analog of deoxy-cytidine and acts as a <strong><em>hypomethylating agent</em></strong>. It inhibits <em>DNA methyltransferase 1</em> by foming an irreversible covalent bond between the drug and the enzyme; It reverses aberrant hypermethylation of genes in cancer and reactivates the genes silenced by this epigenetic error. Hypermethylation of tumor suppressor genes can propagate carcinogenesis by silencing these tumor suppressor and demethylase treatments like administration of decitabine can reverse this. It can restore susceptibility to apoptosis and cell cycle control in the tumor. It is FDA approved for use in hematological malignancies like acute myeloid leukemia and myelodysplastic syndromes.</p>
<p><span style="text-decoration:underline;"><strong>REFERENCES:</strong></span></p>
<p><strong>[1] Blewitt M, Lecture notes, Coursera</strong></p>
<p><strong>[2] Momparler,2013, <em>Front Oncol</em>.</strong></p>
<p><strong>[3] Stresemann and Lyko, 2008, <em>Int. Journal of Cancer</em></strong></p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Although, altering DNA methylation with drugs like Decitabine has proven to be quite successful in patients with myelodysplastic syndromes and in clinical trials again hematopoietic malignancies, the possible long term effects of using such drugs must be a consideration. It is a concern that prolonged use of hypomethylating agents may also activate oncogenes by reversing their DNA methyation  and may produce a secondary tumor. Aberrant activation of normally silenced genes and overexpression of imprint control genes are possible side effects to this. Research has also revealed that prolonged use  can cause epigenetic effects on stem-cell like cancer cells which may quickly evolve to produce a drug-resistant tumour. </p>
<p>Treating patients during sensitive periods might cause irreparable changes in their epigenetic control which may also affect future generations. Sensitive periods are those periods in the development of an organism wherein it is most susceptible to epigenetic changes and reprogramming. Since epigenetic mechanisms are mitotically heritable, these periods may influence to a large extent the epigenetic machinery in the future F1 and F2 generations. During the sensitive periods, environmental changes can alter the epigenetic control. The sensitive periods of development are during the blastocyst stage and development of primordial germ cells.</p>
<p><span style="text-decoration:underline;"><strong>REFERENCES:</strong></span></p>
<p><strong>[1] Blewitt M, Lecture notes, Coursera</strong></p>
<p><strong>[2] Kuzawa, 2011, Epigenomics</strong></p>
<p><strong>[3] NCI Cancer Bulletin</strong></p>
<p><strong>[4] Garcia-Manero, Horizon Symposia</strong></p></div>
  </body>
</html>