<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1256 from Anon (session_user_id: cc2e1ce0699fb2d18ea38724b83f45fa800d8630)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1256 from Anon (session_user_id: cc2e1ce0699fb2d18ea38724b83f45fa800d8630)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p><span style="font-size:14px;line-height:21px;">Regions rich in CpG dinucleotides (CpG islands) tend to be protected from DNA methylation. CpG islands are found in about 60% of promoters and are usually kept free of DNA methylation, which doesn't depends on their activity state. In cancer cells, the CpG islands found in the promoters of tumour suppressor genes tend to become hypermethylated, silencing the underlying gene. This mechanism, frequently found in tumours is a very effective way of silencing tumour suppressor genes, since DNA methylation is mitoticaly heritable.<br />Normally, DNA methylation is found in intergenic regions and repetitive elements, which may be related with repression of transcriptional noise from alternative start sites, inhibition of antisense transcription or direct RNA splicing and replication timing (maintenance of genomic stability). In cancer cells, we have hypomethylation of intergenic regions and repetitive elements leading to genomic instability in that specific tissue, causing a particular type of cancer.<br />Less common than hypomethylation of repetitive elements, there is another mechanism which consists in hypomethylation of CpG poor promoters. This hypomethylation is associated with activation of the associated genes.<br />To sum up, in the case of cancer we may find genome-wide hypomethylation and tumor supressor hypermethylation, which depends on the context since different tumours have different dependencies.</span></p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>The imprinting of the H19/IgF2 cluster is such that there is expression only of the H19 alelle inherited from the mother and expression only of the IgF2 alelle inherited from the father. Both H19 and IgF2 genes share enhancers and their imprinting is controlled by a CTCF-dependent insulator. In this case, there is a imprinted control region which is unmethylated in the maternal alelle and methylated in the paternal alelle (imprinted). The unmethylated control region in the maternal alelle is bound by the CTCF protein which  prevents Igf2 from accessing the shared enhancers. On the paternal chromosome, there is methylation at the ICR and also methylation at the H19 promoter (silencing H19), leading to Igf2 expression.<br />In Wilm's tumour there is a loss of IgF2 imprinting, leading to the <span>abnormal activation of the normally silenced maternally inherited allele. IgF2 promoves growth, consisting in an oncogene, therefore its excessive activation leads to the known Wilm's tumour.</span></p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine belongs to the class of the DNA-methyltransferase inhibitors (DNMTi). This drug acts as a nucleoside analogue, which irreversibly binds to DNMTs after they are incorporated into the DNA. This action is division dependent and once cancer cells divides much more rapidly than the other body cells, they will be more affected. The anti-tumour effect of Decitabine is because of the inhibition of the DNMT1, therefore supressing this enzyme capacity to methylate CpG islands, decreasing the number of CpG island hypermethylation and tumour suppressor gene hypermethylation (which is related to some types of cancer such as haematological malignancies).</p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Its expected that drugs that alter DNA methylation have effects beyond the period of treatment because DNA methylation is mitotically heritable. Therefore, after the the usage of the drug, there will be still maintanence of these epigenetics changes caused by the drug. Sensitive period is defined by the critical period when enviroment may be able to influence epigenetic marks. During development, the sensitive periods are those when there are active remodelling of the epigenome (removal and lay down of epigenetic marks). The sensitive periods are: the period of primordial germ cells development until the production of mature eggs and sperms and pre implanted and early post implantation period. Therefore, its inadvisable to treat patients with DMNTi drugs during sensitive periods, because these drugs won't allow the normal methylation of the DNA and it will affect the remodelling of the epigenome (lay down of the normal epigenetic marks).</p></div>
  </body>
</html>