<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 28 from Anon (session_user_id: d412a456dda2a8ef55d97b2a2fd16ab7447e1005)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 28 from Anon (session_user_id: d412a456dda2a8ef55d97b2a2fd16ab7447e1005)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><div>CPG islands tend to be found at the promoters of gene's so the function of  methylation at CpG islands will silence gene expression.  That is, DNA methylation is an inactive epigenetic mark.  The function of DNA methylation at repetitive elements is to maintain genomic integrity so in cancer we tend to see genomic instability. There are two types DNA methylation abnormalities in cancer, which are DNA hypermethylation that is locus specific and genome-wide hypomethylation. CpG islands are usually hypomethylated in a normal cell but the genome in general is methylated in their repetitive elements through the intergenic regions and even in the introns of genes. In contrast, if you look in general at a cancer cell, the CpG islands are more likely to be methylated. And the rest of the genome in general,including the repetitive elements in these intergenic elements and introns are hypomethylated.</div></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><div>The paternal and maternal pronuclei exist as two separate small nuclei before they fuse together. The paternal genome is actively and rapidly demethylated after fertilization and before cell division or DNA replication. In contrast, maternal is demethylated more slowly and reaches a global low of blastocyst. After implantating the epigenetic marks are re-established in a lineage specific fashion, these are associated with the particular gene expression patterns that are required with each of lineages.</div><div><div>Igf2 is controlled by enhancer blocking and disrupted as imprinting disorder. It's unmethylated on the maternal allele and methylated on the paternal allele so it's paternally imprinted. H19 is produced from the maternal allele. When this in imprint control region is unmethylated as a maternal allele, it is bound by an insulator protein CTCF. When CTCF blocks this by binding, the enhancers loop to H19 enhancing H19 expression. When this imprint control region on the patternal allele is methlyated, CTCF can no longer bind. Igf2 is expressed only from the paternal allele. Igf2 is an oncogene promoteing growth. Two paternal-like alleles and loss of the maternal like allele will get too much Igf2, which happens due to mutations or deletions. That's how they contribute to cancer.</div></div></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value">Decitabine belongs to DNA-demethylating agents and is used to treat myelodysplastic syndromes. <span>Modification of 5 mC via oxidation by the TET family of proteins to 5-hydroxymethyl cytosine (5hmC) or via deamination to thymine by AID/APOBEC1 followed by base excision repair (BER) was proposed as a possible mechanism for DNA demethylation. </span><span>DNA demethylation will impact gene expression. I</span>n cancer cells, promoter CpG islands tend to become hypermethylated, which then causes silencing of the underlying gene. If the cancer patients take Decitabine, it will prevent CpG islands from hypermethylation, which can have an anti-tumour effect.<br /></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><span>Epigenetic changes will be passed on during cell division to daughter and granddaughter cells until they are actively erased. Therefore, epigenetic therapies can effect changes which stop a cancer growing without having to kill all its cells so they can have enduring effects on the epigenome.<br /></span><div>In general, there are two rounds of epigenetic reprogramming, early development and primordial germ cell development (PGD), which are the potential sensitive periods of development. In the primordial germ cell development, imprinted genes undergo reprogramming and are cleared a little bit later than the rest of the genome, but their resetting happens at about the same rate. they maintain their level of methylation during early embryonic development. When the primordial germ cells are developing, they erase the DNA methylation marks because this resetting only happens during primordial germ cell </div><div><div><div><div>development. Clearing happens just once but escapes re-programming in early development. If treating patients during the sensitive periods, the epigenetic marks will be erased and won't have effect. That's why it's inadvisable.</div></div></div></div></div>
  </body>
</html>