<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 489 from Anon (session_user_id: f3c8659beee71ecc98339d09754f2f574bd4c5b7)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 489 from Anon (session_user_id: f3c8659beee71ecc98339d09754f2f574bd4c5b7)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>DNA methylation is altered in cancer. You can find either hypermethylation (for example in CpG islands, CpG island shores or ICRs) or hypomethylation (for example in repetitive regions, CpG poor promoters or ICRs)</p>
<p>Q1a: The function of DNA methylation at CpG Island is the control gene expression and maintenance genomic stability. Methylation of CpG sites in the promoter of a gene may inhibit gene expression and it usually is hypomethylated in normal cells.</p>
<p>Q1b: In cancer the cells show hypermethylation in CpG islands of tumour suppressor genes (promoter). So if a promoter is hypermethylated, it is unlikely to be expressed.</p>
<p>Q1c. The hypermethylation silences these antioncogenes (tumor suppressor genes) and it permits the decontrolled proliferation characteristic in cancer.  For instance, the methylation in CpG island forms a repressive chromatin structure that inhibits transcription silencing the gene expression. So, cells lose genomic stability  </p>
<p>Q1d: The normal function of DNA methylation in intergenic regions and repetitive elements is genomic stability. These regions/elements in normal cells are methylated. </p>
<p>Q1e: The cells show hypomethylation in intergenic regions and repetitive elements in cancer</p>
<p>Q1f: The methylation in intergenic regions silences cryptic transcription start sites or cryptic splice sites. On the other hand, the methylation in repetitive elements silences repeats to: prevent transposition, avoid transcriptional interference from strong promoters, and prevent illegitimate recombination. So, the hypomethylation in these regions/elements in cancer lead to genomic instability. It permits illegitimate recombination between repeats, activations of repeats and transposition and activation of cryptic promoters and disruption to neighboring genes</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>Disruption of imprinting can contribute to cancer. Alterations in DNA methylation at ICR can result in loss of expression of growth restricting genes or overexpression of growth promoting genes.</p>
<p>Q2a. The paternal allele is methylated.  DNA methylation at ICR blocks binding of CTCF binding (CTCF is an insulator protein that insulates Igf2 from downstream enhancers). Without CTCF, DNA methylation spreads to H19 promoter to silence and enhancers can access Igf2 to activate. So, DNA methylation of paternal allele activates Igf2 expression.</p>
<p>Q2b. The maternal allele is unmethylated permitting binding of CTCF binding, generating an insulator that prevents Ifg2 from accessing the shared enhancers that are located on the H19 side of the insulator. The insulator acts as an enhancer-blocker. So, DNA unmethylation of maternal allele prevents Igf2 expression.</p>
<p>Q2c. The H19/Igf2 cluster is disrupted in Wilm’s tumour and ICR is methylated in both, maternal and paternal alleles.</p>
<p>Q2d. Disrupting imprinting at the H19/Igf2 cluster increased Igf2 expression to a double dose in comparison to normal cells. Igf2 is a oncogene that promotes growth, so upregulation of Igf2 predisposes to tumors.  </p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Q3a. Decitabine belongs to DNA-demethylating agents. It is approved for use in myelodysplastic syndrome that has progressed to acute myeloid leukemia (AML)</p>
<p>Q3b.  Decitabine inhibits a DNA methyltransferase (DNMT1). DNMT1 maintains DNA methylation during cell division. The substrate of DNMT1 is the hemi-methylated DNA. Decitabine is a nucleoside analogue incorporated in DNA on replication. Then DNMT1 is bound irreversibly and it cannot be released. So, DNMT1 cannot methyilate during cell division showing a replication dependent effect. So, Decitabine is a hypomethylating agent.</p>
<p>Q3c. Anti-tumor effect is because it kills tumor cells because it is a hypomethylating agent. The problem is that this drug lacks of specificity affecting tumor cells but also normal cells.</p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Q4a. The methylation is established by DNA methyltransferases. DNA methylation is laid down by de novo methyltransferases, DNMT3a and DNMT3b, and maintained by DNMT1. If you change <em>de novo</em> methylations, for example modify functions of DNMT3a or DNMT3b, then DNMT1will maintain the changes during cell division, so you will have enduring effects on the epigenome. Other possibility could be to alter functions of DNMT1, so new cells would show different pattern of methylation. </p>
<p>Q4b. Sensitive periods are when epigenetic marks are established rather than maintained. They happen during embryogenesis/gametogenesis and in these periods environments have an effect on epigenetic control.</p>
<p>Q4c. The sensitive periods of development are the period of primordial germ cell development and pre-implanted and early post-implantation period.</p>
<p>Q4d. Treating patients during sensitive periods would be inadvisable because you would affect all germ layer established during early development and futures individuals.</p></div>
  </body>
</html>