<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 2163 from Anon (session_user_id: 53223a94e86e081ebd48f2b9e50f51fc83b38c74)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 2163 from Anon (session_user_id: 53223a94e86e081ebd48f2b9e50f51fc83b38c74)</h1>
    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p>Decitabine is a nucleoside analogues, part of the DNMTi (DNA methyltransferases inhibitors) groups. Once incorporated into DNA, decitabine is recognized by DNMT as substrate to be methylated but this linkage is irreversible. The action of these drugs is division dependent, they need cell replicating with active DNMT, therefore they are more effective on cancer cell than normal cells because they are replicating faster.</p></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p>Drug that alter DNA methylation pattern are part of the epigenetic drugs, a new treatment for cancers. In particular, DNA methylation is the modification of the c5 position of the cytosine base, a stable epigenetic mark found in 70-80% of CpG dinucleotide in somatic cells. It is found in promoter regions at CpG islands; it is the base of X chromosome inactivation; it is found in repetitive sequences to stabilize them avoinding possibilities of recombination or trasposition etc.</p>
<p>Thanks to symmetry of this dinucleotide on the two strands of DNA and the action of DNA methyltrasnferases (DNMT), that incorporate methyl groups on the hemi-methylated DNA, methylation signal is maintained through cell division without changes. Therefore drugs, altering the methylation status of a cell, alter gene expression and genome stability of all its daughter cells.</p>
<p>The methylation pattern is established in two sensitive periods of development: early embrionic development and germ cell development (midgestation embryos and among 12-14 years of age). In these periods, methyl groups are removed and re-set. Therefore interfering with DNA methylating and demethylating enzymes can be very deleterious.</p></div>

    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p>DNA methylation is tha addition of methyl groups to DNA. Normally it is found at 70-80% of CpG dinucleotides and is maintained through cell division, thanks to the symmetry of this dinucleotide, on the two strands of DNA by DNA methyltrasnferases (DNMT). This dinucleotide is usually found in promoter regions called CpG islands, in intergenic intervals and repetitive elements.</p>
<p>CpG island are in 60% of gene promoter. In normal cells, they are unmethylated, independently of the gene activity state. In cancer, global changes of DNA methylation occur. In this case, methyl groups are added to promoter regions. This signal induces silencing of the underling genes. Therefore potentially also tumor suppressor genes are silenced, giving another hit to the tumor development. When an unusually high frequency of DNA methylation at CpG site occur, is used the term CIMP, CpG island methylator phenotype. In addition to CpG island, their shores (about 2 kb surrounding them) are also target of hypermethylation.</p>
<p>On the contrary, CpGs at intergenic regions and repetitive sequences are methylated. Here the addition of methyl groups is necessary to maintain genomic stability and integrity. Methylation at intergenic regions and repetitive sequences is necessary to coil strictly the DNA on histone proteins, avoiding the possibility of the DNA to recombine, accumulating inversion, deletion or insertion. In cancer, a general hypomethylation occur. More than half of the genome is hypomethyled, resulting in high genomic instability.</p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>The H19/Igf2 cluster is a gene cluster regulated by an enhancer domain downstream the H19 and an ICR (imprinted controlled region) among them. This last region is responsible for the differential expression in maternal and paternal alleles.</p>
<p>The paternal alleles have methylated ICR, therefore an insulator protein (CTCF) cannot bind to this region, the methylation status spread on H19 promoter blocking its expression. In this case a DNA loop is formed among the enhancer region and Igf2 gene promoter, leading to Igf2 exression; on the contrary, the maternal alleles show unmethylated ICR, the insulator protein can bind to it and the enhancer is involved in another DNA loop that now switch on the epression of H19.</p>
<p>In Wilm’s tumor loss of imprinting result in methylation of maternal allele and so the individual have a double dose to Igf2, in comparison to normal cell. Because Igf2 is a growth promoting, it act as oncogene inducing tumor growth.</p></div>
  </body>
</html>