<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 261 from Anon (session_user_id: f70e041417d4ca53d51ff23cd48a36770d5e3846)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 261 from Anon (session_user_id: f70e041417d4ca53d51ff23cd48a36770d5e3846)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p><span>CpG islands tend to be found in
the promoter regions of tumor suppressor genes.  In a normal genome there is very little methylation at these
CpG islands.  The lack of
methylation does not mean that the gene is active.  When CpG islands are hypomethylated the tumor suppressor
genes may or may not be expressed depending upon the action of other epigenetic
regulators.  </span></p>

<p> </p>

<p><span>However, in the context of
cancer, there is hypermethylation at CpG islands and tumor suppressor genes are
silenced.  Silencing of these genes
is one of the ‘hits’ in the Knudson hypothesis that is necessary for
development of cancer.  The lack of
expression of tumor suppressor genes increases the probability of cancer.  It has also been noted that the
particular CpG island or set of CpG islands that are hypermethylated vary by tumor
type.</span></p>

<p> </p>

<p><span>In the normal genome intergenic
regions and repetitive elements are methylated and this silences expression of
these areas.   In the context
of cancer, the intergenic regions and repetitive elements are hypomethylated.  This allows the expression of
repetitive elements which can lead to genomic instability through
transpositions and activation of cryptic promoters that could disrupt the
expression of neighboring genes.</span></p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p>IIn the normal cell, the ICR for
one parental allele is methylated and the other is unmethylated.  In the case of the H19/Igf2 cluster the
ICR for the imprinted paternal allele is methylated, while the ICR for the unimprinted
maternal allele is unmethylated. </p>
















<p> </p>

<p><span>For the maternal allele, the
insulator protein - CTCF - binds to the unmethylated ICR.  CTCF insulates Igf2 from downstream
enhancers and allows the enhancers to express H19.  There is no expression of Igf2. </span></p>

<p> </p>

<p><span>In the paternal allele, the
insulator protein - CTCF - cannot bind to the methylated ICR.  This allows the enhancers to interact
with Igf2 and cause Igf2 expression. 
Also  methylation spreads
from the ICR to the H19 promoter and blocks expression of H19.</span></p>

<p> </p>

<p><span>Wilm’s tumor occurs when the
ICR is methylated for both parents.  
The maternal allele will act like the paternal allele since both ICR’s
are methylated and both alleles will produce Igf2 and no H19.  Igf2 is a growth promoting protein and
its overexpression causes Wilm’s syndrome, a childhood kidney tumor.  The fact that a growth gene is
overexpressed is a hit that can lead to cancer.  In general cancer has too many growth promoters and not
enough growth suppressors.</span></p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p><span>Decitabine is a nucleoside
analog that acts as a DNA demethylation agent.  The decitabine becomes incorporated into the DNA at cell
replication.  Instead of methylating
the nucleoside analog, DNMT1 becomes bound to the DNA.  </span></p>

<p> </p>

<p><span>If a cancer is caused by
hypermethylation of CpG islands then a DNA demethylation agent could be
useful.  As cells divide the amount
of CpG methylation would decrease.</span></p>

<span>This has been found
useful for treatment of myelodysplastic syndrome</span></div>

    <div class="field-name" id="10aa9d89e69a5f7a">Q4. Dr Stephen Baylin speculates in the Economist article that "epigenetic
drugs altered the tumour cells in some lasting way that made them more
susceptible to standard chemotherapy." How can drugs that alter DNA methylation
have effects that last beyond the period of drug treatment? Discuss whether
there are any periods of development when you would avoid treating patients
with such drugs. In your answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe how altering DNA methylation can have enduring effects on the
        epigenome.
        <br>
    </li>
    <li>Define what is meant by a sensitive period.
        <br>
    </li>
    <li>Identify sensitive periods of development.
        <br>
    </li>
    <li>Explain why treating patients during sensitive periods would be inadvisable.
    <br>
</li>
</ul></div>
    <div class="field-value"><p><span>DNA methylation is mitotically
transmitted.  At cell replication,
DNMT1 recognizes the hemimethylated DNA strand and duplicates the methylation
from the parent strand to the daughter strand.   So, if you treat a patient with a drug that alters DNA
methylation, those changes in DNA methylation will be passed on through mitotic
replication throughout the lifetime of the individual.   </span></p><p> </p><p><span>Sensitive periods are periods
where the environment is going to have the biggest effect.  The two most important sensitive
periods for epigenetic markers are primoridal germ cell development and
pre-implantation development of the embryo.  During this time period epigenetic marks are being removed
and reestablished.  Other more
restrictive sensitive periods occur at different, limited times as specific
tissue differentiation occurs. </span></p><p> </p><p>
























</p><p><span>In general it would be
inadvisable to treat patients with DNA methylation altering agents during
epigenetic sensitive periods. The treatment could adversely affect germ cell
development by disrupting normal epigenetic marks that would affect future
embryos. For a pregnant mother treatment during the embryos early development
sensitive period could be catastrophic for the embryo since their epigenetic
marks could be altered in a variety of negative ways.</span></p></div>
  </body>
</html>