<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Submission 1757 from Anon (session_user_id: ee80b19abf23e41b266219269b080cad48d55622)</title>
    <link href="../../../export.css" rel="stylesheet">
  </head>
  <body>
    <h1>Submission 1757 from Anon (session_user_id: ee80b19abf23e41b266219269b080cad48d55622)</h1>
    <div class="field-name" id="d6002606a5ec3d03">Q1. Describe how DNA methylation is altered in cancer.
In your answer include the following points:
<br><p></p><ul><li>Describe the normal
function of DNA methylation at CpG islands.<br></li><li>Describe how DNA
methylation of CpG islands is disrupted in cancer.<br></li><li>Explain how disruption of
DNA methylation at CpG islands contributes to cancer.<br></li><li>Describe the normal
function of DNA methylation in intergenic regions and repetitive elements.<br></li><li>Describe how DNA
methylation in intergenic regions and repetitive elements is disrupted in cancer.<br></li><li>Explain how disruption
of DNA methylation in intergenic regions and repetitive elements contributes to
cancer.<br></li></ul><p></p></div>
    <div class="field-value"><p class="s8"><span><span class="s2">Normally </span><span class="s2">CpG</span><span class="s2"> islands are</span><span class="s2"> </span><span class="s2">usually </span><span class="s2">unmethylated</span><span class="s2">,</span><span class="s2">Intergenic</span><span class="s2"> regions are</span><span class="s2"> usually methylated r</span></span><span class="s2">epetitive elements are </span><span class="s2">usually </span><span class="s2">methylated </span><span class="s12">to m</span><span class="s12">aintain </span><span class="s12">genomic integrity </span><span class="s12">and to </span><span class="s13">protect the genome </span><span class="s13">from</span><span class="s13"> transposable elements.</span></p>
<p class="s8"><span class="s2">DNA methylation is associated with histone modifications and the interplay of these epigenetic modifications is crucial to regulate the functioning of the genome by changing chromatin architecture. The covalent addition of a methyl group occurs generally in cytosine within </span><span class="s2">CpG</span><span class="s2"> </span><span class="s2">dinucleotides</span><span class="s2"> which are concentrated in large clusters called </span><span class="s2">CpG</span><span class="s2"> islands.</span><span class="s2"> </span><span class="s2">CpG</span><span class="s2">methylation </span><span class="s2">is</span><span class="s2"> s</span><span class="s2">ymmetric</span><span class="s2">.</span><span class="s2"> </span><span class="s14">DNA</span><span class="s2"> </span><span class="s2">methyltransferases</span><span class="s2"> </span><span class="s2">3A &amp;3B </span><span class="s2">are responsible for establishing and</span><span class="s2"> </span><span class="s14">DNA</span><span class="s2">methyltransferases</span><span class="s2">1</span><span class="s2"> </span><span class="s2">is</span><span class="s2"> responsible </span><span class="s2">maintenance of methylation pattern. It is commonly known that inactivation of certain tumor-suppressor genes occurs as a </span><span class="s2">consequence of </span><span class="s2">hypermethylation</span><span class="s2"> within the promoter regions and a numerous studies have demonstrated a broad range of genes silenced by DNA methylation in different cancer types.</span><span class="s2"> </span><span class="s2">For example p16 methylation is associated  </span><span class="s2"> </span><span class="s2">with </span><span class="s2">many malignancies</span><span class="s2"> like HCC</span><span class="s2">.</span><span class="s2"> </span><span class="s2">On the other hand, global </span><span class="s2">hypomethylation</span><span class="s2">, inducing genomic instability, also contributes to cell transfor</span><span class="s2">mation.T</span><span class="s15">he</span><span class="s15"> earliest epigenetic aberration found was a genome-</span><span class="s15">wide</span><span class="s15"> lack of methylation,  </span>hypomethylatio at intergenic <span class="s15">regions</span><span class="s15">/ repeats, and genomic instability</span></p></div>

    <div class="field-name" id="323cdd9789545f83">Q2. Describe how disruption of imprinting can contribute to cancer, using the example of the H19/Igf2 cluster. In your
answer include the following points:
<br>
<p></p>
<ul>
    <li>Describe the methylation pattern of the paternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe the methylation pattern of the maternal allele and how this determines
        Igf2 expression status.&nbsp;
        <br>
    </li>
    <li>Describe how imprinting at the H19/Igf2 cluster is disrupted in Wilm’s tumour.&nbsp;
        <br>
    </li>
    <li>Explain how disrupting imprinting at the H19/Igf2 cluster contributes to cancer.
        <br>
    </li>
</ul></div>
    <div class="field-value"><p class="s11"> </p>
<p class="s11"><span class="s2">One critical attribute of imprinted genes is that they must be marked with their parental </span><span class="s2">ori</span><span class="s2">gin</span><span class="s2"> so that the correct allele-</span><span class="s2">s</span><span class="s2">pecific expression patterns are observed in somatic tissue.</span></p>
<p class="s19"><span><span class="s18">The insulator model of imprinting, which is employed by the H19/Igf2 imprinted locus. The</span><span class="s18"> </span><span class="s18">maternally-expressed H19 gene and paternally-expressed Igf2 gene share enhancers and</span><span class="s18"> </span><span class="s18">their reciprocal imprinting </span><span class="s18">is</span><span class="s18"> governed by a CTCF-dependent insulator that is located</span><span class="s18"> </span><span class="s18">between the genes.</span></span></p>
<p class="s19"><span><span class="s20">On the paternal allele the ICR which located between H19 and Igf2 genes is methylated leading to prevention of the binding of CTCF and its cofactors ( p68/SRA and cohesin) therfore the enhancer will act on the promotor of Igf2 gene leading to its expression.</span></span></p>
<p class="s19"><span><span class="s20">On the maternal allele , CTCF binds to 4 binding sites within</span><span class="s20"> </span><span class="s20">the ICR, generating an insulator that prevents Igf2 from accessing the shared enhancers that</span><span class="s20"> </span><span class="s20">are located on the H19 side of the insulator. Thus, this</span></span> interaction mediates enhancer blocking allowing downstrea <span class="s20">enhancers</span><span class="s20"> </span><span class="s20">to</span><span class="s20"> access the H19 promoter.</span></p>
<p class="s19"><span class="s2">H19, a candidate tumor suppressor gene, and IGF2, a candidate dominant oncogene, the majority of WTs show a </span><span class="s2">bipaternal</span><span class="s2">epigenotype</span><span class="s2"> at these loci, with H19 inactive and IGF2</span><span class="s2">biallelically</span><span class="s2"> active.</span></p></div>

    <div class="field-name" id="961ab4c640df430a">Q3. The Economist article “<a href="http://www.economist.com/node/21552168" target="_blank">Cancer’s epicentre</a>” describes several drugs that affect
    epigenetic processes. Explain how Decitabine may be used to treat cancer,
    with reference to effects on the epigenome. In your answer include the
    following points:
<br>
<p></p>
    <ul>
        <li>Identify the class of epigenetic inhibitors that Decitabine belongs to.&nbsp;
            <br>
        </li>
        <li>Describe the impact of Decitabine on DNA methylation.
            <br>
        </li>
        <li>Describe how Decitabine can have an anti-tumour effect.&nbsp;
            <br>
        </li>
    </ul></div>
    <div class="field-value"><p class="s4"><span class="s3">Decitabine</span></p>
<p class="s4"><span><span class="s5">Chemically, it is a </span><span class="s6">cytidine</span><span class="s5"> </span><span class="s6">analog</span><span class="s6">,</span><span class="s3"> </span><span class="s2">its </span><span class="s2">chemical structure</span><span class="s2"> is </span><span class="s2">4-Amino-1-(2-deoxy-β-D-erythro-pentofuranosyl)-1</span><span class="s2">,3,5</span><span class="s2">-triazin-2(1H)-one</span></span></p>
<table><tbody><tr><td class="s7">
<p class="s8"><span class="s2"><span style="color:#333333;font-family:'Helvetica Neue', Helvetica, Arial, sans-serif;">Formula</span></span></p>
</td>
<td class="s7">
<p class="s8"><span style="color:#333333;font-family:'Helvetica Neue', Helvetica, Arial, sans-serif;"><span><span class="s2">C</span><span class="s2">8</span><span class="s2">H</span><span class="s2">12</span><span class="s2">N</span><span class="s2">4</span><span class="s2">O</span><span class="s2">4 </span></span></span></p>
</td>
</tr></tbody></table><p class="s8"><span><span class="s9">It is </span><span class="s2">DNA-</span><span class="s2">demethylating</span><span class="s2"> agents,</span><span class="s5"> </span><span class="s5">it </span><span class="s5">inhibts</span><span class="s5"> </span><span class="s6">DNA </span><span class="s6">methyltransferase</span><span class="s6"> so it</span><span class="s2"> </span><span class="s5">can only be incorporated into DNA </span><span class="s5">strands</span><span class="s2">. S</span><span class="s2">old as </span><span class="s2">Dacogen</span><span class="s2"> by Eisai, a Japanese company, are used to treat</span><span class="s2">myelodysplastic</span><span class="s2"> syndromes</span><span class="s5"> </span><span class="s5">including previously treated and untreated, de novo and secondary MDS</span><span class="s5"> and for </span><span class="s5">acute myeloid leukemi</span><span class="s5">a</span><span class="s5"> (AML).</span></span></p>
<p class="s4"><span><span class="s5">In the treatment of</span><span class="s5"> patients with renal insufficiency, </span><span class="s5">some researchers </span><span class="s5">reported the first case series on the feasibility of therapy with </span><span class="s5">hypomethylating</span><span class="s5"> agents in patients with renal insufficiency.</span></span></p></div>
  </body>
</html>